<!DOCTYPE html>
<html class="js" lang="en">
  <head>
    <style></style>
    <meta http-equiv="content-type" content="text/html; charset=UTF-8" />
    <!-- <link
      rel="preload"
      href="https://www.nature.com/static/fonts/HardingText-Regular-Web.b167e675dc.woff2"
      as="font"
      type="font/woff2"
      crossorigin=""
    /> -->
    <title>
      scRNA 结题报告 | BGI & SJTU
    </title>

    <script async="" src="report_files/gpt.js"></script>
    <script
      type="text/javascript"
      async=""
      src="report_files/linkid.js"
    ></script>
    <script
      type="text/javascript"
      async=""
      src="report_files/analytics.js"
    ></script>
    <script src="report_files/bat.js" async=""></script>
    <script
      type="text/javascript"
      async=""
      src="report_files/beacon.js"
    ></script>
    <script src="report_files/hm.js"></script>
    <script async="" src="report_files/uwt.js"></script>
    <script async="" src="report_files/fbevents.js"></script>
    <script type="text/javascript" async="" src="report_files/ld.js"></script>
    <!-- <script async="" src="report_files/MathJax.js"></script> -->
    <script
      async=""
      src="report_files/93dabb8d80079a87fec7bb6f67b807fce90e1688f8957ad7ad152bfd58ea01c2"
    ></script>
    <!-- <script
      src="report_files/otSDKStub.js"
      data-domain-script="83f2c78a-6cbc-4d1a-9088-3f8e8c4c7460"
    ></script> -->
    <script async="" src="report_files/gtm.js"></script>
    <script id="save-data-connection-testing">
      function hasConnection() {
        return (
          navigator.connection ||
          navigator.mozConnection ||
          navigator.webkitConnection ||
          navigator.msConnection
        );
      }

      function createLink(src) {
        var preloadLink = document.createElement("link");
        preloadLink.rel = "preload";
        preloadLink.href = src;
        preloadLink.as = "font";
        preloadLink.type = "font/woff2";
        preloadLink.crossOrigin = "";
        document.head.insertBefore(preloadLink, document.head.firstChild);
      }

      var connectionDetail = {
        saveDataEnabled: false,
        slowConnection: false,
      };

      var connection = hasConnection();
      if (connection) {
        connectionDetail.saveDataEnabled = connection.saveData;
        if (/\slow-2g|2g/.test(connection.effectiveType)) {
          connectionDetail.slowConnection = true;
        }
      }

      if (
        !(connectionDetail.saveDataEnabled || connectionDetail.slowConnection)
      ) {
        createLink("/static/fonts/HardingText-Regular-Web.b167e675dc.woff2");
      } else {
        document.documentElement.classList.add("save-data");
      }
    </script>

    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <meta name="applicable-device" content="pc,mobile" />
    <meta
      name="viewport"
      content="width=device-width,initial-scale=1.0,maximum-scale=5,user-scalable=yes"
    />



    <script>
      (function (w, d) {
        w.config = w.config || {};
        w.config.mustardcut = false;

        if (
          w.matchMedia &&
          w.matchMedia(
            "only screen and (-webkit-min-device-pixel-ratio:0) and (min-color-index:0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio:0) and (min-resolution: 3e1dpcm)"
          ).matches
        ) {
          w.config.mustardcut = true;
          d.classList.add("js");
          d.classList.remove("grade-c");
        }
      })(window, document.documentElement);
    </script>

    <style>
      html {
        text-size-adjust: 100%;
        box-sizing: border-box;
        font-family: sans-serif;
        font-size: 62.5%;
        height: 100%;
        line-height: 1.15;
        overflow-y: scroll;
      }
      body {
        background: #eee;
        color: #222;
        font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto,
          Oxygen-Sans, Ubuntu, Cantarell, Helvetica Neue, sans-serif;
        line-height: 1.76;
        min-height: 100%;
      }
      article,
      aside,
      header,
      main,
      nav,
      section {
        display: block;
      }
      h1 {
        font-size: 30px;
      }
      a,
      sup {
        vertical-align: baseline;
      }
      a {
        background-color: transparent;
        color: #069;
        text-decoration: underline;
      }
      b {
        font-weight: bolder;
      }
      sup {
        font-size: 75%;
        line-height: 0;
        position: relative;
        top: -0.5em;
      }
      img {
        border: 0;
        height: auto;
        max-width: 100%;
        vertical-align: middle;
      }
      svg:not(:root) {
        overflow: hidden;
      }
      button,
      input,
      select {
        font-family: sans-serif;
        font-size: 100%;
        line-height: 1.15;
      }
      select {
        margin: 0;
      }
      button,
      input {
        overflow: visible;
      }
      button,
      select {
        text-transform: none;
      }
      [type="submit"],
      button {
        -webkit-appearance: button;
      }
      [hidden] {
        display: none;
      }
      button {
        border-radius: 0;
        cursor: pointer;
        font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto,
          Oxygen-Sans, Ubuntu, Cantarell, Helvetica Neue, sans-serif;
      }
      h2 {
        font-size: 24px;
      }
      h3 {
        font-size: 17px;
      }
      h1 {
        -webkit-font-smoothing: antialiased;
        font-family: Harding, Palatino, serif;
        font-size: 3.2rem;
        font-weight: 700;
        letter-spacing: -0.0625rem;
        line-height: 3.6rem;
      }
      .c-card--major .c-card__title,
      .u-h2,
      h2 {
        font-family: Harding, Palatino, serif;
        font-size: 2.4rem;
        line-height: 2.6rem;
      }
      .c-card--major .c-card__title,
      .u-h2 {
        letter-spacing: -0.01875rem;
        margin-bottom: 8px;
      }
      h2,
      h3 {
        letter-spacing: -0.01875rem;
      }
      .c-card--major .c-card__title,
      .c-card__title,
      .c-reading-companion__figure-title,
      .u-h2,
      .u-h3,
      .u-h4,
      h2,
      h3,
      h4,
      h5,
      h6 {
        -webkit-font-smoothing: antialiased;
        font-weight: 700;
      }
      .c-card__title,
      .c-reading-companion__figure-title,
      .u-h3,
      .u-h4,
      h3,
      h4,
      h5,
      h6 {
        line-height: 2.2rem;
      }
      .c-card__title,
      h3 {
        font-family: Harding, Palatino, serif;
        font-size: 2rem;
      }
      .c-reading-companion__figure-title,
      .u-h3,
      .u-h4 {
        letter-spacing: -0.01875rem;
      }
      .u-h3 {
        font-family: Harding, Palatino, serif;
        font-size: 2rem;
        margin-bottom: 8px;
      }
      .c-reading-companion__figure-title,
      .u-h4 {
        font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto,
          Oxygen-Sans, Ubuntu, Cantarell, Helvetica Neue, sans-serif;
        font-size: 1.8rem;
      }
      .u-h4 {
        margin-bottom: 8px;
      }
      body,
      button,
      div,
      form,
      h1,
      h2,
      h3,
      input {
        margin: 0;
        padding: 0;
      }
      p {
        padding: 0;
      }
      nav ol,
      nav ul {
        list-style: none none;
      }
      body {
        font-size: 1.8em;
      }
      p {
        margin: 0 0 28px;
      }
      ol,
      ul {
        margin-bottom: 28px;
        margin-top: 0;
      }
      p:empty {
        display: none;
      }
      .sans-serif {
        font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto,
          Oxygen-Sans, Ubuntu, Cantarell, Helvetica Neue, sans-serif;
      }
      .article-page {
        background: #fff;
      }
      .c-article-header {
        font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto,
          Oxygen-Sans, Ubuntu, Cantarell, Helvetica Neue, sans-serif;
        margin-bottom: 40px;
      }
      .c-article-identifiers {
        color: #6f6f6f;
        display: flex;
        flex-wrap: wrap;
        font-size: 1.6rem;
        line-height: 1.3;
        list-style: none;
        margin: 0 0 8px;
        padding: 0;
      }
      .c-article-identifiers__item {
        border-right: 1px solid #6f6f6f;
        list-style: none;
        margin-right: 8px;
        padding-right: 8px;
      }
      .c-article-identifiers__item:last-child {
        border-right: 0;
        margin-right: 0;
        padding-right: 0;
      }
      .c-article-identifiers__open {
        color: #b74616;
      }
      .c-article-title {
        font-size: 2.4rem;
        line-height: 1.25;
        margin-bottom: 16px;
      }
      @media only screen and (min-width: 768px) {
        .c-article-title {
          font-size: 3rem;
          line-height: 1.2;
        }
      }
      .c-article-author-list {
        font-size: 1.6rem;
        list-style: none;
        margin-bottom: 0;
        padding: 0;
        width: 100%;
      }
      .c-article-author-list__item {
        margin-left: 0;
      }
      .c-article-author-list__item,
      .c-article-author-list li {
        display: inline;
        padding-right: 0;
      }
      .c-article-author-list__item svg,
      .c-reading-companion__figure-full-link svg {
        margin-left: 4px;
      }
      .c-article-author-list__show-less {
        margin-left: 8px;
      }
      .js .js-author-etal {
        display: none;
        visibility: hidden;
      }
      @media only screen and (max-width: 539px) {
        .js .js-smaller-author-etal {
          display: none;
          visibility: hidden;
        }
      }
      .c-article-info-details {
        font-size: 1.6rem;
        margin-bottom: 8px;
        margin-top: 16px;
      }
      .c-article-info-details__cite-as {
        border-left: 1px solid #6f6f6f;
        margin-left: 8px;
        padding-left: 8px;
      }
      .c-article-metrics-bar {
        display: flex;
        flex-wrap: wrap;
        font-size: 1.6rem;
        line-height: 1.3;
      }
      .c-article-metrics-bar__wrapper {
        margin: 0 0 16px;
      }
      .c-article-metrics-bar__item {
        -webkit-box-align: baseline;
        align-items: baseline;
        border-right: 1px solid #6f6f6f;
        margin-right: 8px;
      }
      .c-article-metrics-bar__item:last-child {
        border-right: 0;
      }
      .c-article-metrics-bar__count {
        font-weight: 700;
        margin: 0;
      }
      .c-article-metrics-bar__label {
        color: #626262;
        font-style: normal;
        font-weight: 400;
        margin: 0 10px 0 5px;
      }
      .c-article-metrics-bar__details {
        margin: 0;
      }
      .c-article-main-column {
        font-family: Harding, Palatino, serif;
        margin-right: 8.6%;
        width: 60.2%;
      }
      @media only screen and (max-width: 1023px) {
        .c-article-main-column {
          margin-right: 0;
          width: 100%;
        }
      }
      .c-article-section {
        clear: both;
      }
      .c-article-associated-content__container
        .c-article-associated-content__title,
      .c-article-section__title {
        border-bottom: 2px solid #d5d5d5;
        font-size: 2rem;
        line-height: 1.3;
        padding-bottom: 8px;
      }
      @media only screen and (min-width: 768px) {
        .c-article-associated-content__container
          .c-article-associated-content__title,
        .c-article-section__title {
          font-size: 2.4rem;
          line-height: 1.24;
        }
      }
      .c-article-section__content {
        margin-bottom: 40px;
        padding-top: 8px;
      }
      @media only screen and (max-width: 1023px) {
        .c-article-section__content {
          padding-left: 0;
        }
      }
      .c-article-authors-search {
        margin-bottom: 24px;
        margin-top: 0;
      }
      .c-article-authors-search__item,
      .c-article-authors-search__title {
        font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto,
          Oxygen-Sans, Ubuntu, Cantarell, Helvetica Neue, sans-serif;
      }
      .c-article-authors-search__title {
        color: #626262;
        font-size: 1.7rem;
        font-weight: 700;
        margin: 0;
        padding: 0;
      }
      .c-article-authors-search__item {
        font-size: 1.6rem;
      }
      .c-article-authors-search__text {
        margin: 0;
      }
      .c-article-share-box__no-sharelink-info {
        font-size: 1.3rem;
        font-weight: 700;
        margin-bottom: 24px;
        padding-top: 4px;
      }
      .c-article-share-box__only-read-input {
        border: 1px solid #d5d5d5;
        display: inline-block;
        font-size: 1.4rem;
        font-weight: 700;
        margin-bottom: 8px;
        padding: 8px 10px;
      }
      .c-article-share-box__button--link-like {
        background-color: transparent;
        border: 0;
        color: #069;
        cursor: pointer;
        font-size: 1.4rem;
        margin-bottom: 8px;
        margin-left: 10px;
      }
      .c-article-body {
        clear: both;
      }
      .c-article-body a,
      .c-article-body p {
        overflow-wrap: break-word;
      }
      .c-article-body a {
        word-break: break-word;
      }
      .c-article-editorial-summary__container {
        border-bottom: 1px solid #d5d5d5;
        clear: both;
        float: left;
        margin-bottom: 16px;
        width: 100%;
      }
      .c-article-editorial-summary__container
        .c-article-editorial-summary__title {
        line-height: 1.4;
        margin-bottom: 16px;
      }
      .c-article-editorial-summary__container
        .c-article-editorial-summary__article-title {
        font-size: 1.4rem;
        line-height: 1.4;
        margin-bottom: 8px;
      }
      .c-article-editorial-summary__container
        .c-article-editorial-summary__content {
        font-family: Harding, Palatino, serif;
        font-size: 1.4rem;
      }
      .c-article-editorial-summary__container
        .c-article-editorial-summary__content
        p:last-child {
        margin-bottom: 0;
      }
      .c-article-editorial-summary__container
        .c-article-editorial-summary__content--less {
        max-height: 9.5rem;
        overflow: hidden;
      }
      .c-article-editorial-summary__container
        .c-article-editorial-summary__button {
        background-color: #fff;
        border: 0;
        color: #069;
        font-size: 1.4rem;
        margin-bottom: 16px;
      }
      .c-article-editorial-summary__container
        .c-article-editorial-summary__button:focus,
      .c-article-editorial-summary__container
        .c-article-editorial-summary__button:hover {
        text-decoration: underline;
      }
      .c-article-associated-content__container
        .c-article-associated-content__collection,
      .c-article-associated-content__container
        .c-article-associated-content__item {
        margin-bottom: 24px;
        overflow: hidden;
      }
      .c-article-associated-content__container
        .c-article-associated-content__collection-label {
        color: inherit;
        font-size: 1.4rem;
        font-weight: 700;
        line-height: 1.4;
        margin-bottom: 8px;
      }
      .c-article-associated-content__container
        .c-article-associated-content__collection-title,
      .c-nature-box {
        line-height: 1.3;
        margin-bottom: 8px;
      }
      @media only screen and (min-width: 1024px) {
        .c-pdf-button__container {
          display: none;
        }
      }
      .c-context-bar {
        box-shadow: 0 0 10px 0 rgba(51, 51, 51, 0.2);
        position: relative;
        width: 100%;
      }
      .c-context-bar__title,
      .c-header__menu--global .c-header__item svg {
        display: none;
      }
      .c-article-extras {
        float: left;
        font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto,
          Oxygen-Sans, Ubuntu, Cantarell, Helvetica Neue, sans-serif;
        width: 31.2%;
      }
      @media only screen and (max-width: 1023px) {
        .c-article-extras {
          display: none;
        }
      }
      .c-reading-companion {
        clear: both;
      }
      .c-reading-companion__sticky {
        max-width: 389px;
      }
      .c-reading-companion__scroll-pane {
        margin: 0;
        min-height: 200px;
        overflow: hidden auto;
      }
      .c-reading-companion__tabs {
        -webkit-box-orient: horizontal;
        -webkit-box-direction: normal;
        display: flex;
        flex-flow: row nowrap;
        font-size: 1.6rem;
        list-style: none;
        margin: 0 0 8px;
        padding: 0;
      }
      .c-reading-companion__tabs > li {
        -webkit-box-flex: 1;
        flex-grow: 1;
      }
      .c-reading-companion__tab {
        background-color: #eee;
        border: 1px solid #d5d5d5;
        border-image: initial;
        border-left-width: 0;
        color: #069;
        font-size: 1.6rem;
        padding: 8px 8px 8px 15px;
        text-align: left;
        width: 100%;
      }
      .c-reading-companion__tabs li:first-child .c-reading-companion__tab {
        border-left-width: 1px;
      }
      .c-reading-companion__tab--active {
        background-color: #fff;
        border-bottom: 1px solid #fff;
        color: #222;
        font-weight: 700;
      }
      .c-reading-companion__figures-list,
      .c-reading-companion__references-list,
      .c-reading-companion__sections-list {
        list-style: none;
        padding: 0;
      }
      .c-reading-companion__references-list--numeric {
        list-style: decimal inside;
      }
      .c-reading-companion__sections-list {
        margin: 0 0 8px;
        min-height: 50px;
      }
      .c-reading-companion__section-item {
        font-size: 1.6rem;
        padding: 0;
      }
      .c-reading-companion__section-item a {
        display: block;
        line-height: 1em;
        overflow: hidden;
        padding: 8px 0 8px 16px;
        text-overflow: ellipsis;
        white-space: nowrap;
      }
      .c-reading-companion__figure-item {
        border-top: 1px solid #d5d5d5;
        font-size: 1.6rem;
        padding: 16px 8px 16px 0;
      }
      .c-reading-companion__figure-item:first-child {
        border-top: none;
        padding-top: 8px;
      }
      .c-reading-companion__reference-item {
        border-top: 1px solid #d5d5d5;
        font-size: 1.6rem;
        padding: 8px 8px 8px 16px;
      }
      .c-reading-companion__reference-item:first-child {
        border-top: none;
      }
      .c-reading-companion__reference-item a {
        word-break: break-word;
      }
      .c-reading-companion__reference-citation {
        display: inline;
      }
      .c-reading-companion__reference-links {
        font-size: 1.3rem;
        font-weight: 700;
        list-style: none;
        margin: 8px 0 0;
        padding: 0;
        text-align: right;
      }
      .c-reading-companion__reference-links > a {
        display: inline-block;
        padding-left: 8px;
      }
      .c-reading-companion__reference-links > a:first-child {
        display: inline-block;
        padding-left: 0;
      }
      .c-reading-companion__figure-title {
        display: block;
        margin: 0 0 8px;
      }
      .c-reading-companion__figure-links {
        -webkit-box-pack: justify;
        display: flex;
        justify-content: space-between;
        margin: 8px 0 0;
      }
      .c-reading-companion__figure-links > a {
        display: inline-block;
      }
      .c-reading-companion__panel {
        border-top: none;
        display: none;
        margin-top: 0;
        padding-top: 0;
      }
      .c-reading-companion__panel--active {
        display: block;
      }
      .c-pdf-download {
        display: flex;
        margin-bottom: 24px;
        max-height: 48px;
      }
      @media only screen and (min-width: 540px) {
        .c-pdf-download {
          max-height: none;
        }
      }
      @media only screen and (min-width: 1024px) {
        .c-pdf-download {
          max-height: 48px;
        }
      }
      .c-pdf-download__link {
        -webkit-box-pack: justify;
        -webkit-box-flex: 1;
        background: #069;
        border: 1px solid #069;
        border-radius: 2px;
        color: #fff;
        display: flex;
        flex: 1 1 0%;
        font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto,
          Oxygen-Sans, Ubuntu, Cantarell, Helvetica Neue, sans-serif;
        font-size: 1.6rem;
        justify-content: space-between;
        line-height: 1.3;
        padding: 13px 24px;
        text-decoration: none;
      }
      .c-article-metrics__heading a,
      .c-article-metrics__posts .c-card__title a {
        color: inherit;
      }
      .c-article-metrics__posts .c-card__title {
        font-size: 1.7rem;
      }
      .c-article-metrics__posts .c-card__title + span {
        color: #6f6f6f;
        font-size: 1.6rem;
      }
      .c-ad {
        text-align: center;
      }
      @media only screen and (min-width: 320px) {
        .c-ad {
          padding: 8px;
        }
      }
      .c-ad--728x90 {
        background-color: #ccc;
        display: none;
      }
      .c-ad--728x90 .c-ad__inner {
        min-height: calc(1.5em + 94px);
      }
      @media only screen and (min-width: 768px) {
        .js .c-ad--728x90 {
          display: none;
        }
      }
      .c-ad__label {
        color: #333;
        font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto,
          Oxygen-Sans, Ubuntu, Cantarell, Helvetica Neue, sans-serif;
        font-size: 1.4rem;
        font-weight: 400;
        line-height: 1.5;
        margin-bottom: 4px;
      }
      .c-author-list {
        color: #6f6f6f;
        font-family: inherit;
        font-size: 1.6rem;
        line-height: inherit;
        list-style: none;
        margin: 0;
        padding: 0;
      }
      .c-author-list > li,
      .c-breadcrumbs > li,
      .c-footer__list--inline > li,
      .u-list-comma-separated > li,
      .u-list-inline > li {
        display: inline;
      }
      .c-author-list > li:not(:first-child):not(:last-child):before {
        content: ", ";
      }
      .c-author-list > li:not(:only-child):last-child:before {
        content: " & ";
      }
      .c-author-list--compact {
        font-size: 1.4rem;
        line-height: 1.4;
      }
      .c-author-list--truncated > li:not(:only-child):last-child:before {
        content: " ... ";
      }
      .c-skip-link {
        background: #069;
        color: #fff;
        font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto,
          Oxygen-Sans, Ubuntu, Cantarell, Helvetica Neue, sans-serif;
        font-size: 1.4rem;
        padding: 8px;
        position: absolute;
        text-align: center;
        transform: translateY(-100%);
        width: 100%;
        z-index: 9999;
      }
      @media (prefers-reduced-motion: reduce) {
        .c-skip-link {
          transition: top 0.3s ease-in-out 0s;
        }
      }
      @media print {
        .c-skip-link {
          display: none;
        }
      }
      .c-skip-link:link {
        color: #fff;
      }
      .c-status-message {
        -webkit-box-align: center;
        align-items: center;
        display: flex;
        font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto,
          Oxygen-Sans, Ubuntu, Cantarell, Helvetica Neue, sans-serif;
        font-size: 16px;
        position: relative;
        width: 100%;
      }
      .c-card__summary > p:last-child,
      .c-status-message :last-child {
        margin-bottom: 0;
      }
      .c-status-message--boxed {
        background-color: #fff;
        border: 1px solid #eee;
        border-radius: 2px;
        line-height: 1.4;
        padding: 16px;
      }
      .c-status-message__heading {
        font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto,
          Oxygen-Sans, Ubuntu, Cantarell, Helvetica Neue, sans-serif;
        font-size: 16px;
        font-weight: 700;
      }
      .c-status-message__icon {
        fill: currentcolor;
        -webkit-box-flex: 0;
        display: inline-block;
        flex: 0 0 auto;
        margin-right: 8px;
        transform: translate(0);
        vertical-align: text-top;
      }
      .c-status-message__icon--top {
        align-self: flex-start;
      }
      .c-status-message--info .c-status-message__icon {
        color: #003f8d;
      }
      .c-status-message--boxed.c-status-message--info {
        border-bottom: 4px solid #003f8d;
      }
      .c-status-message--error .c-status-message__icon {
        color: #c40606;
      }
      .c-status-message--boxed.c-status-message--error {
        border-bottom: 4px solid #c40606;
      }
      .c-status-message--success .c-status-message__icon {
        color: #00b8b0;
      }
      .c-status-message--boxed.c-status-message--success {
        border-bottom: 4px solid #00b8b0;
      }
      .c-status-message--warning .c-status-message__icon {
        color: #edbc53;
      }
      .c-status-message--boxed.c-status-message--warning {
        border-bottom: 4px solid #edbc53;
      }
      .c-breadcrumbs {
        color: #000;
        font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto,
          Oxygen-Sans, Ubuntu, Cantarell, Helvetica Neue, sans-serif;
        font-size: 16px;
        list-style: none;
        margin: 0;
        padding: 0;
      }
      .c-breadcrumbs__link {
        color: #666;
      }
      .c-breadcrumbs__chevron {
        margin: 4px 4px 0;
      }
      .c-card__body {
        -webkit-box-flex: 1;
        flex: 1 1 auto;
        padding: 16px;
      }
      .c-card__title {
        letter-spacing: -0.01875rem;
        margin-bottom: 8px;
      }
      .c-card--flush .c-card__body {
        padding: 0;
      }
      .c-card--dark .c-card__title {
        color: #fff;
      }
      .c-header {
        background-color: #fff;
        border-bottom: 5px solid #000;
        font-size: 1.6rem;
        line-height: 1.4;
        margin-bottom: 16px;
      }
      .c-header__row {
        padding: 8px 0;
        position: relative;
      }
      .c-header__row:not(:last-child) {
        border-bottom: 1px solid #eee;
      }
      .c-header__row--flush {
        padding: 0;
      }
      .c-header__split {
        -webkit-box-align: center;
        -webkit-box-pack: justify;
        align-items: center;
        display: flex;
        justify-content: space-between;
      }
      .c-header__logo-container {
        -webkit-box-flex: 1;
        flex: 1 1 0px;
        line-height: 0;
        margin-right: 24px;
      }
      .c-header__logo {
        transform: translateZ(0);
      }
      .c-header__logo img {
        max-height: 32px;
      }
      .c-header__container {
        margin: 0 auto;
        max-width: 1280px;
        padding: 0 16px;
      }
      .c-header__menu {
        -webkit-box-align: center;
        -webkit-box-flex: 0;
        align-items: center;
        display: flex;
        flex: 0 1 auto;
        flex-wrap: wrap;
        font-weight: 700;
        line-height: 1.4;
        list-style: none;
        margin: 0 -8px;
        padding: 0;
      }
      .c-header__menu--global {
        -webkit-box-pack: end;
        font-weight: 400;
        justify-content: flex-end;
      }
      @media only screen and (min-width: 540px) {
        .c-header__menu--global .c-header__item svg {
          display: block;
        }
      }
      .c-header__menu--journal {
        font-size: 1.4rem;
      }
      @media only screen and (min-width: 540px) {
        .c-header__menu--journal {
          flex-wrap: nowrap;
          font-size: 1.6rem;
        }
      }
      @media only screen and (max-width: 1023px) {
        .c-header__menu--tools {
          display: none;
        }
      }
      .c-header__item {
        display: flex;
      }
      @media only screen and (min-width: 540px) {
        .c-header__item:not(:first-child) {
          margin-left: 8px;
        }
      }
      .c-header__item--pipe {
        border-left: 2px solid #eee;
        padding-left: 8px;
      }
      @media only screen and (max-width: 767px) {
        .c-header__item--nature-research {
          display: none;
        }
      }
      .c-header__row--flush .c-header__item {
        padding-bottom: 8px;
        padding-top: 8px;
      }
      .c-header__link {
        -webkit-box-align: center;
        align-items: center;
        color: inherit;
        display: flex;
        padding: 8px;
        white-space: nowrap;
      }
      .c-header-expander a {
        color: inherit;
      }
      .c-header__link > svg {
        margin-left: 2px;
        transition-duration: 0.2s;
      }
      @media only screen and (min-width: 540px) {
        .c-header__link > svg {
          margin-left: 4px;
        }
      }
      .c-header__show-text {
        display: none;
      }
      @media only screen and (min-width: 540px) {
        .c-header__show-text {
          display: inline;
        }
        .c-header__item--dropdown-menu {
          position: relative;
        }
      }
      @media only screen and (min-width: 540px) {
        .c-header__menu--journal
          .c-header__item--dropdown-menu:last-child
          .c-header-expander.has-tethered {
          left: auto;
          right: 0;
        }
      }
      @media only screen and (min-width: 768px) {
        .c-header__menu--journal
          .c-header__item--dropdown-menu:last-child
          .c-header-expander.has-tethered {
          left: 0;
          right: auto;
        }
      }
      .c-header-expander {
        background-color: #000;
        border-bottom: 1px solid #2f2f2f;
        color: #eee;
        font-size: 1.4rem;
        line-height: 1.2;
        padding: 16px 0;
      }
      .c-header-expander button {
        background-color: transparent;
        border: 1px solid #fff;
        color: #fff;
      }
      .c-header-expander__container {
        margin: 0 auto;
        max-width: 1280px;
        padding: 0 16px;
      }
      .c-header-expander__keyline {
        border-bottom: 1px solid #555;
        margin-bottom: 0;
        padding-bottom: 16px;
      }
      .c-header-expander__heading {
        display: inline-block;
        font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto,
          Oxygen-Sans, Ubuntu, Cantarell, Helvetica Neue, sans-serif;
        font-size: 2rem;
        font-weight: 400;
        line-height: 1.4;
      }
      .c-header-expander__list {
        display: flex;
        flex-wrap: wrap;
        margin: 0;
        padding: 0;
      }
      .c-header-expander__item {
        margin: 0 16px 8px 0;
      }
      .c-header-expander__link {
        -webkit-box-align: center;
        align-items: center;
        color: inherit;
        display: flex;
        white-space: nowrap;
      }
      .c-header-expander__link > svg {
        margin-left: 8px;
      }
      .c-header-expander.has-tethered {
        border-bottom: 0;
        border-radius: 0 0 2px 2px;
        left: 0;
        position: absolute;
        top: 100%;
        transform: translateY(5px);
        width: 100%;
        z-index: 1;
      }
      @media only screen and (min-width: 540px) {
        .c-header-expander.has-tethered {
          transform: translateY(8px);
          width: auto;
        }
      }
      @media only screen and (min-width: 768px) {
        .c-header-expander.has-tethered {
          min-width: 225px;
        }
      }
      .c-header-expander.has-tethered .c-header-expander__list {
        display: block;
      }
      .c-header-expander.has-tethered .c-header-expander__item {
        margin: 0;
        padding: 8px 0;
      }
      .c-header-expander.has-tethered .c-header-expander__item--keyline {
        border-top: 1px solid #fff;
        margin-top: 8px;
        padding-top: 16px;
      }
      .c-header-expander.has-tethered
        .c-header-expander__item--keyline-first-item-only
        ~ .c-header-expander__item--keyline-first-item-only {
        border-top: none;
        padding-top: 0;
      }
      .c-header-expander--tray.has-tethered {
        padding: 32px 0 24px;
        transform: none;
        width: 100%;
      }
      .c-search {
        font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto,
          Oxygen-Sans, Ubuntu, Cantarell, Helvetica Neue, sans-serif;
        font-size: 2rem;
        font-weight: 400;
        line-height: 1.3;
      }
      .c-search--max-width {
        max-width: 720px;
      }
      .c-search__field {
        -webkit-box-orient: horizontal;
        -webkit-box-direction: normal;
        -webkit-box-align: center;
        align-items: center;
        display: flex;
        flex-flow: row wrap;
      }
      .c-search__input-container {
        -webkit-box-flex: 1;
        flex: 1 0 100%;
        margin-bottom: 8px;
      }
      @media only screen and (min-width: 768px) {
        .c-search__input-container--md {
          -webkit-box-flex: 999;
          flex: 999 1 auto;
          margin-bottom: 0;
          margin-right: 16px;
        }
      }
      .c-search__select-container {
        -webkit-box-flex: 1;
        flex: 1 1 auto;
        margin-bottom: 8px;
      }
      @media only screen and (min-width: 540px) {
        .c-search__select-container {
          -webkit-box-flex: 999;
          flex: 999 1 auto;
          margin-bottom: 0;
          margin-right: 16px;
        }
      }
      @media only screen and (min-width: 768px) {
        .c-search__select-container {
          -webkit-box-flex: 1;
          flex: 1 0 auto;
        }
      }
      .c-search__button-container {
        -webkit-box-flex: 1;
        flex: 1 0 100%;
      }
      @media only screen and (min-width: 540px) {
        .c-search__button-container {
          -webkit-box-flex: 1;
          flex: 1 0 auto;
        }
      }
      .c-search__input,
      .c-search__select {
        background-color: #fff;
        border: 1px solid;
        border-radius: 3px;
        box-sizing: border-box;
        font-size: 1.6rem;
        padding: 0.6em 1em;
        width: 100%;
      }
      .c-search__select {
        -webkit-appearance: none;
        background-image: url("data:image/svg+xml,%3Csvg height='16' viewBox='0 0 16 16' width='16' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z' fill='%23333' fill-rule='evenodd' transform='matrix(0 1 -1 0 11 3)'/%3E%3C/svg%3E");
        background-position: right 0.7em top 50%;
        background-repeat: no-repeat;
        background-size: 1em;
        box-shadow: 0 1px 0 1px rgba(0, 0, 0, 0.04);
        display: block;
        margin: 0;
        max-width: 100%;
        min-width: 150px;
      }
      .u-button {
        -webkit-box-align: center;
        -webkit-box-pack: center;
        align-items: center;
        background-color: transparent;
        background-image: none;
        border: 1px solid #069;
        border-radius: 2px;
        color: #069;
        cursor: pointer;
        display: inline-flex;
        font-family: sans-serif;
        font-size: 1.6rem;
        justify-content: center;
        line-height: 1.3;
        margin: 0;
        padding: 8px;
        position: relative;
        text-decoration: none;
        transition: all 0.25s ease 0s, color 0.25s ease 0s,
          border-color 0.25s ease 0s;
        width: auto;
      }
      .u-button--primary {
        background-color: #069;
        background-image: none;
        border: 1px solid #069;
        color: #fff;
      }
      .u-button--full-width {
        display: flex;
        width: 100%;
      }
      .u-display-none {
        display: none;
      }
      .js .u-js-hide,
      .u-hide {
        display: none;
        visibility: hidden;
      }
      .u-visually-hidden {
        clip: rect(0, 0, 0, 0);
        border: 0;
        height: 1px;
        margin: -100%;
        overflow: hidden;
        padding: 0;
        position: absolute !important;
        width: 1px;
      }
      @media print {
        .u-hide-print {
          display: none;
        }
      }
      @media only screen and (min-width: 768px) {
        .u-show-at-md {
          display: block;
          visibility: visible;
        }
      }
      @media only screen and (min-width: 1024px) {
        .u-hide-at-lg {
          display: none;
          visibility: hidden;
        }
      }
      .u-clearfix:after,
      .u-clearfix:before {
        content: "";
        display: table;
      }
      .u-clearfix:after {
        clear: both;
      }
      .u-float-left {
        float: left;
      }
      .u-icon {
        fill: currentcolor;
        display: inline-block;
        transform: translate(0);
        vertical-align: text-top;
      }
      .u-full-height {
        height: 100%;
      }
      .u-list-reset {
        list-style: none;
        margin: 0;
        padding: 0;
      }
      .u-container {
        margin: 0 auto;
        max-width: 1280px;
        padding: 0 16px;
      }
      .u-display-inline-block {
        display: inline-block;
      }
      .u-ma-0 {
        margin: 0;
      }
      .u-mt-32 {
        margin-top: 32px;
      }
      .u-mr-2 {
        margin-right: 2px;
      }
      .u-mr-24 {
        margin-right: 24px;
      }
      .u-mb-8 {
        margin-bottom: 8px;
      }
      .u-mb-16 {
        margin-bottom: 16px;
      }
      .u-mb-24 {
        margin-bottom: 24px;
      }
      .u-mb-32 {
        margin-bottom: 32px;
      }
      html *,
      html :after,
      html :before {
        box-sizing: inherit;
      }
      .c-article-section__title,
      .c-article-title {
        font-weight: 700;
      }
      .c-card__title {
        line-height: 1.4em;
      }
      .c-header__link {
        text-decoration: inherit;
      }
      .grade-c-hide {
        display: block;
      }
      .u-lazy-ad-wrapper {
        background-color: #ccc;
        display: none;
        min-height: 137px;
      }
      @media only screen and (min-width: 768px) {
        .u-lazy-ad-wrapper {
          display: block;
        }
      }
    </style>

    <link
      data-inline-css-source="critical-css"
      rel="stylesheet"
      href="report_files/article-nature-branded.css"
      media="only screen and (-webkit-min-device-pixel-ratio:0) and (min-color-index:0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio:0) and (min-resolution: 3e1dpcm)"
      onload="this.media='only screen and (-webkit-min-device-pixel-ratio:0) and (min-color-index:0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio:0) and (min-resolution: 3e1dpcm)';this.onload=null"
    />
    <noscript>
      <link
        rel="stylesheet"
        type="text/css"
        href="/static/css/article-nature-branded.17525d86ef.css"
        media="only screen and (-webkit-min-device-pixel-ratio:0) and (min-color-index:0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio:0) and (min-resolution: 3e1dpcm)"
      />
    </noscript>

    <link
      rel="stylesheet"
      type="text/css"
      href="report_files/article-print.css"
      media="print"
    />

    <link
      rel="apple-touch-icon"
      sizes="180x180"
      href="https://www.nature.com/static/images/favicons/nature/apple-touch-icon.f39cb19454.png"
    />
    <link
      rel="icon"
      type="image/png"
      sizes="32x32"
      href="https://www.nature.com/static/images/favicons/nature/favicon-32x32.3fe59ece92.png"
    />
    <link
      rel="icon"
      type="image/png"
      sizes="16x16"
      href="https://www.nature.com/static/images/favicons/nature/favicon-16x16.951651ab72.png"
    />
    <link
      rel="manifest"
      href="https://www.nature.com/static/manifest.1a481c42b1.json"
      crossorigin="use-credentials"
    />
    <link
      rel="mask-icon"
      href="https://www.nature.com/static/images/favicons/nature/safari-pinned-tab.69bff48fe6.svg"
      color="#000000"
    />
    <link
      rel="shortcut icon"
      href="https://www.nature.com/static/images/favicons/nature/favicon.62367f778b.ico"
    />
    <meta name="msapplication-TileColor" content="#000000" />
    <meta
      name="msapplication-config"
      content="/static/browserconfig.e35b3b052c.xml"
    />
    <meta name="theme-color" content="#000000" />
    <meta name="application-name" content="Nature" />

    <script>
      (function () {
        if (typeof window.CustomEvent === "function") return false;
        function CustomEvent(event, params) {
          params = params || {
            bubbles: false,
            cancelable: false,
            detail: null,
          };
          var evt = document.createEvent("CustomEvent");
          evt.initCustomEvent(
            event,
            params.bubbles,
            params.cancelable,
            params.detail
          );
          return evt;
        }

        CustomEvent.prototype = window.Event.prototype;

        window.CustomEvent = CustomEvent;
      })();
    </script>

    <script>
      (function (w, d, t) {
        function cc() {
          var h = w.location.hostname;
          if (h.indexOf("preview-www.nature.com") > -1) return;

          var e = d.createElement(t),
            s = d.getElementsByTagName(t)[0];

          if (h.indexOf("nature.com") > -1) {
            e.src = "https://cdn.cookielaw.org/scripttemplates/otSDKStub.js";
            e.setAttribute(
              "data-domain-script",
              "83f2c78a-6cbc-4d1a-9088-3f8e8c4c7460"
            );
          } else {
            e.src = "/static/js/cookie-consent-es5-bundle.34a145d526.js";
            e.setAttribute("data-consent", h);
          }
          s.parentNode.insertBefore(e, s);
        }

        !!w.google_tag_manager
          ? cc()
          : window.addEventListener("gtm_loaded", function () {
              cc();
            });
      })(window, document, "script");
    </script>
    <script>
      function OptanonWrapper() {
        window.dataLayer.push({ event: "OneTrustGroupsUpdated" });
        document.activeElement.blur();
      }
    </script>

    <!-- <script
      src="report_files/f81201b2ef1d4895848163b43aeaecb4.js"
      crossorigin="anonymous"
    ></script> -->
    <!-- 重要 -->
    <script id="js-position0">
      (function (w, d) {
        w.idpVerifyPrefix = "https://verify.nature.com";
        w.ra21Host = "https://wayf.springernature.com";

        if (w.Sentry) {
          w.Sentry.onLoad(function () {
            w.Sentry.init({
              allowUrls: [/^https?:\/\/www\.nature\.com\//i],
              ignoreErrors: [
                "Non-Error exception captured",
                "Non-Error promise rejection captured",
                "Illegal invocation",
                "Failed to fetch",
              ],
            });
          });
        }

        var moduleSupport = (function () {
          return "noModule" in d.createElement("script");
        })();

        var polyfillsUrl = function () {
          var features = {
            IntersectionObserver: window.IntersectionObserver,
            Promise: window.Promise,
            URLSearchParams: window.URLSearchParams,
            "Symbol.iterator": window.Symbol && Symbol.iterator,
            "Array.from": Array.from,
            "Array.prototype.includes": Array.prototype.includes,
            "Array.prototype.find": Array.prototype.find,
            "Array.prototype.forEach": Array.prototype.forEach,
            "NodeList.prototype.forEach": NodeList.prototype.forEach,
            "Element.prototype.closest": Element.prototype.closest,
            "Element.prototype.prepend": Element.prototype.prepend,
            "Element.prototype.remove": Element.prototype.remove,
            "Object.assign": Object.assign,
          };
          var req = [];
          for (var feature in features) {
            if (
              Object.prototype.hasOwnProperty.call(features, feature) &&
              !features[feature]
            ) {
              req.push(feature);
            }
          }
          if (req.length) {
            return (
              "https://polyfill.io/v3/polyfill.min.js?features=" +
              req.join("%2C") +
              "&flags=always"
            );
          }
          return null;
        };

        if (w.config.mustardcut === true) {
          w.loader = {
            index: 0,
            registered: [],
            scripts: [
              { src: polyfillsUrl(), test: "polyfills-js", noinit: true },

              {
                src: "/static/js/global-article-es6-bundle.7dc3dce534.js",
                test: "global-article-js",
                module: true,
              },
              {
                src: "/static/js/global-article-es5-bundle.6c16769b0e.js",
                test: "global-article-js",
                nomodule: true,
              },
              {
                src: "/static/js/shared-es6-bundle.bc701b2283.js",
                test: "shared-js",
                module: true,
              },
              {
                src: "/static/js/shared-es5-bundle.6b94ed7e0e.js",
                test: "shared-js",
                nomodule: true,
              },
              {
                src: "/static/js/header-150-es6-bundle.785ec93d66.js",
                test: "header-150-js",
                module: true,
              },
              {
                src: "/static/js/header-150-es5-bundle.4f4f49f383.js",
                test: "header-150-js",
                nomodule: true,
              },
            ].filter(function (s) {
              if (s.src === null) return false;
              if (moduleSupport && s.nomodule) return false;
              return !(!moduleSupport && s.module);
            }),

            register: function (value) {
              this.registered.push(value);
            },

            ready: function () {
              if (this.registered.length === this.scripts.length) {
                this.registered.forEach(function (fn) {
                  if (typeof fn === "function") {
                    setTimeout(fn, 0);
                  }
                });
                this.ready = function () {};
              }
            },

            insert: function (s) {
              var t = d.getElementById("js-position" + this.index);
              if (t && t.insertAdjacentElement) {
                t.insertAdjacentElement("afterend", s);
              } else {
                d.head.appendChild(s);
              }
              ++this.index;
            },

            createScript: function (script, beforeLoad) {
              var s = d.createElement("script");
              s.id = "js-position" + (this.index + 1);
              s.setAttribute("data-test", script.test);
              if (beforeLoad) {
                s.defer = "defer";
                s.onload = function () {
                  if (script.noinit) {
                    loader.register(true);
                  }
                  if (
                    d.readyState === "interactive" ||
                    d.readyState === "complete"
                  ) {
                    loader.ready();
                  }
                };
              } else {
                s.async = "async";
              }
              s.src = script.src;
              return s;
            },

            init: function () {
              this.scripts.forEach(function (s) {
                loader.insert(loader.createScript(s, true));
              });

              d.addEventListener(
                "DOMContentLoaded",
                function () {
                  loader.ready();

                  var conditionalScripts = [
                    {
                      match: "div[data-pan-container]",
                      src: "/static/js/pan-zoom-es6-bundle.b5d6c3b38e.js",
                      test: "pan-zoom-js",
                      module: true,
                    },
                    {
                      match: "div[data-pan-container]",
                      src: "/static/js/pan-zoom-es5-bundle.40944b99f9.js",
                      test: "pan-zoom-js",
                      nomodule: true,
                    },
                    // {
                    //   match: "math,span.mathjax-tex",
                    //   src: "/static/js/math.6a7821f32b.js",
                    //   test: "math-js",
                    // },
                  ];

                  if (conditionalScripts) {
                    conditionalScripts
                      .filter(function (script) {
                        return (
                          !!document.querySelector(script.match) &&
                          !(
                            (moduleSupport && script.nomodule) ||
                            (!moduleSupport && script.module)
                          )
                        );
                      })
                      .forEach(function (script) {
                        loader.insert(loader.createScript(script));
                      });
                  }
                },
                false
              );
            },
          };
          loader.init();
        }
      })(window, document);
    </script>
    <script
      id="js-position1"
      data-test="global-article-js"
      defer="defer"
      src="report_files/global-article-es6-bundle.js"
    ></script>
    <script
      id="js-position2"
      data-test="shared-js"
      defer="defer"
      src="report_files/shared-es6-bundle.js"
    ></script>
    <script
      id="js-position3"
      data-test="header-150-js"
      defer="defer"
      src="report_files/header-150-es6-bundle.js"
    ></script>
    <script
      id="js-position4"
      data-test="math-js"
      async=""
      src="report_files/math.js"
    ></script>

    <script>
      (function (w, d, s, l, i) {
        w[l] = w[l] || [];
        w[l].push({ "gtm.start": new Date().getTime(), event: "gtm.js" });
        var f = d.getElementsByTagName(s)[0],
          j = d.createElement(s),
          dl = l != "dataLayer" ? "&l=" + l : "";
        j.async = true;
        j.src = "https://www.googletagmanager.com/gtm.js?id=" + i + dl;

        j.addEventListener("load", function () {
          var _ge = new CustomEvent("gtm_loaded", { bubbles: true });
          d.dispatchEvent(_ge);
        });

        f.parentNode.insertBefore(j, f);
      })(window, document, "script", "dataLayer", "GTM-NWDMT9Q");
    </script>


    <script>
      window.eligibleForRa21 = "false"; // required by js files for displaying the cobranding box (entitlement-box.js)
    </script>
    <style type="text/css">
      .MathJax_Hover_Frame {
        border-radius: 0.25em;
        -webkit-border-radius: 0.25em;
        -moz-border-radius: 0.25em;
        -khtml-border-radius: 0.25em;
        box-shadow: 0px 0px 15px #83a;
        -webkit-box-shadow: 0px 0px 15px #83a;
        -moz-box-shadow: 0px 0px 15px #83a;
        -khtml-box-shadow: 0px 0px 15px #83a;
        border: 1px solid #a6d !important;
        display: inline-block;
        position: absolute;
      }
      .MathJax_Menu_Button .MathJax_Hover_Arrow {
        position: absolute;
        cursor: pointer;
        display: inline-block;
        border: 2px solid #aaa;
        border-radius: 4px;
        -webkit-border-radius: 4px;
        -moz-border-radius: 4px;
        -khtml-border-radius: 4px;
        font-family: "Courier New", Courier;
        font-size: 9px;
        color: #f0f0f0;
      }
      .MathJax_Menu_Button .MathJax_Hover_Arrow span {
        display: block;
        background-color: #aaa;
        border: 1px solid;
        border-radius: 3px;
        line-height: 0;
        padding: 4px;
      }
      .MathJax_Hover_Arrow:hover {
        color: white !important;
        border: 2px solid #ccc !important;
      }
      .MathJax_Hover_Arrow:hover span {
        background-color: #ccc !important;
      }
    </style>
    <style type="text/css">
      #MathJax_About {
        position: fixed;
        left: 50%;
        width: auto;
        text-align: center;
        border: 3px outset;
        padding: 1em 2em;
        background-color: #dddddd;
        color: black;
        cursor: default;
        font-family: message-box;
        font-size: 120%;
        font-style: normal;
        text-indent: 0;
        text-transform: none;
        line-height: normal;
        letter-spacing: normal;
        word-spacing: normal;
        word-wrap: normal;
        white-space: nowrap;
        float: none;
        z-index: 201;
        border-radius: 15px;
        -webkit-border-radius: 15px;
        -moz-border-radius: 15px;
        -khtml-border-radius: 15px;
        box-shadow: 0px 10px 20px #808080;
        -webkit-box-shadow: 0px 10px 20px #808080;
        -moz-box-shadow: 0px 10px 20px #808080;
        -khtml-box-shadow: 0px 10px 20px #808080;
        filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true');
      }
      #MathJax_About.MathJax_MousePost {
        outline: none;
      }
      .MathJax_Menu {
        position: absolute;
        background-color: white;
        color: black;
        width: auto;
        padding: 2px;
        border: 1px solid #cccccc;
        margin: 0;
        cursor: default;
        font: menu;
        text-align: left;
        text-indent: 0;
        text-transform: none;
        line-height: normal;
        letter-spacing: normal;
        word-spacing: normal;
        word-wrap: normal;
        white-space: nowrap;
        float: none;
        z-index: 201;
        box-shadow: 0px 10px 20px #808080;
        -webkit-box-shadow: 0px 10px 20px #808080;
        -moz-box-shadow: 0px 10px 20px #808080;
        -khtml-box-shadow: 0px 10px 20px #808080;
        filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true');
      }
      .MathJax_MenuItem {
        padding: 2px 2em;
        background: transparent;
      }
      .MathJax_MenuArrow {
        position: absolute;
        right: 0.5em;
        padding-top: 0.25em;
        color: #666666;
        font-size: 0.75em;
      }
      .MathJax_MenuActive .MathJax_MenuArrow {
        color: white;
      }
      .MathJax_MenuArrow.RTL {
        left: 0.5em;
        right: auto;
      }
      .MathJax_MenuCheck {
        position: absolute;
        left: 0.7em;
      }
      .MathJax_MenuCheck.RTL {
        right: 0.7em;
        left: auto;
      }
      .MathJax_MenuRadioCheck {
        position: absolute;
        left: 1em;
      }
      .MathJax_MenuRadioCheck.RTL {
        right: 1em;
        left: auto;
      }
      .MathJax_MenuLabel {
        padding: 2px 2em 4px 1.33em;
        font-style: italic;
      }
      .MathJax_MenuRule {
        border-top: 1px solid #cccccc;
        margin: 4px 1px 0px;
      }
      .MathJax_MenuDisabled {
        color: GrayText;
      }
      .MathJax_MenuActive {
        background-color: Highlight;
        color: HighlightText;
      }
      .MathJax_MenuDisabled:focus,
      .MathJax_MenuLabel:focus {
        background-color: #e8e8e8;
      }
      .MathJax_ContextMenu:focus {
        outline: none;
      }
      .MathJax_ContextMenu .MathJax_MenuItem:focus {
        outline: none;
      }
      #MathJax_AboutClose {
        top: 0.2em;
        right: 0.2em;
      }
      .MathJax_Menu .MathJax_MenuClose {
        top: -10px;
        left: -10px;
      }
      .MathJax_MenuClose {
        position: absolute;
        cursor: pointer;
        display: inline-block;
        border: 2px solid #aaa;
        border-radius: 18px;
        -webkit-border-radius: 18px;
        -moz-border-radius: 18px;
        -khtml-border-radius: 18px;
        font-family: "Courier New", Courier;
        font-size: 24px;
        color: #f0f0f0;
      }
      .MathJax_MenuClose span {
        display: block;
        background-color: #aaa;
        border: 1.5px solid;
        border-radius: 18px;
        -webkit-border-radius: 18px;
        -moz-border-radius: 18px;
        -khtml-border-radius: 18px;
        line-height: 0;
        padding: 8px 0 6px;
      }
      .MathJax_MenuClose:hover {
        color: white !important;
        border: 2px solid #ccc !important;
      }
      .MathJax_MenuClose:hover span {
        background-color: #ccc !important;
      }
      .MathJax_MenuClose:hover:focus {
        outline: none;
      }
    </style>
    <style type="text/css">
      .MJX_Assistive_MathML {
        position: absolute !important;
        top: 0;
        left: 0;
        clip: rect(1px, 1px, 1px, 1px);
        padding: 1px 0 0 0 !important;
        border: 0 !important;
        height: 1px !important;
        width: 1px !important;
        overflow: hidden !important;
        display: block !important;
        -webkit-touch-callout: none;
        -webkit-user-select: none;
        -khtml-user-select: none;
        -moz-user-select: none;
        -ms-user-select: none;
        user-select: none;
      }
      .MJX_Assistive_MathML.MJX_Assistive_MathML_Block {
        width: 100% !important;
      }
    </style>
    <style type="text/css">
      #MathJax_Zoom {
        position: absolute;
        background-color: #f0f0f0;
        overflow: auto;
        display: block;
        z-index: 301;
        padding: 0.5em;
        border: 1px solid black;
        margin: 0;
        font-weight: normal;
        font-style: normal;
        text-align: left;
        text-indent: 0;
        text-transform: none;
        line-height: normal;
        letter-spacing: normal;
        word-spacing: normal;
        word-wrap: normal;
        white-space: nowrap;
        float: none;
        -webkit-box-sizing: content-box;
        -moz-box-sizing: content-box;
        box-sizing: content-box;
        box-shadow: 5px 5px 15px #aaaaaa;
        -webkit-box-shadow: 5px 5px 15px #aaaaaa;
        -moz-box-shadow: 5px 5px 15px #aaaaaa;
        -khtml-box-shadow: 5px 5px 15px #aaaaaa;
        filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true');
      }
      #MathJax_ZoomOverlay {
        position: absolute;
        left: 0;
        top: 0;
        z-index: 300;
        display: inline-block;
        width: 100%;
        height: 100%;
        border: 0;
        padding: 0;
        margin: 0;
        background-color: white;
        opacity: 0;
        filter: alpha(opacity=0);
      }
      #MathJax_ZoomFrame {
        position: relative;
        display: inline-block;
        height: 0;
        width: 0;
      }
      #MathJax_ZoomEventTrap {
        position: absolute;
        left: 0;
        top: 0;
        z-index: 302;
        display: inline-block;
        border: 0;
        padding: 0;
        margin: 0;
        background-color: white;
        opacity: 0;
        filter: alpha(opacity=0);
      }
    </style>
    <style type="text/css">
      .MathJax_Preview {
        color: #888;
      }
      #MathJax_Message {
        position: fixed;
        left: 1px;
        bottom: 2px;
        background-color: #e6e6e6;
        border: 1px solid #959595;
        margin: 0px;
        padding: 2px 8px;
        z-index: 102;
        color: black;
        font-size: 80%;
        width: auto;
        white-space: nowrap;
      }
      #MathJax_MSIE_Frame {
        position: absolute;
        top: 0;
        left: 0;
        width: 0px;
        z-index: 101;
        border: 0px;
        margin: 0px;
        padding: 0px;
      }
      .MathJax_Error {
        color: #cc0000;
        font-style: italic;
      }
    </style>
    <style type="text/css">
      .MJXp-script {
        font-size: 0.8em;
      }
      .MJXp-right {
        -webkit-transform-origin: right;
        -moz-transform-origin: right;
        -ms-transform-origin: right;
        -o-transform-origin: right;
        transform-origin: right;
      }
      .MJXp-bold {
        font-weight: bold;
      }
      .MJXp-italic {
        font-style: italic;
      }
      .MJXp-scr {
        font-family: MathJax_Script, "Times New Roman", Times, STIXGeneral,
          serif;
      }
      .MJXp-frak {
        font-family: MathJax_Fraktur, "Times New Roman", Times, STIXGeneral,
          serif;
      }
      .MJXp-sf {
        font-family: MathJax_SansSerif, "Times New Roman", Times, STIXGeneral,
          serif;
      }
      .MJXp-cal {
        font-family: MathJax_Caligraphic, "Times New Roman", Times, STIXGeneral,
          serif;
      }
      .MJXp-mono {
        font-family: MathJax_Typewriter, "Times New Roman", Times, STIXGeneral,
          serif;
      }
      .MJXp-largeop {
        font-size: 150%;
      }
      .MJXp-largeop.MJXp-int {
        vertical-align: -0.2em;
      }
      .MJXp-math {
        display: inline-block;
        line-height: 1.2;
        text-indent: 0;
        font-family: "Times New Roman", Times, STIXGeneral, serif;
        white-space: nowrap;
        border-collapse: collapse;
      }
      .MJXp-display {
        display: block;
        text-align: center;
        margin: 1em 0;
      }
      .MJXp-math span {
        display: inline-block;
      }
      .MJXp-box {
        display: block !important;
        text-align: center;
      }
      .MJXp-box:after {
        content: " ";
      }
      .MJXp-rule {
        display: block !important;
        margin-top: 0.1em;
      }
      .MJXp-char {
        display: block !important;
      }
      .MJXp-mo {
        margin: 0 0.15em;
      }
      .MJXp-mfrac {
        margin: 0 0.125em;
        vertical-align: 0.25em;
      }
      .MJXp-denom {
        display: inline-table !important;
        width: 100%;
      }
      .MJXp-denom > * {
        display: table-row !important;
      }
      .MJXp-surd {
        vertical-align: top;
      }
      .MJXp-surd > * {
        display: block !important;
      }
      .MJXp-script-box > * {
        display: table !important;
        height: 50%;
      }
      .MJXp-script-box > * > * {
        display: table-cell !important;
        vertical-align: top;
      }
      .MJXp-script-box > *:last-child > * {
        vertical-align: bottom;
      }
      .MJXp-script-box > * > * > * {
        display: block !important;
      }
      .MJXp-mphantom {
        visibility: hidden;
      }
      .MJXp-munderover,
      .MJXp-munder {
        display: inline-table !important;
      }
      .MJXp-over {
        display: inline-block !important;
        text-align: center;
      }
      .MJXp-over > * {
        display: block !important;
      }
      .MJXp-munderover > *,
      .MJXp-munder > * {
        display: table-row !important;
      }
      .MJXp-mtable {
        vertical-align: 0.25em;
        margin: 0 0.125em;
      }
      .MJXp-mtable > * {
        display: inline-table !important;
        vertical-align: middle;
      }
      .MJXp-mtr {
        display: table-row !important;
      }
      .MJXp-mtd {
        display: table-cell !important;
        text-align: center;
        padding: 0.5em 0 0 0.5em;
      }
      .MJXp-mtr > .MJXp-mtd:first-child {
        padding-left: 0;
      }
      .MJXp-mtr:first-child > .MJXp-mtd {
        padding-top: 0;
      }
      .MJXp-mlabeledtr {
        display: table-row !important;
      }
      .MJXp-mlabeledtr > .MJXp-mtd:first-child {
        padding-left: 0;
      }
      .MJXp-mlabeledtr:first-child > .MJXp-mtd {
        padding-top: 0;
      }
      .MJXp-merror {
        background-color: #ffff88;
        color: #cc0000;
        border: 1px solid #cc0000;
        padding: 1px 3px;
        font-style: normal;
        font-size: 90%;
      }
      .MJXp-scale0 {
        -webkit-transform: scaleX(0);
        -moz-transform: scaleX(0);
        -ms-transform: scaleX(0);
        -o-transform: scaleX(0);
        transform: scaleX(0);
      }
      .MJXp-scale1 {
        -webkit-transform: scaleX(0.1);
        -moz-transform: scaleX(0.1);
        -ms-transform: scaleX(0.1);
        -o-transform: scaleX(0.1);
        transform: scaleX(0.1);
      }
      .MJXp-scale2 {
        -webkit-transform: scaleX(0.2);
        -moz-transform: scaleX(0.2);
        -ms-transform: scaleX(0.2);
        -o-transform: scaleX(0.2);
        transform: scaleX(0.2);
      }
      .MJXp-scale3 {
        -webkit-transform: scaleX(0.3);
        -moz-transform: scaleX(0.3);
        -ms-transform: scaleX(0.3);
        -o-transform: scaleX(0.3);
        transform: scaleX(0.3);
      }
      .MJXp-scale4 {
        -webkit-transform: scaleX(0.4);
        -moz-transform: scaleX(0.4);
        -ms-transform: scaleX(0.4);
        -o-transform: scaleX(0.4);
        transform: scaleX(0.4);
      }
      .MJXp-scale5 {
        -webkit-transform: scaleX(0.5);
        -moz-transform: scaleX(0.5);
        -ms-transform: scaleX(0.5);
        -o-transform: scaleX(0.5);
        transform: scaleX(0.5);
      }
      .MJXp-scale6 {
        -webkit-transform: scaleX(0.6);
        -moz-transform: scaleX(0.6);
        -ms-transform: scaleX(0.6);
        -o-transform: scaleX(0.6);
        transform: scaleX(0.6);
      }
      .MJXp-scale7 {
        -webkit-transform: scaleX(0.7);
        -moz-transform: scaleX(0.7);
        -ms-transform: scaleX(0.7);
        -o-transform: scaleX(0.7);
        transform: scaleX(0.7);
      }
      .MJXp-scale8 {
        -webkit-transform: scaleX(0.8);
        -moz-transform: scaleX(0.8);
        -ms-transform: scaleX(0.8);
        -o-transform: scaleX(0.8);
        transform: scaleX(0.8);
      }
      .MJXp-scale9 {
        -webkit-transform: scaleX(0.9);
        -moz-transform: scaleX(0.9);
        -ms-transform: scaleX(0.9);
        -o-transform: scaleX(0.9);
        transform: scaleX(0.9);
      }
      .MathJax_PHTML .noError {
        vertical-align: ;
        font-size: 90%;
        text-align: left;
        color: black;
        padding: 1px 3px;
        border: 1px solid;
      }
    </style>
    <script
      async=""
      type="text/javascript"
      src="report_files/inject.js"
    ></script>
    <link
      type="text/css"
      rel="stylesheet"
      href="report_files/nature_checkout.css"
    />
    <script
      src="report_files/26100523.js"
      type="text/javascript"
      async=""
    ></script>
    <script
      src="report_files/12000044.js"
      type="text/javascript"
      async=""
    ></script>
    <script
      src="report_files/otBannerSdk.js"
      async=""
      type="text/javascript"
    ></script>
    <style id="onetrust-style">
      #onetrust-banner-sdk {
        -ms-text-size-adjust: 100%;
        -webkit-text-size-adjust: 100%;
      }
      #onetrust-banner-sdk .onetrust-vendors-list-handler {
        cursor: pointer;
        color: #1f96db;
        font-size: inherit;
        font-weight: bold;
        text-decoration: none;
        margin-left: 5px;
      }
      #onetrust-banner-sdk .onetrust-vendors-list-handler:hover {
        color: #1f96db;
      }
      #onetrust-banner-sdk .ot-close-icon,
      #onetrust-pc-sdk .ot-close-icon,
      #ot-sync-ntfy .ot-close-icon {
        background-image: url("data:image/svg+xml;base64,PHN2ZyB2ZXJzaW9uPSIxLjEiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgeG1sbnM6eGxpbms9Imh0dHA6Ly93d3cudzMub3JnLzE5OTkveGxpbmsiIHg9IjBweCIgeT0iMHB4IiB3aWR0aD0iMzQ4LjMzM3B4IiBoZWlnaHQ9IjM0OC4zMzNweCIgdmlld0JveD0iMCAwIDM0OC4zMzMgMzQ4LjMzNCIgc3R5bGU9ImVuYWJsZS1iYWNrZ3JvdW5kOm5ldyAwIDAgMzQ4LjMzMyAzNDguMzM0OyIgeG1sOnNwYWNlPSJwcmVzZXJ2ZSI+PGc+PHBhdGggZmlsbD0iIzU2NTY1NiIgZD0iTTMzNi41NTksNjguNjExTDIzMS4wMTYsMTc0LjE2NWwxMDUuNTQzLDEwNS41NDljMTUuNjk5LDE1LjcwNSwxNS42OTksNDEuMTQ1LDAsNTYuODVjLTcuODQ0LDcuODQ0LTE4LjEyOCwxMS43NjktMjguNDA3LDExLjc2OWMtMTAuMjk2LDAtMjAuNTgxLTMuOTE5LTI4LjQxOS0xMS43NjlMMTc0LjE2NywyMzEuMDAzTDY4LjYwOSwzMzYuNTYzYy03Ljg0Myw3Ljg0NC0xOC4xMjgsMTEuNzY5LTI4LjQxNiwxMS43NjljLTEwLjI4NSwwLTIwLjU2My0zLjkxOS0yOC40MTMtMTEuNzY5Yy0xNS42OTktMTUuNjk4LTE1LjY5OS00MS4xMzksMC01Ni44NWwxMDUuNTQtMTA1LjU0OUwxMS43NzQsNjguNjExYy0xNS42OTktMTUuNjk5LTE1LjY5OS00MS4xNDUsMC01Ni44NDRjMTUuNjk2LTE1LjY4Nyw0MS4xMjctMTUuNjg3LDU2LjgyOSwwbDEwNS41NjMsMTA1LjU1NEwyNzkuNzIxLDExLjc2N2MxNS43MDUtMTUuNjg3LDQxLjEzOS0xNS42ODcsNTYuODMyLDBDMzUyLjI1OCwyNy40NjYsMzUyLjI1OCw1Mi45MTIsMzM2LjU1OSw2OC42MTF6Ii8+PC9nPjwvc3ZnPg==");
        background-size: contain;
        background-repeat: no-repeat;
        background-position: center;
        height: 12px;
        width: 12px;
      }
      #onetrust-banner-sdk .powered-by-logo,
      #onetrust-banner-sdk .ot-pc-footer-logo a,
      #onetrust-pc-sdk .powered-by-logo,
      #onetrust-pc-sdk .ot-pc-footer-logo a,
      #ot-sync-ntfy .powered-by-logo,
      #ot-sync-ntfy .ot-pc-footer-logo a {
        background-size: contain;
        background-repeat: no-repeat;
        background-position: center;
        height: 25px;
        width: 152px;
        display: block;
      }
      #onetrust-banner-sdk h3 *,
      #onetrust-banner-sdk h4 *,
      #onetrust-banner-sdk h6 *,
      #onetrust-banner-sdk button *,
      #onetrust-banner-sdk a[data-parent-id] *,
      #onetrust-pc-sdk h3 *,
      #onetrust-pc-sdk h4 *,
      #onetrust-pc-sdk h6 *,
      #onetrust-pc-sdk button *,
      #onetrust-pc-sdk a[data-parent-id] *,
      #ot-sync-ntfy h3 *,
      #ot-sync-ntfy h4 *,
      #ot-sync-ntfy h6 *,
      #ot-sync-ntfy button *,
      #ot-sync-ntfy a[data-parent-id] * {
        font-size: inherit;
        font-weight: inherit;
        color: inherit;
      }
      #onetrust-banner-sdk .ot-hide,
      #onetrust-pc-sdk .ot-hide,
      #ot-sync-ntfy .ot-hide {
        display: none !important;
      }
      #onetrust-pc-sdk .ot-sdk-row .ot-sdk-column {
        padding: 0;
      }
      #onetrust-pc-sdk .ot-sdk-container {
        padding-right: 0;
      }
      #onetrust-pc-sdk .ot-sdk-row {
        flex-direction: initial;
        width: 100%;
      }
      #onetrust-pc-sdk [type="checkbox"]:checked,
      #onetrust-pc-sdk [type="checkbox"]:not(:checked) {
        pointer-events: initial;
      }
      #onetrust-pc-sdk [type="checkbox"]:disabled + label::before,
      #onetrust-pc-sdk [type="checkbox"]:disabled + label:after,
      #onetrust-pc-sdk [type="checkbox"]:disabled + label {
        pointer-events: none;
        opacity: 0.7;
      }
      #onetrust-pc-sdk #vendor-list-content {
        transform: translate3d(0, 0, 0);
      }
      #onetrust-pc-sdk li input[type="checkbox"] {
        z-index: 1;
      }
      #onetrust-pc-sdk li .ot-checkbox label {
        z-index: 2;
      }
      #onetrust-pc-sdk li .ot-checkbox input[type="checkbox"] {
        height: auto;
        width: auto;
      }
      #onetrust-pc-sdk li .host-title a,
      #onetrust-pc-sdk li .ot-host-name a,
      #onetrust-pc-sdk li .accordion-text,
      #onetrust-pc-sdk li .ot-acc-txt {
        z-index: 2;
        position: relative;
      }
      #onetrust-pc-sdk input {
        margin: 3px 0.1ex;
      }
      #onetrust-pc-sdk .toggle-always-active {
        opacity: 0.6;
        cursor: default;
      }
      #onetrust-pc-sdk .pc-logo,
      #onetrust-pc-sdk .ot-pc-logo {
        height: 60px;
        width: 180px;
        background-position: center;
        background-size: contain;
        background-repeat: no-repeat;
      }
      #onetrust-pc-sdk .ot-tooltip .ot-tooltiptext {
        visibility: hidden;
        width: 120px;
        background-color: #555;
        color: #fff;
        text-align: center;
        padding: 5px 0;
        border-radius: 6px;
        position: absolute;
        z-index: 1;
        bottom: 125%;
        left: 50%;
        margin-left: -60px;
        opacity: 0;
        transition: opacity 0.3s;
      }
      #onetrust-pc-sdk .ot-tooltip .ot-tooltiptext::after {
        content: "";
        position: absolute;
        top: 100%;
        left: 50%;
        margin-left: -5px;
        border-width: 5px;
        border-style: solid;
        border-color: #555 transparent transparent transparent;
      }
      #onetrust-pc-sdk .ot-tooltip:hover .ot-tooltiptext {
        visibility: visible;
        opacity: 1;
      }
      #onetrust-pc-sdk .ot-tooltip {
        position: relative;
        display: inline-block;
        z-index: 3;
      }
      #onetrust-pc-sdk .ot-tooltip svg {
        color: grey;
        height: 20px;
        width: 20px;
      }
      #onetrust-pc-sdk .screen-reader-only,
      #onetrust-pc-sdk .ot-scrn-rdr,
      .ot-sdk-cookie-policy .screen-reader-only,
      .ot-sdk-cookie-policy .ot-scrn-rdr {
        border: 0;
        clip: rect(0 0 0 0);
        height: 1px;
        margin: -1px;
        overflow: hidden;
        padding: 0;
        position: absolute;
        width: 1px;
      }
      #onetrust-pc-sdk.ot-fade-in,
      .onetrust-pc-dark-filter.ot-fade-in {
        animation-name: onetrust-fade-in;
        animation-duration: 400ms;
        animation-timing-function: ease-in-out;
      }
      #onetrust-pc-sdk.ot-hide {
        display: none !important;
      }
      .onetrust-pc-dark-filter.ot-hide {
        display: none !important;
      }
      #ot-sdk-btn.ot-sdk-show-settings,
      #ot-sdk-btn.optanon-show-settings {
        color: #68b631;
        border: 1px solid #68b631;
        height: auto;
        white-space: normal;
        word-wrap: break-word;
        padding: 0.8em 2em;
        font-size: 0.8em;
        line-height: 1.2;
        cursor: pointer;
        -moz-transition: 0.1s ease;
        -o-transition: 0.1s ease;
        -webkit-transition: 1s ease;
        transition: 0.1s ease;
      }
      #ot-sdk-btn.ot-sdk-show-settings:hover,
      #ot-sdk-btn.optanon-show-settings:hover {
        color: #fff;
        background-color: #68b631;
      }
      .onetrust-pc-dark-filter {
        background: rgba(0, 0, 0, 0.5);
        z-index: 2147483646;
        width: 100%;
        height: 100%;
        overflow: hidden;
        position: fixed;
        top: 0;
        bottom: 0;
        left: 0;
      }
      @keyframes onetrust-fade-in {
        0% {
          opacity: 0;
        }
        100% {
          opacity: 1;
        }
      }
      @media only screen and (min-width: 426px) and (max-width: 896px) and (orientation: landscape) {
        #onetrust-pc-sdk p {
          font-size: 0.75em;
        }
      }
      #onetrust-banner-sdk .banner-option-input:focus + label {
        outline: 1px solid #000;
        outline-style: auto;
      }
      #onetrust-banner-sdk,
      #onetrust-pc-sdk,
      #ot-sdk-cookie-policy,
      #ot-sync-ntfy {
        font-size: 16px;
      }
      #onetrust-banner-sdk *,
      #onetrust-banner-sdk ::after,
      #onetrust-banner-sdk ::before,
      #onetrust-pc-sdk *,
      #onetrust-pc-sdk ::after,
      #onetrust-pc-sdk ::before,
      #ot-sdk-cookie-policy *,
      #ot-sdk-cookie-policy ::after,
      #ot-sdk-cookie-policy ::before,
      #ot-sync-ntfy *,
      #ot-sync-ntfy ::after,
      #ot-sync-ntfy ::before {
        -webkit-box-sizing: content-box;
        -moz-box-sizing: content-box;
        box-sizing: content-box;
      }
      #onetrust-banner-sdk div,
      #onetrust-banner-sdk span,
      #onetrust-banner-sdk h1,
      #onetrust-banner-sdk h2,
      #onetrust-banner-sdk h3,
      #onetrust-banner-sdk h4,
      #onetrust-banner-sdk h5,
      #onetrust-banner-sdk h6,
      #onetrust-banner-sdk p,
      #onetrust-banner-sdk img,
      #onetrust-banner-sdk svg,
      #onetrust-banner-sdk button,
      #onetrust-banner-sdk section,
      #onetrust-banner-sdk a,
      #onetrust-banner-sdk label,
      #onetrust-banner-sdk input,
      #onetrust-banner-sdk ul,
      #onetrust-banner-sdk li,
      #onetrust-banner-sdk nav,
      #onetrust-banner-sdk table,
      #onetrust-banner-sdk thead,
      #onetrust-banner-sdk tr,
      #onetrust-banner-sdk td,
      #onetrust-banner-sdk tbody,
      #onetrust-banner-sdk .ot-main-content,
      #onetrust-banner-sdk .ot-toggle,
      #onetrust-banner-sdk #ot-content,
      #onetrust-banner-sdk #ot-pc-content,
      #onetrust-banner-sdk .checkbox,
      #onetrust-pc-sdk div,
      #onetrust-pc-sdk span,
      #onetrust-pc-sdk h1,
      #onetrust-pc-sdk h2,
      #onetrust-pc-sdk h3,
      #onetrust-pc-sdk h4,
      #onetrust-pc-sdk h5,
      #onetrust-pc-sdk h6,
      #onetrust-pc-sdk p,
      #onetrust-pc-sdk img,
      #onetrust-pc-sdk svg,
      #onetrust-pc-sdk button,
      #onetrust-pc-sdk section,
      #onetrust-pc-sdk a,
      #onetrust-pc-sdk label,
      #onetrust-pc-sdk input,
      #onetrust-pc-sdk ul,
      #onetrust-pc-sdk li,
      #onetrust-pc-sdk nav,
      #onetrust-pc-sdk table,
      #onetrust-pc-sdk thead,
      #onetrust-pc-sdk tr,
      #onetrust-pc-sdk td,
      #onetrust-pc-sdk tbody,
      #onetrust-pc-sdk .ot-main-content,
      #onetrust-pc-sdk .ot-toggle,
      #onetrust-pc-sdk #ot-content,
      #onetrust-pc-sdk #ot-pc-content,
      #onetrust-pc-sdk .checkbox,
      #ot-sdk-cookie-policy div,
      #ot-sdk-cookie-policy span,
      #ot-sdk-cookie-policy h1,
      #ot-sdk-cookie-policy h2,
      #ot-sdk-cookie-policy h3,
      #ot-sdk-cookie-policy h4,
      #ot-sdk-cookie-policy h5,
      #ot-sdk-cookie-policy h6,
      #ot-sdk-cookie-policy p,
      #ot-sdk-cookie-policy img,
      #ot-sdk-cookie-policy svg,
      #ot-sdk-cookie-policy button,
      #ot-sdk-cookie-policy section,
      #ot-sdk-cookie-policy a,
      #ot-sdk-cookie-policy label,
      #ot-sdk-cookie-policy input,
      #ot-sdk-cookie-policy ul,
      #ot-sdk-cookie-policy li,
      #ot-sdk-cookie-policy nav,
      #ot-sdk-cookie-policy table,
      #ot-sdk-cookie-policy thead,
      #ot-sdk-cookie-policy tr,
      #ot-sdk-cookie-policy td,
      #ot-sdk-cookie-policy tbody,
      #ot-sdk-cookie-policy .ot-main-content,
      #ot-sdk-cookie-policy .ot-toggle,
      #ot-sdk-cookie-policy #ot-content,
      #ot-sdk-cookie-policy #ot-pc-content,
      #ot-sdk-cookie-policy .checkbox,
      #ot-sync-ntfy div,
      #ot-sync-ntfy span,
      #ot-sync-ntfy h1,
      #ot-sync-ntfy h2,
      #ot-sync-ntfy h3,
      #ot-sync-ntfy h4,
      #ot-sync-ntfy h5,
      #ot-sync-ntfy h6,
      #ot-sync-ntfy p,
      #ot-sync-ntfy img,
      #ot-sync-ntfy svg,
      #ot-sync-ntfy button,
      #ot-sync-ntfy section,
      #ot-sync-ntfy a,
      #ot-sync-ntfy label,
      #ot-sync-ntfy input,
      #ot-sync-ntfy ul,
      #ot-sync-ntfy li,
      #ot-sync-ntfy nav,
      #ot-sync-ntfy table,
      #ot-sync-ntfy thead,
      #ot-sync-ntfy tr,
      #ot-sync-ntfy td,
      #ot-sync-ntfy tbody,
      #ot-sync-ntfy .ot-main-content,
      #ot-sync-ntfy .ot-toggle,
      #ot-sync-ntfy #ot-content,
      #ot-sync-ntfy #ot-pc-content,
      #ot-sync-ntfy .checkbox {
        font-family: inherit;
        font-weight: normal;
        -webkit-font-smoothing: auto;
        letter-spacing: normal;
        line-height: normal;
        padding: 0;
        margin: 0;
        height: auto;
        min-height: 0;
        max-height: none;
        width: auto;
        min-width: 0;
        max-width: none;
        border-radius: 0;
        border: none;
        clear: none;
        float: none;
        position: static;
        bottom: auto;
        left: auto;
        right: auto;
        top: auto;
        text-align: left;
        text-decoration: none;
        text-indent: 0;
        text-shadow: none;
        text-transform: none;
        white-space: normal;
        background: none;
        overflow: visible;
        vertical-align: baseline;
        visibility: visible;
        z-index: auto;
        box-shadow: none;
      }
      #onetrust-banner-sdk label:before,
      #onetrust-banner-sdk label:after,
      #onetrust-banner-sdk .checkbox:after,
      #onetrust-banner-sdk .checkbox:before,
      #onetrust-pc-sdk label:before,
      #onetrust-pc-sdk label:after,
      #onetrust-pc-sdk .checkbox:after,
      #onetrust-pc-sdk .checkbox:before,
      #ot-sdk-cookie-policy label:before,
      #ot-sdk-cookie-policy label:after,
      #ot-sdk-cookie-policy .checkbox:after,
      #ot-sdk-cookie-policy .checkbox:before,
      #ot-sync-ntfy label:before,
      #ot-sync-ntfy label:after,
      #ot-sync-ntfy .checkbox:after,
      #ot-sync-ntfy .checkbox:before {
        content: "";
        content: none;
      }
      #onetrust-banner-sdk .ot-sdk-container,
      #onetrust-pc-sdk .ot-sdk-container,
      #ot-sdk-cookie-policy .ot-sdk-container {
        position: relative;
        width: 100%;
        max-width: 100%;
        margin: 0 auto;
        padding: 0 20px;
        box-sizing: border-box;
      }
      #onetrust-banner-sdk .ot-sdk-column,
      #onetrust-banner-sdk .ot-sdk-columns,
      #onetrust-pc-sdk .ot-sdk-column,
      #onetrust-pc-sdk .ot-sdk-columns,
      #ot-sdk-cookie-policy .ot-sdk-column,
      #ot-sdk-cookie-policy .ot-sdk-columns {
        width: 100%;
        float: left;
        box-sizing: border-box;
        padding: 0;
        display: initial;
      }
      @media (min-width: 400px) {
        #onetrust-banner-sdk .ot-sdk-container,
        #onetrust-pc-sdk .ot-sdk-container,
        #ot-sdk-cookie-policy .ot-sdk-container {
          width: 90%;
          padding: 0;
        }
      }
      @media (min-width: 550px) {
        #onetrust-banner-sdk .ot-sdk-container,
        #onetrust-pc-sdk .ot-sdk-container,
        #ot-sdk-cookie-policy .ot-sdk-container {
          width: 100%;
        }
        #onetrust-banner-sdk .ot-sdk-column,
        #onetrust-banner-sdk .ot-sdk-columns,
        #onetrust-pc-sdk .ot-sdk-column,
        #onetrust-pc-sdk .ot-sdk-columns,
        #ot-sdk-cookie-policy .ot-sdk-column,
        #ot-sdk-cookie-policy .ot-sdk-columns {
          margin-left: 4%;
        }
        #onetrust-banner-sdk .ot-sdk-column:first-child,
        #onetrust-banner-sdk .ot-sdk-columns:first-child,
        #onetrust-pc-sdk .ot-sdk-column:first-child,
        #onetrust-pc-sdk .ot-sdk-columns:first-child,
        #ot-sdk-cookie-policy .ot-sdk-column:first-child,
        #ot-sdk-cookie-policy .ot-sdk-columns:first-child {
          margin-left: 0;
        }
        #onetrust-banner-sdk .ot-sdk-one.ot-sdk-column,
        #onetrust-banner-sdk .ot-sdk-one.ot-sdk-columns,
        #onetrust-pc-sdk .ot-sdk-one.ot-sdk-column,
        #onetrust-pc-sdk .ot-sdk-one.ot-sdk-columns,
        #ot-sdk-cookie-policy .ot-sdk-one.ot-sdk-column,
        #ot-sdk-cookie-policy .ot-sdk-one.ot-sdk-columns {
          width: 4.66666666667%;
        }
        #onetrust-banner-sdk .ot-sdk-two.ot-sdk-columns,
        #onetrust-pc-sdk .ot-sdk-two.ot-sdk-columns,
        #ot-sdk-cookie-policy .ot-sdk-two.ot-sdk-columns {
          width: 13.3333333333%;
        }
        #onetrust-banner-sdk .ot-sdk-three.ot-sdk-columns,
        #onetrust-pc-sdk .ot-sdk-three.ot-sdk-columns,
        #ot-sdk-cookie-policy .ot-sdk-three.ot-sdk-columns {
          width: 22%;
        }
        #onetrust-banner-sdk .ot-sdk-four.ot-sdk-columns,
        #onetrust-pc-sdk .ot-sdk-four.ot-sdk-columns,
        #ot-sdk-cookie-policy .ot-sdk-four.ot-sdk-columns {
          width: 30.6666666667%;
        }
        #onetrust-banner-sdk .ot-sdk-five.ot-sdk-columns,
        #onetrust-pc-sdk .ot-sdk-five.ot-sdk-columns,
        #ot-sdk-cookie-policy .ot-sdk-five.ot-sdk-columns {
          width: 39.3333333333%;
        }
        #onetrust-banner-sdk .ot-sdk-six.ot-sdk-columns,
        #onetrust-pc-sdk .ot-sdk-six.ot-sdk-columns,
        #ot-sdk-cookie-policy .ot-sdk-six.ot-sdk-columns {
          width: 48%;
        }
        #onetrust-banner-sdk .ot-sdk-seven.ot-sdk-columns,
        #onetrust-pc-sdk .ot-sdk-seven.ot-sdk-columns,
        #ot-sdk-cookie-policy .ot-sdk-seven.ot-sdk-columns {
          width: 56.6666666667%;
        }
        #onetrust-banner-sdk .ot-sdk-eight.ot-sdk-columns,
        #onetrust-pc-sdk .ot-sdk-eight.ot-sdk-columns,
        #ot-sdk-cookie-policy .ot-sdk-eight.ot-sdk-columns {
          width: 65.3333333333%;
        }
        #onetrust-banner-sdk .ot-sdk-nine.ot-sdk-columns,
        #onetrust-pc-sdk .ot-sdk-nine.ot-sdk-columns,
        #ot-sdk-cookie-policy .ot-sdk-nine.ot-sdk-columns {
          width: 74%;
        }
        #onetrust-banner-sdk .ot-sdk-ten.ot-sdk-columns,
        #onetrust-pc-sdk .ot-sdk-ten.ot-sdk-columns,
        #ot-sdk-cookie-policy .ot-sdk-ten.ot-sdk-columns {
          width: 82.6666666667%;
        }
        #onetrust-banner-sdk .ot-sdk-eleven.ot-sdk-columns,
        #onetrust-pc-sdk .ot-sdk-eleven.ot-sdk-columns,
        #ot-sdk-cookie-policy .ot-sdk-eleven.ot-sdk-columns {
          width: 91.3333333333%;
        }
        #onetrust-banner-sdk .ot-sdk-twelve.ot-sdk-columns,
        #onetrust-pc-sdk .ot-sdk-twelve.ot-sdk-columns,
        #ot-sdk-cookie-policy .ot-sdk-twelve.ot-sdk-columns {
          width: 100%;
          margin-left: 0;
        }
        #onetrust-banner-sdk .ot-sdk-one-third.ot-sdk-column,
        #onetrust-pc-sdk .ot-sdk-one-third.ot-sdk-column,
        #ot-sdk-cookie-policy .ot-sdk-one-third.ot-sdk-column {
          width: 30.6666666667%;
        }
        #onetrust-banner-sdk .ot-sdk-two-thirds.ot-sdk-column,
        #onetrust-pc-sdk .ot-sdk-two-thirds.ot-sdk-column,
        #ot-sdk-cookie-policy .ot-sdk-two-thirds.ot-sdk-column {
          width: 65.3333333333%;
        }
        #onetrust-banner-sdk .ot-sdk-one-half.ot-sdk-column,
        #onetrust-pc-sdk .ot-sdk-one-half.ot-sdk-column,
        #ot-sdk-cookie-policy .ot-sdk-one-half.ot-sdk-column {
          width: 48%;
        }
        #onetrust-banner-sdk .ot-sdk-offset-by-one.ot-sdk-column,
        #onetrust-banner-sdk .ot-sdk-offset-by-one.ot-sdk-columns,
        #onetrust-pc-sdk .ot-sdk-offset-by-one.ot-sdk-column,
        #onetrust-pc-sdk .ot-sdk-offset-by-one.ot-sdk-columns,
        #ot-sdk-cookie-policy .ot-sdk-offset-by-one.ot-sdk-column,
        #ot-sdk-cookie-policy .ot-sdk-offset-by-one.ot-sdk-columns {
          margin-left: 8.66666666667%;
        }
        #onetrust-banner-sdk .ot-sdk-offset-by-two.ot-sdk-column,
        #onetrust-banner-sdk .ot-sdk-offset-by-two.ot-sdk-columns,
        #onetrust-pc-sdk .ot-sdk-offset-by-two.ot-sdk-column,
        #onetrust-pc-sdk .ot-sdk-offset-by-two.ot-sdk-columns,
        #ot-sdk-cookie-policy .ot-sdk-offset-by-two.ot-sdk-column,
        #ot-sdk-cookie-policy .ot-sdk-offset-by-two.ot-sdk-columns {
          margin-left: 17.3333333333%;
        }
        #onetrust-banner-sdk .ot-sdk-offset-by-three.ot-sdk-column,
        #onetrust-banner-sdk .ot-sdk-offset-by-three.ot-sdk-columns,
        #onetrust-pc-sdk .ot-sdk-offset-by-three.ot-sdk-column,
        #onetrust-pc-sdk .ot-sdk-offset-by-three.ot-sdk-columns,
        #ot-sdk-cookie-policy .ot-sdk-offset-by-three.ot-sdk-column,
        #ot-sdk-cookie-policy .ot-sdk-offset-by-three.ot-sdk-columns {
          margin-left: 26%;
        }
        #onetrust-banner-sdk .ot-sdk-offset-by-four.ot-sdk-column,
        #onetrust-banner-sdk .ot-sdk-offset-by-four.ot-sdk-columns,
        #onetrust-pc-sdk .ot-sdk-offset-by-four.ot-sdk-column,
        #onetrust-pc-sdk .ot-sdk-offset-by-four.ot-sdk-columns,
        #ot-sdk-cookie-policy .ot-sdk-offset-by-four.ot-sdk-column,
        #ot-sdk-cookie-policy .ot-sdk-offset-by-four.ot-sdk-columns {
          margin-left: 34.6666666667%;
        }
        #onetrust-banner-sdk .ot-sdk-offset-by-five.ot-sdk-column,
        #onetrust-banner-sdk .ot-sdk-offset-by-five.ot-sdk-columns,
        #onetrust-pc-sdk .ot-sdk-offset-by-five.ot-sdk-column,
        #onetrust-pc-sdk .ot-sdk-offset-by-five.ot-sdk-columns,
        #ot-sdk-cookie-policy .ot-sdk-offset-by-five.ot-sdk-column,
        #ot-sdk-cookie-policy .ot-sdk-offset-by-five.ot-sdk-columns {
          margin-left: 43.3333333333%;
        }
        #onetrust-banner-sdk .ot-sdk-offset-by-six.ot-sdk-column,
        #onetrust-banner-sdk .ot-sdk-offset-by-six.ot-sdk-columns,
        #onetrust-pc-sdk .ot-sdk-offset-by-six.ot-sdk-column,
        #onetrust-pc-sdk .ot-sdk-offset-by-six.ot-sdk-columns,
        #ot-sdk-cookie-policy .ot-sdk-offset-by-six.ot-sdk-column,
        #ot-sdk-cookie-policy .ot-sdk-offset-by-six.ot-sdk-columns {
          margin-left: 52%;
        }
        #onetrust-banner-sdk .ot-sdk-offset-by-seven.ot-sdk-column,
        #onetrust-banner-sdk .ot-sdk-offset-by-seven.ot-sdk-columns,
        #onetrust-pc-sdk .ot-sdk-offset-by-seven.ot-sdk-column,
        #onetrust-pc-sdk .ot-sdk-offset-by-seven.ot-sdk-columns,
        #ot-sdk-cookie-policy .ot-sdk-offset-by-seven.ot-sdk-column,
        #ot-sdk-cookie-policy .ot-sdk-offset-by-seven.ot-sdk-columns {
          margin-left: 60.6666666667%;
        }
        #onetrust-banner-sdk .ot-sdk-offset-by-eight.ot-sdk-column,
        #onetrust-banner-sdk .ot-sdk-offset-by-eight.ot-sdk-columns,
        #onetrust-pc-sdk .ot-sdk-offset-by-eight.ot-sdk-column,
        #onetrust-pc-sdk .ot-sdk-offset-by-eight.ot-sdk-columns,
        #ot-sdk-cookie-policy .ot-sdk-offset-by-eight.ot-sdk-column,
        #ot-sdk-cookie-policy .ot-sdk-offset-by-eight.ot-sdk-columns {
          margin-left: 69.3333333333%;
        }
        #onetrust-banner-sdk .ot-sdk-offset-by-nine.ot-sdk-column,
        #onetrust-banner-sdk .ot-sdk-offset-by-nine.ot-sdk-columns,
        #onetrust-pc-sdk .ot-sdk-offset-by-nine.ot-sdk-column,
        #onetrust-pc-sdk .ot-sdk-offset-by-nine.ot-sdk-columns,
        #ot-sdk-cookie-policy .ot-sdk-offset-by-nine.ot-sdk-column,
        #ot-sdk-cookie-policy .ot-sdk-offset-by-nine.ot-sdk-columns {
          margin-left: 78%;
        }
        #onetrust-banner-sdk .ot-sdk-offset-by-ten.ot-sdk-column,
        #onetrust-banner-sdk .ot-sdk-offset-by-ten.ot-sdk-columns,
        #onetrust-pc-sdk .ot-sdk-offset-by-ten.ot-sdk-column,
        #onetrust-pc-sdk .ot-sdk-offset-by-ten.ot-sdk-columns,
        #ot-sdk-cookie-policy .ot-sdk-offset-by-ten.ot-sdk-column,
        #ot-sdk-cookie-policy .ot-sdk-offset-by-ten.ot-sdk-columns {
          margin-left: 86.6666666667%;
        }
        #onetrust-banner-sdk .ot-sdk-offset-by-eleven.ot-sdk-column,
        #onetrust-banner-sdk .ot-sdk-offset-by-eleven.ot-sdk-columns,
        #onetrust-pc-sdk .ot-sdk-offset-by-eleven.ot-sdk-column,
        #onetrust-pc-sdk .ot-sdk-offset-by-eleven.ot-sdk-columns,
        #ot-sdk-cookie-policy .ot-sdk-offset-by-eleven.ot-sdk-column,
        #ot-sdk-cookie-policy .ot-sdk-offset-by-eleven.ot-sdk-columns {
          margin-left: 95.3333333333%;
        }
        #onetrust-banner-sdk .ot-sdk-offset-by-one-third.ot-sdk-column,
        #onetrust-banner-sdk .ot-sdk-offset-by-one-third.ot-sdk-columns,
        #onetrust-pc-sdk .ot-sdk-offset-by-one-third.ot-sdk-column,
        #onetrust-pc-sdk .ot-sdk-offset-by-one-third.ot-sdk-columns,
        #ot-sdk-cookie-policy .ot-sdk-offset-by-one-third.ot-sdk-column,
        #ot-sdk-cookie-policy .ot-sdk-offset-by-one-third.ot-sdk-columns {
          margin-left: 34.6666666667%;
        }
        #onetrust-banner-sdk .ot-sdk-offset-by-two-thirds.ot-sdk-column,
        #onetrust-banner-sdk .ot-sdk-offset-by-two-thirds.ot-sdk-columns,
        #onetrust-pc-sdk .ot-sdk-offset-by-two-thirds.ot-sdk-column,
        #onetrust-pc-sdk .ot-sdk-offset-by-two-thirds.ot-sdk-columns,
        #ot-sdk-cookie-policy .ot-sdk-offset-by-two-thirds.ot-sdk-column,
        #ot-sdk-cookie-policy .ot-sdk-offset-by-two-thirds.ot-sdk-columns {
          margin-left: 69.3333333333%;
        }
        #onetrust-banner-sdk .ot-sdk-offset-by-one-half.ot-sdk-column,
        #onetrust-banner-sdk .ot-sdk-offset-by-one-half.ot-sdk-columns,
        #onetrust-pc-sdk .ot-sdk-offset-by-one-half.ot-sdk-column,
        #onetrust-pc-sdk .ot-sdk-offset-by-one-half.ot-sdk-columns,
        #ot-sdk-cookie-policy .ot-sdk-offset-by-one-half.ot-sdk-column,
        #ot-sdk-cookie-policy .ot-sdk-offset-by-one-half.ot-sdk-columns {
          margin-left: 52%;
        }
      }
      #onetrust-banner-sdk h1,
      #onetrust-banner-sdk h2,
      #onetrust-banner-sdk h3,
      #onetrust-banner-sdk h4,
      #onetrust-banner-sdk h5,
      #onetrust-banner-sdk h6,
      #onetrust-pc-sdk h1,
      #onetrust-pc-sdk h2,
      #onetrust-pc-sdk h3,
      #onetrust-pc-sdk h4,
      #onetrust-pc-sdk h5,
      #onetrust-pc-sdk h6,
      #ot-sdk-cookie-policy h1,
      #ot-sdk-cookie-policy h2,
      #ot-sdk-cookie-policy h3,
      #ot-sdk-cookie-policy h4,
      #ot-sdk-cookie-policy h5,
      #ot-sdk-cookie-policy h6 {
        margin-top: 0;
        font-weight: 600;
        font-family: inherit;
      }
      #onetrust-banner-sdk h1,
      #onetrust-pc-sdk h1,
      #ot-sdk-cookie-policy h1 {
        font-size: 1.5rem;
        line-height: 1.2;
      }
      #onetrust-banner-sdk h2,
      #onetrust-pc-sdk h2,
      #ot-sdk-cookie-policy h2 {
        font-size: 1.5rem;
        line-height: 1.25;
      }
      #onetrust-banner-sdk h3,
      #onetrust-pc-sdk h3,
      #ot-sdk-cookie-policy h3 {
        font-size: 1.5rem;
        line-height: 1.3;
      }
      #onetrust-banner-sdk h4,
      #onetrust-pc-sdk h4,
      #ot-sdk-cookie-policy h4 {
        font-size: 1.5rem;
        line-height: 1.35;
      }
      #onetrust-banner-sdk h5,
      #onetrust-pc-sdk h5,
      #ot-sdk-cookie-policy h5 {
        font-size: 1.5rem;
        line-height: 1.5;
      }
      #onetrust-banner-sdk h6,
      #onetrust-pc-sdk h6,
      #ot-sdk-cookie-policy h6 {
        font-size: 1.5rem;
        line-height: 1.6;
      }
      @media (min-width: 550px) {
        #onetrust-banner-sdk h1,
        #onetrust-pc-sdk h1,
        #ot-sdk-cookie-policy h1 {
          font-size: 1.5rem;
        }
        #onetrust-banner-sdk h2,
        #onetrust-pc-sdk h2,
        #ot-sdk-cookie-policy h2 {
          font-size: 1.5rem;
        }
        #onetrust-banner-sdk h3,
        #onetrust-pc-sdk h3,
        #ot-sdk-cookie-policy h3 {
          font-size: 1.5rem;
        }
        #onetrust-banner-sdk h4,
        #onetrust-pc-sdk h4,
        #ot-sdk-cookie-policy h4 {
          font-size: 1.5rem;
        }
        #onetrust-banner-sdk h5,
        #onetrust-pc-sdk h5,
        #ot-sdk-cookie-policy h5 {
          font-size: 1.5rem;
        }
        #onetrust-banner-sdk h6,
        #onetrust-pc-sdk h6,
        #ot-sdk-cookie-policy h6 {
          font-size: 1.5rem;
        }
      }
      #onetrust-banner-sdk p,
      #onetrust-pc-sdk p,
      #ot-sdk-cookie-policy p {
        margin: 0 0 1em 0;
        font-family: inherit;
        line-height: normal;
      }
      #onetrust-banner-sdk a,
      #onetrust-pc-sdk a,
      #ot-sdk-cookie-policy a {
        color: #565656;
        text-decoration: underline;
      }
      #onetrust-banner-sdk a:hover,
      #onetrust-pc-sdk a:hover,
      #ot-sdk-cookie-policy a:hover {
        color: #565656;
        text-decoration: none;
      }
      #onetrust-banner-sdk .ot-sdk-button,
      #onetrust-banner-sdk button,
      #onetrust-pc-sdk .ot-sdk-button,
      #onetrust-pc-sdk button,
      #ot-sdk-cookie-policy .ot-sdk-button,
      #ot-sdk-cookie-policy button {
        margin-bottom: 1rem;
        font-family: inherit;
      }
      #onetrust-banner-sdk .ot-sdk-button,
      #onetrust-banner-sdk button,
      #onetrust-banner-sdk input[type="submit"],
      #onetrust-banner-sdk input[type="reset"],
      #onetrust-banner-sdk input[type="button"],
      #onetrust-pc-sdk .ot-sdk-button,
      #onetrust-pc-sdk button,
      #onetrust-pc-sdk input[type="submit"],
      #onetrust-pc-sdk input[type="reset"],
      #onetrust-pc-sdk input[type="button"],
      #ot-sdk-cookie-policy .ot-sdk-button,
      #ot-sdk-cookie-policy button,
      #ot-sdk-cookie-policy input[type="submit"],
      #ot-sdk-cookie-policy input[type="reset"],
      #ot-sdk-cookie-policy input[type="button"] {
        display: inline-block;
        height: 38px;
        padding: 0 30px;
        color: #555;
        text-align: center;
        font-size: 0.9em;
        font-weight: 400;
        line-height: 38px;
        letter-spacing: 0.01em;
        text-decoration: none;
        white-space: nowrap;
        background-color: transparent;
        border-radius: 2px;
        border: 1px solid #bbb;
        cursor: pointer;
        box-sizing: border-box;
      }
      #onetrust-banner-sdk .ot-sdk-button:hover,
      #onetrust-banner-sdk :not(.ot-leg-btn-container) > button:hover,
      #onetrust-banner-sdk input[type="submit"]:hover,
      #onetrust-banner-sdk input[type="reset"]:hover,
      #onetrust-banner-sdk input[type="button"]:hover,
      #onetrust-banner-sdk .ot-sdk-button:focus,
      #onetrust-banner-sdk :not(.ot-leg-btn-container) > button:focus,
      #onetrust-banner-sdk input[type="submit"]:focus,
      #onetrust-banner-sdk input[type="reset"]:focus,
      #onetrust-banner-sdk input[type="button"]:focus,
      #onetrust-pc-sdk .ot-sdk-button:hover,
      #onetrust-pc-sdk :not(.ot-leg-btn-container) > button:hover,
      #onetrust-pc-sdk input[type="submit"]:hover,
      #onetrust-pc-sdk input[type="reset"]:hover,
      #onetrust-pc-sdk input[type="button"]:hover,
      #onetrust-pc-sdk .ot-sdk-button:focus,
      #onetrust-pc-sdk :not(.ot-leg-btn-container) > button:focus,
      #onetrust-pc-sdk input[type="submit"]:focus,
      #onetrust-pc-sdk input[type="reset"]:focus,
      #onetrust-pc-sdk input[type="button"]:focus,
      #ot-sdk-cookie-policy .ot-sdk-button:hover,
      #ot-sdk-cookie-policy :not(.ot-leg-btn-container) > button:hover,
      #ot-sdk-cookie-policy input[type="submit"]:hover,
      #ot-sdk-cookie-policy input[type="reset"]:hover,
      #ot-sdk-cookie-policy input[type="button"]:hover,
      #ot-sdk-cookie-policy .ot-sdk-button:focus,
      #ot-sdk-cookie-policy :not(.ot-leg-btn-container) > button:focus,
      #ot-sdk-cookie-policy input[type="submit"]:focus,
      #ot-sdk-cookie-policy input[type="reset"]:focus,
      #ot-sdk-cookie-policy input[type="button"]:focus {
        color: #333;
        border-color: #888;
        opacity: 0.7;
      }
      #onetrust-banner-sdk .ot-sdk-button:focus,
      #onetrust-banner-sdk :not(.ot-leg-btn-container) > button:focus,
      #onetrust-banner-sdk input[type="submit"]:focus,
      #onetrust-banner-sdk input[type="reset"]:focus,
      #onetrust-banner-sdk input[type="button"]:focus,
      #onetrust-pc-sdk .ot-sdk-button:focus,
      #onetrust-pc-sdk :not(.ot-leg-btn-container) > button:focus,
      #onetrust-pc-sdk input[type="submit"]:focus,
      #onetrust-pc-sdk input[type="reset"]:focus,
      #onetrust-pc-sdk input[type="button"]:focus,
      #ot-sdk-cookie-policy .ot-sdk-button:focus,
      #ot-sdk-cookie-policy :not(.ot-leg-btn-container) > button:focus,
      #ot-sdk-cookie-policy input[type="submit"]:focus,
      #ot-sdk-cookie-policy input[type="reset"]:focus,
      #ot-sdk-cookie-policy input[type="button"]:focus {
        outline: 2px solid #000;
      }
      #onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary,
      #onetrust-banner-sdk button.ot-sdk-button-primary,
      #onetrust-banner-sdk input[type="submit"].ot-sdk-button-primary,
      #onetrust-banner-sdk input[type="reset"].ot-sdk-button-primary,
      #onetrust-banner-sdk input[type="button"].ot-sdk-button-primary,
      #onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary,
      #onetrust-pc-sdk button.ot-sdk-button-primary,
      #onetrust-pc-sdk input[type="submit"].ot-sdk-button-primary,
      #onetrust-pc-sdk input[type="reset"].ot-sdk-button-primary,
      #onetrust-pc-sdk input[type="button"].ot-sdk-button-primary,
      #ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary,
      #ot-sdk-cookie-policy button.ot-sdk-button-primary,
      #ot-sdk-cookie-policy input[type="submit"].ot-sdk-button-primary,
      #ot-sdk-cookie-policy input[type="reset"].ot-sdk-button-primary,
      #ot-sdk-cookie-policy input[type="button"].ot-sdk-button-primary {
        color: #fff;
        background-color: #33c3f0;
        border-color: #33c3f0;
      }
      #onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:hover,
      #onetrust-banner-sdk button.ot-sdk-button-primary:hover,
      #onetrust-banner-sdk input[type="submit"].ot-sdk-button-primary:hover,
      #onetrust-banner-sdk input[type="reset"].ot-sdk-button-primary:hover,
      #onetrust-banner-sdk input[type="button"].ot-sdk-button-primary:hover,
      #onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:focus,
      #onetrust-banner-sdk button.ot-sdk-button-primary:focus,
      #onetrust-banner-sdk input[type="submit"].ot-sdk-button-primary:focus,
      #onetrust-banner-sdk input[type="reset"].ot-sdk-button-primary:focus,
      #onetrust-banner-sdk input[type="button"].ot-sdk-button-primary:focus,
      #onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:hover,
      #onetrust-pc-sdk button.ot-sdk-button-primary:hover,
      #onetrust-pc-sdk input[type="submit"].ot-sdk-button-primary:hover,
      #onetrust-pc-sdk input[type="reset"].ot-sdk-button-primary:hover,
      #onetrust-pc-sdk input[type="button"].ot-sdk-button-primary:hover,
      #onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:focus,
      #onetrust-pc-sdk button.ot-sdk-button-primary:focus,
      #onetrust-pc-sdk input[type="submit"].ot-sdk-button-primary:focus,
      #onetrust-pc-sdk input[type="reset"].ot-sdk-button-primary:focus,
      #onetrust-pc-sdk input[type="button"].ot-sdk-button-primary:focus,
      #ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:hover,
      #ot-sdk-cookie-policy button.ot-sdk-button-primary:hover,
      #ot-sdk-cookie-policy input[type="submit"].ot-sdk-button-primary:hover,
      #ot-sdk-cookie-policy input[type="reset"].ot-sdk-button-primary:hover,
      #ot-sdk-cookie-policy input[type="button"].ot-sdk-button-primary:hover,
      #ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:focus,
      #ot-sdk-cookie-policy button.ot-sdk-button-primary:focus,
      #ot-sdk-cookie-policy input[type="submit"].ot-sdk-button-primary:focus,
      #ot-sdk-cookie-policy input[type="reset"].ot-sdk-button-primary:focus,
      #ot-sdk-cookie-policy input[type="button"].ot-sdk-button-primary:focus {
        color: #fff;
        background-color: #1eaedb;
        border-color: #1eaedb;
      }
      #onetrust-banner-sdk input[type="email"],
      #onetrust-banner-sdk input[type="number"],
      #onetrust-banner-sdk input[type="search"],
      #onetrust-banner-sdk input[type="text"],
      #onetrust-banner-sdk input[type="tel"],
      #onetrust-banner-sdk input[type="url"],
      #onetrust-banner-sdk input[type="password"],
      #onetrust-banner-sdk textarea,
      #onetrust-banner-sdk select,
      #onetrust-pc-sdk input[type="email"],
      #onetrust-pc-sdk input[type="number"],
      #onetrust-pc-sdk input[type="search"],
      #onetrust-pc-sdk input[type="text"],
      #onetrust-pc-sdk input[type="tel"],
      #onetrust-pc-sdk input[type="url"],
      #onetrust-pc-sdk input[type="password"],
      #onetrust-pc-sdk textarea,
      #onetrust-pc-sdk select,
      #ot-sdk-cookie-policy input[type="email"],
      #ot-sdk-cookie-policy input[type="number"],
      #ot-sdk-cookie-policy input[type="search"],
      #ot-sdk-cookie-policy input[type="text"],
      #ot-sdk-cookie-policy input[type="tel"],
      #ot-sdk-cookie-policy input[type="url"],
      #ot-sdk-cookie-policy input[type="password"],
      #ot-sdk-cookie-policy textarea,
      #ot-sdk-cookie-policy select {
        height: 38px;
        padding: 6px 10px;
        background-color: #fff;
        border: 1px solid #d1d1d1;
        border-radius: 4px;
        box-shadow: none;
        box-sizing: border-box;
      }
      #onetrust-banner-sdk input[type="email"],
      #onetrust-banner-sdk input[type="number"],
      #onetrust-banner-sdk input[type="search"],
      #onetrust-banner-sdk input[type="text"],
      #onetrust-banner-sdk input[type="tel"],
      #onetrust-banner-sdk input[type="url"],
      #onetrust-banner-sdk input[type="password"],
      #onetrust-banner-sdk textarea,
      #onetrust-pc-sdk input[type="email"],
      #onetrust-pc-sdk input[type="number"],
      #onetrust-pc-sdk input[type="search"],
      #onetrust-pc-sdk input[type="text"],
      #onetrust-pc-sdk input[type="tel"],
      #onetrust-pc-sdk input[type="url"],
      #onetrust-pc-sdk input[type="password"],
      #onetrust-pc-sdk textarea,
      #ot-sdk-cookie-policy input[type="email"],
      #ot-sdk-cookie-policy input[type="number"],
      #ot-sdk-cookie-policy input[type="search"],
      #ot-sdk-cookie-policy input[type="text"],
      #ot-sdk-cookie-policy input[type="tel"],
      #ot-sdk-cookie-policy input[type="url"],
      #ot-sdk-cookie-policy input[type="password"],
      #ot-sdk-cookie-policy textarea {
        -webkit-appearance: none;
        -moz-appearance: none;
        appearance: none;
      }
      #onetrust-banner-sdk textarea,
      #onetrust-pc-sdk textarea,
      #ot-sdk-cookie-policy textarea {
        min-height: 65px;
        padding-top: 6px;
        padding-bottom: 6px;
      }
      #onetrust-banner-sdk input[type="email"]:focus,
      #onetrust-banner-sdk input[type="number"]:focus,
      #onetrust-banner-sdk input[type="search"]:focus,
      #onetrust-banner-sdk input[type="text"]:focus,
      #onetrust-banner-sdk input[type="tel"]:focus,
      #onetrust-banner-sdk input[type="url"]:focus,
      #onetrust-banner-sdk input[type="password"]:focus,
      #onetrust-banner-sdk textarea:focus,
      #onetrust-banner-sdk select:focus,
      #onetrust-pc-sdk input[type="email"]:focus,
      #onetrust-pc-sdk input[type="number"]:focus,
      #onetrust-pc-sdk input[type="search"]:focus,
      #onetrust-pc-sdk input[type="text"]:focus,
      #onetrust-pc-sdk input[type="tel"]:focus,
      #onetrust-pc-sdk input[type="url"]:focus,
      #onetrust-pc-sdk input[type="password"]:focus,
      #onetrust-pc-sdk textarea:focus,
      #onetrust-pc-sdk select:focus,
      #ot-sdk-cookie-policy input[type="email"]:focus,
      #ot-sdk-cookie-policy input[type="number"]:focus,
      #ot-sdk-cookie-policy input[type="search"]:focus,
      #ot-sdk-cookie-policy input[type="text"]:focus,
      #ot-sdk-cookie-policy input[type="tel"]:focus,
      #ot-sdk-cookie-policy input[type="url"]:focus,
      #ot-sdk-cookie-policy input[type="password"]:focus,
      #ot-sdk-cookie-policy textarea:focus,
      #ot-sdk-cookie-policy select:focus {
        border: 1px solid #000;
        outline: 0;
      }
      #onetrust-banner-sdk label,
      #onetrust-banner-sdk legend,
      #onetrust-pc-sdk label,
      #onetrust-pc-sdk legend,
      #ot-sdk-cookie-policy label,
      #ot-sdk-cookie-policy legend {
        display: block;
        margin-bottom: 0.5rem;
        font-weight: 600;
      }
      #onetrust-banner-sdk fieldset,
      #onetrust-pc-sdk fieldset,
      #ot-sdk-cookie-policy fieldset {
        padding: 0;
        border-width: 0;
      }
      #onetrust-banner-sdk input[type="checkbox"],
      #onetrust-banner-sdk input[type="radio"],
      #onetrust-pc-sdk input[type="checkbox"],
      #onetrust-pc-sdk input[type="radio"],
      #ot-sdk-cookie-policy input[type="checkbox"],
      #ot-sdk-cookie-policy input[type="radio"] {
        display: inline;
      }
      #onetrust-banner-sdk label > .label-body,
      #onetrust-pc-sdk label > .label-body,
      #ot-sdk-cookie-policy label > .label-body {
        display: inline-block;
        margin-left: 0.5rem;
        font-weight: normal;
      }
      #onetrust-banner-sdk ul,
      #onetrust-pc-sdk ul,
      #ot-sdk-cookie-policy ul {
        list-style: circle inside;
      }
      #onetrust-banner-sdk ol,
      #onetrust-pc-sdk ol,
      #ot-sdk-cookie-policy ol {
        list-style: decimal inside;
      }
      #onetrust-banner-sdk ol,
      #onetrust-banner-sdk ul,
      #onetrust-pc-sdk ol,
      #onetrust-pc-sdk ul,
      #ot-sdk-cookie-policy ol,
      #ot-sdk-cookie-policy ul {
        padding-left: 0;
        margin-top: 0;
      }
      #onetrust-banner-sdk ul ul,
      #onetrust-banner-sdk ul ol,
      #onetrust-banner-sdk ol ol,
      #onetrust-banner-sdk ol ul,
      #onetrust-pc-sdk ul ul,
      #onetrust-pc-sdk ul ol,
      #onetrust-pc-sdk ol ol,
      #onetrust-pc-sdk ol ul,
      #ot-sdk-cookie-policy ul ul,
      #ot-sdk-cookie-policy ul ol,
      #ot-sdk-cookie-policy ol ol,
      #ot-sdk-cookie-policy ol ul {
        margin: 1.5rem 0 1.5rem 3rem;
        font-size: 90%;
      }
      #onetrust-banner-sdk li,
      #onetrust-pc-sdk li,
      #ot-sdk-cookie-policy li {
        margin-bottom: 1rem;
      }
      #onetrust-banner-sdk code,
      #onetrust-pc-sdk code,
      #ot-sdk-cookie-policy code {
        padding: 0.2rem 0.5rem;
        margin: 0 0.2rem;
        font-size: 90%;
        white-space: nowrap;
        background: #f1f1f1;
        border: 1px solid #e1e1e1;
        border-radius: 4px;
      }
      #onetrust-banner-sdk pre > code,
      #onetrust-pc-sdk pre > code,
      #ot-sdk-cookie-policy pre > code {
        display: block;
        padding: 1rem 1.5rem;
        white-space: pre;
      }
      #onetrust-banner-sdk th,
      #onetrust-banner-sdk td,
      #onetrust-pc-sdk th,
      #onetrust-pc-sdk td,
      #ot-sdk-cookie-policy th,
      #ot-sdk-cookie-policy td {
        padding: 12px 15px;
        text-align: left;
        border-bottom: 1px solid #e1e1e1;
      }
      #onetrust-banner-sdk .ot-sdk-u-full-width,
      #onetrust-pc-sdk .ot-sdk-u-full-width,
      #ot-sdk-cookie-policy .ot-sdk-u-full-width {
        width: 100%;
        box-sizing: border-box;
      }
      #onetrust-banner-sdk .ot-sdk-u-max-full-width,
      #onetrust-pc-sdk .ot-sdk-u-max-full-width,
      #ot-sdk-cookie-policy .ot-sdk-u-max-full-width {
        max-width: 100%;
        box-sizing: border-box;
      }
      #onetrust-banner-sdk .ot-sdk-u-pull-right,
      #onetrust-pc-sdk .ot-sdk-u-pull-right,
      #ot-sdk-cookie-policy .ot-sdk-u-pull-right {
        float: right;
      }
      #onetrust-banner-sdk .ot-sdk-u-pull-left,
      #onetrust-pc-sdk .ot-sdk-u-pull-left,
      #ot-sdk-cookie-policy .ot-sdk-u-pull-left {
        float: left;
      }
      #onetrust-banner-sdk hr,
      #onetrust-pc-sdk hr,
      #ot-sdk-cookie-policy hr {
        margin-top: 3rem;
        margin-bottom: 3.5rem;
        border-width: 0;
        border-top: 1px solid #e1e1e1;
      }
      #onetrust-banner-sdk .ot-sdk-container:after,
      #onetrust-banner-sdk .ot-sdk-row:after,
      #onetrust-banner-sdk .ot-sdk-u-cf,
      #onetrust-pc-sdk .ot-sdk-container:after,
      #onetrust-pc-sdk .ot-sdk-row:after,
      #onetrust-pc-sdk .ot-sdk-u-cf,
      #ot-sdk-cookie-policy .ot-sdk-container:after,
      #ot-sdk-cookie-policy .ot-sdk-row:after,
      #ot-sdk-cookie-policy .ot-sdk-u-cf {
        content: "";
        display: table;
        clear: both;
      }
      #onetrust-banner-sdk .ot-sdk-row,
      #onetrust-pc-sdk .ot-sdk-row,
      #ot-sdk-cookie-policy .ot-sdk-row {
        margin: 0;
        max-width: none;
        display: block;
      }
      #onetrust-banner-sdk {
        box-shadow: 0 0 18px rgba(0, 0, 0, 0.2);
      }
      #onetrust-banner-sdk.otFlat {
        position: fixed;
        z-index: 2147483645;
        bottom: 0;
        right: 0;
        left: 0;
        background-color: #fff;
        max-height: 90%;
        overflow-x: hidden;
        overflow-y: auto;
      }
      #onetrust-banner-sdk > .ot-sdk-container {
        overflow: hidden;
      }
      #onetrust-banner-sdk::-webkit-scrollbar {
        width: 11px;
      }
      #onetrust-banner-sdk::-webkit-scrollbar-thumb {
        border-radius: 10px;
        background: #c1c1c1;
      }
      #onetrust-banner-sdk {
        scrollbar-arrow-color: #c1c1c1;
        scrollbar-darkshadow-color: #c1c1c1;
        scrollbar-face-color: #c1c1c1;
        scrollbar-shadow-color: #c1c1c1;
      }
      #onetrust-banner-sdk #onetrust-policy {
        margin: 1.25em 0 0.625em 2em;
        overflow: hidden;
      }
      #onetrust-banner-sdk #onetrust-policy .ot-gv-list-handler {
        float: left;
        font-size: 0.82em;
        padding: 0;
        margin-bottom: 0;
        border: 0;
        line-height: normal;
        height: auto;
        width: auto;
      }
      #onetrust-banner-sdk #onetrust-policy-title {
        font-size: 1.2em;
        line-height: 1.3;
        margin-bottom: 10px;
      }
      #onetrust-banner-sdk #onetrust-policy-text {
        clear: both;
        text-align: left;
        font-size: 0.88em;
        line-height: 1.4;
      }
      #onetrust-banner-sdk #onetrust-policy-text * {
        font-size: inherit;
        line-height: inherit;
      }
      #onetrust-banner-sdk #onetrust-policy-text a {
        font-weight: bold;
        margin-left: 5px;
      }
      #onetrust-banner-sdk #onetrust-policy-title,
      #onetrust-banner-sdk #onetrust-policy-text {
        color: dimgray;
        float: left;
      }
      #onetrust-banner-sdk #onetrust-button-group-parent {
        min-height: 1px;
        text-align: center;
      }
      #onetrust-banner-sdk #onetrust-button-group {
        display: inline-block;
      }
      #onetrust-banner-sdk #onetrust-accept-btn-handler,
      #onetrust-banner-sdk #onetrust-reject-all-handler,
      #onetrust-banner-sdk #onetrust-pc-btn-handler {
        background-color: #68b631;
        color: #fff;
        border-color: #68b631;
        margin-right: 1em;
        min-width: 125px;
        height: auto;
        white-space: normal;
        word-break: break-word;
        word-wrap: break-word;
        padding: 12px 10px;
        line-height: 1.2;
        font-size: 0.813em;
        font-weight: 600;
      }
      #onetrust-banner-sdk #onetrust-pc-btn-handler.cookie-setting-link {
        background-color: #fff;
        border: none;
        color: #68b631;
        text-decoration: underline;
        padding-left: 0;
        padding-right: 0;
      }
      #onetrust-banner-sdk #onetrust-close-btn-container {
        text-align: center;
      }
      #onetrust-banner-sdk .onetrust-close-btn-ui {
        width: 44px;
        height: 44px;
        background-size: 12px;
        margin: 50% 0 0 50%;
        border: none;
      }
      #onetrust-banner-sdk .onetrust-close-btn-ui.onetrust-lg {
        top: 50%;
        margin: auto;
        transform: translate(-50%, -50%);
        position: absolute;
        padding: 0;
      }
      #onetrust-banner-sdk .banner_logo {
        display: none;
      }
      #onetrust-banner-sdk .ot-b-addl-desc {
        clear: both;
        float: left;
        display: block;
      }
      #onetrust-banner-sdk #banner-options {
        float: left;
        display: table;
        margin-right: 0;
        margin-left: 1em;
        width: calc(100% - 1em);
      }
      #onetrust-banner-sdk .banner-option-input {
        cursor: pointer;
        width: auto;
        height: auto;
        border: none;
        padding: 0;
        padding-right: 3px;
        margin: 0 0 10px;
        font-size: 0.82em;
        line-height: 1.4;
      }
      #onetrust-banner-sdk .banner-option-input * {
        pointer-events: none;
        font-size: inherit;
        line-height: inherit;
      }
      #onetrust-banner-sdk
        .banner-option-input[aria-expanded="true"]
        ~ .banner-option-details {
        display: block;
        height: auto;
      }
      #onetrust-banner-sdk
        .banner-option-input[aria-expanded="true"]
        .ot-arrow-container {
        transform: rotate(90deg);
      }
      #onetrust-banner-sdk .banner-option {
        margin-bottom: 12px;
        margin-left: 0;
        border: none;
        float: left;
        padding: 0;
      }
      #onetrust-banner-sdk .banner-option:first-child {
        padding-left: 2px;
      }
      #onetrust-banner-sdk .banner-option:not(:first-child) {
        padding: 0;
        border: none;
      }
      #onetrust-banner-sdk .banner-option-header {
        cursor: pointer;
        display: inline-block;
      }
      #onetrust-banner-sdk .banner-option-header :first-child {
        color: dimgray;
        font-weight: bold;
        float: left;
      }
      #onetrust-banner-sdk .banner-option-header .ot-arrow-container {
        display: inline-block;
        border-top: 6px solid transparent;
        border-bottom: 6px solid transparent;
        border-left: 6px solid dimgray;
        margin-left: 10px;
        vertical-align: middle;
      }
      #onetrust-banner-sdk .banner-option-details {
        display: none;
        font-size: 0.83em;
        line-height: 1.5;
        padding: 10px 0px 5px 10px;
        margin-right: 10px;
        height: 0px;
      }
      #onetrust-banner-sdk .banner-option-details * {
        font-size: inherit;
        line-height: inherit;
        color: dimgray;
      }
      #onetrust-banner-sdk .ot-arrow-container,
      #onetrust-banner-sdk .banner-option-details {
        transition: all 300ms ease-in 0s;
        -webkit-transition: all 300ms ease-in 0s;
        -moz-transition: all 300ms ease-in 0s;
        -o-transition: all 300ms ease-in 0s;
      }
      #onetrust-banner-sdk .ot-dpd-container {
        float: left;
      }
      #onetrust-banner-sdk .ot-dpd-title {
        margin-bottom: 10px;
      }
      #onetrust-banner-sdk .ot-dpd-title,
      #onetrust-banner-sdk .ot-dpd-desc {
        font-size: 0.88em;
        line-height: 1.4;
        color: dimgray;
      }
      #onetrust-banner-sdk .ot-dpd-title *,
      #onetrust-banner-sdk .ot-dpd-desc * {
        font-size: inherit;
        line-height: inherit;
      }
      #onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text * {
        margin-bottom: 0;
      }
      #onetrust-banner-sdk.ot-iab-2 .onetrust-vendors-list-handler {
        display: block;
        margin-left: 0;
        margin-top: 5px;
        clear: both;
        margin-bottom: 0;
        padding: 0;
        border: 0;
        height: auto;
        width: auto;
      }
      #onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button {
        display: block;
      }
      #onetrust-banner-sdk #onetrust-policy-text,
      #onetrust-banner-sdk .ot-dpd-desc,
      #onetrust-banner-sdk .ot-b-addl-desc {
        font-size: 0.813em;
        line-height: 1.5;
      }
      #onetrust-banner-sdk .ot-dpd-desc {
        margin-bottom: 10px;
      }
      #onetrust-banner-sdk .ot-dpd-desc > .ot-b-addl-desc {
        margin-top: 10px;
        margin-bottom: 10px;
        font-size: 1em;
      }
      @media only screen and (max-width: 425px) {
        #onetrust-banner-sdk #onetrust-policy {
          margin-left: 0;
        }
        #onetrust-banner-sdk .ot-hide-small {
          display: none;
        }
        #onetrust-banner-sdk #onetrust-button-group {
          display: block;
        }
        #onetrust-banner-sdk #onetrust-accept-btn-handler,
        #onetrust-banner-sdk #onetrust-reject-all-handler,
        #onetrust-banner-sdk #onetrust-pc-btn-handler {
          width: 100%;
        }
        #onetrust-banner-sdk .onetrust-close-btn-ui {
          margin: -10px -10px 0 0;
          float: right;
          padding: 0;
        }
        #onetrust-banner-sdk #onetrust-close-btn-container-mobile,
        #onetrust-banner-sdk #onetrust-policy-title {
          display: inline;
          float: none;
        }
        #onetrust-banner-sdk #banner-options {
          margin: 0;
          padding: 0;
          width: 100%;
        }
      }
      @media only screen and (min-width: 426px) and (max-width: 896px) {
        #onetrust-banner-sdk #onetrust-policy {
          margin-left: 1em;
          margin-right: 1em;
        }
        #onetrust-banner-sdk .onetrust-close-btn-ui.onetrust-lg {
          top: 25px;
          right: -15px;
        }
        #onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container {
          width: 95%;
        }
        #onetrust-banner-sdk.ot-iab-2 #onetrust-group-container {
          width: 100%;
        }
        #onetrust-banner-sdk #onetrust-button-group-parent {
          width: 100%;
          position: relative;
          margin-left: 0;
        }
        #onetrust-banner-sdk .ot-hide-large {
          display: none;
        }
        #onetrust-banner-sdk #onetrust-button-group button {
          display: inline-block;
        }
        #onetrust-banner-sdk #onetrust-button-group {
          margin-right: 0;
          text-align: center;
        }
        #onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler {
          float: left;
        }
        #onetrust-banner-sdk
          .has-reject-all-button
          #onetrust-reject-all-handler,
        #onetrust-banner-sdk
          .has-reject-all-button
          #onetrust-accept-btn-handler {
          float: right;
        }
        #onetrust-banner-sdk .has-reject-all-button #onetrust-button-group {
          width: calc(100% - 2em);
          margin-right: 0;
        }
        #onetrust-banner-sdk
          .has-reject-all-button
          #onetrust-pc-btn-handler.cookie-setting-link {
          padding-left: 0px;
          text-align: left;
        }
        #onetrust-banner-sdk.ot-buttons-fw .ot-sdk-three button {
          width: 100%;
          text-align: center;
        }
        #onetrust-banner-sdk.ot-buttons-fw
          #onetrust-button-group-parent
          button {
          float: none;
        }
        #onetrust-banner-sdk.ot-buttons-fw
          #onetrust-pc-btn-handler.cookie-setting-link {
          text-align: center;
        }
      }
      @media only screen and (min-width: 550px) {
        #onetrust-banner-sdk .banner-option:not(:first-child) {
          border-left: 1px solid #d8d8d8;
          padding-left: 25px;
        }
      }
      @media only screen and (min-width: 425px) and (max-width: 550px) {
        #onetrust-banner-sdk.ot-iab-2 #onetrust-button-group,
        #onetrust-banner-sdk.ot-iab-2 #onetrust-policy,
        #onetrust-banner-sdk.ot-iab-2 .banner-option {
          width: 100%;
        }
      }
      @media only screen and (min-width: 769px) {
        #onetrust-banner-sdk .ot-hide-large {
          display: none;
        }
        #onetrust-banner-sdk #onetrust-button-group {
          margin-right: 30%;
        }
        #onetrust-banner-sdk #banner-options {
          margin-left: 2em;
          margin-right: 5em;
          margin-bottom: 1.25em;
          width: calc(100% - 7em);
        }
      }
      @media only screen and (min-width: 897px) and (max-width: 1023px) {
        #onetrust-banner-sdk.vertical-align-content
          #onetrust-button-group-parent {
          position: absolute;
          top: 50%;
          left: 75%;
          transform: translateY(-50%);
        }
        #onetrust-banner-sdk #onetrust-close-btn-container {
          float: right;
        }
      }
      @media only screen and (min-width: 1024px) {
        #onetrust-banner-sdk #onetrust-policy {
          margin-left: 2em;
        }
        #onetrust-banner-sdk.vertical-align-content
          #onetrust-button-group-parent {
          position: absolute;
          top: 50%;
          left: 60%;
          transform: translateY(-50%);
        }
        #onetrust-banner-sdk.ot-iab-2 #onetrust-policy-title {
          width: 50%;
        }
        #onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text,
        #onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc) > .ot-b-addl-desc {
          margin-bottom: 1em;
          width: 50%;
          border-right: 1px solid #d8d8d8;
          padding-right: 1rem;
        }
        #onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text {
          margin-bottom: 0;
          padding-bottom: 1em;
        }
        #onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc) > .ot-b-addl-desc {
          margin-bottom: 0;
          padding-bottom: 1em;
        }
        #onetrust-banner-sdk.ot-iab-2 .ot-dpd-container {
          width: 45%;
          padding-left: 1rem;
          display: inline-block;
          float: none;
        }
        #onetrust-banner-sdk.ot-iab-2 .ot-dpd-title {
          line-height: 1.7;
        }
        #onetrust-banner-sdk.ot-iab-2 #onetrust-button-group-parent {
          left: auto;
          right: 4%;
          margin-left: 0;
        }
        #onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button {
          display: block;
        }
        #onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent {
          margin: auto;
          width: 30%;
        }
        #onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container {
          width: 60%;
        }
        #onetrust-banner-sdk #onetrust-button-group {
          margin-right: auto;
        }
        #onetrust-banner-sdk #onetrust-close-btn-container {
          float: right;
        }
        #onetrust-banner-sdk #onetrust-accept-btn-handler,
        #onetrust-banner-sdk #onetrust-reject-all-handler,
        #onetrust-banner-sdk #onetrust-pc-btn-handler {
          margin-top: 1em;
        }
      }
      @media only screen and (min-width: 890px) {
        #onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2)
          #onetrust-button-group-parent {
          padding-left: 4%;
          margin-left: 0;
        }
        #onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2)
          #onetrust-button-group {
          margin-right: 0;
          margin-top: 1.25em;
          width: 100%;
        }
        #onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2)
          #onetrust-button-group
          button {
          width: 100%;
          margin-bottom: 5px;
          margin-top: 5px;
        }
        #onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2)
          #onetrust-button-group
          button:last-of-type {
          margin-bottom: 20px;
        }
      }
      @media only screen and (min-width: 1280px) {
        #onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container {
          width: 55%;
        }
        #onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent {
          width: 44%;
          padding-left: 2%;
          padding-right: 2%;
        }
        #onetrust-banner-sdk:not(.ot-iab-2).vertical-align-content
          #onetrust-button-group-parent {
          position: absolute;
          left: 55%;
        }
      }
      #onetrust-consent-sdk #onetrust-banner-sdk {
        background-color: #486a7e;
      }
      #onetrust-consent-sdk #onetrust-policy-title,
      #onetrust-consent-sdk #onetrust-policy-text,
      #onetrust-consent-sdk .ot-b-addl-desc,
      #onetrust-consent-sdk .ot-dpd-desc,
      #onetrust-consent-sdk .ot-dpd-title,
      #onetrust-consent-sdk
        #onetrust-policy-text
        *:not(.onetrust-vendors-list-handler),
      #onetrust-consent-sdk .ot-dpd-desc *:not(.onetrust-vendors-list-handler),
      #onetrust-consent-sdk #onetrust-banner-sdk #banner-options * {
        color: #ffffff;
      }
      #onetrust-consent-sdk #onetrust-banner-sdk .banner-option-details {
        background-color: #e9e9e9;
      }
      #onetrust-consent-sdk #onetrust-accept-btn-handler,
      #onetrust-banner-sdk #onetrust-reject-all-handler {
        background-color: #ffffff;
        border-color: #ffffff;
        color: #222;
      }
      #onetrust-consent-sdk #onetrust-pc-btn-handler,
      #onetrust-consent-sdk #onetrust-pc-btn-handler.cookie-setting-link {
        color: #ffffff;
        border-color: #ffffff;
        background-color: #486a7e;
      } /* scrap ot grid column layout */
      #onetrust-banner-sdk .ot-sdk-columns {
        width: auto !important;
        margin: 0 !important;
      }

      #onetrust-banner-sdk .ot-sdk-container {
        font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto,
          Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif !important;
        width: auto !important;
        max-width: 1280px !important;
        margin: auto !important;
        padding: 16px !important;
      }

      #onetrust-banner-sdk #onetrust-policy {
        margin: auto !important;
      }

      #onetrust-banner-sdk #onetrust-group-container {
        width: auto !important;
      }

      #onetrust-banner-sdk #onetrust-policy-title {
        display: block !important;
        font-size: 22px !important;
        margin-bottom: 16px !important;
        margin-bottom: 0.25em !important;
      }

      #onetrust-banner-sdk #onetrust-policy-text {
        font-size: 14px !important;
      }

      #onetrust-banner-sdk #onetrust-button-group-parent {
        text-align: left !important;
        position: static !important;
        -webkit-transform: translate(0) !important;
        -ms-transform: translate(0) !important;
        transform: translate(0) !important;
      }

      #onetrust-banner-sdk #onetrust-button-group {
        margin: auto !important;
      }

      #onetrust-banner-sdk #onetrust-button-group > button {
        font-size: 14px !important;
        padding: 1em 1.2em !important;
        margin: 0 0 1em !important;
        font-weight: bold;
      }

      #onetrust-banner-sdk #onetrust-button-group > button:focus {
        outline: 4px solid #fc0 !important;
        opacity: 1 !important;
      }

      #onetrust-banner-sdk #onetrust-close-btn-container {
        position: absolute !important;
        top: 16px !important;
        right: 16px !important;
      }

      /* Media Queries */
      @media only screen and (min-width: 426px) {
        #onetrust-banner-sdk #onetrust-button-group > button + button {
          margin-left: 1em !important;
        }
      }

      @media screen and (min-width: 1024px) {
        #onetrust-banner-sdk .ot-sdk-container {
          padding: 32px 16px !important;
        }

        #onetrust-banner-sdk .ot-sdk-container .ot-sdk-row {
          display: block !important;
        }

        #onetrust-banner-sdk #onetrust-policy-title {
          font-size: 32px !important;
        }

        #onetrust-banner-sdk #onetrust-button-group-parent {
          padding-left: 0 !important;
        }

        #onetrust-banner-sdk #onetrust-button-group > button,
        #onetrust-banner-sdk #onetrust-policy-text {
          font-size: 16px !important;
        }
      }

      @media screen and (min-width: 1280px) {
        #onetrust-banner-sdk .ot-sdk-container {
          padding: 48px 16px !important;
        }
      }
      #onetrust-pc-sdk {
        position: fixed;
        width: 730px;
        max-width: 730px;
        height: 610px;
        left: 0;
        right: 0;
        top: 0;
        bottom: 0;
        margin: auto;
        font-size: 16px;
        z-index: 2147483647;
        border-radius: 2px;
        background-color: #fff;
        box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0),
          0 7px 14px 0 rgba(50, 50, 93, 0.1);
      }
      #onetrust-pc-sdk *,
      #onetrust-pc-sdk ::after,
      #onetrust-pc-sdk ::before {
        box-sizing: content-box;
      }
      #onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr,
      #onetrust-pc-sdk .ot-hide-tgl {
        visibility: hidden;
      }
      #onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr *,
      #onetrust-pc-sdk .ot-hide-tgl * {
        visibility: hidden;
      }
      #onetrust-pc-sdk #ot-pc-content,
      #onetrust-pc-sdk #ot-pc-lst {
        height: calc(100% - 185px);
      }
      #onetrust-pc-sdk li {
        list-style: none;
      }
      #onetrust-pc-sdk ul,
      #onetrust-pc-sdk li {
        margin: 0;
      }
      #onetrust-pc-sdk a {
        text-decoration: none;
      }
      #onetrust-pc-sdk .ot-link-btn {
        padding: 0;
        margin-bottom: 0;
        border: 0;
        font-weight: normal;
        line-height: normal;
        width: auto;
        height: auto;
      }
      #onetrust-pc-sdk .ot-grps-cntr *::-webkit-scrollbar,
      #onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar {
        width: 11px;
      }
      #onetrust-pc-sdk .ot-grps-cntr *::-webkit-scrollbar-thumb,
      #onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-thumb {
        border-radius: 10px;
        background: #c1c1c1;
      }
      #onetrust-pc-sdk .ot-grps-cntr *,
      #onetrust-pc-sdk .ot-pc-scrollbar {
        scrollbar-arrow-color: #c1c1c1;
        scrollbar-darkshadow-color: #c1c1c1;
        scrollbar-face-color: #c1c1c1;
        scrollbar-shadow-color: #c1c1c1;
      }
      #onetrust-pc-sdk .ot-pc-header {
        height: auto;
        padding: 10px;
        display: table;
        vertical-align: middle;
        width: calc(100% - 20px);
        min-height: 52px;
        border-bottom: 1px solid #d8d8d8;
        position: relative;
      }
      #onetrust-pc-sdk .ot-pc-logo {
        display: table-cell;
        vertical-align: middle;
        width: 180px;
        height: 40px;
      }
      #onetrust-pc-sdk .ot-title-cntr {
        position: relative;
        display: table-cell;
        vertical-align: middle;
        width: calc(100% - 190px);
        padding-left: 10px;
      }
      #onetrust-pc-sdk .ot-always-active {
        font-size: 0.813em;
        line-height: 1.5;
        font-weight: 700;
        color: #3860be;
      }
      #onetrust-pc-sdk .ot-close-cntr {
        float: right;
        position: absolute;
        right: -9px;
        top: 50%;
        transform: translateY(-50%);
      }
      #onetrust-pc-sdk #ot-pc-content {
        position: relative;
        overflow-y: auto;
        overflow-x: hidden;
      }
      #onetrust-pc-sdk .ot-grps-cntr,
      #onetrust-pc-sdk .ot-grps-cntr > * {
        height: 100%;
        overflow-y: auto;
      }
      #onetrust-pc-sdk .category-menu-switch-handler {
        cursor: pointer;
        border-left: 10px solid transparent;
        background-color: #f4f4f4;
        border-bottom: 1px solid #d7d7d7;
        padding-top: 12px;
        padding-right: 5px;
        padding-bottom: 12px;
        padding-left: 12px;
        overflow: hidden;
      }
      #onetrust-pc-sdk .category-menu-switch-handler h3 {
        float: left;
        text-align: left;
        margin: 0;
        color: dimgray;
        line-height: 1.4;
        font-size: 0.875em;
        word-break: break-word;
        word-wrap: break-word;
      }
      #onetrust-pc-sdk .ot-active-menu {
        border-left: 10px solid #68b631;
        background-color: #fff;
        border-bottom: none;
        position: relative;
      }
      #onetrust-pc-sdk .ot-active-menu h3 {
        color: #263238;
        font-weight: bold;
      }
      #onetrust-pc-sdk .ot-desc-cntr {
        word-break: break-word;
        word-wrap: break-word;
        padding-top: 20px;
        padding-right: 16px;
        padding-bottom: 15px;
      }
      #onetrust-pc-sdk .ot-grp-desc {
        word-break: break-word;
        word-wrap: break-word;
        text-align: left;
        font-size: 0.813em;
        line-height: 1.5;
        margin: 0;
      }
      #onetrust-pc-sdk .ot-grp-desc * {
        font-size: inherit;
        line-height: inherit;
      }
      #onetrust-pc-sdk #ot-pc-desc a {
        color: #3860be;
        cursor: pointer;
        font-size: 1em;
      }
      #onetrust-pc-sdk #ot-pc-desc a:hover {
        color: #1883fd;
      }
      #onetrust-pc-sdk #ot-pc-desc * {
        font-size: inherit;
      }
      #onetrust-pc-sdk #ot-pc-desc ul li {
        padding: 10px 0px;
        border-bottom: 1px solid #e2e2e2;
      }
      #onetrust-pc-sdk .ot-btn-subcntr {
        float: right;
      }
      #onetrust-pc-sdk .ot-close-icon {
        background-image: url("data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHhtbG5zOnhsaW5rPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5L3hsaW5rIiB4PSIwcHgiIHk9IjBweCIgd2lkdGg9IjQ3Ljk3MSIgaGVpZ2h0PSI0Ny45NzEiIHZpZXdCb3g9IjAgMCA0Ny45NzEgNDcuOTcxIiBzdHlsZT0iZW5hYmxlLWJhY2tncm91bmQ6bmV3IDAgMCA0Ny45NzEgNDcuOTcxOyIgeG1sOnNwYWNlPSJwcmVzZXJ2ZSI+PGc+PHBhdGggZD0iTTI4LjIyOCwyMy45ODZMNDcuMDkyLDUuMTIyYzEuMTcyLTEuMTcxLDEuMTcyLTMuMDcxLDAtNC4yNDJjLTEuMTcyLTEuMTcyLTMuMDctMS4xNzItNC4yNDIsMEwyMy45ODYsMTkuNzQ0TDUuMTIxLDAuODhjLTEuMTcyLTEuMTcyLTMuMDctMS4xNzItNC4yNDIsMGMtMS4xNzIsMS4xNzEtMS4xNzIsMy4wNzEsMCw0LjI0MmwxOC44NjUsMTguODY0TDAuODc5LDQyLjg1Yy0xLjE3MiwxLjE3MS0xLjE3MiwzLjA3MSwwLDQuMjQyQzEuNDY1LDQ3LjY3NywyLjIzMyw0Ny45NywzLDQ3Ljk3czEuNTM1LTAuMjkzLDIuMTIxLTAuODc5bDE4Ljg2NS0xOC44NjRMNDIuODUsNDcuMDkxYzAuNTg2LDAuNTg2LDEuMzU0LDAuODc5LDIuMTIxLDAuODc5czEuNTM1LTAuMjkzLDIuMTIxLTAuODc5YzEuMTcyLTEuMTcxLDEuMTcyLTMuMDcxLDAtNC4yNDJMMjguMjI4LDIzLjk4NnoiLz48L2c+PC9zdmc+");
        background-size: 12px;
        background-repeat: no-repeat;
        background-position: center;
        height: 44px;
        width: 44px;
        display: inline-block;
      }
      #onetrust-pc-sdk .ot-tgl {
        float: right;
        position: relative;
        z-index: 1;
      }
      #onetrust-pc-sdk .ot-tgl input:checked + .ot-switch .ot-switch-nob {
        background-color: #cddcf2;
      }
      #onetrust-pc-sdk
        .ot-tgl
        input:checked
        + .ot-switch
        .ot-switch-nob:before {
        -webkit-transform: translateX(16px);
        -ms-transform: translateX(16px);
        transform: translateX(16px);
        background-color: #4285f4;
      }
      #onetrust-pc-sdk .ot-tgl input:focus + .ot-switch .ot-switch-nob:before {
        box-shadow: 0 0 1px #2196f3;
        outline: #000 auto 5px;
      }
      #onetrust-pc-sdk .ot-switch {
        position: relative;
        display: inline-block;
        width: 35px;
        height: 10px;
        margin-bottom: 0;
      }
      #onetrust-pc-sdk .ot-switch-nob {
        position: absolute;
        cursor: pointer;
        top: 0;
        left: 0;
        right: 0;
        bottom: 0;
        background-color: #f2f1f1;
        border: none;
        transition: all 0.2s ease-in 0s;
        -moz-transition: all 0.2s ease-in 0s;
        -o-transition: all 0.2s ease-in 0s;
        -webkit-transition: all 0.2s ease-in 0s;
        border-radius: 46px;
      }
      #onetrust-pc-sdk .ot-switch-nob:before {
        position: absolute;
        content: "";
        height: 20px;
        width: 20px;
        bottom: 1px;
        background-color: #7d7d7d;
        -webkit-transition: 0.4s;
        border-radius: 100%;
        top: -5px;
        transition: 0.4s;
      }
      #onetrust-pc-sdk .ot-chkbox {
        z-index: 1;
        position: relative;
      }
      #onetrust-pc-sdk .ot-chkbox input:checked ~ label::before {
        background-color: #3860be;
      }
      #onetrust-pc-sdk .ot-chkbox input + label::after {
        content: none;
        color: #fff;
      }
      #onetrust-pc-sdk .ot-chkbox input:checked + label::after {
        content: "";
      }
      #onetrust-pc-sdk .ot-chkbox input:focus + label::before {
        outline-style: solid;
        outline-width: 2px;
        outline-style: auto;
      }
      #onetrust-pc-sdk .ot-chkbox label {
        position: relative;
        height: 20px;
        padding-left: 30px;
        display: inline-block;
        cursor: pointer;
      }
      #onetrust-pc-sdk .ot-chkbox label::before,
      #onetrust-pc-sdk .ot-chkbox label::after {
        position: absolute;
        content: "";
        display: inline-block;
        border-radius: 3px;
      }
      #onetrust-pc-sdk .ot-chkbox label::before {
        height: 18px;
        width: 18px;
        border: 1px solid #3860be;
        left: 0px;
      }
      #onetrust-pc-sdk .ot-chkbox label::after {
        height: 5px;
        width: 9px;
        border-left: 3px solid;
        border-bottom: 3px solid;
        transform: rotate(-45deg);
        -o-transform: rotate(-45deg);
        -ms-transform: rotate(-45deg);
        -webkit-transform: rotate(-45deg);
        left: 4px;
        top: 5px;
      }
      #onetrust-pc-sdk .ot-label-txt {
        display: none;
      }
      #onetrust-pc-sdk .ot-fltr-opt .ot-label-txt {
        display: block;
      }
      #onetrust-pc-sdk .ot-chkbox input,
      #onetrust-pc-sdk .ot-tgl input {
        position: absolute;
        opacity: 0;
        width: 0;
        height: 0;
      }
      #onetrust-pc-sdk .ot-arw-cntr {
        float: right;
        position: relative;
      }
      #onetrust-pc-sdk .ot-arw {
        width: 16px;
        height: 16px;
        margin-left: 5px;
        color: dimgray;
        display: inline-block;
        vertical-align: middle;
        -webkit-transition: all 300ms ease-in 0s;
        -moz-transition: all 300ms ease-in 0s;
        -o-transition: all 300ms ease-in 0s;
        transition: all 300ms ease-in 0s;
      }
      #onetrust-pc-sdk input:checked ~ .ot-acc-hdr .ot-arw {
        transform: rotate(90deg);
        -o-transform: rotate(90deg);
        -ms-transform: rotate(90deg);
        -webkit-transform: rotate(90deg);
      }
      #onetrust-pc-sdk .ot-label-status {
        font-size: 0.75em;
        position: relative;
        top: 2px;
        display: none;
        padding-right: 5px;
        float: left;
      }
      #onetrust-pc-sdk #ot-lst-cnt .ot-label-status {
        top: -6px;
      }
      #onetrust-pc-sdk .ot-fltr-btns {
        margin-left: 15px;
        overflow: hidden;
        margin-right: 15px;
      }
      #onetrust-pc-sdk .ot-fltr-btns button {
        padding: 12px 30px;
      }
      #onetrust-pc-sdk .ot-pc-footer {
        position: absolute;
        bottom: 0px;
        width: 100%;
        max-height: 160px;
        border-top: 1px solid #d8d8d8;
      }
      #onetrust-pc-sdk .ot-pc-footer button {
        margin-top: 20px;
        margin-bottom: 20px;
        font-weight: 600;
        font-size: 0.813em;
        min-height: 40px;
        height: auto;
        line-height: normal;
        padding: 10px 30px;
      }
      #onetrust-pc-sdk .ot-tab-desc {
        margin-left: 3%;
      }
      #onetrust-pc-sdk .ot-grp-hdr1 {
        display: inline-block;
        width: 100%;
        margin-bottom: 10px;
      }
      #onetrust-pc-sdk .ot-desc-cntr h3 {
        color: #263238;
        display: inline-block;
        vertical-align: middle;
        margin: 0;
        font-weight: bold;
        font-size: 0.875em;
        line-height: 1.3;
        max-width: 70%;
      }
      #onetrust-pc-sdk #ot-pvcy-hdr {
        margin-bottom: 10px;
      }
      #onetrust-pc-sdk .ot-vlst-cntr {
        overflow: hidden;
      }
      #onetrust-pc-sdk .category-vendors-list-handler,
      #onetrust-pc-sdk .category-host-list-handler,
      #onetrust-pc-sdk .category-vendors-list-handler + a {
        display: block;
        float: left;
        color: #3860be;
        font-size: 0.813em;
        font-weight: 400;
        line-height: 1.1;
        cursor: pointer;
      }
      #onetrust-pc-sdk .category-vendors-list-handler:hover,
      #onetrust-pc-sdk .category-host-list-handler:hover,
      #onetrust-pc-sdk .category-vendors-list-handler + a:hover {
        color: #1883fd;
      }
      #onetrust-pc-sdk .category-vendors-list-handler + a::after {
        content: "";
        height: 15px;
        width: 15px;
        background-repeat: no-repeat;
        margin-left: 5px;
        float: right;
        background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 511.626 511.627'%3E%3Cg fill='%231276CE'%3E%3Cpath d='M392.857 292.354h-18.274c-2.669 0-4.859.855-6.563 2.573-1.718 1.708-2.573 3.897-2.573 6.563v91.361c0 12.563-4.47 23.315-13.415 32.262-8.945 8.945-19.701 13.414-32.264 13.414H82.224c-12.562 0-23.317-4.469-32.264-13.414-8.945-8.946-13.417-19.698-13.417-32.262V155.31c0-12.562 4.471-23.313 13.417-32.259 8.947-8.947 19.702-13.418 32.264-13.418h200.994c2.669 0 4.859-.859 6.57-2.57 1.711-1.713 2.566-3.9 2.566-6.567V82.221c0-2.662-.855-4.853-2.566-6.563-1.711-1.713-3.901-2.568-6.57-2.568H82.224c-22.648 0-42.016 8.042-58.102 24.125C8.042 113.297 0 132.665 0 155.313v237.542c0 22.647 8.042 42.018 24.123 58.095 16.086 16.084 35.454 24.13 58.102 24.13h237.543c22.647 0 42.017-8.046 58.101-24.13 16.085-16.077 24.127-35.447 24.127-58.095v-91.358c0-2.669-.856-4.859-2.574-6.57-1.713-1.718-3.903-2.573-6.565-2.573z'/%3E%3Cpath d='M506.199 41.971c-3.617-3.617-7.905-5.424-12.85-5.424H347.171c-4.948 0-9.233 1.807-12.847 5.424-3.617 3.615-5.428 7.898-5.428 12.847s1.811 9.233 5.428 12.85l50.247 50.248-186.147 186.151c-1.906 1.903-2.856 4.093-2.856 6.563 0 2.479.953 4.668 2.856 6.571l32.548 32.544c1.903 1.903 4.093 2.852 6.567 2.852s4.665-.948 6.567-2.852l186.148-186.148 50.251 50.248c3.614 3.617 7.898 5.426 12.847 5.426s9.233-1.809 12.851-5.426c3.617-3.616 5.424-7.898 5.424-12.847V54.818c-.001-4.952-1.814-9.232-5.428-12.847z'/%3E%3C/g%3E%3C/svg%3E");
      }
      #onetrust-pc-sdk .category-host-list-handler,
      #onetrust-pc-sdk .ot-vlst-cntr,
      #onetrust-pc-sdk #ot-pc-desc + .category-vendors-list-handler {
        margin-top: 8px;
      }
      #onetrust-pc-sdk .ot-grp-hdr1 + .ot-vlst-cntr {
        margin-top: 0px;
        margin-bottom: 10px;
      }
      #onetrust-pc-sdk .ot-always-active-group h3.ot-cat-header,
      #onetrust-pc-sdk .ot-subgrp.ot-always-active-group > h4 {
        max-width: 70%;
      }
      #onetrust-pc-sdk .ot-always-active-group .ot-tgl-cntr {
        max-width: 28%;
      }
      #onetrust-pc-sdk .ot-grp-desc ul,
      #onetrust-pc-sdk li.ot-subgrp p ul {
        margin: 0px;
        margin-left: 15px;
        padding-bottom: 8px;
      }
      #onetrust-pc-sdk .ot-grp-desc ul li,
      #onetrust-pc-sdk li.ot-subgrp p ul li {
        font-size: inherit;
        padding-top: 8px;
        display: list-item;
        list-style: disc;
      }
      #onetrust-pc-sdk ul.ot-subgrps {
        margin: 0;
        font-size: inherit;
      }
      #onetrust-pc-sdk ul.ot-subgrps li {
        padding: 0;
        border: none;
        position: relative;
      }
      #onetrust-pc-sdk ul.ot-subgrps li h4,
      #onetrust-pc-sdk ul.ot-subgrps li p {
        font-size: 0.82em;
        line-height: 1.4;
      }
      #onetrust-pc-sdk ul.ot-subgrps li p {
        color: dimgray;
        clear: both;
        float: left;
        margin-top: 10px;
        margin-bottom: 0;
        word-break: break-word;
        word-wrap: break-word;
      }
      #onetrust-pc-sdk ul.ot-subgrps li h4 {
        color: #263238;
        font-weight: bold;
        margin-bottom: 0;
        float: left;
        position: relative;
        top: 3px;
      }
      #onetrust-pc-sdk li.ot-subgrp {
        margin-left: 30px;
        display: inline-block;
        width: calc(100% - 30px);
      }
      #onetrust-pc-sdk .ot-subgrp-tgl {
        float: right;
      }
      #onetrust-pc-sdk .ot-subgrp-tgl.ot-always-active-subgroup {
        width: auto;
      }
      #onetrust-pc-sdk .ot-pc-footer-logo {
        height: 30px;
        width: 100%;
        text-align: right;
        background: #f4f4f4;
        border-radius: 0 0 2px 2px;
      }
      #onetrust-pc-sdk .ot-pc-footer-logo a {
        display: inline-block;
        margin-top: 5px;
        margin-right: 10px;
      }
      #onetrust-pc-sdk #accept-recommended-btn-handler {
        float: right;
        text-align: center;
      }
      #onetrust-pc-sdk .save-preference-btn-handler {
        min-width: 155px;
        background-color: #68b631;
        border-radius: 2px;
        color: #fff;
        font-size: 0.9em;
        line-height: 1.1;
        text-align: center;
        margin-left: 15px;
        margin-right: 15px;
      }
      #onetrust-pc-sdk .ot-btn-subcntr button {
        margin-right: 16px;
      }
      #onetrust-pc-sdk.ot-ftr-stacked .save-preference-btn-handler,
      #onetrust-pc-sdk.ot-ftr-stacked .ot-btn-subcntr {
        max-width: 40%;
        white-space: normal;
        text-align: center;
      }
      #onetrust-pc-sdk.ot-ftr-stacked .ot-btn-subcntr button {
        margin-left: auto;
        margin-right: auto;
        min-width: 60%;
        max-width: 90%;
      }
      #onetrust-pc-sdk.ot-ftr-stacked .ot-btn-subcntr button:nth-child(2) {
        margin-top: 0;
      }
      #onetrust-pc-sdk.ot-ftr-stacked #accept-recommended-btn-handler {
        float: none;
      }
      #onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container {
        overflow: hidden;
      }
      #onetrust-pc-sdk #ot-pc-title {
        float: left;
        margin-left: 10px;
        max-width: 85%;
        overflow: hidden;
        position: relative;
        line-height: 1.2;
        max-height: 2.4em;
        padding-right: 1em;
        font-size: 1.37em;
      }
      #onetrust-pc-sdk #ot-pc-title:before {
        content: "...";
        right: 0px;
        bottom: 0px;
        position: absolute;
      }
      #onetrust-pc-sdk #ot-pc-title:after {
        position: absolute;
        content: "";
        width: 1em;
        height: 1em;
        right: 0px;
        background: #fff;
      }
      #onetrust-pc-sdk #ot-pc-lst {
        width: 100%;
        position: relative;
      }
      #onetrust-pc-sdk #ot-pc-lst .ot-acc-hdr {
        padding-top: 17px;
        padding-right: 15px;
        padding-bottom: 17px;
        padding-left: 20px;
        display: inline-block;
        width: calc(100% - 35px);
        vertical-align: middle;
      }
      #onetrust-pc-sdk #ot-pc-lst .ot-acc-txt {
        padding-top: 6px;
        padding-right: 15px;
        padding-bottom: 10px;
        padding-left: 20px;
      }
      #onetrust-pc-sdk .ot-lst-cntr {
        height: 100%;
      }
      #onetrust-pc-sdk #ot-pc-hdr {
        padding-top: 15px;
        padding-right: 30px;
        padding-bottom: 15px;
        padding-left: 20px;
        display: inline-block;
        width: calc(100% - 50px);
        height: 20px;
        border-bottom: 1px solid #d8d8d8;
      }
      #onetrust-pc-sdk #ot-pc-hdr input {
        height: 32px;
        width: 100%;
        border-radius: 50px;
        font-size: 0.8em;
        padding-right: 35px;
        padding-left: 15px;
        float: left;
      }
      #onetrust-pc-sdk #ot-pc-hdr input::placeholder {
        color: #d4d4d4;
        font-style: italic;
      }
      #onetrust-pc-sdk #ot-lst-cnt {
        height: calc(100% - 86px);
        padding-left: 30px;
        padding-right: 27px;
        padding-top: 20px;
        margin-top: 8px;
        margin-right: 3px;
        margin-bottom: 4px;
        margin-left: 0;
        overflow-x: hidden;
        overflow-y: auto;
        transform: translate3d(0, 0, 0);
      }
      #onetrust-pc-sdk #ot-back-arw {
        height: 12px;
        width: 12px;
      }
      #onetrust-pc-sdk #ot-lst-title {
        display: inline-block;
        font-size: 1em;
      }
      #onetrust-pc-sdk #ot-lst-title span {
        color: dimgray;
        font-weight: bold;
        margin-left: 10px;
      }
      #onetrust-pc-sdk #ot-lst-title span * {
        font-size: inherit;
      }
      #onetrust-pc-sdk .ot-lst-subhdr {
        float: right;
        position: relative;
        bottom: 6px;
      }
      #onetrust-pc-sdk #ot-search-cntr {
        display: inline-block;
        vertical-align: middle;
        position: relative;
        width: 300px;
      }
      #onetrust-pc-sdk #ot-search-cntr svg {
        position: absolute;
        right: 0px;
        width: 30px;
        height: 30px;
        font-size: 1em;
        line-height: 1;
        top: 2px;
      }
      #onetrust-pc-sdk #ot-fltr-cntr {
        display: inline-block;
        position: relative;
        margin-left: 20px;
        vertical-align: middle;
        font-size: 0;
      }
      #onetrust-pc-sdk #filter-btn-handler {
        background-color: #3860be;
        border-radius: 17px;
        -moz-transition: 0.1s ease;
        -o-transition: 0.1s ease;
        -webkit-transition: 1s ease;
        transition: 0.1s ease;
        width: 32px;
        height: 32px;
        padding: 0;
        margin: 0;
        position: relative;
      }
      #onetrust-pc-sdk #filter-btn-handler svg {
        cursor: pointer;
        width: 15px;
        height: 15px;
        position: absolute;
        left: 50%;
        top: 50%;
        transform: translate(-50%, -50%);
        padding-top: 5px;
      }
      #onetrust-pc-sdk #filter-btn-handler path {
        fill: #fff;
      }
      #onetrust-pc-sdk #ot-sel-blk {
        min-width: 200px;
        min-height: 30px;
        padding-left: 20px;
      }
      #onetrust-pc-sdk #ot-selall-vencntr,
      #onetrust-pc-sdk #ot-selall-adtlvencntr {
        float: left;
        height: 100%;
      }
      #onetrust-pc-sdk #ot-selall-vencntr label,
      #onetrust-pc-sdk #ot-selall-adtlvencntr label {
        height: 100%;
        padding-left: 0;
      }
      #onetrust-pc-sdk #ot-selall-hostcntr {
        width: 21px;
        height: 21px;
        position: relative;
        left: 20px;
      }
      #onetrust-pc-sdk #ot-selall-vencntr.line-through label::after,
      #onetrust-pc-sdk #ot-selall-adtlvencntr.line-through label::after,
      #onetrust-pc-sdk #ot-selall-licntr.line-through label::after,
      #onetrust-pc-sdk #ot-selall-hostcntr.line-through label::after,
      #onetrust-pc-sdk #ot-selall-gnvencntr.line-through label::after {
        height: auto;
        border-left: 0;
        left: 5px;
        top: 10.5px;
        transform: none;
        -o-transform: none;
        -ms-transform: none;
        -webkit-transform: none;
      }
      #onetrust-pc-sdk .ot-ven-name,
      #onetrust-pc-sdk .ot-host-name {
        color: #2c3643;
        font-weight: bold;
        font-size: 0.813em;
        line-height: 1.2;
        margin: 0;
        height: auto;
        text-align: left;
        word-break: break-word;
        word-wrap: break-word;
      }
      #onetrust-pc-sdk .ot-ven-name *,
      #onetrust-pc-sdk .ot-host-name * {
        font-size: inherit;
      }
      #onetrust-pc-sdk .ot-host-desc {
        font-size: 0.69em;
        line-height: 1.4;
        margin-top: 5px;
        margin-bottom: 5px;
        float: left;
        color: dimgray;
      }
      #onetrust-pc-sdk .ot-host-name > a {
        text-decoration: underline;
        position: relative;
        z-index: 2;
        float: left;
        margin-bottom: 5px;
        font-weight: bold;
      }
      #onetrust-pc-sdk .ot-host-hdr .ot-host-name + a {
        margin-top: 5px;
        font-size: 0.813em;
        text-decoration: underline;
      }
      #onetrust-pc-sdk .ot-ven-hdr {
        width: 88%;
        float: right;
      }
      #onetrust-pc-sdk input:focus + .ot-acc-hdr {
        outline: #000 solid 1px !important;
      }
      #onetrust-pc-sdk #ot-selall-hostcntr input[type="checkbox"],
      #onetrust-pc-sdk #ot-selall-vencntr input[type="checkbox"],
      #onetrust-pc-sdk #ot-selall-adtlvencntr input[type="checkbox"] {
        position: absolute;
      }
      #onetrust-pc-sdk .ot-host-item .ot-chkbox {
        float: left;
      }
      #onetrust-pc-sdk.ot-addtl-vendors
        #ot-lst-cnt:not(.ot-host-cnt)
        .ot-sel-all-hdr {
        right: 38px;
      }
      #onetrust-pc-sdk.ot-addtl-vendors
        #ot-lst-cnt:not(.ot-host-cnt)
        #ot-sel-blk {
        background-color: #f9f9fc;
        border: 1px solid #e2e2e2;
        width: auto;
        padding-bottom: 5px;
        padding-top: 5px;
      }
      #onetrust-pc-sdk.ot-addtl-vendors
        #ot-lst-cnt:not(.ot-host-cnt)
        .ot-sel-all-chkbox {
        right: 2px;
        width: auto;
      }
      #onetrust-pc-sdk #ot-pc-lst .ot-acc-cntr {
        position: relative;
        border-left: 1px solid #e2e2e2;
        border-right: 1px solid #e2e2e2;
        border-bottom: 1px solid #e2e2e2;
      }
      #onetrust-pc-sdk #ot-pc-lst .ot-acc-cntr input {
        z-index: 1;
      }
      #onetrust-pc-sdk #ot-pc-lst .ot-acc-cntr > .ot-acc-hdr {
        background: #f9f9fc;
        padding-top: 10px;
        padding-bottom: 10px;
        background-color: #f9f9fc;
      }
      #onetrust-pc-sdk #ot-pc-lst .ot-acc-cntr > .ot-acc-hdr input {
        z-index: 2;
      }
      #onetrust-pc-sdk
        #ot-pc-lst
        .ot-acc-cntr
        > input[type="checkbox"]:checked
        ~ .ot-acc-hdr {
        border-bottom: 1px solid #e2e2e2;
      }
      #onetrust-pc-sdk #ot-pc-lst .ot-acc-cntr .ot-addtl-venbox {
        display: none;
      }
      #onetrust-pc-sdk #ot-addtl-venlst .ot-tgl-cntr {
        margin-right: 13px;
      }
      #onetrust-pc-sdk .ot-vensec-title {
        font-size: 0.813em;
        display: inline-block;
      }
      #onetrust-pc-sdk .ot-ven-item > button:focus,
      #onetrust-pc-sdk .ot-host-item > button:focus,
      #onetrust-pc-sdk .ot-acc-cntr > button:focus {
        outline: #000 solid 2px;
      }
      #onetrust-pc-sdk .ot-ven-item > button,
      #onetrust-pc-sdk .ot-host-item > button,
      #onetrust-pc-sdk .ot-acc-cntr > button {
        position: absolute;
        cursor: pointer;
        width: 100%;
        height: 100%;
        border: 0;
        opacity: 0;
        margin: 0;
        top: 0;
        left: 0;
      }
      #onetrust-pc-sdk .ot-ven-item > button ~ .ot-acc-hdr,
      #onetrust-pc-sdk .ot-host-item > button ~ .ot-acc-hdr,
      #onetrust-pc-sdk .ot-acc-cntr > button ~ .ot-acc-hdr {
        cursor: pointer;
      }
      #onetrust-pc-sdk
        .ot-ven-item
        > button[aria-expanded="false"]
        ~ .ot-acc-txt,
      #onetrust-pc-sdk
        .ot-host-item
        > button[aria-expanded="false"]
        ~ .ot-acc-txt,
      #onetrust-pc-sdk
        .ot-acc-cntr
        > button[aria-expanded="false"]
        ~ .ot-acc-txt {
        margin-top: 0;
        max-height: 0;
        opacity: 0;
        overflow: hidden;
        width: 100%;
        transition: 0.25s ease-out;
        display: none;
      }
      #onetrust-pc-sdk
        .ot-ven-item
        > button[aria-expanded="true"]
        ~ .ot-acc-txt,
      #onetrust-pc-sdk
        .ot-host-item
        > button[aria-expanded="true"]
        ~ .ot-acc-txt,
      #onetrust-pc-sdk
        .ot-acc-cntr
        > button[aria-expanded="true"]
        ~ .ot-acc-txt {
        transition: 0.1s ease-in;
        display: block;
      }
      #onetrust-pc-sdk #ot-ven-lst,
      #onetrust-pc-sdk #ot-host-lst,
      #onetrust-pc-sdk #ot-addtl-venlst,
      #onetrust-pc-sdk #ot-gn-venlst {
        width: 100%;
      }
      #onetrust-pc-sdk #ot-ven-lst li,
      #onetrust-pc-sdk #ot-host-lst li,
      #onetrust-pc-sdk #ot-addtl-venlst li,
      #onetrust-pc-sdk #ot-gn-venlst li {
        border: 1px solid #d7d7d7;
        border-radius: 2px;
        position: relative;
        margin-top: 10px;
      }
      #onetrust-pc-sdk #ot-ven-lst .ot-tgl-cntr {
        width: 65%;
      }
      #onetrust-pc-sdk #ot-host-lst .ot-tgl-cntr {
        width: 65%;
        float: left;
      }
      #onetrust-pc-sdk label {
        margin-bottom: 0;
      }
      #onetrust-pc-sdk .ot-host-notice {
        float: right;
      }
      #onetrust-pc-sdk .ot-ven-link,
      #onetrust-pc-sdk .ot-host-expand {
        color: dimgray;
        font-size: 0.75em;
        line-height: 0.9;
        display: inline-block;
      }
      #onetrust-pc-sdk .ot-ven-link *,
      #onetrust-pc-sdk .ot-host-expand * {
        font-size: inherit;
      }
      #onetrust-pc-sdk .ot-ven-link {
        position: relative;
        z-index: 2;
      }
      #onetrust-pc-sdk .ot-ven-link:hover {
        text-decoration: underline;
      }
      #onetrust-pc-sdk .ot-ven-dets {
        border-radius: 2px;
        background-color: #f8f8f8;
      }
      #onetrust-pc-sdk .ot-ven-dets div:first-child p:first-child {
        border-top: none;
      }
      #onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:not(:first-child) {
        border-top: 1px solid #e9e9e9;
      }
      #onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n + 3) p {
        display: inline-block;
      }
      #onetrust-pc-sdk
        .ot-ven-dets
        .ot-ven-disc:nth-child(n + 3)
        p:nth-of-type(odd) {
        width: 30%;
      }
      #onetrust-pc-sdk
        .ot-ven-dets
        .ot-ven-disc:nth-child(n + 3)
        p:nth-of-type(even) {
        width: 50%;
        word-break: break-word;
        word-wrap: break-word;
      }
      #onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p {
        padding-top: 5px;
        padding-bottom: 5px;
        display: block;
      }
      #onetrust-pc-sdk .ot-ven-dets p {
        font-size: 0.69em;
        text-align: left;
        vertical-align: middle;
        word-break: break-word;
        word-wrap: break-word;
        margin: 0;
        padding-bottom: 10px;
        padding-left: 15px;
        color: #2e3644;
      }
      #onetrust-pc-sdk .ot-ven-dets p:first-child {
        padding-top: 5px;
      }
      #onetrust-pc-sdk .ot-ven-dets .ot-ven-pur p:first-child {
        border-top: 1px solid #e9e9e9;
        border-bottom: 1px solid #e9e9e9;
        padding-bottom: 5px;
        margin-bottom: 5px;
        font-weight: bold;
      }
      #onetrust-pc-sdk .ot-host-name {
        float: left;
        width: calc(100% - 50px);
      }
      #onetrust-pc-sdk .ot-host-opt {
        display: inline-block;
        width: 100%;
        margin: 0;
        font-size: inherit;
      }
      #onetrust-pc-sdk .ot-host-opt li > div div {
        font-size: 0.81em;
        padding: 5px 0;
      }
      #onetrust-pc-sdk .ot-host-opt li > div div:nth-child(1) {
        width: 30%;
        float: left;
      }
      #onetrust-pc-sdk .ot-host-opt li > div div:nth-child(2) {
        width: 70%;
        float: left;
        word-break: break-word;
        word-wrap: break-word;
      }
      #onetrust-pc-sdk #ot-host-lst li.ot-host-info {
        border: none;
        font-size: 0.8em;
        color: dimgray;
        float: left;
        text-align: left;
        padding: 10px;
        margin-bottom: 10px;
        width: calc(100% - 10px);
        background-color: #f8f8f8;
      }
      #onetrust-pc-sdk #ot-host-lst li.ot-host-info a {
        color: dimgray;
      }
      #onetrust-pc-sdk #no-results {
        text-align: center;
        margin-top: 30px;
      }
      #onetrust-pc-sdk #no-results p {
        font-size: 1em;
        color: #2e3644;
        word-break: break-word;
        word-wrap: break-word;
      }
      #onetrust-pc-sdk #no-results p span {
        font-weight: bold;
      }
      #onetrust-pc-sdk .ot-tgl-cntr {
        display: inline-block;
        vertical-align: middle;
      }
      #onetrust-pc-sdk .ot-arw-cntr,
      #onetrust-pc-sdk .ot-tgl-cntr {
        float: right;
      }
      #onetrust-pc-sdk .ot-desc-cntr {
        padding-top: 0px;
        margin-top: 20px;
        padding-right: 0px;
        border-radius: 3px;
        overflow: hidden;
        padding-bottom: 10px;
      }
      #onetrust-pc-sdk .ot-leg-border-color {
        border: 1px solid #e9e9e9;
      }
      #onetrust-pc-sdk .ot-leg-border-color .ot-subgrp-cntr {
        border-top: 1px solid #e9e9e9;
        padding-bottom: 10px;
      }
      #onetrust-pc-sdk .ot-category-desc {
        padding-bottom: 10px;
      }
      #onetrust-pc-sdk .ot-grp-hdr1 {
        padding-left: 10px;
        width: calc(100% - 20px);
        padding-top: 10px;
        margin-bottom: 0px;
        padding-bottom: 8px;
      }
      #onetrust-pc-sdk .ot-subgrp-cntr {
        padding-top: 10px;
      }
      #onetrust-pc-sdk .ot-desc-cntr > *:not(.ot-grp-hdr1) {
        padding-left: 10px;
        padding-right: 10px;
      }
      #onetrust-pc-sdk .ot-pli-hdr {
        overflow: hidden;
        padding-top: 7.5px;
        padding-bottom: 7.5px;
        background-color: #f8f8f8;
        border: none;
        border-bottom: 1px solid #e9e9e9;
      }
      #onetrust-pc-sdk .ot-pli-hdr span:first-child {
        text-align: left;
        max-width: 80px;
        padding-right: 5px;
      }
      #onetrust-pc-sdk .ot-pli-hdr span:last-child {
        padding-right: 20px;
        text-align: center;
      }
      #onetrust-pc-sdk .ot-li-title {
        float: right;
        font-size: 0.813em;
      }
      #onetrust-pc-sdk .ot-desc-cntr .ot-tgl-cntr:first-of-type,
      #onetrust-pc-sdk .ot-cat-header + .ot-tgl {
        padding-left: 55px;
        padding-right: 7px;
      }
      #onetrust-pc-sdk
        .ot-always-active-group
        .ot-grp-hdr1
        .ot-tgl-cntr:first-of-type {
        padding-left: 0px;
      }
      #onetrust-pc-sdk .ot-cat-header,
      #onetrust-pc-sdk .ot-subgrp h4 {
        max-width: calc(100% - 133px);
      }
      #onetrust-pc-sdk #ot-lst-cnt #ot-sel-blk {
        width: 100%;
        display: inline-block;
        padding: 0;
      }
      #onetrust-pc-sdk .ot-sel-all {
        display: inline-block;
        width: 100%;
      }
      #onetrust-pc-sdk .ot-sel-all-hdr,
      #onetrust-pc-sdk .ot-sel-all-chkbox {
        width: 100%;
        float: right;
        position: relative;
      }
      #onetrust-pc-sdk .ot-sel-all-chkbox {
        z-index: 1;
      }
      #onetrust-pc-sdk :not(.ot-hosts-ui) .ot-sel-all-hdr,
      #onetrust-pc-sdk :not(.ot-hosts-ui) .ot-sel-all-chkbox {
        right: 23px;
        width: calc(100% - 23px);
      }
      #onetrust-pc-sdk .ot-consent-hdr,
      #onetrust-pc-sdk .ot-li-hdr {
        float: right;
        font-size: 0.813em;
        position: relative;
        line-height: normal;
        text-align: center;
        word-break: break-word;
        word-wrap: break-word;
      }
      #onetrust-pc-sdk .ot-hosts-ui .ot-consent-hdr {
        float: left;
        position: relative;
        left: 5px;
      }
      #onetrust-pc-sdk .ot-li-hdr {
        max-width: 100px;
        margin-right: 10px;
      }
      #onetrust-pc-sdk .ot-consent-hdr {
        max-width: 55px;
      }
      #onetrust-pc-sdk .ot-ven-ctgl {
        margin-left: 10px;
      }
      #onetrust-pc-sdk .ot-ven-litgl {
        margin-right: 55px;
      }
      #onetrust-pc-sdk .ot-ven-litgl.ot-ven-litgl-only {
        margin-right: 86px;
      }
      #onetrust-pc-sdk .ot-ven-ctgl,
      #onetrust-pc-sdk .ot-ven-litgl,
      #onetrust-pc-sdk .ot-ven-gvctgl {
        float: left;
      }
      #onetrust-pc-sdk .ot-ven-ctgl label,
      #onetrust-pc-sdk .ot-ven-litgl label,
      #onetrust-pc-sdk .ot-ven-gvctgl label {
        width: 22px;
        padding: 0;
      }
      #onetrust-pc-sdk #ot-selall-licntr {
        display: block;
        width: 21px;
        height: 21px;
        position: relative;
        float: right;
        right: 80px;
      }
      #onetrust-pc-sdk #ot-selall-licntr input {
        position: absolute;
      }
      #onetrust-pc-sdk #ot-selall-vencntr,
      #onetrust-pc-sdk #ot-selall-adtlvencntr,
      #onetrust-pc-sdk #ot-selall-gnvencntr {
        float: right;
        width: 21px;
        height: 21px;
        position: relative;
        right: 15px;
      }
      #onetrust-pc-sdk #ot-ven-lst .ot-tgl-cntr {
        float: right;
        width: auto;
      }
      #onetrust-pc-sdk .ot-ven-hdr {
        float: left;
        width: 60%;
      }
      #onetrust-pc-sdk #vdr-lst-dsc {
        font-size: 0.812em;
        line-height: 1.5;
        padding: 10px 15px 5px 15px;
      }
      #onetrust-pc-sdk #ot-anchor {
        border: 12px solid transparent;
        display: none;
        position: absolute;
        z-index: 2147483647;
        top: 40px;
        right: 35px;
        transform: rotate(45deg);
        -o-transform: rotate(45deg);
        -ms-transform: rotate(45deg);
        -webkit-transform: rotate(45deg);
        background-color: #fff;
        -webkit-box-shadow: -3px -3px 5px -2px #c7c5c7;
        -moz-box-shadow: -3px -3px 5px -2px #c7c5c7;
        box-shadow: -3px -3px 5px -2px #c7c5c7;
      }
      #onetrust-pc-sdk #ot-fltr-modal {
        width: 300px;
        position: absolute;
        z-index: 2147483646;
        top: 46px;
        height: 90%;
        max-height: 350px;
        display: none;
        -moz-transition: 0.2s ease;
        -o-transition: 0.2s ease;
        -webkit-transition: 2s ease;
        transition: 0.2s ease;
        opacity: 1;
        right: 0;
      }
      #onetrust-pc-sdk #ot-fltr-modal button {
        max-width: 200px;
        line-height: 1;
        word-break: break-word;
        white-space: normal;
        height: auto;
        font-weight: bold;
      }
      #onetrust-pc-sdk #ot-fltr-cnt {
        background-color: #fff;
        margin: 5px;
        border-radius: 3px;
        height: 100%;
        margin-right: 10px;
        padding-right: 10px;
        -webkit-box-shadow: 0px 0px 12px 2px #c7c5c7;
        -moz-box-shadow: 0px 0px 12px 2px #c7c5c7;
        box-shadow: 0px 0px 12px 2px #c7c5c7;
      }
      #onetrust-pc-sdk .ot-fltr-scrlcnt {
        overflow-y: auto;
        overflow-x: hidden;
        clear: both;
        max-height: calc(100% - 60px);
      }
      #onetrust-pc-sdk .ot-fltr-opt {
        margin-bottom: 25px;
        margin-left: 15px;
        clear: both;
      }
      #onetrust-pc-sdk .ot-fltr-opt label {
        height: auto;
      }
      #onetrust-pc-sdk .ot-fltr-opt span {
        cursor: pointer;
        color: dimgray;
        font-size: 0.8em;
        line-height: 1.1;
        font-weight: normal;
      }
      #onetrust-pc-sdk #clear-filters-handler {
        float: right;
        margin-top: 15px;
        margin-bottom: 10px;
        text-decoration: none;
        color: #3860be;
        font-size: 0.9em;
        border: none;
        padding: 1px;
      }
      #onetrust-pc-sdk #clear-filters-handler:hover {
        color: #1883fd;
      }
      #onetrust-pc-sdk #clear-filters-handler:focus {
        outline: #000 solid 1px;
      }
      #onetrust-pc-sdk #filter-apply-handler {
        margin-right: 10px;
      }
      #onetrust-pc-sdk .ot-grp-desc + .ot-leg-btn-container {
        margin-top: 0;
      }
      #onetrust-pc-sdk .ot-leg-btn-container {
        display: inline-block;
        width: 100%;
        margin-top: 10px;
      }
      #onetrust-pc-sdk .ot-leg-btn-container button {
        height: auto;
        padding: 6.5px 8px;
        margin-bottom: 0;
        line-height: normal;
        letter-spacing: 0;
      }
      #onetrust-pc-sdk .ot-leg-btn-container svg {
        display: none;
        height: 14px;
        width: 14px;
        padding-right: 5px;
        vertical-align: sub;
      }
      #onetrust-pc-sdk .ot-active-leg-btn {
        cursor: default;
        pointer-events: none;
      }
      #onetrust-pc-sdk .ot-active-leg-btn svg {
        display: inline-block;
      }
      #onetrust-pc-sdk .ot-remove-objection-handler {
        border: none;
        text-decoration: underline;
        padding: 0;
        font-size: 0.82em;
        font-weight: 600;
        line-height: 1.4;
        padding-left: 10px;
      }
      #onetrust-pc-sdk .ot-obj-leg-btn-handler span {
        font-weight: bold;
        text-align: center;
        font-size: 0.91em;
        line-height: 1.5;
      }
      #onetrust-pc-sdk.otPcTab[dir="rtl"] input ~ .ot-acc-hdr .ot-arw,
      #onetrust-pc-sdk.otPcTab[dir="rtl"] #ot-back-arw {
        transform: rotate(180deg);
        -o-transform: rotate(180deg);
        -ms-transform: rotate(180deg);
        -webkit-transform: rotate(180deg);
      }
      #onetrust-pc-sdk.otPcTab[dir="rtl"] input:checked ~ .ot-acc-hdr .ot-arw {
        transform: rotate(270deg);
        -o-transform: rotate(270deg);
        -ms-transform: rotate(270deg);
        -webkit-transform: rotate(270deg);
      }
      #onetrust-pc-sdk.otPcTab[dir="rtl"] #ot-search-cntr svg {
        right: 15px;
      }
      #onetrust-pc-sdk.otPcTab[dir="rtl"] .ot-chkbox label::after {
        transform: rotate(45deg);
        -webkit-transform: rotate(45deg);
        -o-transform: rotate(45deg);
        -ms-transform: rotate(45deg);
        border-left: 0;
        border-right: 3px solid;
      }
      #onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon {
        padding: 0;
        background-color: transparent;
        border: none;
        margin: 0;
      }
      @media (max-width: 767px) {
        #onetrust-pc-sdk {
          width: 100%;
          border: none;
        }
        #onetrust-pc-sdk .ot-sdk-container,
        #onetrust-pc-sdk .ot-sdk-container {
          padding: 0;
          margin: 0;
        }
        #onetrust-pc-sdk #ot-pc-title {
          margin-left: 10px;
          max-width: 60%;
        }
        #onetrust-pc-sdk .ot-desc-cntr {
          margin: 0;
          padding-top: 20px;
          padding-right: 20px;
          padding-bottom: 15px;
          padding-left: 20px;
          position: relative;
          left: auto;
        }
        #onetrust-pc-sdk .ot-desc-cntr {
          margin-top: 20px;
          margin-left: 20px;
          padding: 0;
          padding-bottom: 10px;
        }
        #onetrust-pc-sdk .ot-grps-cntr {
          max-height: none;
          overflow: hidden;
        }
        #onetrust-pc-sdk #accept-recommended-btn-handler {
          float: none;
        }
      }
      @media (min-width: 768px) {
        #onetrust-pc-sdk.ot-tgl-with-label .ot-label-status {
          display: inline;
        }
        #onetrust-pc-sdk.ot-tgl-with-label #ot-pc-lst .ot-label-status {
          display: none;
        }
        #onetrust-pc-sdk.ot-tgl-with-label.ot-leg-opt-out .ot-pli-hdr {
          padding-right: 8%;
        }
        #onetrust-pc-sdk.ot-tgl-with-label .ot-cat-header {
          max-width: 60%;
        }
        #onetrust-pc-sdk.ot-tgl-with-label .ot-subgrp h4 {
          max-width: 58%;
        }
        #onetrust-pc-sdk.ot-tgl-with-label
          .ot-desc-cntr
          .ot-tgl-cntr:first-of-type,
        #onetrust-pc-sdk.ot-tgl-with-label .ot-cat-header + .ot-tgl {
          padding-left: 15px;
        }
      }
      @media (max-width: 640px) {
        #onetrust-pc-sdk {
          height: 100%;
        }
        #onetrust-pc-sdk .ot-pc-header {
          padding: 10px;
          width: calc(100% - 20px);
        }
        #onetrust-pc-sdk #ot-pc-content {
          overflow: auto;
        }
        #onetrust-pc-sdk .ot-sdk-row .ot-sdk-columns {
          width: 100%;
        }
        #onetrust-pc-sdk .ot-desc-cntr {
          margin: 0;
          overflow: hidden;
        }
        #onetrust-pc-sdk .ot-desc-cntr {
          margin-left: 10px;
          width: calc(100% - 15px);
          margin-top: 5px;
          margin-bottom: 5px;
        }
        #onetrust-pc-sdk .ot-ven-hdr {
          max-width: 80%;
        }
        #onetrust-pc-sdk #ot-lst-cnt {
          width: calc(100% - 18px);
          padding-top: 13px;
          padding-right: 5px;
          padding-left: 10px;
        }
        #onetrust-pc-sdk .ot-grps-cntr {
          width: 100%;
        }
        #onetrust-pc-sdk .ot-pc-footer {
          max-height: 300px;
        }
        #onetrust-pc-sdk #ot-pc-content,
        #onetrust-pc-sdk #ot-pc-lst {
          height: calc(100% - 322px);
        }
      }
      @media (max-width: 640px) and (orientation: portrait) {
        #onetrust-pc-sdk #ot-pc-hdr {
          height: 70px;
          padding: 15px 0;
          width: 100%;
        }
        #onetrust-pc-sdk .ot-lst-subhdr {
          width: calc(100% - 15px);
          float: none;
          bottom: auto;
          display: inline-block;
          padding-top: 8px;
          padding-left: 15px;
        }
        #onetrust-pc-sdk .ot-btn-subcntr {
          float: none;
        }
        #onetrust-pc-sdk #ot-search-cntr {
          display: inline-block;
          width: calc(100% - 55px);
          position: relative;
        }
        #onetrust-pc-sdk #ot-anchor {
          top: 75px;
          right: 30px;
        }
        #onetrust-pc-sdk #ot-fltr-modal {
          top: 81px;
        }
        #onetrust-pc-sdk #ot-fltr-cntr {
          float: right;
          right: 15px;
        }
        #onetrust-pc-sdk #ot-lst-title {
          padding-left: 15px;
        }
        #onetrust-pc-sdk .ot-lst-cntr {
          overflow-y: scroll;
        }
        #onetrust-pc-sdk #ot-lst-cnt {
          height: auto;
          overflow: hidden;
        }
        #onetrust-pc-sdk .save-preference-btn-handler,
        #onetrust-pc-sdk #accept-recommended-btn-handler,
        #onetrust-pc-sdk .ot-pc-refuse-all-handler {
          width: calc(100% - 33px);
        }
        #onetrust-pc-sdk.ot-ftr-stacked .save-preference-btn-handler,
        #onetrust-pc-sdk.ot-ftr-stacked .ot-btn-subcntr {
          max-width: none;
        }
        #onetrust-pc-sdk.ot-ftr-stacked .ot-pc-footer button {
          margin: 15px;
        }
        #onetrust-pc-sdk.ot-ftr-stacked .ot-btn-subcntr button {
          min-width: none;
          max-width: none;
        }
        #onetrust-pc-sdk.ot-ftr-stacked .ot-btn-subcntr button:nth-child(2) {
          margin-top: 15px;
        }
        #onetrust-pc-sdk.ot-ftr-stacked
          .ot-btn-container
          button:not(:last-child) {
          margin-bottom: 0;
        }
      }
      @media (max-width: 425px) {
        #onetrust-pc-sdk #ot-pc-lst .ot-acc-txt {
          padding-top: 6px;
          padding-bottom: 10px;
        }
        #onetrust-pc-sdk #ot-pc-lst .ot-host-notice {
          float: left;
          margin-left: 30px;
        }
        #onetrust-pc-sdk #ot-pc-lst .ot-arw-cntr {
          float: none;
          display: inline;
        }
        #onetrust-pc-sdk #ot-pc-lst .ot-ven-hdr {
          float: left;
          width: 100%;
          max-width: 85%;
        }
        #onetrust-pc-sdk.ot-addtl-vendors
          #ot-pc-lst
          .ot-acc-cntr
          .ot-arw-cntr:first-of-type {
          float: right;
        }
        #onetrust-pc-sdk #ot-pc-title {
          max-width: 100%;
        }
        #onetrust-pc-sdk .ot-subgrp-cntr li.ot-subgrp {
          margin-left: 10px;
          width: calc(100% - 10px);
        }
        #onetrust-pc-sdk #ot-ven-lst .ot-tgl-cntr {
          width: auto;
          float: right;
        }
        #onetrust-pc-sdk #ot-ven-lst .ot-arw-cntr {
          float: right;
        }
        #onetrust-pc-sdk .ot-ven-hdr {
          max-width: 47%;
        }
        #onetrust-pc-sdk .ot-always-active-group .ot-tgl-cntr:first-of-type {
          max-width: none;
          padding-left: 20px;
        }
      }
      @media only screen and (max-height: 425px) and (max-width: 896px) and (orientation: landscape) {
        #onetrust-pc-sdk {
          height: 100%;
          width: 100%;
          max-width: none;
        }
        #onetrust-pc-sdk .ot-always-active-group .ot-tgl-cntr {
          max-width: none;
        }
        #onetrust-pc-sdk .ot-pc-header {
          padding: 10px;
          width: calc(100% - 20px);
        }
        #onetrust-pc-sdk .ot-lst-cntr {
          overflow-y: scroll;
        }
        #onetrust-pc-sdk #ot-lst-cnt {
          height: auto;
          overflow: hidden;
        }
        #onetrust-pc-sdk #accept-recommended-btn-handler {
          float: right;
        }
        #onetrust-pc-sdk .save-preference-btn-handler,
        #onetrust-pc-sdk #accept-recommended-btn-handler,
        #onetrust-pc-sdk .ot-pc-refuse-all-handler {
          width: auto;
        }
        #onetrust-pc-sdk #ot-pc-content,
        #onetrust-pc-sdk #ot-pc-lst {
          height: calc(100% - 155px);
        }
        #onetrust-pc-sdk .ot-pc-footer-logo {
          display: none;
        }
        #onetrust-pc-sdk.ot-shw-fltr .ot-lst-cntr {
          overflow: hidden;
        }
        #onetrust-pc-sdk.ot-shw-fltr #ot-pc-lst {
          position: static;
        }
        #onetrust-pc-sdk.ot-shw-fltr #ot-fltr-modal {
          top: 0;
          width: 100%;
          height: 100%;
          max-height: none;
        }
        #onetrust-pc-sdk.ot-shw-fltr #ot-fltr-modal > div {
          margin: 0;
          box-sizing: initial;
          height: 100%;
          max-height: none;
        }
        #onetrust-pc-sdk.ot-shw-fltr #clear-filters-handler {
          padding-right: 20px;
        }
        #onetrust-pc-sdk.ot-shw-fltr #ot-anchor {
          display: none !important;
        }
        #onetrust-pc-sdk .ot-pc-footer button {
          margin: 10px;
        }
      }
      @media (max-width: 425px),
        (max-width: 896px) and (max-height: 425px) and (orientation: landscape) {
        #onetrust-pc-sdk .ot-pc-header {
          padding-right: 20px;
        }
        #onetrust-pc-sdk .ot-pc-logo {
          margin-left: 0px;
          margin-top: 5px;
          width: 150px;
        }
        #onetrust-pc-sdk .ot-close-icon {
          width: 44px;
          height: 44px;
          background-size: 12px;
        }
        #onetrust-pc-sdk .ot-grp-hdr1 {
          float: right;
          margin-left: 10px;
          width: auto;
        }
        #onetrust-pc-sdk .ot-grp-hdr1 {
          margin-left: 0px;
          padding-right: 10px;
        }
        #onetrust-pc-sdk #ot-pvcy-hdr,
        #onetrust-pc-sdk .ot-grp-hdr1 .ot-cat-header {
          display: none;
        }
        #onetrust-pc-sdk .ot-grp-hdr1 + .ot-vlst-cntr {
          padding-top: 10px;
        }
      }
      @media only screen and (max-height: 610px) {
        #onetrust-pc-sdk {
          max-height: 100%;
        }
      }
      #onetrust-consent-sdk #onetrust-pc-sdk,
      #onetrust-consent-sdk #ot-search-cntr,
      #onetrust-consent-sdk #onetrust-pc-sdk .ot-switch.ot-toggle,
      #onetrust-consent-sdk #onetrust-pc-sdk ot-grp-hdr1 .checkbox,
      #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title:after,
      #onetrust-consent-sdk #onetrust-pc-sdk #ot-sel-blk,
      #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-cnt,
      #onetrust-consent-sdk #onetrust-pc-sdk #ot-anchor {
        background-color: #ffffff;
      }

      #onetrust-consent-sdk #onetrust-pc-sdk h3,
      #onetrust-consent-sdk #onetrust-pc-sdk h4,
      #onetrust-consent-sdk #onetrust-pc-sdk h5,
      #onetrust-consent-sdk #onetrust-pc-sdk h6,
      #onetrust-consent-sdk #onetrust-pc-sdk p,
      #onetrust-consent-sdk #onetrust-pc-sdk #ot-ven-lst .ot-ven-opts p,
      #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-desc,
      #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title,
      #onetrust-consent-sdk #onetrust-pc-sdk .ot-li-title,
      #onetrust-consent-sdk #onetrust-pc-sdk .ot-sel-all-hdr span,
      #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
      #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-modal #modal-header,
      #onetrust-consent-sdk #onetrust-pc-sdk .ot-checkbox label span,
      #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-sel-blk p,
      #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-lst-title span,
      #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .back-btn-handler p,
      #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .ot-ven-name,
      #onetrust-consent-sdk
        #onetrust-pc-sdk
        #ot-pc-lst
        #ot-ven-lst
        .consent-category,
      #onetrust-consent-sdk
        #onetrust-pc-sdk
        .ot-leg-btn-container
        .ot-inactive-leg-btn,
      #onetrust-consent-sdk #onetrust-pc-sdk .ot-label-status,
      #onetrust-consent-sdk #onetrust-pc-sdk .ot-chkbox label span,
      #onetrust-consent-sdk #onetrust-pc-sdk #clear-filters-handler {
        color: #222222;
      }
      #onetrust-consent-sdk #onetrust-pc-sdk .privacy-notice-link,
      #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler,
      #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler + a,
      #onetrust-consent-sdk #onetrust-pc-sdk .category-host-list-handler,
      #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-link,
      #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-name a,
      #onetrust-consent-sdk
        #onetrust-pc-sdk
        #ot-host-lst
        .ot-acc-hdr
        .ot-host-expand,
      #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info a {
        color: #004b83;
      }
      #onetrust-consent-sdk #onetrust-banner-sdk a[href],
      #onetrust-consent-sdk #onetrust-banner-sdk .ot-link-btn {
        color: #ffffff;
      }
      #onetrust-consent-sdk
        #onetrust-pc-sdk
        .category-vendors-list-handler:hover {
        opacity: 0.7;
      }
      #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt,
      #onetrust-consent-sdk
        #onetrust-pc-sdk
        .ot-acc-txt
        .ot-subgrp-tgl
        .ot-switch.ot-toggle {
        background-color: #f8f8f8;
      }

      #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
      #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-ven-dets {
        background-color: #f8f8f8;
      }
      #onetrust-consent-sdk
        #onetrust-pc-sdk
        button:not(#clear-filters-handler):not(.ot-close-icon):not(#filter-btn-handler):not(.ot-remove-objection-handler):not(.ot-obj-leg-btn-handler):not([aria-expanded]):not(.ot-link-btn),
      #onetrust-consent-sdk
        #onetrust-pc-sdk
        .ot-leg-btn-container
        .ot-active-leg-btn {
        background-color: #183642;
        border-color: #183642;
        color: #fff;
      }
      #onetrust-consent-sdk #onetrust-pc-sdk .ot-active-menu {
        border-color: #183642;
      }

      #onetrust-consent-sdk
        #onetrust-pc-sdk
        .ot-leg-btn-container
        .ot-remove-objection-handler {
        background-color: transparent;
        border: 1px solid transparent;
      }
      #onetrust-consent-sdk
        #onetrust-pc-sdk
        .ot-leg-btn-container
        .ot-inactive-leg-btn {
        background-color: #ffffff;
        color: #78808e;
        border-color: #78808e;
      }
      #onetrust-consent-sdk #onetrust-pc-sdk .category-menu-switch-handler {
        background-color: #f2f2f2;
      }
      #onetrust-consent-sdk #onetrust-pc-sdk .ot-active-menu {
        background-color: #ffffff;
      }
      .ot-sdk-cookie-policy {
        font-family: inherit;
        font-size: 16px;
      }
      .ot-sdk-cookie-policy h3,
      .ot-sdk-cookie-policy h4,
      .ot-sdk-cookie-policy h6,
      .ot-sdk-cookie-policy p,
      .ot-sdk-cookie-policy li,
      .ot-sdk-cookie-policy a,
      .ot-sdk-cookie-policy th,
      .ot-sdk-cookie-policy #cookie-policy-description,
      .ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,
      .ot-sdk-cookie-policy #cookie-policy-title {
        color: dimgray;
      }
      .ot-sdk-cookie-policy #cookie-policy-description {
        margin-bottom: 1em;
      }
      .ot-sdk-cookie-policy h4 {
        font-size: 1.2em;
      }
      .ot-sdk-cookie-policy h6 {
        font-size: 1em;
        margin-top: 2em;
      }
      .ot-sdk-cookie-policy th {
        min-width: 75px;
      }
      .ot-sdk-cookie-policy a,
      .ot-sdk-cookie-policy a:hover {
        background: #fff;
      }
      .ot-sdk-cookie-policy thead {
        background-color: #f6f6f4;
        font-weight: bold;
      }
      .ot-sdk-cookie-policy .ot-mobile-border {
        display: none;
      }
      .ot-sdk-cookie-policy section {
        margin-bottom: 2em;
      }
      .ot-sdk-cookie-policy table {
        border-collapse: inherit;
      }
      #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy {
        font-family: inherit;
        font-size: 16px;
      }
      #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h3,
      #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h4,
      #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,
      #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,
      #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,
      #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,
      #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,
      #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,
      #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,
      #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title {
        color: dimgray;
      }
      #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description {
        margin-bottom: 1em;
      }
      #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup {
        margin-left: 1.5em;
      }
      #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,
      #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy
        .ot-sdk-cookie-policy-group-desc,
      #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-table-header,
      #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,
      #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,
      #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td {
        font-size: 0.9em;
      }
      #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td span,
      #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td a {
        font-size: inherit;
      }
      #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy
        .ot-sdk-cookie-policy-group {
        font-size: 1em;
        margin-bottom: 0.6em;
      }
      #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy
        .ot-sdk-cookie-policy-title {
        margin-bottom: 1.2em;
      }
      #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy > section {
        margin-bottom: 1em;
      }
      #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th {
        min-width: 75px;
      }
      #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,
      #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a:hover {
        background: #fff;
      }
      #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead {
        background-color: #f6f6f4;
        font-weight: bold;
      }
      #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-mobile-border {
        display: none;
      }
      #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy section {
        margin-bottom: 2em;
      }
      #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li {
        list-style: disc;
        margin-left: 1.5em;
      }
      #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li h4 {
        display: inline-block;
      }
      #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table {
        border-collapse: inherit;
        margin: auto;
        border: 1px solid #d7d7d7;
        border-radius: 5px;
        border-spacing: initial;
        width: 100%;
        overflow: hidden;
      }
      #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th,
      #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td {
        border-bottom: 1px solid #d7d7d7;
        border-right: 1px solid #d7d7d7;
      }
      #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td {
        border-bottom: 0px;
      }
      #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr th:last-child,
      #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr td:last-child {
        border-right: 0px;
      }
      #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,
      #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type {
        width: 25%;
      }
      .ot-sdk-cookie-policy[dir="rtl"] {
        text-align: left;
      }
      #ot-sdk-cookie-policy h3 {
        font-size: 1.5em;
      }
      @media only screen and (max-width: 530px) {
        .ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) table,
        .ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead,
        .ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tbody,
        .ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) th,
        .ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td,
        .ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr {
          display: block;
        }
        .ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead tr {
          position: absolute;
          top: -9999px;
          left: -9999px;
        }
        .ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr {
          margin: 0 0 1em 0;
        }
        .ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd),
        .ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2)
          tr:nth-child(odd)
          a {
          background: #f6f6f4;
        }
        .ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td {
          border: none;
          border-bottom: 1px solid #eee;
          position: relative;
          padding-left: 50%;
        }
        .ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before {
          position: absolute;
          height: 100%;
          left: 6px;
          width: 40%;
          padding-right: 10px;
        }
        .ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) .ot-mobile-border {
          display: inline-block;
          background-color: #e4e4e4;
          position: absolute;
          height: 100%;
          top: 0;
          left: 45%;
          width: 2px;
        }
        .ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before {
          content: attr(data-label);
          font-weight: bold;
        }
        .ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) li {
          word-break: break-word;
          word-wrap: break-word;
        }
        #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table {
          overflow: hidden;
        }
        #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td {
          border: none;
          border-bottom: 1px solid #d7d7d7;
        }
        #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table,
        #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead,
        #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tbody,
        #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,
        #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,
        #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr {
          display: block;
        }
        #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,
        #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type {
          width: auto;
        }
        #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr {
          margin: 0 0 1em 0;
        }
        #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before {
          height: 100%;
          width: 40%;
          padding-right: 10px;
        }
        #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before {
          content: attr(data-label);
          font-weight: bold;
        }
        #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li {
          word-break: break-word;
          word-wrap: break-word;
        }
        #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead tr {
          position: absolute;
          top: -9999px;
          left: -9999px;
          z-index: -9999;
        }
        #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td {
          border-bottom: 1px solid #d7d7d7;
          border-right: 0px;
        }
        #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy
          table
          tr:last-child
          td:last-child {
          border-bottom: 0px;
        }
      }

      #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h5,
      #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,
      #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,
      #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,
      #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,
      #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,
      #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,
      #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description {
        color: #696969;
      }
      #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th {
        color: #696969;
      }
      #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy
        .ot-sdk-cookie-policy-group {
        color: #696969;
      }

      #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title {
        color: #696969;
      }

      #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th {
        background-color: #f8f8f8;
      }
    </style>
    <meta
      http-equiv="origin-trial"
      content="AxujKG9INjsZ8/gUq8+dTruNvk7RjZQ1oFhhgQbcTJKDnZfbzSTE81wvC2Hzaf3TW4avA76LTZEMdiedF1vIbA4AAABueyJvcmlnaW4iOiJodHRwczovL2ltYXNkay5nb29nbGVhcGlzLmNvbTo0NDMiLCJmZWF0dXJlIjoiVHJ1c3RUb2tlbnMiLCJleHBpcnkiOjE2NTI3NzQ0MDAsImlzVGhpcmRQYXJ0eSI6dHJ1ZX0="
    />
    <meta
      http-equiv="origin-trial"
      content="Azuce85ORtSnWe1MZDTv68qpaW3iHyfL9YbLRy0cwcCZwVnePnOmkUJlG8HGikmOwhZU22dElCcfrfX2HhrBPAkAAAB7eyJvcmlnaW4iOiJodHRwczovL2RvdWJsZWNsaWNrLm5ldDo0NDMiLCJmZWF0dXJlIjoiVHJ1c3RUb2tlbnMiLCJleHBpcnkiOjE2NTI3NzQ0MDAsImlzU3ViZG9tYWluIjp0cnVlLCJpc1RoaXJkUGFydHkiOnRydWV9"
    />
    <meta
      http-equiv="origin-trial"
      content="A16nvcdeoOAqrJcmjLRpl1I6f3McDD8EfofAYTt/P/H4/AWwB99nxiPp6kA0fXoiZav908Z8etuL16laFPUdfQsAAACBeyJvcmlnaW4iOiJodHRwczovL2dvb2dsZXRhZ3NlcnZpY2VzLmNvbTo0NDMiLCJmZWF0dXJlIjoiVHJ1c3RUb2tlbnMiLCJleHBpcnkiOjE2NTI3NzQ0MDAsImlzU3ViZG9tYWluIjp0cnVlLCJpc1RoaXJkUGFydHkiOnRydWV9"
    />
    <meta
      http-equiv="origin-trial"
      content="AxBHdr0J44vFBQtZUqX9sjiqf5yWZ/OcHRcRMN3H9TH+t90V/j3ENW6C8+igBZFXMJ7G3Pr8Dd13632aLng42wgAAACBeyJvcmlnaW4iOiJodHRwczovL2dvb2dsZXN5bmRpY2F0aW9uLmNvbTo0NDMiLCJmZWF0dXJlIjoiVHJ1c3RUb2tlbnMiLCJleHBpcnkiOjE2NTI3NzQ0MDAsImlzU3ViZG9tYWluIjp0cnVlLCJpc1RoaXJkUGFydHkiOnRydWV9"
    />
    <meta
      http-equiv="origin-trial"
      content="A88BWHFjcawUfKU3lIejLoryXoyjooBXLgWmGh+hNcqMK44cugvsI5YZbNarYvi3roc1fYbHA1AVbhAtuHZflgEAAAB2eyJvcmlnaW4iOiJodHRwczovL2dvb2dsZS5jb206NDQzIiwiZmVhdHVyZSI6IlRydXN0VG9rZW5zIiwiZXhwaXJ5IjoxNjUyNzc0NDAwLCJpc1N1YmRvbWFpbiI6dHJ1ZSwiaXNUaGlyZFBhcnR5Ijp0cnVlfQ=="
    />
    <meta
      http-equiv="origin-trial"
      content="A/OOU4XAFfeAV4kM4+W9WBwNAHqq/bvtrRcJ1wpnNyO/i076BSUy1d14l2kBEgVmEuvxojSpD23172C6hBg2AQYAAACWeyJvcmlnaW4iOiJodHRwczovL2RvdWJsZWNsaWNrLm5ldDo0NDMiLCJmZWF0dXJlIjoiQ29udmVyc2lvbk1lYXN1cmVtZW50IiwiZXhwaXJ5IjoxNjM0MDgzMTk5LCJpc1N1YmRvbWFpbiI6dHJ1ZSwiaXNUaGlyZFBhcnR5Ijp0cnVlLCJ1c2FnZSI6InN1YnNldCJ9"
    />
    <meta
      http-equiv="origin-trial"
      content="AwrB+XVH/KV6HfZNVtSEqlUJi3yUbtCp0/TJRj+38NDIw19/9P1h7ECTtdLdhIzG0Bsl4n/0rVmttJtGUCcewgAAAACceyJvcmlnaW4iOiJodHRwczovL2dvb2dsZXN5bmRpY2F0aW9uLmNvbTo0NDMiLCJmZWF0dXJlIjoiQ29udmVyc2lvbk1lYXN1cmVtZW50IiwiZXhwaXJ5IjoxNjM0MDgzMTk5LCJpc1N1YmRvbWFpbiI6dHJ1ZSwiaXNUaGlyZFBhcnR5Ijp0cnVlLCJ1c2FnZSI6InN1YnNldCJ9"
    />
    <meta
      http-equiv="origin-trial"
      content="Ax15QOERqai2A5XWrDY38Eg07xh2T0pkhpDPJuDlxr7D2Ka8wHvklgK7tTPZOnT+8H31lwHto5JpvYV8jRn1WgIAAACceyJvcmlnaW4iOiJodHRwczovL2dvb2dsZXRhZ3NlcnZpY2VzLmNvbTo0NDMiLCJmZWF0dXJlIjoiQ29udmVyc2lvbk1lYXN1cmVtZW50IiwiZXhwaXJ5IjoxNjM0MDgzMTk5LCJpc1N1YmRvbWFpbiI6dHJ1ZSwiaXNUaGlyZFBhcnR5Ijp0cnVlLCJ1c2FnZSI6InN1YnNldCJ9"
    />
    <!-- <script src="report_files/pubads_impl_2021100701.js" async=""></script> -->
    <link rel="preload" href="report_files/integrator.js" as="script" />
    <script type="text/javascript" src="report_files/integrator.js"></script>
    <link rel="preload" href="report_files/integrator.js" as="script" />
    <script type="text/javascript" src="report_files/integrator.js"></script>
    <link rel="preload" href="report_files/integrator.js" as="script" />
    <script type="text/javascript" src="report_files/integrator.js"></script>
    <script charset="utf-8" src="report_files/12.js"></script>
    <link rel="preload" href="report_files/integrator.js" as="script" />
    <script type="text/javascript" src="report_files/integrator.js"></script>
  </head>
  <body class="article-page">
    <div id="MathJax_Message" style="display: none"></div>

    <noscript
      ><iframe
        src="https://www.googletagmanager.com/ns.html?id=GTM-NWDMT9Q"
        height="0"
        width="0"
        style="display: none; visibility: hidden"
      ></iframe
    ></noscript>


    <div
      class="u-container u-mt-32 u-mb-32 u-clearfix"
      id="content"
      data-component="article-container"
      data-container-type="article"
    >
      <main
        class="c-article-main-column u-float-left js-main-column"
        data-track-component="article body"
      >
        <div
          class="c-context-bar u-hide"
          data-component="context-bar"
          aria-hidden="true"
        >
          <div class="c-context-bar__container u-container">
            <div class="c-context-bar__title">
              <!-- Inference and analysis of cell-cell communication using CellChat -->
              scRNA 结题报告
              {{ ceshi }}
            </div>

            <div class="c-pdf-download u-clear-both">
              <a
                href="https://www.nature.com/articles/s41467-021-21246-9.pdf"
                class="c-pdf-download__link"
                data-article-pdf="true"
                data-readcube-pdf-url="true"
                data-test="download-pdf"
                data-draft-ignore="true"
                data-track="click"
                data-track-action="download pdf"
                data-track-label="link"
                data-track-external=""
              >
                <span>Download PDF</span>
                <svg
                  aria-hidden="true"
                  focusable="false"
                  width="16"
                  height="16"
                  class="u-icon"
                >
                  <use xlink:href="#global-icon-download"></use>
                </svg>
              </a>
            </div>
          </div>
        </div>

        <article lang="en">
          <div class="c-article-header">
            <header>
              <ul class="c-article-identifiers" data-test="article-identifier">
                <li
                  class="c-article-identifiers__item"
                  data-test="article-category"
                >
                  Report
                </li>

                <li class="c-article-identifiers__item">
                  <span
                    class="c-article-identifiers__open"
                    data-test="open-access"
                    >scRNA</span
                  >
                </li>

                <li class="c-article-identifiers__item">
                  <a
                    href="#article-info"
                    data-track="click"
                    data-track-action="publication date"
                    data-track-label="link"
                    >Published:
                    <time datetime="2021-10-09">09 October 2021</time></a
                  >
                </li>
              </ul>

              <h1
                class="c-article-title"
                data-test="article-title"
                data-article-title=""
              >
                scRNA 结题报告
              </h1>

            </header>
          </div>

          <div class="c-article-body">
            <section
              aria-labelledby="Abs1"
              data-title="Abstract"
              data-gtm-vis-recent-on-screen-10482319_393="475"
              data-gtm-vis-first-on-screen-10482319_393="475"
              data-gtm-vis-total-visible-time-10482319_393="10000"
              data-gtm-vis-recent-on-screen-10482319_401="475"
              data-gtm-vis-first-on-screen-10482319_401="475"
              data-gtm-vis-total-visible-time-10482319_401="10000"
              data-gtm-vis-has-fired-10482319_393="1"
              data-gtm-vis-has-fired-10482319_401="1"
              lang="en"
            >
              <div class="c-article-section" id="Abs1-section">
                <h2
                  class="
                    c-article-section__title
                    js-section-title js-c-reading-companion-sections-item
                  "
                  id="Abs1"
                >
                结果概述
                </h2>
                <div class="c-article-section__content" id="Abs1-content">
                  <p>
                    本项目中分析物种为：human;
                    分析时样本总数为：6个样本；
                    所有样本总细胞数为：1431个cells；
                    平均每个样本的细胞数为：238.50个cells；
                    与参考基因组比对后，平均每个样本的比对率为：3.64%。
                    详细结果见细胞信息统计总表。
                  </p>
                  <div
                    class="c-article-table"
                    data-test="inline-table"
                    data-container-section="table"
                    id="table-2"
                  >
                    <figure>
                      <figcaption class="c-article-table__figcaption">
                        <b id="Tab2" data-test="table-caption"
                          >表1 细胞信息统计总表</b
                        >
                      </figcaption>
                      <div class="c-article-table-container">
                        <div class="c-article-table-border c-table-scroll-wrapper">
                          <div
                            class="
                              c-table-scroll-wrapper__content
                              c-table-scroll-wrapper__fade--transparent
                            "
                            data-component-scroll-wrapper=""
                          >
                            <table class="data last-table">
                              <thead class="c-article-table-head">
                                <tr>
                                  <th class="u-text-left">SampleID</th>
                                  <th class="u-text-left">
                                    Number of Reads
                                  </th>
                                  <th class="u-text-left">Reads Mapped to Genome</th>
                                  <th class="u-text-left">Estimated Number of Cells</th>
                                  <th class="u-text-left">
                                    Mean Reads per Cell
                                  </th>
                                  <th class="u-text-left">Median UMI Counts per Cell</th>
                                  <th class="u-text-left">Median Genes per Cell</th>
                                  <th class="u-text-left">Total Genes Detected</th>
                                </tr>
                              </thead>
                              <tbody>
                                <tr>
                                  <td class="u-text-left">L01</td>
                                  <td class="u-text-left">404342813</td>
                                  <td class="u-text-left">21.85%</td>
                                  <td class="u-text-left">1431</td>
                                  <td class="u-text-left">133650</td>
                                  <td class="u-text-left">15982</td>
                                  <td class="u-text-left">2834</td>
                                  <td class="u-text-left">17647</td>
                                </tr>
                              </tbody>
                            </table>
                          </div>
                        </div>
                      </div>
                      <div class="u-text-right u-hide-print">
                        <a
                          class="c-article__pill-button"
                          data-test="table-link"
                          data-track="click"
                          data-track-action="view table"
                          data-track-label="button"
                          rel="nofollow"
                          href="/articles/s41467-021-26027-y/tables/2"
                          aria-label="Full size table 2"
                          ><span>Full size table</span
                          ><svg
                            width="16"
                            height="16"
                            focusable="false"
                            role="img"
                            aria-hidden="true"
                            class="u-icon"
                          >
                            <use
                              xmlns:xlink="http://www.w3.org/1999/xlink"
                              xlink:href="#global-icon-chevron-right"
                            ></use>
                            </svg></a>
                      </div>
                    </figure>
                  </div>
                </div>
              </div>
            </section>
            <noscript> </noscript>

            <div class="c-pdf-button__container">
              <div class="c-pdf-download u-clear-both">
                <a
                  href="https://www.nature.com/articles/s41467-021-21246-9.pdf"
                  class="c-pdf-download__link"
                  data-article-pdf="true"
                  data-readcube-pdf-url="true"
                  data-test="download-pdf"
                  data-draft-ignore="true"
                  data-track="click"
                  data-track-action="download pdf"
                  data-track-label="link"
                  data-track-external=""
                >
                  <span>Download PDF</span>
                  <svg
                    aria-hidden="true"
                    focusable="false"
                    width="16"
                    height="16"
                    class="u-icon"
                  >
                    <use xlink:href="#global-icon-download"></use>
                  </svg>
                </a>
              </div>
            </div>

            <section data-title="Introduction">
              <div class="c-article-section" id="Sec1-section">
                <h2
                  class="
                    c-article-section__title
                    js-section-title js-c-reading-companion-sections-item
                  "
                  id="Sec1"
                >
                细胞过滤
                </h2>
                <div class="c-article-section__content" id="Sec1-content">
                  <p>
                    原始数据中存在一些低质量的数据信息，会影响生信分析的结果可靠性。因此，在进行下游分析前需要对细胞数据进行过滤。主要根据：基因数、UMI数、线粒体含量、双胞数等指标进行过滤。
                  </p>
                  <p>
                    一般过滤指标我们可以选择细胞中最低表达的基因数在200-500之间，MI数至少在100以上，线粒体含量小于20%。具体项目分析时设置的过滤参数如下：
                  </p>
                  <p>
                    (1) 基因表达的细胞数：>= 10；<br/>
                    (2) 线粒体含量(%)：<= 5；<br/>
                    (3) UMI数目：>= 100；<br/>
                    (4) 基因数目：200 ~ 25000。<br/>
                  </p>
                  <p>过滤后的细胞相关详细结果信息见过滤后的细胞统计表。</p>
                  <div
                  class="c-article-table"
                  data-test="inline-table"
                  data-container-section="table"
                  id="table-2"
                >
                  <figure>
                    <figcaption class="c-article-table__figcaption">
                      <b id="Tab2" data-test="table-caption"
                        >表2 过滤后的细胞统计表</b
                      >
                    </figcaption>
                    <div class="c-article-table-container">
                      <div class="c-article-table-border c-table-scroll-wrapper">
                        <div
                          class="
                            c-table-scroll-wrapper__content
                            c-table-scroll-wrapper__fade--transparent
                          "
                          data-component-scroll-wrapper=""
                        >
                          <table class="data last-table">
                            <thead class="c-article-table-head">
                              <tr>
                                <th class="u-text-left">Sample</th>
                                <th class="u-text-left">
                                  All_cells_number
                                </th>
                                <th class="u-text-left">filter by Mitochondria <= 5</th>
                                <th class="u-text-left">filter by UMI >= 100</th>
                                <th class="u-text-left">
                                  filter by minimal gene number >= 200
                                </th>
                                <th class="u-text-left">filter by maximum gene number <= 25000</th>
                                <th class="u-text-left">filter by double</th>
                                <th class="u-text-left">Final_cells_number</th>
                                <th class="u-text-left">percent(%)</th>
                              </tr>
                            </thead>
                            <tbody>
                              <tr>
                                <td class="u-text-left">pbmc1</td>
                                <td class="u-text-left">1725</td>
                                <td class="u-text-left">758</td>
                                <td class="u-text-left">0</td>
                                <td class="u-text-left">4</td>
                                <td class="u-text-left">0</td>
                                <td class="u-text-left">52</td>
                                <td class="u-text-left">911</td>
                                <td class="u-text-left">52.81</td>
                              </tr>
                              <tr>
                                <td class="u-text-left">pbmc1</td>
                                <td class="u-text-left">1725</td>
                                <td class="u-text-left">758</td>
                                <td class="u-text-left">0</td>
                                <td class="u-text-left">4</td>
                                <td class="u-text-left">0</td>
                                <td class="u-text-left">52</td>
                                <td class="u-text-left">911</td>
                                <td class="u-text-left">52.81</td>
                              </tr>
                              <tr>
                                <td class="u-text-left">pbmc2</td>
                                <td class="u-text-left">1725</td>
                                <td class="u-text-left">758</td>
                                <td class="u-text-left">0</td>
                                <td class="u-text-left">4</td>
                                <td class="u-text-left">0</td>
                                <td class="u-text-left">52</td>
                                <td class="u-text-left">911</td>
                                <td class="u-text-left">52.81</td>
                              </tr>
                              <tr>
                                <td class="u-text-left">pbmc3</td>
                                <td class="u-text-left">1725</td>
                                <td class="u-text-left">758</td>
                                <td class="u-text-left">0</td>
                                <td class="u-text-left">4</td>
                                <td class="u-text-left">0</td>
                                <td class="u-text-left">52</td>
                                <td class="u-text-left">911</td>
                                <td class="u-text-left">52.81</td>
                              </tr>
                              <tr>
                                <td class="u-text-left">TIL1</td>
                                <td class="u-text-left">1725</td>
                                <td class="u-text-left">758</td>
                                <td class="u-text-left">0</td>
                                <td class="u-text-left">4</td>
                                <td class="u-text-left">0</td>
                                <td class="u-text-left">52</td>
                                <td class="u-text-left">911</td>
                                <td class="u-text-left">52.81</td>
                              </tr>
                              <tr>
                                <td class="u-text-left">TIL2</td>
                                <td class="u-text-left">1725</td>
                                <td class="u-text-left">758</td>
                                <td class="u-text-left">0</td>
                                <td class="u-text-left">4</td>
                                <td class="u-text-left">0</td>
                                <td class="u-text-left">52</td>
                                <td class="u-text-left">911</td>
                                <td class="u-text-left">52.81</td>
                              </tr>
                              <tr>
                                <td class="u-text-left">TIL3</td>
                                <td class="u-text-left">1725</td>
                                <td class="u-text-left">758</td>
                                <td class="u-text-left">0</td>
                                <td class="u-text-left">4</td>
                                <td class="u-text-left">0</td>
                                <td class="u-text-left">52</td>
                                <td class="u-text-left">911</td>
                                <td class="u-text-left">52.81</td>
                              </tr>
                            </tbody>
                          </table>
                        </div>
                      </div>
                    </div>
                    <div class="u-text-right u-hide-print">
                      <a
                        class="c-article__pill-button"
                        data-test="table-link"
                        data-track="click"
                        data-track-action="view table"
                        data-track-label="button"
                        rel="nofollow"
                        href="/articles/s41467-021-26027-y/tables/2"
                        aria-label="Full size table 2"
                        ><span>Full size table</span
                        ><svg
                          width="16"
                          height="16"
                          focusable="false"
                          role="img"
                          aria-hidden="true"
                          class="u-icon"
                        >
                          <use
                            xmlns:xlink="http://www.w3.org/1999/xlink"
                            xlink:href="#global-icon-chevron-right"
                          ></use>
                          </svg></a>
                    </div>
                  </figure>
                </div>
                  <p>对各个样本过滤前后整体的nGene（number of Gene），nUMI（number of UMI），percent.mt（线粒体基因占比）和 percent.ribosome（核糖体含量）做了统计。如下小提琴图所示：</p>
                  <div class="
                      c-article-section__figure
                      js-c-reading-companion-figures-item
                    " data-test="figure" data-container-section="figure" id="figure-0" data-title="过滤前的小提琴图" data-gtm-vis-first-on-screen-10482319_399="746637" data-gtm-vis-total-visible-time-10482319_399="10000" data-gtm-vis-recent-on-screen-10482319_399="4647217" data-gtm-vis-has-fired-10482319_399="1">
                    <figure>
                      <figcaption>
                        <b id="Fig0" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 0: 过滤前的小提琴图</b>
                      </figcaption>
                      <div class="c-article-section__figure-content">
                        <div class="c-article-section__figure-item">
                          <a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/s41467-021-21246-9/figures/1" rel="nofollow"><picture><source type="image/webp" srcset="
                            img/res.jpg 1x
                                ">
                              <img aria-describedby="Fig0" src="img/res.jpg" alt="figure0" loading="lazy" width="685" height="519"></picture></a>
                        </div>
                        <div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-0-desc">
                          <p>
                            <b>a</b> 横坐标为样本，纵坐标分别为基因数，UMI数，线粒体含量、核糖体含量 
                            <b>b</b> 空 <b>c</b> 空 <b>d</b> 空 <b>e</b> 空
                          </p>
                        </div>
                      </div>
                      <div class="u-text-right u-hide-print">
                        <a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="https://www.nature.com/articles/s41467-021-21246-9/figures/1" data-track-dest="link:Figure1 Full size image" aria-label="Full size image figure 1" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon">
                            <use xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-chevron-right"></use></svg></a>
                      </div>
                    </figure>
                  </div>
                </div>
              </div>
            </section>
            <section data-title="Results">
              <div class="c-article-section" id="Sec2-section">
                <h2
                  class="
                    c-article-section__title
                    js-section-title js-c-reading-companion-sections-item
                  "
                  id="Sec2"
                >
                  Results
                </h2>
                <div class="c-article-section__content" id="Sec2-content">
                  <h3 class="c-article__sub-heading" id="Sec3">
                    Overview of CellChat
                  </h3>
                  <p>
                    CellChat requires gene expression data from cells as the
                    user input and models the probability of cell–cell
                    communication by integrating gene expression with prior
                    knowledge of the interactions between signaling ligands,
                    receptors and their cofactors (Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="figure anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#Fig1"
                      >1a</a
                    >). To establish intercellular communications, CellChat can
                    operate in label-based and label-free modes (Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="figure anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#Fig1"
                      >1b</a
                    >). In its label-based mode, CellChat requires user-assigned
                    cell labels as the input. In its label-free mode, CellChat
                    requires user input in form of a low-dimensional
                    representation of the data, such as principal component
                    analysis or diffusion map. For the latter, CellChat
                    automatically groups cells by building a shared neighbor
                    graph based on the cell–cell distance in the low-dimensional
                    space or the pseudotemporal trajectory space (see “Methods”
                    section). Upon receiving input data, CellChat models
                    intercellular communications via the following three
                    modules:
                  </p>
                  <div
                    class="
                      c-article-section__figure
                      js-c-reading-companion-figures-item
                    "
                    data-test="figure"
                    data-container-section="figure"
                    id="figure-1"
                    data-title="Overview of CellChat."
                  >
                    <figure>
                      <figcaption>
                        <b
                          id="Fig1"
                          class="c-article-section__figure-caption"
                          data-test="figure-caption-text"
                          >Fig. 1: Overview of CellChat.</b
                        >
                      </figcaption>
                      <div class="c-article-section__figure-content">
                        <div class="c-article-section__figure-item">
                          <a
                            class="c-article-section__figure-link"
                            data-test="img-link"
                            data-track="click"
                            data-track-label="image"
                            data-track-action="view figure"
                            href="https://www.nature.com/articles/s41467-021-21246-9/figures/1"
                            rel="nofollow"
                            ><picture
                              ><source
                                type="image/webp"
                                srcset="
                                  report_files/41467_2021_21246_Fig1_HTML.webp 1x
                                " />
                              <img
                                aria-describedby="Fig1"
                                src="report_files/41467_2021_21246_Fig1_HTML.png"
                                alt="figure1"
                                loading="lazy"
                                width="685"
                                height="519" /></picture
                          ></a>
                        </div>
                        <div
                          class="c-article-section__figure-description"
                          data-test="bottom-caption"
                          id="figure-1-desc"
                        >
                          <p>
                            <b>a</b> Overview of the ligand-receptor interaction
                            database. CellChatDB takes into account known
                            composition of the ligand-receptor complexes,
                            including complexes with multimeric ligands and
                            receptors, as well as several cofactor types:
                            soluble agonists, antagonists, co-stimulatory and
                            co-inhibitory membrane-bound receptors. CellChatDB
                            contains 2021 validated interactions, including 60%
                            of secreting interactions. In addition, 48% of the
                            interactions involve heteromeric molecular
                            complexes. <b>b</b> CellChat either requires user
                            assigned cell labels as input or automatically
                            groups cells based on the low-dimensional data
                            representation supplied as input. <b>c</b> CellChat
                            models the communication probability and identifies
                            significant communications. <b>d</b> CellChat offers
                            several visualization outputs for different
                            analytical tasks. Different colors in the hierarchy
                            plot and circle plot represent different cell
                            groups. Colors in the bubble plot are proportional
                            to the communication probability, where dark and
                            yellow colors correspond to the smallest and largest
                            values. <b>e</b> CellChat quantitatively measures
                            networks through approaches from graph theory,
                            pattern recognition and manifold learning, to better
                            facilitate the interpretation of intercellular
                            communication networks and the identification of
                            design principles. In addition to analyzing
                            individual dataset, CellChat also delineates
                            signaling changes across different contexts, such as
                            different developmental stages and biological
                            conditions.
                          </p>
                        </div>
                      </div>
                      <div class="u-text-right u-hide-print">
                        <a
                          class="c-article__pill-button"
                          data-test="article-link"
                          data-track="click"
                          data-track-label="button"
                          data-track-action="view figure"
                          href="https://www.nature.com/articles/s41467-021-21246-9/figures/1"
                          data-track-dest="link:Figure1 Full size image"
                          aria-label="Full size image figure 1"
                          rel="nofollow"
                          ><span>Full size image</span
                          ><svg
                            width="16"
                            height="16"
                            focusable="false"
                            role="img"
                            aria-hidden="true"
                            class="u-icon"
                          >
                            <use
                              xlink="http://www.w3.org/1999/xlink"
                              xlink:href="#global-icon-chevron-right"
                            ></use></svg></a>
                      </div>
                    </figure>
                  </div>
                  <p>
                    Cross-referencing ligand-receptor interaction database. The
                    accuracy of the assigned roles for the signaling molecules
                    and their interactions is crucial for predicting
                    biologically meaningful intercellular communications. We
                    manually curated a literature-supported signaling molecule
                    interaction database, called CellChatDB, which takes into
                    account the known composition of ligand-receptor complexes,
                    including complexes with multimeric ligands and receptors,
                    as well as several cofactors: soluble agonists, antagonists,
                    co-stimulatory and co-inhibitory membrane-bound receptors
                    (Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="figure anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#Fig1"
                      >1a</a
                    >, Supplementary Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="supplementary material anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#MOESM1"
                      >1a</a
                    >, Supplementary Note&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="supplementary material anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#MOESM1"
                      >1</a
                    >). CellChatDB incorporates signaling molecule interaction
                    information from the KEGG Pathway database<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 21"
                        title="Kanehisa, M., Furumichi, M., Tanabe, M., Sato, Y. &amp; Morishima, K. KEGG: new perspectives on genomes, pathways, diseases and drugs. Nucleic Acids Res. 45, D353–D361 (2017)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR21"
                        id="ref-link-section-d38771043e776"
                        >21</a
                      ></sup
                    >, a collection of manually drawn signaling pathway maps
                    assembled by expert curators based on existing literature.
                    It also includes information from recent experimental
                    studies. CellChatDB contains 2,021 validated molecular
                    interactions, including 60% of paracrine/autocrine signaling
                    interactions, 21% of extracellular matrix (ECM)-receptor
                    interactions and 19% of cell–cell contact interactions. 48%
                    of the interactions involve heteromeric molecular complexes
                    and 25% of the interactions are curated by us from recent
                    literature (Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="figure anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#Fig1"
                      >1a</a
                    >). Furthermore, each interaction is manually classified
                    into one of the 229 functionally related signaling pathways
                    based on the literature.
                  </p>
                  <p>
                    Inference and visualization of intercellular communications.
                    To predict significant communications, CellChat identifies
                    differentially over-expressed ligands and receptors for each
                    cell group (Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="figure anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#Fig1"
                      >1b</a
                    >; also see “Methods” section). To quantify communications
                    between two cell groups mediated by these signaling genes,
                    CellChat associates each interaction with a probability
                    value. The latter is modeled by the law of mass action based
                    on the average expression values of a ligand by one cell
                    group and that of a receptor by another cell group, as well
                    as their cofactors (see “Methods” scetion). Significant
                    interactions are identified on the basis of a statistical
                    test that randomly permutes the group labels of cells and
                    then recalculates the interaction probability (Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="figure anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#Fig1"
                      >1c</a
                    >, see “Methods” section). An intercellular communication
                    network is a weighted directed graph composed of significant
                    connections between interacting cell groups. CellChat also
                    provides an informative and intuitive visualization method,
                    called hierarchical plot, to highlight autocrine and
                    paracrine signaling communications between cell groups of
                    interest. This hierarchical plot provides an overview of
                    inferred intercellular communication network for each
                    signaling pathway or ligand-receptor pair, consisting of two
                    components: the left portion shows autocrine and paracrine
                    signaling to certain cell groups of interest, and the right
                    portion shows autocrine and paracrine signaling to the
                    remaining cell groups in the dataset. In addition, CellChat
                    implements several other visualization ways, including
                    circle plot and bubble plot (Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="figure anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#Fig1"
                      >1d</a
                    >, see “Methods” section).
                  </p>
                  <p>
                    Quantitative analysis of intercellular communications. To
                    facilitate the interpretation of the complex intercellular
                    communication networks, CellChat
                    quantitatively&nbsp;measures networks through methods
                    abstracted from graph theory, pattern recognition and
                    manifold learning (see “Methods” section). CellChat performs
                    a variety of analyses in an unsupervised manner
                    (Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="figure anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#Fig1"
                      >1e</a
                    >). First, it can determine major signaling sources and
                    targets, as well as mediators and influencers within a given
                    signaling network using centrality measures from network
                    analysis, such as out-degree, in-degree, betweenness, and
                    information metrics (see “Methods” section). Second, it can
                    predict key incoming and outgoing signals for specific cell
                    types, as well as coordinated responses among different cell
                    types by leveraging pattern recognition approaches. Outgoing
                    patterns reveal how the sender cells (i.e., cells as signal
                    source) coordinate with each other, as well as how they
                    coordinate with certain signaling pathways to drive
                    communication. Incoming patterns show how the target cells
                    (i.e., cells as signal receivers) coordinate with each
                    other, as well as how they coordinate with certain signaling
                    pathways to respond to incoming signals. Third, it can group
                    signaling pathways by defining similarity measures and
                    performing manifold learning from both functional and
                    topological perspectives. Fourth, it can delineate conserved
                    and context-specific signaling pathways by joint manifold
                    learning of multiple networks across datasets. Overall,
                    these functionalities allow CellChat to deconvolute complex
                    intercellular communications in an easily interpretable way
                    and predict biologically&nbsp;meaningful discoveries from
                    scRNA-seq data.
                  </p>
                  <h3 class="c-article__sub-heading" id="Sec4">
                    CellChat identifies communication patterns and predicts
                    functions for poorly studied pathways
                  </h3>
                  <p>
                    We showcase CellChat functionalities by applying it to
                    several recently published mouse skin scRNA-seq datasets
                    from embryonic development<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 22"
                        title="Gupta, K. et al. Single-cell analysis reveals a hair follicle dermal niche molecular differentiation trajectory that begins prior to morphogenesis. Dev. Cell 48, 17–31 e16 (2019)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR22"
                        id="ref-link-section-d38771043e809"
                        >22</a
                      ></sup
                    >
                    and adult wound healing stages<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 23"
                        title="Guerrero-Juarez, C. F. et al. Single-cell analysis reveals fibroblast heterogeneity and myeloid-derived adipocyte progenitors in murine skin wounds. Nat. Commun. 10, 650 (2019)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR23"
                        id="ref-link-section-d38771043e813"
                        >23</a
                      ></sup
                    >. Choice of skin was determined by our prior expertise on
                    the aspects of skin morphogenesis and regeneration, its
                    complex cellular make-up and the fact that the role of many
                    signaling pathways in skin is well-established, which
                    enables meaningful literature-based interpretation of a
                    portion of CellChat predictions. First, we ran CellChat
                    analysis on scRNA-seq dataset for day 12 mouse skin wound
                    tissue<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 23"
                        title="Guerrero-Juarez, C. F. et al. Single-cell analysis reveals fibroblast heterogeneity and myeloid-derived adipocyte progenitors in murine skin wounds. Nat. Commun. 10, 650 (2019)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR23"
                        id="ref-link-section-d38771043e817"
                        >23</a
                      ></sup
                    >. This dataset contains 21,898 cells, which cluster into 25
                    cell groups, including nine fibroblast (FIB), five myeloid
                    (MYL) and six endothelial (ENDO) groups, as well as several
                    other cell types such as T cells (TC), B cells (BC),
                    dendritic cells (DC), and lymphatic endothelial cells (LYME)
                    (Supplementary Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="supplementary material anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#MOESM1"
                      >2a–h</a
                    >; see “Methods” section).
                  </p>
                  <p>
                    CellChat detected 60 significant ligand-receptor pairs among
                    the 25 cell groups, which were further categorized into 25
                    signaling pathways, including TGFβ, non-canonical WNT
                    (ncWNT), TNF, SPP1, PTN, PDGF, CXCL, CCL, and MIF pathways.
                    Network centrality analysis of the inferred TGFβ signaling
                    network identified that several myeloid cell populations are
                    the most prominent sources for TGFβ ligands acting onto
                    fibroblasts (Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="figure anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#Fig2"
                      >2a, b</a
                    >). Of note one myeloid population MYL-A is also the
                    dominant mediator, suggesting its role as a gatekeeper of
                    cell–cell communication. These findings are consistent with
                    the known critical role played by myeloid cells in
                    initiating inflammation during skin wound healing and
                    driving activation of skin-resident fibroblasts via TGFβ
                    signaling<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        title="Peters, T. et al. Wound-healing defect of CD18(-/-) mice due to a decrease in TGF-beta1 and myofibroblast differentiation. EMBO J. 24, 3400–3410 (2005)."
                        href="#ref-CR24"
                        id="ref-link-section-d38771043e830"
                        >24</a
                      >,<a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        title="Rappolee, D. A., Mark, D., Banda, M. J. &amp; Werb, Z. Wound macrophages express TGF-alpha and other growth factors in vivo: analysis by mRNA phenotyping. Science 241, 708–712 (1988)."
                        href="#ref-CR25"
                        id="ref-link-section-d38771043e830_1"
                        >25</a
                      >,<a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        title="Shah, M., Foreman, D. M. &amp; Ferguson, M. W. Neutralising antibody to TGF-beta 1,2 reduces cutaneous scarring in adult rodents. J. Cell Sci. 107, 1137–1157 (1994)."
                        href="#ref-CR26"
                        id="ref-link-section-d38771043e830_2"
                        >26</a
                      >,<a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        title="Wu, L., Yu, Y. L., Galiano, R. D., Roth, S. I. &amp; Mustoe, T. A. Macrophage colony-stimulating factor accelerates wound healing and upregulates TGF-beta1 mRNA levels through tissue macrophages. J. Surg. Res. 72, 162–169 (1997)."
                        href="#ref-CR27"
                        id="ref-link-section-d38771043e830_3"
                        >27</a
                      >,<a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        title="Ishida, Y., Gao, J. L. &amp; Murphy, P. M. Chemokine receptor CX3CR1 mediates skin wound healing by promoting macrophage and fibroblast accumulation and function. J. Immunol. 180, 569–579 (2008)."
                        href="#ref-CR28"
                        id="ref-link-section-d38771043e830_4"
                        >28</a
                      >,<a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 29"
                        title="Border, W. A. &amp; Noble, N. A. Transforming growth factor beta in tissue fibrosis. N. Engl. J. Med. 331, 1286–1292 (1994)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR29"
                        id="ref-link-section-d38771043e833"
                        >29</a
                      ></sup
                    >. Importantly, CellChat also predicted that certain
                    endothelial cell populations, as well as several fibroblast
                    populations, both known sources of TGFβ ligands,
                    significantly contribute to myeloid-dominated TGFβ signal
                    production in the wound. This reveals that TGFβ signaling
                    network in skin wounds is complex and highly redundant with
                    multiple ligand sources targeting large portion of wound
                    fibroblasts. Interestingly, CellChat shows that the majority
                    of TGFβ interactions among wound cells are paracrine, with
                    only one fibroblast and one myeloid population demonstrating
                    significant autocrine signaling (Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="figure anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#Fig2"
                      >2b</a
                    >). Notably, among all known ligand-receptor pairs, wound
                    TGFβ signaling is dominated by <i>Tgfb1</i> ligand and its
                    multimeric <i>Tgfbr1/Tgfbr2</i> receptor (Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="figure anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#Fig2"
                      >2c</a
                    >). In contrast with TGFβ, CellChat analysis of inferred
                    ncWNT signaling network revealed its very distinct,
                    non-redundant structure with only one ligand
                    <i>(Wnt5a)</i> and only one population of fibroblasts
                    (FIB-D) driving largely fibroblast-to-fibroblast,
                    fibroblast-to-endothelial and to a lesser extent
                    fibroblast-to-myeloid signaling (Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="figure anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#Fig2"
                      >2d–f</a
                    >). FIB-D cells highly expressed <i>Crabp1</i> and were
                    enriched for cell cycle genes (Supplementary Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="supplementary material anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#MOESM1"
                      >2d</a
                    >), which likely represent an actively cycling subset of
                    <i>Crabp1</i>-positive cells in upper wound dermis<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 23"
                        title="Guerrero-Juarez, C. F. et al. Single-cell analysis reveals fibroblast heterogeneity and myeloid-derived adipocyte progenitors in murine skin wounds. Nat. Commun. 10, 650 (2019)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR23"
                        id="ref-link-section-d38771043e866"
                        >23</a
                      >,<a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 30"
                        title="Abbasi, S. et al. Distinct regulatory programs control the latent regenerative potential of dermal fibroblasts during wound healing. Cell Stem Cell 27, 396–412 e396 (2020)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR30"
                        id="ref-link-section-d38771043e869"
                        >30</a
                      ></sup
                    >. Network centrality analysis confirmed that FIB-D is a
                    prominent influencer controlling the communications
                    (Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="figure anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#Fig2"
                      >2e</a
                    >). Importantly, elevated expression of WNT5A in fibroblasts
                    and its role in scarring has recently been reported<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        title="Lee, Y. S. et al. STAT3 signalling pathway is implicated in keloid pathogenesis by preliminary transcriptome and open chromatin analyses. Exp. Dermatol. 28, 480–484 (2019)."
                        href="#ref-CR31"
                        id="ref-link-section-d38771043e876"
                        >31</a
                      >,<a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        title="Feng, Y. et al. The signaling protein Wnt5a promotes TGFbeta1-mediated macrophage polarization and kidney fibrosis by inducing the transcriptional regulators Yap/Taz. J. Biol. Chem. 293, 19290–19302 (2018)."
                        href="#ref-CR32"
                        id="ref-link-section-d38771043e876_1"
                        >32</a
                      >,<a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        title="Abraityte, A. et al. Wnt5a is elevated in heart failure and affects cardiac fibroblast function. J. Mol. Med. 95, 767–777 (2017)."
                        href="#ref-CR33"
                        id="ref-link-section-d38771043e876_2"
                        >33</a
                      >,<a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 34"
                        title="Vuga, L. J. et al. WNT5A is a regulator of fibroblast proliferation and resistance to apoptosis. Am. J. Respir. Cell Mol. Biol. 41, 583–589 (2009)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR34"
                        id="ref-link-section-d38771043e879"
                        >34</a
                      ></sup
                    >.
                  </p>
                  <div
                    class="
                      c-article-section__figure
                      js-c-reading-companion-figures-item
                    "
                    data-test="figure"
                    data-container-section="figure"
                    id="figure-2"
                    data-title="CellChat analysis of the communications between skin cells during wound repair."
                  >
                    <figure>
                      <figcaption>
                        <b
                          id="Fig2"
                          class="c-article-section__figure-caption"
                          data-test="figure-caption-text"
                          >Fig. 2: CellChat analysis of the communications
                          between skin cells during wound repair.</b
                        >
                      </figcaption>
                      <div class="c-article-section__figure-content">
                        <div class="c-article-section__figure-item">
                          <a
                            class="c-article-section__figure-link"
                            data-test="img-link"
                            data-track="click"
                            data-track-label="image"
                            data-track-action="view figure"
                            href="https://www.nature.com/articles/s41467-021-21246-9/figures/2"
                            rel="nofollow"
                            ><picture
                              ><source
                                type="image/webp"
                                srcset="
                                  report_files/41467_2021_21246_Fig2_HTML.webp 1x
                                " />
                              <img
                                aria-describedby="Fig2"
                                src="report_files/41467_2021_21246_Fig2_HTML.png"
                                alt="figure2"
                                loading="lazy"
                                width="685"
                                height="876" /></picture
                          ></a>
                        </div>
                        <div
                          class="c-article-section__figure-description"
                          data-test="bottom-caption"
                          id="figure-2-desc"
                        >
                          <p>
                            <b>a</b> Hierarchical plot shows the inferred
                            intercellular communication network for TGFβ
                            signaling. This plot consists of two parts: Left and
                            right portions highlight the autocrine and paracrine
                            signaling to fibroblast states and to other
                            non-fibroblast skin cell states, respectively. Solid
                            and open circles represent source and target,
                            respectively. Circle sizes are proportional to the
                            number of cells in each cell group and edge width
                            represents the communication probability. Edge
                            colors are consistent with the signaling source.
                            FIB-A – I: nine fibroblast cell groups; MYL-A – E:
                            five myeloid cell groups; ENDO-A – F: six
                            endothelial cell groups; TC: T cell; BC: B cell;
                            SCH: Schwan cell; DC: Dendritic cell, LYME:
                            Lymphatic endothelial cell; <b>(b)</b> Heatmap shows
                            the relative importance of each cell group based on
                            the computed four network centrality measures of
                            TGFβ signaling network. <b>c</b> Relative
                            contribution of each ligand-receptor pair to the
                            overall communication network of TGFβ signaling
                            pathway, which is the ratio of the total
                            communication probability of the inferred network of
                            each ligand-receptor pair to that of TGFβ signaling
                            pathway. <b>d</b> The inferred ncWNT signaling
                            network. <b>e</b> Relative contribution of each
                            ncWNT ligand-receptor pair. <b>f</b> The computed
                            network centrality measures of ncWNT signaling.
                            <b>g</b> The inferred outgoing communication
                            patterns of secreting cells, which shows the
                            correspondence between the inferred latent patterns
                            and cell groups, as well as signaling pathways. The
                            thickness of the flow indicates the contribution of
                            the cell group or signaling pathway to each latent
                            pattern. <b>h</b> The inferred incoming
                            communication patterns of target cells.
                            <b>i</b> Projecting signaling pathways onto a
                            two-dimensional manifold according to their
                            functional similarity. Each dot represents the
                            communication network of one signaling pathway. Dot
                            size is proportional to the overall communication
                            probability. Different colors represent different
                            groups of signaling pathways. <b>j</b> Two different
                            similarity measures are used to quantify the
                            similarity among the inferred networks. Examples
                            showing the functional similarity with similar major
                            sources/targets, and structural similarity with
                            similar network topology. <b>k</b> Projecting
                            signaling pathways onto a two-dimensional manifold
                            according to their structural similarity.
                          </p>
                        </div>
                      </div>
                      <div class="u-text-right u-hide-print">
                        <a
                          class="c-article__pill-button"
                          data-test="article-link"
                          data-track="click"
                          data-track-label="button"
                          data-track-action="view figure"
                          href="https://www.nature.com/articles/s41467-021-21246-9/figures/2"
                          data-track-dest="link:Figure2 Full size image"
                          aria-label="Full size image figure 2"
                          rel="nofollow"
                          ><span>Full size image</span
                          ><svg
                            width="16"
                            height="16"
                            focusable="false"
                            role="img"
                            aria-hidden="true"
                            class="u-icon"
                          >
                            <use
                              xlink="http://www.w3.org/1999/xlink"
                              xlink:href="#global-icon-chevron-right"
                            ></use></svg></a>
                      </div>
                    </figure>
                  </div>
                  <p>
                    In addition to exploring detailed communications for
                    individual pathways, an important question is how multiple
                    cell groups and signaling pathways coordinate to function.
                    To address this question, CellChat employs a pattern
                    recognition method based on non-negative matrix
                    factorization to identify the global communication patterns,
                    as well as the key signals in different cell groups (see
                    “Methods” section). The output of this analysis is a set of
                    the so-called communication patterns that connect cell
                    groups with signaling pathways either in the context of
                    outgoing signaling (i.e., treating cells as senders) or
                    incoming signaling (i.e., treating cells as receivers).
                    Application of this analysis uncovered five patterns for
                    outgoing signaling (Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="figure anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#Fig2"
                      >2g</a
                    >) and five patterns for incoming signaling (Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="figure anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#Fig2"
                      >2h</a
                    >). This output, for example, reveals that a large portion
                    of outgoing fibroblast signaling is characterized by pattern
                    #4, which represents multiple pathways, including but not
                    limited to ncWNT, SPP1, MK, and PROS (Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="figure anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#Fig2"
                      >2g</a
                    >). All of the outgoing myeloid cell signaling is
                    characterized by pattern #2, representing such pathways as
                    TGFβ, TNF, CSF, IL1, and RANKL. On the other hand, the
                    communication patterns of target cells (Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="figure anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#Fig2"
                      >2h</a
                    >) shows that incoming fibroblast signaling is dominated by
                    two patterns #1 and #3, which include signaling pathways
                    such as TGFβ and ncWNT, as well as PDGF, TNF, MK, and PTN
                    among others. Majority of incoming myeloid cell signaling is
                    characterized by the pattern #4, driven by CSF and CXCL
                    pathways. Notably, both incoming and outgoing signaling by
                    Schwann cells share the same pattern #1 with wound
                    fibroblasts (Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="figure anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#Fig2"
                      >2g-h</a
                    >). These results show that: (1) two distinct cell types in
                    the same tissue can rely on largely overlapping signaling
                    networks; and that (2) certain cell types, such as
                    fibroblasts, simultaneously activate multiple signaling
                    patterns and pathways, while other cell types, such as
                    myeloid cells or B cells, rely on fewer and more homogeneous
                    communication patterns. Moreover, cross-referencing outgoing
                    and incoming signaling patterns also provides a quick
                    insight into the autocrine-acting vs. paracrine-acting
                    pathways for a given cell type. For example, major
                    autocrine-acting pathways between wound fibroblasts are MK,
                    SEMA3, PROS, and ncWNT, and major paracrine-acting
                    myeloid-to-fibroblasts pathways are TGFβ and TNF
                    (Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="figure anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#Fig2"
                      >2g-h</a
                    >).
                  </p>
                  <p>
                    Further, CellChat is able to quantify the similarity between
                    all significant signaling pathways and then group them based
                    on their cellular communication network similarity. Grouping
                    can be done either based on functional or structural
                    similarity (see “Methods” section). Application of
                    functional similarity grouping identified four groups of
                    pathways (Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="figure anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#Fig2"
                      >2i</a
                    >). Group #1 is dominated by inflammatory pathways (e.g.,
                    TGFβ, TNF, IL, CCL) and largely represents paracrine
                    signaling from myeloid and endothelial cells to fibroblasts.
                    Group #2, which includes ncWNT, EGF, GAS, and PROS pathways,
                    largely represents autocrine signaling between wound
                    fibroblasts. Group #3, which includes CXCL and APELIN
                    pathways, represents signaling from endothelial cells, while
                    group #4, which includes MK, PTN, and SPP1 pathways,
                    represents promiscuous signaling (i.e., signaling with high
                    connectivity) and is dominated by signals from certain
                    fibroblast populations and myeloid cells. By identifying
                    poorly studied pathways that group together with other
                    pathways, whose role is well known, this CellChat analysis
                    can predict putative functions of the former. Different from
                    grouping on the basis of functional similarity, which
                    heavily weighs in similarity between sender and receiver
                    cell groups, grouping based on structural similarity is
                    primarily driven by the similarity of signaling network
                    topology (Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="figure anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#Fig2"
                      >2j</a
                    >; see “Methods” section). Structural similarity grouping
                    also identified four groups of signaling pathways
                    (Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="figure anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#Fig2"
                      >2k</a
                    >). Group #1 represents pathways that have very few senders
                    and numerous receivers, such as ncWNT; group #2 represents
                    pathways with numerous senders and receivers, such as TGFβ
                    and PTN; group #3 represents pathways with numerous senders
                    and few receivers, such as CCL and IL1; and group #4
                    represents pathways with few senders and few receivers, such
                    as PROS, IL2, and CXCL. Thus, grouping based on structural
                    similarity reveals general mode of how sender and receiver
                    cells utilize a given signaling pathway. Collectively,
                    CellChat can identify key features of intercellular
                    communications within a given scRNA-seq dataset and predict
                    putative functions for poorly understood signaling pathways.
                  </p>
                  <h3 class="c-article__sub-heading" id="Sec5">
                    CellChat reveals continuous cell lineage-associated
                    signaling events
                  </h3>
                  <p>
                    In addition to discrete cell states, our framework can be
                    applied to continuous cell states along the pseudotemporal
                    trajectory (see “Methods” section). We demonstrate this
                    utility using scRNA-seq data on embryonic day E14.5 mouse
                    skin<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 22"
                        title="Gupta, K. et al. Single-cell analysis reveals a hair follicle dermal niche molecular differentiation trajectory that begins prior to morphogenesis. Dev. Cell 48, 17–31 e16 (2019)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR22"
                        id="ref-link-section-d38771043e978"
                        >22</a
                      ></sup
                    >, when both dermal and epidermal cell lineages undergo
                    rapid specification and give rise to new cell types within
                    the developing hair follicles<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 22"
                        title="Gupta, K. et al. Single-cell analysis reveals a hair follicle dermal niche molecular differentiation trajectory that begins prior to morphogenesis. Dev. Cell 48, 17–31 e16 (2019)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR22"
                        id="ref-link-section-d38771043e982"
                        >22</a
                      >,<a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 35"
                        title="Saxena, N., Mok, K. W. &amp; Rendl, M. An updated classification of hair follicle morphogenesis. Exp. Dermatol 28, 332–344 (2019)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR35"
                        id="ref-link-section-d38771043e985"
                        >35</a
                      >,<a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 36"
                        title="Mok, K. W. et al. Dermal condensate niche fate specification occurs prior to formation and is placode progenitor dependent. Dev. Cell 48, 32–48 e35 (2019)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR36"
                        id="ref-link-section-d38771043e988"
                        >36</a
                      ></sup
                    >. First, we inferred pseudotemporal trajectories for dermal
                    and epidermal embryonic skin cells using the diffusion map
                    approach (Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="figure anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#Fig3"
                      >3a-b</a
                    >; Supplementary Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="supplementary material anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#MOESM1"
                      >3a–d</a
                    >; see “Methods” section). Dermal cell trajectory, which on
                    one end contains <i>Sox2</i>-high hair follicle dermal
                    condensate (DC) cells, was divided into seven groups, that
                    include five fibroblast states (FIB-A, FIB-B, FIB-C, FIB-D,
                    FIB-E) and two DC states (DC-A and DC-B). A linear sequence
                    of these trajectory groups recapitulates sequential stages
                    of embryonic skin fibroblast lineage specification process
                    (Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="figure anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#Fig3"
                      >3a</a
                    >). Embryonic epidermal cell trajectory starts from basal
                    epidermal cells and progresses either toward
                    <i>Edar</i>-high epithelial placode cells or toward
                    <i>Krt1</i>-high and <i>Lor</i>-high suprabasal epidermal
                    cells. Collectively, epidermal lineage specification events
                    are represented by two basal, one placode and three
                    suprabasal trajectory groups (Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="figure anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#Fig3"
                      >3b</a
                    >).
                  </p>
                  <div
                    class="
                      c-article-section__figure
                      js-c-reading-companion-figures-item
                    "
                    data-test="figure"
                    data-container-section="figure"
                    id="figure-3"
                    data-title="Application of CellChat to continuous cell states along pseudotemporal trajectories during embryonic skin development."
                  >
                    <figure>
                      <figcaption>
                        <b
                          id="Fig3"
                          class="c-article-section__figure-caption"
                          data-test="figure-caption-text"
                          >Fig. 3: Application of CellChat to continuous cell
                          states along pseudotemporal trajectories during
                          embryonic skin development.</b
                        >
                      </figcaption>
                      <div class="c-article-section__figure-content">
                        <div class="c-article-section__figure-item">
                          <a
                            class="c-article-section__figure-link"
                            data-test="img-link"
                            data-track="click"
                            data-track-label="image"
                            data-track-action="view figure"
                            href="https://www.nature.com/articles/s41467-021-21246-9/figures/3"
                            rel="nofollow"
                            ><picture
                              ><source
                                type="image/webp"
                                srcset="
                                  report_files/41467_2021_21246_Fig3_HTML.webp 1x
                                " />
                              <img
                                aria-describedby="Fig3"
                                src="report_files/41467_2021_21246_Fig3_HTML.png"
                                alt="figure3"
                                loading="lazy"
                                width="685"
                                height="804" /></picture
                          ></a>
                        </div>
                        <div
                          class="c-article-section__figure-description"
                          data-test="bottom-caption"
                          id="figure-3-desc"
                        >
                          <p>
                            <b>a</b> Left: Diffusion map projecting dermal skin
                            cells onto the low-dimensional space and showing the
                            dermal differentiation from fibroblasts to DC
                            (dermal condensate) cells. Cells are grouped based
                            on their location in this space. Right: Density plot
                            showing the distribution of expression for selected
                            marker genes in each cell group/population.
                            <b>b</b> Diffusion map showing the epidermal
                            trajectory and associated density plot for selected
                            marker genes. <b>c</b>
                            Hierarchical plot showing dermal and epidermal
                            interactions via canonical WNT signaling. Left and
                            right portions show the autocrine and paracrine
                            signaling to dermal trajectory and epidermal
                            trajectory, respectively. Circle sizes are
                            proportional to the number of cells in each cell
                            group and edge width represents the communication
                            probability.
                            <b>d</b> Violin plot showing the expression
                            distribution of signaling genes involved in the
                            inferred WNT signaling network. <b>e</b> The dermal
                            and epidermal interactions via ncWNT signaling.
                            <b>f</b> The expression distribution of signaling
                            genes involved in the inferred ncWNT signaling
                            network. <b>g</b> The outgoing signaling patterns of
                            secreting cells visualized by alluvial plot, which
                            shows the correspondence between the inferred latent
                            patterns and cell groups, as well as signaling
                            pathways. The thickness of the flow indicates the
                            contribution of the cell group or signaling pathway
                            to each latent pattern. The height of each pattern
                            is proportional to the number of its associated cell
                            groups or signaling pathways. Outgoing patterns
                            reveal how the sender cells coordinate with each
                            other, as well as how they coordinate with certain
                            signaling pathways to drive communication.
                            <b>h</b> Incoming signaling patterns of target
                            cells. Incoming patterns show how the target cells
                            coordinate with each other, as well as how they
                            coordinate with certain signaling pathways to
                            respond to incoming signaling.
                          </p>
                        </div>
                      </div>
                      <div class="u-text-right u-hide-print">
                        <a
                          class="c-article__pill-button"
                          data-test="article-link"
                          data-track="click"
                          data-track-label="button"
                          data-track-action="view figure"
                          href="https://www.nature.com/articles/s41467-021-21246-9/figures/3"
                          data-track-dest="link:Figure3 Full size image"
                          aria-label="Full size image figure 3"
                          rel="nofollow"
                          ><span>Full size image</span
                          ><svg
                            width="16"
                            height="16"
                            focusable="false"
                            role="img"
                            aria-hidden="true"
                            class="u-icon"
                          >
                            <use
                              xlink="http://www.w3.org/1999/xlink"
                              xlink:href="#global-icon-chevron-right"
                            ></use></svg></a>
                      </div>
                    </figure>
                  </div>
                  <p>
                    We applied CellChat to study dermal-epidermal communication
                    along these sequential cell lineage states. 88 significant
                    ligand-receptor interactions within 22 signaling pathways
                    were predicted, including WNT, ncWNT, TGFβ, PDGF, NGF, FGF,
                    and SEMA3. Previous studies showed that activation of
                    canonical WNT signaling is required for DC cell
                    specification in the embryonic skin<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 22"
                        title="Gupta, K. et al. Single-cell analysis reveals a hair follicle dermal niche molecular differentiation trajectory that begins prior to morphogenesis. Dev. Cell 48, 17–31 e16 (2019)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR22"
                        id="ref-link-section-d38771043e1064"
                        >22</a
                      >,<a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        title="Mok, K. W. et al. Dermal condensate niche fate specification occurs prior to formation and is placode progenitor dependent. Dev. Cell 48, 32–48 e35 (2019)."
                        href="#ref-CR36"
                        id="ref-link-section-d38771043e1067"
                        >36</a
                      >,<a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        title="Glover, J. D. et al. Hierarchical patterning modes orchestrate hair follicle morphogenesis. PLoS Biol. 15, e2002117 (2017)."
                        href="#ref-CR37"
                        id="ref-link-section-d38771043e1067_1"
                        >37</a
                      >,<a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        title="Tsai, S. Y. et al. Wnt/beta-catenin signaling in dermal condensates is required for hair follicle formation. Dev. Biol. 385, 179–188 (2014)."
                        href="#ref-CR38"
                        id="ref-link-section-d38771043e1067_2"
                        >38</a
                      >,<a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        title="Fu, J. &amp; Hsu, W. Epidermal Wnt controls hair follicle induction by orchestrating dynamic signaling crosstalk between the epidermis and dermis. J. Invest. Dermatol. 133, 890–898 (2013)."
                        href="#ref-CR39"
                        id="ref-link-section-d38771043e1067_3"
                        >39</a
                      >,<a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 40"
                        title="Chen, D., Jarrell, A., Guo, C., Lang, R. &amp; Atit, R. Dermal beta-catenin activity in response to epidermal Wnt ligands is required for fibroblast proliferation and hair follicle initiation. Development 139, 1522–1533 (2012)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR40"
                        id="ref-link-section-d38771043e1070"
                        >40</a
                      ></sup
                    >. Indeed, CellChat-inferred canonical WNT signaling network
                    indicates that epidermal cells are the primary ligand
                    source, which acts both in autocrine manner between
                    epidermal cell populations, as well as in paracrine way from
                    epidermal to dermal cells (Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="figure anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#Fig3"
                      >3c</a
                    >). Notably, two WNT ligand-receptor pairs, namely
                    <i>Wnt6–Fzd10/Lrp6</i> and <i>Wnt6–Fzd2/Lrp6</i> were the
                    dominant contributors to this communication network
                    (Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="figure anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#Fig3"
                      >3d</a
                    >
                    and Supplementary Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="supplementary material anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#MOESM1"
                      >4a</a
                    >), which is consistent with the previous report that
                    <i>Wnt6</i> is the highest expressed canonical WNT ligand in
                    embryonic mouse skin<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 41"
                        title="Andl, T., Reddy, S. T., Gaddapara, T. &amp; Millar, S. E. WNT signals are required for the initiation of hair follicle development. Dev. Cell 2, 643–653 (2002)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR41"
                        id="ref-link-section-d38771043e1093"
                        >41</a
                      ></sup
                    >. Signaling communication network for ncWNT pathway differs
                    substantially from that of canonical WNT pathway. Late stage
                    fibroblast state FIB-E was the primary ncWNT source,
                    signaling both in autocrine and paracrine manner
                    (Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="figure anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#Fig3"
                      >3e</a
                    >) with <i>Wnt5a–Fzd2</i> and
                    <i>Wnt5–Fzd10</i> ligand-receptor pairs driving the
                    signaling (Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="figure anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#Fig3"
                      >3f</a
                    >
                    and Supplementary Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="supplementary material anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#MOESM1"
                      >4b–c</a
                    >). These results suggest distinct roles for canonical WNT
                    and ncWNT pathways in skin morphogenesis. In another
                    example, we analyzed the FGF signaling network
                    (Supplementary Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="supplementary material anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#MOESM1"
                      >4d–h</a
                    >) and found it to be similar to the ncWNT signaling
                    network, with the additional epithelial placode-derived
                    <i>Fgf20</i> signaling (Supplementary Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="supplementary material anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#MOESM1"
                      >4e and h</a
                    >). This is consistent with the known role of
                    placode-derived FGF20 signaling in hair follicle
                    morphogenesis<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 22"
                        title="Gupta, K. et al. Single-cell analysis reveals a hair follicle dermal niche molecular differentiation trajectory that begins prior to morphogenesis. Dev. Cell 48, 17–31 e16 (2019)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR22"
                        id="ref-link-section-d38771043e1122"
                        >22</a
                      >,<a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 42"
                        title="Biggs, L. C. et al. Hair follicle dermal condensation forms via Fgf20 primed cell cycle exit, cell motility, and aggregation. Elife 7, e36468 (2018)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR42"
                        id="ref-link-section-d38771043e1125"
                        >42</a
                      >,<a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 43"
                        title="Huh, S. H. et al. Fgf20 governs formation of primary and secondary dermal condensations in developing hair follicles. Genes Dev. 27, 450–458 (2013)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR43"
                        id="ref-link-section-d38771043e1128"
                        >43</a
                      ></sup
                    >. In another distinct example of TGFβ pathway, epithelial
                    placode cells and to a lesser extent early DC-A cells were
                    the driving sources of TGFβ ligands to dermal cells
                    (Supplementary Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="supplementary material anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#MOESM1"
                      >4i–k</a
                    >). These findings are consistent with the known role for
                    TGFβ signaling in early hair follicle morphogenesis<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 44"
                        title="Jamora, C. et al. A signaling pathway involving TGF-beta2 and snail in hair follicle morphogenesis. PLoS Biol. 3, e11 (2005)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR44"
                        id="ref-link-section-d38771043e1136"
                        >44</a
                      >,<a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 45"
                        title="Foitzik, K., Paus, R., Doetschman, T. &amp; Dotto, G. P. The TGF-beta2 isoform is both a required and sufficient inducer of murine hair follicle morphogenesis. Dev. Biol. 212, 278–289 (1999)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR45"
                        id="ref-link-section-d38771043e1139"
                        >45</a
                      ></sup
                    >.
                  </p>
                  <p>
                    We then ran CellChat pattern recognition module to uncover
                    the key sequential signaling events along the process of
                    skin morphogenesis. To predict the sequential signaling
                    events, we combined the communication pattern analysis with
                    the inferred pseudotemporal cell events. The dermal and
                    epidermal trajectory analysis potentially revealed the
                    pseudotemporal order of different cell types, and the
                    communication pattern analysis identified strong signals
                    that were sent or received by certain cell types. At the
                    outgoing end of signaling, we predicted that FGF and
                    GALECTIN signals are first secreted by FIB-A cells
                    (Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="figure anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#Fig3"
                      >3g</a
                    >). FIB-B and FIB-C cells then coordinate production of GAS
                    signaling. Next, FIB-D, and FIB-E fibroblasts along with
                    suprabasal epidermal cells coordinate secretion of numerous
                    ligands for pathways such as ncWNT, EGF, IGF, CXCL, and
                    SEMA3; while DC-A and epithelial placode cells jointly
                    secrete ligands for TGFβ pathway. At the same time, basal
                    epidermal cells dominantly drive WNT, PDGF, NGF, and
                    VISFATIN signaling pathways. On the other hand, at the
                    incoming end of signaling, fibroblasts are driven by
                    patterns #1 and #2 involving pathways such as FGF, PDGF,
                    SEMA3, TGFβ, IGF, and GALECTIN (Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="figure anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#Fig3"
                      >3h</a
                    >). DC and epithelial placode cells are driven by the
                    pattern #4, which includes HH and CXCL signaling; basal
                    epidermal cells are dominated by pattern #3 pathways—WNT,
                    ncWNT, and EGF; while suprabasal epidermal cells are the
                    primary target for GRN (granulin) signaling within pattern
                    #5. Together, CellChat analysis faithfully recovers many
                    signaling events with well-established roles in embryonic
                    skin and hair follicle morphogenesis and systematically
                    predicts a number of additional signaling patterns along
                    dermal and epidermal cell lineage trajectories.
                  </p>
                  <h3 class="c-article__sub-heading" id="Sec6">
                    CellChat predicts key signaling events between spatially
                    colocalized cell populations
                  </h3>
                  <p>
                    To further demonstrate the predictive nature of CellChat, we
                    studied signaling communication between E14.5 dermal
                    condensate (DC) and epithelial placode cells, since these
                    cells spatially colocalize and actively signal to each other
                    during the initial stages of embryonic hair follicle
                    formation (Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="figure anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#Fig4"
                      >4a</a
                    >). Three DC states—pre-DC, DC1, and DC2, and one placode
                    state were identified (Supplementary Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="supplementary material anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#MOESM1"
                      >3e–f</a
                    >; see “Methods” section). CellChat analysis on these four
                    cell states identified placode cells as the dominant
                    communication “hub”, which secretes and receives signals via
                    44 and 19 ligand-receptor pairs, respectively (Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="figure anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#Fig4"
                      >4b</a
                    >). Prominent bidirectional forward and reverse signals were
                    identified for DC states, suggesting that the cell state
                    transition along pre-DC-DC1-DC2 cell lineage trajectory is
                    highly regulated. Specifically, FGF pathway exhibited
                    abundant signaling interactions among all four states with
                    FGF ligands being dominantly secreted by pre-DC and DC2
                    states (Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="figure anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#Fig4"
                      >4c</a
                    >). <i>Fgf10</i> was the major ligand contributing to dermal
                    FGF signaling (Supplementary Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="supplementary material anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#MOESM1"
                      >5a</a
                    >), which is the known DC signature gene<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 36"
                        title="Mok, K. W. et al. Dermal condensate niche fate specification occurs prior to formation and is placode progenitor dependent. Dev. Cell 48, 32–48 e35 (2019)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR36"
                        id="ref-link-section-d38771043e1179"
                        >36</a
                      ></sup
                    >. Epithelial placode cells distinctly secreted
                    <i>Fgf20</i> both in autocrine and in paracrine manner to
                    all three DC states (Supplementary Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="supplementary material anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#MOESM1"
                      >5a</a
                    >), which is consistent with the known role of
                    placode-derived FGF20 signaling in hair follicle
                    morphogenesis<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 22"
                        title="Gupta, K. et al. Single-cell analysis reveals a hair follicle dermal niche molecular differentiation trajectory that begins prior to morphogenesis. Dev. Cell 48, 17–31 e16 (2019)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR22"
                        id="ref-link-section-d38771043e1189"
                        >22</a
                      >,<a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 42"
                        title="Biggs, L. C. et al. Hair follicle dermal condensation forms via Fgf20 primed cell cycle exit, cell motility, and aggregation. Elife 7, e36468 (2018)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR42"
                        id="ref-link-section-d38771043e1192"
                        >42</a
                      >,<a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 43"
                        title="Huh, S. H. et al. Fgf20 governs formation of primary and secondary dermal condensations in developing hair follicles. Genes Dev. 27, 450–458 (2013)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR43"
                        id="ref-link-section-d38771043e1195"
                        >43</a
                      ></sup
                    >. For another major signaling pathway in early hair
                    follicle morphogenesis—canonical WNT, epithelial placode
                    cells were the major source of ligands (Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="figure anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#Fig4"
                      >4c</a
                    >), prominently expressing primarily autocrine
                    <i>Wnt3</i> and <i>Wnt6</i>. CellChat also predicted that
                    this dominant epithelial autocrine WNT signaling was
                    supplemented by a minor DC-derived <i>Wnt9a</i> paracrine
                    signaling (Supplementary Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="supplementary material anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#MOESM1"
                      >5b-d</a
                    >). In contrast with canonical WNT, the inferred ncWNT
                    signaling network revealed that DC cells express only one
                    ligand, <i>Wnt5a</i>, that drives paracrine DC-to-placode
                    and autocrine DC-to-DC signaling (Supplementary Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="supplementary material anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#MOESM1"
                      >5e</a
                    >). This result implies distinct roles for canonical WNT and
                    ncWNT pathways in hair follicle morphogenesis.
                  </p>
                  <div
                    class="
                      c-article-section__figure
                      js-c-reading-companion-figures-item
                    "
                    data-test="figure"
                    data-container-section="figure"
                    id="figure-4"
                    data-title="Application of CellChat to communications between spatially colocalized cell populations."
                  >
                    <figure>
                      <figcaption>
                        <b
                          id="Fig4"
                          class="c-article-section__figure-caption"
                          data-test="figure-caption-text"
                          >Fig. 4: Application of CellChat to communications
                          between spatially colocalized cell populations.</b
                        >
                      </figcaption>
                      <div class="c-article-section__figure-content">
                        <div class="c-article-section__figure-item">
                          <a
                            class="c-article-section__figure-link"
                            data-test="img-link"
                            data-track="click"
                            data-track-label="image"
                            data-track-action="view figure"
                            href="https://www.nature.com/articles/s41467-021-21246-9/figures/4"
                            rel="nofollow"
                            ><picture
                              ><source
                                type="image/webp"
                                srcset="
                                  report_files/41467_2021_21246_Fig4_HTML.webp 1x
                                " />
                              <img
                                aria-describedby="Fig4"
                                src="report_files/41467_2021_21246_Fig4_HTML.png"
                                alt="figure4"
                                loading="lazy"
                                width="685"
                                height="847" /></picture
                          ></a>
                        </div>
                        <div
                          class="c-article-section__figure-description"
                          data-test="bottom-caption"
                          id="figure-4-desc"
                        >
                          <p>
                            <b>a</b> Spatial diagram of placode, pre-DC, DC1 and
                            DC2 cells during hair follicle (HF) development at
                            E14.5. DC: dermal condensate. <b>b</b> Number of
                            significant ligand-receptor pairs between any pair
                            of two&nbsp;cell populations. The edge width is
                            proportional to the indicated number of
                            ligand-receptor pairs. <b>c</b> The inferred FGF and
                            WNT signaling networks. Circle sizes are
                            proportional to the number of cells in each cell
                            group and edge width represents the communication
                            probability. <b>d</b> All the significant
                            ligand-receptor pairs that contribute to the
                            signaling sending from placode to three DC states.
                            The dot color and size represent the calculated
                            communication probability and <i>p</i>-values.
                            <i>p</i>-values are computed from one-sided
                            permutation test. <b>e</b> The outgoing
                            communication patterns of secreting cells, which
                            shows the correspondence between the inferred latent
                            patterns and cell groups, as well as signaling
                            pathways. <b>f</b> Incoming communication patterns
                            of target cells. <b>g</b> The inferred Pros1-Axl
                            signaling network, as well as the scRNA-seq
                            expression distribution of the <i>Pros1</i> ligand,
                            the <i>Axl</i> receptor and cell migration marker
                            <i>Thy1</i>. The edge width represents the
                            communication probability. <b>h</b> RNAscope data
                            (<i>n</i>  =4 independent experiments) showing
                            spatial distribution of <i>Edn3 (red)</i>,
                            <i>Axl (green),</i> and
                            <i>Thy1 (white)</i> transcripts in early-stage
                            developing hair follicle from E14.5 embryonic mouse
                            skin. Epithelial placode and dermal condensate (DC)
                            are annotated and outlined with dashed lines. Solid
                            white curved arrows in the bottom-right panel mark
                            CellChat-predicted Pros1-Axl signaling within skin
                            space. DAPI (teal) stains nuclei. Scale bar: 50 μm.
                            <b>i</b> The inferred Edn3-Ednrb signaling network,
                            as well as the scRNA-seq expression distribution of
                            the melanocyte marker <i>Dct</i>, <i>Edn3</i> ligand
                            and its receptor <i>Ednrb</i>. DC: dermal
                            condensate; MELA: melanocytes; (<b>j</b>) RNAscope
                            data (<i>n</i> = 4 independent experiments) showing
                            spatial distribution of
                            <i>Dct (green), Edn3 (red),</i> and
                            <i>Ednrb (white)</i> transcripts in early-stage
                            developing hair follicle from E14.5 embryonic mouse
                            skin. Arrowheads mark possible melanocyte
                            populations. Solid white curved arrows in the
                            top-right panel mark CellChat-predicted Edn3-Ednrb
                            signaling within skin space. DAPI (teal) stains
                            nuclei. Scale bar: 50 μm.
                          </p>
                        </div>
                      </div>
                      <div class="u-text-right u-hide-print">
                        <a
                          class="c-article__pill-button"
                          data-test="article-link"
                          data-track="click"
                          data-track-label="button"
                          data-track-action="view figure"
                          href="https://www.nature.com/articles/s41467-021-21246-9/figures/4"
                          data-track-dest="link:Figure4 Full size image"
                          aria-label="Full size image figure 4"
                          rel="nofollow"
                          ><span>Full size image</span
                          ><svg
                            width="16"
                            height="16"
                            focusable="false"
                            role="img"
                            aria-hidden="true"
                            class="u-icon"
                          >
                            <use
                              xlink="http://www.w3.org/1999/xlink"
                              xlink:href="#global-icon-chevron-right"
                            ></use></svg></a>
                      </div>
                    </figure>
                  </div>
                  <p>
                    By systematically investigating the predicted placode-to-DC
                    signals, we found 21 ligand-receptor pairs implicating WNT,
                    TGFβ, SEMA3, PTN, PDGF, MK, and FGF signaling pathways in
                    the process of DC specification (Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="figure anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#Fig4"
                      >4d</a
                    >). Pattern recognition analysis further revealed that
                    pre-DC and DC2 states jointly coordinate outgoing signals
                    for ncWNT, FGF, IGF, EDN, and SEMA3 pathways (pattern #1 in
                    Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="figure anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#Fig4"
                      >4e</a
                    >). DC1 dominantly drives PROS signaling (pattern #3), while
                    epithelial placode cells drive outgoing WNT, TGFβ, PDGF, MK,
                    PTN, and PTH signaling (pattern #2, Supplementary
                    Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="supplementary material anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#MOESM1"
                      >5f</a
                    >). At the incoming end of signaling, pre-DC cells respond
                    to SEMA3 and PTH signaling (pattern #3 in Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="figure anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#Fig4"
                      >4f</a
                    >); DC1 and DC2 cells respond to TGFβ, PDGF, EDN, and PROS
                    signaling (pattern #1) and epithelial placode cells respond
                    to WNT, ncWNT, IGF, MK, and PTN signaling (pattern #2,
                    Supplementary Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="supplementary material anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#MOESM1"
                      >5f</a
                    >).
                  </p>
                  <p>
                    CellChat revealed that at E14.5, DC cells respond to
                    autocrine PROS signaling (Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="figure anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#Fig4"
                      >4g</a
                    >). <i>Pros1</i> is the ligand for the pathway, which
                    signals via the receptor tyrosine kinase <i>Axl</i>.
                    Signaling via <i>Axl</i> has been implicated in conferring
                    cells with migratory properties in different biological
                    context, including EMT-mediated cancer invasion<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        title="Scutera, S. et al. Survival and migration of human dendritic cells are regulated by an IFN-alpha-inducible Axl/Gas6 pathway. J. Immunol. 183, 3004–3013 (2009)."
                        href="#ref-CR46"
                        id="ref-link-section-d38771043e1353"
                        >46</a
                      >,<a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        title="Abu-Thuraia, A. et al. AXL confers cell migration and invasion by hijacking a PEAK1-regulated focal adhesion protein network. Nat. Commun. 11, 3586 (2020)."
                        href="#ref-CR47"
                        id="ref-link-section-d38771043e1353_1"
                        >47</a
                      >,<a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 48"
                        title="Tieche, C. C. et al. Tumor initiation capacity and therapy resistance are differential features of EMT-related subpopulations in the NSCLC Cell Line A549. Neoplasia 21, 185–196 (2019)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR48"
                        id="ref-link-section-d38771043e1356"
                        >48</a
                      ></sup
                    >, and directional migration has been recently shown to be
                    crucial for normal dermal condensate formation upon hair
                    follicle morphogenesis<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 42"
                        title="Biggs, L. C. et al. Hair follicle dermal condensation forms via Fgf20 primed cell cycle exit, cell motility, and aggregation. Elife 7, e36468 (2018)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR42"
                        id="ref-link-section-d38771043e1361"
                        >42</a
                      ></sup
                    >. We examined CellChat’s prediction of active PROS
                    signaling in DC cells by RNAscope technique&nbsp;for
                    <i>Edn3</i> as DC marker, <i>Axl</i> and <i>Thy1</i> (Cd90)
                    as a marker of cell migration<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 49"
                        title="Kumar, A., Bhanja, A., Bhattacharyya, J. &amp; Jaganathan, B. G. Multiple roles of CD90 in cancer. Tumour Biol. 37, 11611–11622 (2016)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR49"
                        id="ref-link-section-d38771043e1374"
                        >49</a
                      >,<a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 50"
                        title="Rege, T. A. et al. Thy-1, via its GPI anchor, modulates Src family kinase and focal adhesion kinase phosphorylation and subcellular localization, and fibroblast migration, in response to thrombospondin-1/hep I. Exp. Cell Res. 312, 3752–3767 (2006)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR50"
                        id="ref-link-section-d38771043e1377"
                        >50</a
                      ></sup
                    >
                    and EMT process<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 51"
                        title="Meshcheryakova, A. et al. Exploring the role of sphingolipid machinery during the epithelial to mesenchymal transition program using an integrative approach. Oncotarget 7, 22295–22323 (2016)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR51"
                        id="ref-link-section-d38771043e1381"
                        >51</a
                      ></sup
                    >. As expected from scRNA-seq, <i>Axl</i> expression was
                    co-localized with <i>Edn3</i> and <i>Thy1</i> expression,
                    which was concentrated in DC with significantly lower levels
                    elsewhere (Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="figure anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#Fig4"
                      >4h</a
                    >). This RNAscope result is consistent with the possibility
                    of autocrine PROS signaling in DC, likely driven via
                    <i>Pros1</i>-<i>Axl</i> signaling.
                  </p>
                  <p>
                    During early hair follicle formation at E14.5, melanoblasts
                    (melanocyte precursor cells) migrate into the hair placode
                    from the dermis and then become differentiated toward
                    melanocytes. However, the mechanisms of melanocyte migration
                    into placode remain incompletely understood<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 52"
                        title="Botchkareva, N. V., Botchkarev, V. A. &amp; Gilchrest, B. A. Fate of melanocytes during development of the hair follicle pigmentary unit. J. Investig. Dermatol Symp. Proc. 8, 76–79 (2003)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR52"
                        id="ref-link-section-d38771043e1407"
                        >52</a
                      ></sup
                    >. Therefore, we further studied the cell–cell communication
                    among placodes, DC cells and melanocyte cells (including
                    three melanocyte subpopulations: MELA-A, MELA-B, and MELA-C;
                    see “Methods” section and Supplementary Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="supplementary material anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#MOESM1"
                      >3g</a
                    >). CellChat revealed that melanocytes strongly respond to
                    DC cells via previously unrecognized EDN signaling
                    (Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="figure anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#Fig4"
                      >4i</a
                    >). <i>Edn3</i> is a ligand of the EDN pathway, which
                    regulates melanocyte migration<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 53"
                        title="Garcia, R. J. et al. Endothelin 3 induces skin pigmentation in a keratin-driven inducible mouse model. J. Invest Dermatol 128, 131–142 (2008)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR53"
                        id="ref-link-section-d38771043e1420"
                        >53</a
                      ></sup
                    >. Therefore, CellChat prediction suggests DC cells induce
                    early directed migration of melanocytes. To experimentally
                    examine this prediction, we used the RNAscope technique to
                    spatially map expression of <i>Dct</i>, which marks
                    late-stage melanocyte precursors, <i>Edn3</i> ligand and its
                    receptor <i>Ednrb</i> in E14.5 embryonic mouse skin. As
                    expected, <i>Dct</i><sup><i>+</i></sup> melanocytes (i.e.,
                    MELA-C subpopulation) localize in and around epithelial
                    placode. They also express <i>Ednrb</i>. In turn,
                    <i>Edn3</i> is specifically enriched in DC cells (preDC,
                    DC1, and DC2 subpopulations), while <i>Ednrb</i> is also
                    enriched in a portion of DC cells (likely DC2
                    subpopulation). Scattered <i>Ednrb</i><sup><i>+</i></sup
                    ><i>/Edn3</i><sup><i>neg</i></sup
                    ><i>/Dct</i><sup><i>neg</i></sup> cells inside dermal
                    condensate are likely undifferentiated melanoblasts (i.e.,
                    MELA-A/B subpopulations) (Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="figure anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#Fig4"
                      >4j</a
                    >). This spatial <i>Edn3, Ednrb, Dct</i> co-expression
                    pattern is highly consistent with the scRNA-seq data
                    (Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="figure anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#Fig4"
                      >4i</a
                    >). Thus, our RNAscope result confirms the CellChat
                    prediction of <i>Edn3</i>-<i>Ednrb</i> signaling from DC
                    cells to melanocytes, implying the roles for DC cells in
                    inducing early-stage directed migration of melanocytes into
                    placodes. It also shows potential autocrine <i>Edn3</i>-<i
                      >Ednrb</i
                    >
                    signaling within the dermal condensate.
                  </p>
                  <h3 class="c-article__sub-heading" id="Sec7">
                    Joint learning of time-course scRNA-seq data to uncover
                    dynamic communication patterns
                  </h3>
                  <p>
                    Next, we demonstrate how CellChat can be applied to studying
                    temporal changes of intercellular communications in the same
                    tissue (Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="figure anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#Fig5"
                      >5a</a
                    >). For this purpose, we performed combined analysis on two
                    embryonic mouse skin scRNA-seq datasets from days E13.5 and
                    E14.5<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 22"
                        title="Gupta, K. et al. Single-cell analysis reveals a hair follicle dermal niche molecular differentiation trajectory that begins prior to morphogenesis. Dev. Cell 48, 17–31 e16 (2019)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR22"
                        id="ref-link-section-d38771043e1501"
                        >22</a
                      ></sup
                    >. Unsupervised clustering of E13.5 and E14.5 datasets
                    identified 11 skin cell populations at E13.5 and E14.5 and
                    additional two populations (i.e., dermal DC and pericytes)
                    specific to E14.5 (Supplementary Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="supplementary material anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#MOESM1"
                      >3a–d</a
                    >; see “Methods” section).
                  </p>
                  <div
                    class="
                      c-article-section__figure
                      js-c-reading-companion-figures-item
                    "
                    data-test="figure"
                    data-container-section="figure"
                    id="figure-5"
                    data-title="Comparison analysis of epidermal-dermal communications between different skin developmental stages."
                  >
                    <figure>
                      <figcaption>
                        <b
                          id="Fig5"
                          class="c-article-section__figure-caption"
                          data-test="figure-caption-text"
                          >Fig. 5: Comparison analysis of epidermal-dermal
                          communications between different skin developmental
                          stages.</b
                        >
                      </figcaption>
                      <div class="c-article-section__figure-content">
                        <div class="c-article-section__figure-item">
                          <a
                            class="c-article-section__figure-link"
                            data-test="img-link"
                            data-track="click"
                            data-track-label="image"
                            data-track-action="view figure"
                            href="https://www.nature.com/articles/s41467-021-21246-9/figures/5"
                            rel="nofollow"
                            ><picture
                              ><source
                                type="image/webp"
                                srcset="
                                  report_files/41467_2021_21246_Fig5_HTML.webp 1x
                                " />
                              <img
                                aria-describedby="Fig5"
                                src="report_files/41467_2021_21246_Fig5_HTML.png"
                                alt="figure5"
                                loading="lazy"
                                width="685"
                                height="972" /></picture
                          ></a>
                        </div>
                        <div
                          class="c-article-section__figure-description"
                          data-test="bottom-caption"
                          id="figure-5-desc"
                        >
                          <p>
                            <b>a</b> Schematic illustration of cellular
                            composition of embryonic skin at E13.5 and E14.5.
                            Different cell populations are color-coded to match
                            colors in panel <b>e</b> and <b>h</b>. FIB-A:
                            fibroblast type A; FIB-B: fibroblast type B; FIB-P:
                            proliferative fibroblasts. MYL: myeloid cell; ENDO:
                            endothelial cell; MELA: melanocytes;
                            <b>b</b> Jointly projecting and clustering signaling
                            pathways from E13.5 and E14.5 into a shared
                            two-dimensional manifold according to their
                            functional similarity. Circle and square symbols
                            represent the signaling networks from E13.5 and
                            E14.5 respectively. Each dot or square represents
                            the communication network of one signaling pathway.
                            Dot or square size is proportional to the total
                            communication probability. Different colors
                            represent different groups of signaling pathways.
                            <b>c</b> Magnified view of each pathway group.
                            <b>d</b> The overlapping signaling pathways between
                            E13.5 and E14.5 were ranked based on their pairwise
                            Euclidean distance in the shared two-dimensional
                            manifold. <b>e</b> The inferred WNT signaling
                            network at E13.5. Left and right portions show the
                            autocrine and paracrine signaling to dermis and
                            epidermis, respectively. Circle sizes are
                            proportional to the number of cells in each cell
                            group and edge width represents the communication
                            probability. <b>f</b> Relative contribution of each
                            ligand-receptor pair to the overall WNT signaling
                            network at E13.5. <b>g</b> Expression distribution
                            of WNT signaling genes at E13.5. <b>h</b> The
                            inferred WNT signaling network at E14.5.
                            <b>i</b> Relative contribution of each
                            ligand-receptor pair at E14.5. <b>j</b> The
                            expression distribution of WNT signaling genes at
                            E14.5. <b>k</b> All significant signaling pathways
                            were ranked based on their differences of overall
                            information flow within the inferred networks
                            between E13.5 and E14.5. The top signaling pathways
                            colored red are more enriched in E13.5, the middle
                            ones colored black are equally enriched in E13.5 and
                            E14.5, and the bottom ones colored green are more
                            enriched in E14.5. <b>l</b> The dot plot showing the
                            comparison of outgoing signaling patterns of
                            secreting cells between E13.5 and E14.5. The dot
                            size is proportional to the contribution score
                            computed from pattern recognition analysis. Higher
                            contribution score implies the signaling pathway is
                            more enriched in the corresponding cell group.
                          </p>
                        </div>
                      </div>
                      <div class="u-text-right u-hide-print">
                        <a
                          class="c-article__pill-button"
                          data-test="article-link"
                          data-track="click"
                          data-track-label="button"
                          data-track-action="view figure"
                          href="https://www.nature.com/articles/s41467-021-21246-9/figures/5"
                          data-track-dest="link:Figure5 Full size image"
                          aria-label="Full size image figure 5"
                          rel="nofollow"
                          ><span>Full size image</span
                          ><svg
                            width="16"
                            height="16"
                            focusable="false"
                            role="img"
                            aria-hidden="true"
                            class="u-icon"
                          >
                            <use
                              xlink="http://www.w3.org/1999/xlink"
                              xlink:href="#global-icon-chevron-right"
                            ></use></svg></a>
                      </div>
                    </figure>
                  </div>
                  <p>
                    We inferred intercellular communications for the above two
                    datasets separately, and then analyzed them together via
                    joint manifold learning and classification of the inferred
                    communication networks based on their functional similarity.
                    The functional similarity analysis requires the same cell
                    population composition between two datasets. Thus, for such
                    analysis we used only 11 common cell populations between
                    E13.5 and E14.5 datasets. As the result, the signaling
                    pathways associated with inferred networks from both
                    datasets were mapped onto a shared two-dimensional manifold
                    and clustered into groups. We identified four pathway groups
                    (Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="figure anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#Fig5"
                      >5b-c</a
                    >). Groups #1 and #3 were dominated by growth factor
                    pathways such as PDGF, NGF, FGF, EGF, and ANGPTL, while
                    groups #2 and #5 dominantly contained inflammation-related
                    pathways such as CCL, IL2, IL4, OSM, LIFR, and VISFATIN. As
                    expected, the majority of the same signaling pathways from
                    E13.5 and E14.5 were grouped together such as CCL, CSF,
                    ANGPTL, PDGF, VEGF, ncWNT, and MK, suggesting that these
                    pathways are essential for skin morphogenesis at both time
                    points and likely do not critically regulate new
                    developmental events at E14.5, such as hair follicle
                    morphogenesis or dermal maturation. However, WNT and KIT
                    signaling were classified into different groups, consistent
                    with profound and multi-faceted role of WNT signaling in
                    skin morphogenesis<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 22"
                        title="Gupta, K. et al. Single-cell analysis reveals a hair follicle dermal niche molecular differentiation trajectory that begins prior to morphogenesis. Dev. Cell 48, 17–31 e16 (2019)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR22"
                        id="ref-link-section-d38771043e1577"
                        >22</a
                      >,<a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 54"
                        title="Rognoni, E. &amp; Watt, F. M. Skin cell heterogeneity in development, wound healing, and cancer. Trends Cell Biol. 28, 709–722 (2018)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR54"
                        id="ref-link-section-d38771043e1580"
                        >54</a
                      ></sup
                    >. By computing the Euclidean distance between any pair of
                    the shared&nbsp;signaling pathways in the shared
                    two-dimensional manifold, we observed a large distance for
                    WNT and KIT and to a lesser extent for RANKL, IL2, FGF,
                    GALECTIN, EGF, TGFβ, and NGF pathways (Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="figure anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#Fig5"
                      >5d</a
                    >, Supplementary Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="supplementary material anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#MOESM1"
                      >6a-d</a
                    >). We specifically examined how WNT communications change
                    over one day of skin development (Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="figure anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#Fig5"
                      >5e-j</a
                    >,&nbsp;Supplementary Data
                    <a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="supplementary material anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#MOESM5"
                      >2</a
                    >). At both embryonic time points, basal epidermal cells
                    were the dominant source of WNT ligands, with further minor
                    contribution from fibroblasts. Yet, compared to E13.5, when
                    only basal epidermal cells were the WNT targets, at E14.5
                    fibroblasts gained WNT responsiveness. Further, melanocytes
                    emerged as the new minor source of WNT signaling, helping to
                    drive an overall increase in WNT communication network
                    complexity. Collectively, the joint manifold learning
                    enables the identification of signaling pathways that
                    undergo embryonic stage-dependent change.
                  </p>
                  <p>
                    Next, we compared the information flow for each signaling
                    pathway between E13.5 and E14.5 time points. The information
                    flow for a given signaling pathway is defined by the sum of
                    communication probability among all pairs of cell groups in
                    the inferred network. We found that some pathways, including
                    ANGPTL, APELIN, CSF, FGF, RANKL, and TGFβ maintain similar
                    flow between the time points (black in Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="figure anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#Fig5"
                      >5k</a
                    >). We interpret that these pathways are equally important
                    in the developing skin at both time points. In contrast,
                    other pathways prominently change their information flow at
                    E14.5 as compared to E13.5: (i) turn off (NT, TWEAK), (ii)
                    decrease (such as PTN, MK), (iii) turn on (TNF), or (iv)
                    increase (such as WNT, GALECTIN, KIT, IGF, VEGF).
                  </p>
                  <p>
                    Moreover, we studied the detailed changes in the outgoing
                    signaling across all significant pathways using pattern
                    recognition analysis (Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="figure anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#Fig5"
                      >5l</a
                    >; see “Methods” section). We found that skin fibroblasts
                    change their major and minor outgoing communication patterns
                    between E13.5 and E14.5. At E13.5, early fibroblast state
                    FIB-A dominates the outgoing signaling. Over one day period,
                    the minor signaling of late fibroblast states FIB-B and
                    FIB-P become major and includes ANGPTL, IGF, VEGF, KIT,
                    SEMA3 pathways (Supplementary Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="supplementary material anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#MOESM1"
                      >6a-h</a
                    >). This suggests the balancing changes in the levels and
                    patterns of ligand expression. On the other hand,
                    endothelial cells (ENDO), melanocytes (MELA) and
                    skin-resident myeloid cells (MYL) maintain their outgoing
                    signaling patterns. Complex outgoing signaling dynamics were
                    observed in the epidermis. Basal epidermal cells at E14.5
                    maintain secreted signaling patterns for NGF, PDGF,
                    VISFATIN, and WNT, yet turn off signaling including for KIT
                    and Neurotrophin (NT), and turn on signaling including for
                    VEGF, PTN and LIFR. On the other hand, spinous epidermal
                    cells prominently redesign their outgoing signaling. They
                    turn off or decrease four pathways, such as PDGF
                    (Supplementary Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="supplementary material anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#MOESM1"
                      >6e</a
                    >
                    and
                    <a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="supplementary material anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#MOESM1"
                      >6g</a
                    >), turn on SEMA3 pathway, and maintain three pathways—IGF,
                    MK, and PTN (Supplementary Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="supplementary material anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#MOESM1"
                      >6</a
                    >f and
                    <a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="supplementary material anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#MOESM1"
                      >6h</a
                    >). Prominent change in spinous cell signaling is consistent
                    with known epidermal stratification event that occurs in
                    mice at the transition between E13.5 and E14.5<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 55"
                        title="Bazzi, H., Fantauzzo, K. A., Richardson, G. D., Jahoda, C. A. &amp; Christiano, A. M. Transcriptional profiling of developing mouse epidermis reveals novel patterns of coordinated gene expression. Dev. Dyn. 236, 961–970 (2007)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR55"
                        id="ref-link-section-d38771043e1625"
                        >55</a
                      >,<a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 56"
                        title="Byrne, C., Tainsky, M. &amp; Fuchs, E. Programming gene expression in developing epidermis. Development 120, 2369–2383 (1994)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR56"
                        id="ref-link-section-d38771043e1628"
                        >56</a
                      ></sup
                    >. Taken together, CellChat analysis on joint scRNA-seq
                    datasets enables multifaceted assessment of intercellular
                    communication patterns across biological times, such as
                    embryonic developmental time scale.
                  </p>
                  <h3 class="c-article__sub-heading" id="Sec8">
                    Joint learning of conserved and context-specific
                    communication patterns between distinct scRNA-seq datasets
                  </h3>
                  <p>
                    We also used CellChat to compare cell–cell communication
                    patterns between two scRNA-seq datasets, one from embryonic
                    day E13.5 skin<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 22"
                        title="Gupta, K. et al. Single-cell analysis reveals a hair follicle dermal niche molecular differentiation trajectory that begins prior to morphogenesis. Dev. Cell 48, 17–31 e16 (2019)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR22"
                        id="ref-link-section-d38771043e1640"
                        >22</a
                      ></sup
                    >
                    and another from adult day 12 wound skin<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 23"
                        title="Guerrero-Juarez, C. F. et al. Single-cell analysis reveals fibroblast heterogeneity and myeloid-derived adipocyte progenitors in murine skin wounds. Nat. Commun. 10, 650 (2019)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR23"
                        id="ref-link-section-d38771043e1644"
                        >23</a
                      ></sup
                    >
                    (Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="figure anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#Fig6"
                      >6a</a
                    >). While representing the same tissue (skin) from the same
                    species (mouse) and containing some of the same principal
                    cell types, such as fibroblasts, these two datasets are from
                    vastly distinct biological contexts— embryonic morphogenesis
                    vs. wound-induced repair. As such, this case study presents
                    an opportunity to discover signaling logic and signal
                    conservation principles. First, we performed joint manifold
                    learning and classification of the inferred communication
                    networks based on their topological similarity (functional
                    similarity cannot be performed because of the vastly
                    different cell type composition). We identified four
                    signaling pathway groups (Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="figure anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#Fig6"
                      >6b–c</a
                    >). Intriguingly, none of the groups are unique to a given
                    dataset, suggesting that the entire spectrum of
                    communications is represented in both skin states. There
                    are, however, dataset-specific enrichments, especially in
                    groups #1 and # 4, which are dominated by signaling networks
                    of the embryonic skin (8 out of 14 and 6 out of 9,
                    respectively). The other two groups #2 and #3 are nearly
                    equally contributed by the communication networks and
                    contain several overlapping pathways from both skin states.
                    By computing the Euclidean distance between any pair of the
                    shared signaling pathways in the shared two-dimensions
                    space, we observed a large distance for signaling pathways
                    like IGF, PDGF, CSF, PROS, and CCL (Supplementary
                    Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="supplementary material anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#MOESM1"
                      >7a-b</a
                    >), suggesting that these pathways exhibit significantly
                    different communication network architectures. However,
                    other signaling pathways show relatively small distances,
                    including ANGPTL, RANKL, TGFb, SEMA3, IL2, PTN, ncWNT, MK,
                    EGF, APELN, and EDN (Supplementary Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="supplementary material anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#MOESM1"
                      >7c</a
                    >), which are also grouped together (Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="figure anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#Fig6"
                      >6c–d</a
                    >). This suggests similar communication network
                    architectures for these overlapping pathways in both skin
                    states. Closer look at the MK (Midkine) pathway
                    (Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="figure anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#Fig6"
                      >6e–f</a
                    >) shows its high signaling redundancy (i.e., multiple
                    signaling sources) and high target promiscuity (i. e. all
                    cell groups can function as MK targets). The latter finding
                    suggests that certain pathways have highly conserved
                    signaling architecture (i.e., high degree of redundancy)
                    which is largely independent of the specific cellular
                    composition of the tissue.
                  </p>
                  <div
                    class="
                      c-article-section__figure
                      js-c-reading-companion-figures-item
                    "
                    data-test="figure"
                    data-container-section="figure"
                    id="figure-6"
                    data-title="Joint identification of conserved and context-specific communication patterns between two skin states."
                  >
                    <figure>
                      <figcaption>
                        <b
                          id="Fig6"
                          class="c-article-section__figure-caption"
                          data-test="figure-caption-text"
                          >Fig. 6: Joint identification of conserved and
                          context-specific communication patterns between two
                          skin states.</b
                        >
                      </figcaption>
                      <div class="c-article-section__figure-content">
                        <div class="c-article-section__figure-item">
                          <a
                            class="c-article-section__figure-link"
                            data-test="img-link"
                            data-track="click"
                            data-track-label="image"
                            data-track-action="view figure"
                            href="https://www.nature.com/articles/s41467-021-21246-9/figures/6"
                            rel="nofollow"
                            ><picture
                              ><source
                                type="image/webp"
                                srcset="
                                  report_files/41467_2021_21246_Fig6_HTML.webp 1x
                                " />
                              <img
                                aria-describedby="Fig6"
                                src="report_files/41467_2021_21246_Fig6_HTML.png"
                                alt="figure6"
                                loading="lazy"
                                width="685"
                                height="652" /></picture
                          ></a>
                        </div>
                        <div
                          class="c-article-section__figure-description"
                          data-test="bottom-caption"
                          id="figure-6-desc"
                        >
                          <p>
                            <b>a</b> Schematic illustration of cellular
                            composition of skin during embryonic morphogenesis
                            at E13.5 and during adult wound-induced repair at
                            day 12. Different cell populations are color-coded
                            to match colors in panel <b>e</b> and <b>f</b>,
                            respectively. <b>b</b> Jointly projecting and
                            clustering signaling pathways from E13.5 and wound
                            onto shared two-dimensional manifold according to
                            their structural similarity of the inferred
                            networks. Circle and square symbols represent the
                            signaling networks from E13.5 and wound
                            respectively. Each circle or square represents the
                            communication network of one signaling pathway.
                            Circle or square size is proportional to the total
                            communication probability of that signaling network.
                            Different colors represent different groups of
                            signaling pathways. <b>c</b> Magnified view of each
                            pathway group. <b>d</b> The overlapping signaling
                            pathways between E13.5 and wound were ranked based
                            on their pairwise Euclidean distance in the shared
                            two-dimensional manifold. Larger distance implies
                            larger difference. <b>e–f</b> Hierarchical plot
                            showing the inferred intercellular communication
                            network of MK signaling pathway at E13.5 and wound,
                            respectively. Circle sizes are proportional to the
                            number of cells in each cell group and edge width
                            represents the communication probability.
                            <b>g</b> All the significant signaling pathways were
                            ranked based on their differences of overall
                            information flow within the inferred networks
                            between E13.5 and wound. The overall information
                            flow of a signaling network is calculated by
                            summarizing all the communication probabilities in
                            that network. The top signaling pathways colored by
                            red are more enriched in E13.5, and the bottom ones
                            colored by green were more enriched in the wound.
                          </p>
                        </div>
                      </div>
                      <div class="u-text-right u-hide-print">
                        <a
                          class="c-article__pill-button"
                          data-test="article-link"
                          data-track="click"
                          data-track-label="button"
                          data-track-action="view figure"
                          href="https://www.nature.com/articles/s41467-021-21246-9/figures/6"
                          data-track-dest="link:Figure6 Full size image"
                          aria-label="Full size image figure 6"
                          rel="nofollow"
                          ><span>Full size image</span
                          ><svg
                            width="16"
                            height="16"
                            focusable="false"
                            role="img"
                            aria-hidden="true"
                            class="u-icon"
                          >
                            <use
                              xlink="http://www.w3.org/1999/xlink"
                              xlink:href="#global-icon-chevron-right"
                            ></use></svg></a>
                      </div>
                    </figure>
                  </div>
                  <p>
                    We also compared the information flow (i.e., the overall
                    communication probability) across the two skin datasets.
                    Intriguingly, 19 out of 34 pathways are highly active,
                    albeit at different levels, both in embryonic skin and in
                    adult skin wounds (Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="figure anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#Fig6"
                      >6g</a
                    >). These likely represent core signaling pathways necessary
                    for skin function independent of the specific point in the
                    biological time scale (i.e., embryonic vs. adult). Nine
                    pathways are active only in embryonic skin. These include
                    such important pathways for skin morphogenesis as FGF<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 37"
                        title="Glover, J. D. et al. Hierarchical patterning modes orchestrate hair follicle morphogenesis. PLoS Biol. 15, e2002117 (2017)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR37"
                        id="ref-link-section-d38771043e1717"
                        >37</a
                      >,<a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 43"
                        title="Huh, S. H. et al. Fgf20 governs formation of primary and secondary dermal condensations in developing hair follicles. Genes Dev. 27, 450–458 (2013)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR43"
                        id="ref-link-section-d38771043e1720"
                        >43</a
                      >,<a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        title="Mukhopadhyay, A. et al. Negative regulation of Shh levels by Kras and Fgfr2 during hair follicle development. Dev. Biol. 373, 373–382 (2013)."
                        href="#ref-CR57"
                        id="ref-link-section-d38771043e1723"
                        >57</a
                      >,<a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        title="Richardson, G. D. et al. KGF and EGF signalling block hair follicle induction and promote interfollicular epidermal fate in developing mouse skin. Development 136, 2153–2164 (2009)."
                        href="#ref-CR58"
                        id="ref-link-section-d38771043e1723_1"
                        >58</a
                      >,<a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        title="Petiot, A. et al. A crucial role for Fgfr2-IIIb signalling in epidermal development and hair follicle patterning. Development 130, 5493–5501 (2003)."
                        href="#ref-CR59"
                        id="ref-link-section-d38771043e1723_2"
                        >59</a
                      >,<a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 60"
                        title="du Cros, D. L. Fibroblast growth factor and epidermal growth factor in hair development. J. Invest. Dermatol. 101, 106S–113S (1993)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR60"
                        id="ref-link-section-d38771043e1726"
                        >60</a
                      ></sup
                    >
                    and WNT<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 22"
                        title="Gupta, K. et al. Single-cell analysis reveals a hair follicle dermal niche molecular differentiation trajectory that begins prior to morphogenesis. Dev. Cell 48, 17–31 e16 (2019)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR22"
                        id="ref-link-section-d38771043e1730"
                        >22</a
                      >,<a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        title="Mok, K. W. et al. Dermal condensate niche fate specification occurs prior to formation and is placode progenitor dependent. Dev. Cell 48, 32–48 e35 (2019)."
                        href="#ref-CR36"
                        id="ref-link-section-d38771043e1733"
                        >36</a
                      >,<a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        title="Glover, J. D. et al. Hierarchical patterning modes orchestrate hair follicle morphogenesis. PLoS Biol. 15, e2002117 (2017)."
                        href="#ref-CR37"
                        id="ref-link-section-d38771043e1733_1"
                        >37</a
                      >,<a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        title="Tsai, S. Y. et al. Wnt/beta-catenin signaling in dermal condensates is required for hair follicle formation. Dev. Biol. 385, 179–188 (2014)."
                        href="#ref-CR38"
                        id="ref-link-section-d38771043e1733_2"
                        >38</a
                      >,<a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        title="Fu, J. &amp; Hsu, W. Epidermal Wnt controls hair follicle induction by orchestrating dynamic signaling crosstalk between the epidermis and dermis. J. Invest. Dermatol. 133, 890–898 (2013)."
                        href="#ref-CR39"
                        id="ref-link-section-d38771043e1733_3"
                        >39</a
                      >,<a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 40"
                        title="Chen, D., Jarrell, A., Guo, C., Lang, R. &amp; Atit, R. Dermal beta-catenin activity in response to epidermal Wnt ligands is required for fibroblast proliferation and hair follicle initiation. Development 139, 1522–1533 (2012)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR40"
                        id="ref-link-section-d38771043e1736"
                        >40</a
                      ></sup
                    >. Four pathways are specifically active in wounded skin,
                    including known regulators of wound-induced skin repair SPP1
                    (osteopontin)<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        title="Buback, F., Renkl, A. C., Schulz, G. &amp; Weiss, J. M. Osteopontin and the skin: multiple emerging roles in cutaneous biology and pathology. Exp. Dermatol. 18, 750–759 (2009)."
                        href="#ref-CR61"
                        id="ref-link-section-d38771043e1740"
                        >61</a
                      >,<a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        title="Mori, R., Shaw, T. J. &amp; Martin, P. Molecular mechanisms linking wound inflammation and fibrosis: knockdown of osteopontin leads to rapid repair and reduced scarring. J. Exp. Med. 205, 43–51 (2008)."
                        href="#ref-CR62"
                        id="ref-link-section-d38771043e1740_1"
                        >62</a
                      >,<a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 63"
                        title="Liaw, L. et al. Altered wound healing in mice lacking a functional osteopontin gene (spp1). J. Clin. Invest. 101, 1468–1478 (1998)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR63"
                        id="ref-link-section-d38771043e1743"
                        >63</a
                      ></sup
                    >, MIF (macrophage migration inhibitory factor)<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        title="Gilliver, S. C., Emmerson, E., Bernhagen, J. &amp; Hardman, M. J. MIF: a key player in cutaneous biology and wound healing. Exp. Dermatol. 20, 1–6 (2011)."
                        href="#ref-CR64"
                        id="ref-link-section-d38771043e1747"
                        >64</a
                      >,<a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        title="Ashcroft, G. S. et al. Estrogen modulates cutaneous wound healing by downregulating macrophage migration inhibitory factor. J. Clin. Invest. 111, 1309–1318 (2003)."
                        href="#ref-CR65"
                        id="ref-link-section-d38771043e1747_1"
                        >65</a
                      >,<a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 66"
                        title="Danchin, A. The specification of the immune response revisited. Surv. Immunol. Res. 1, 173–183 (1982)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR66"
                        id="ref-link-section-d38771043e1750"
                        >66</a
                      ></sup
                    >
                    and IL1<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        title="Mirza, R. E., Fang, M. M., Ennis, W. J. &amp; Koh, T. J. Blocking interleukin-1beta induces a healing-associated wound macrophage phenotype and improves healing in type 2 diabetes. Diabetes 62, 2579–2587 (2013)."
                        href="#ref-CR67"
                        id="ref-link-section-d38771043e1755"
                        >67</a
                      >,<a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        title="Thomay, A. A. et al. Disruption of interleukin-1 signaling improves the quality of wound healing. Am. J. Pathol. 174, 2129–2136 (2009)."
                        href="#ref-CR68"
                        id="ref-link-section-d38771043e1755_1"
                        >68</a
                      >,<a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 69"
                        title="Sauder, D. N. et al. Interleukin-1 enhances epidermal wound healing. Lymphokine Res. 9, 465–473 (1990)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR69"
                        id="ref-link-section-d38771043e1758"
                        >69</a
                      ></sup
                    >. Taken together, this CellChat approach allows
                    system-level classification and discovery of signaling
                    communication network architecture principles.
                  </p>
                  <h3 class="c-article__sub-heading" id="Sec9">
                    Joint learning of normal and diseased human skin to discover
                    major signaling changes in response to disease
                  </h3>
                  <p>
                    As CellChatDB also includes curated ligand-receptor
                    interactions of human, we next employed CellChat to detect
                    the signaling changes between so-called lesional (diseased)
                    and nonlesional (normal) skin from patients with atopic
                    dermatitis (AD) using recently published human skin
                    scRNA-seq dataset<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 70"
                        title="He, H. et al. Single-cell transcriptome analysis of human skin identifies novel fibroblast subpopulation and enrichment of immune subsets in atopic dermatitis. J. Allergy Clin. Immunol. 145, 1615–1628 (2020)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR70"
                        id="ref-link-section-d38771043e1770"
                        >70</a
                      ></sup
                    >
                    (Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="figure anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#Fig7"
                      >7a</a
                    >). The original study revealed that lesional skin was
                    enriched for chemokine signals (including <i>CCL19</i>) from
                    inflammatory fibroblasts to inflammatory immune cells,
                    including dendritic cells (DC) and T cells (TC). This was
                    validated using immunofluorescence staining<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 70"
                        title="He, H. et al. Single-cell transcriptome analysis of human skin identifies novel fibroblast subpopulation and enrichment of immune subsets in atopic dermatitis. J. Allergy Clin. Immunol. 145, 1615–1628 (2020)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR70"
                        id="ref-link-section-d38771043e1780"
                        >70</a
                      ></sup
                    >. Therefore, we used CellChat to study the intercellular
                    communication among fibroblasts (four subpopulations:
                    <i>APOE</i> + FIB, <i>FBN1</i> + FIB, <i>COL11A</i> + FIB,
                    and Inflam.FIB), DCs (four subpopulations: cDC1, cDC2, LC,
                    and Inflam.DC), and TCs (four subpopulations: TC, Inflam.TC,
                    <i>CD40LG</i> + TC and NKT) (Supplementary Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="supplementary material anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#MOESM1"
                      >8a–e</a
                    >, see “Methods” section).
                  </p>
                  <div
                    class="
                      c-article-section__figure
                      js-c-reading-companion-figures-item
                    "
                    data-test="figure"
                    data-container-section="figure"
                    id="figure-7"
                    data-title="Identification of major signaling changes in diseased human skin."
                  >
                    <figure>
                      <figcaption>
                        <b
                          id="Fig7"
                          class="c-article-section__figure-caption"
                          data-test="figure-caption-text"
                          >Fig. 7: Identification of major signaling changes in
                          diseased human skin.</b
                        >
                      </figcaption>
                      <div class="c-article-section__figure-content">
                        <div class="c-article-section__figure-item">
                          <a
                            class="c-article-section__figure-link"
                            data-test="img-link"
                            data-track="click"
                            data-track-label="image"
                            data-track-action="view figure"
                            href="https://www.nature.com/articles/s41467-021-21246-9/figures/7"
                            rel="nofollow"
                            ><picture
                              ><source
                                type="image/webp"
                                srcset="
                                  report_files/41467_2021_21246_Fig7_HTML.webp 1x
                                " />
                              <img
                                aria-describedby="Fig7"
                                src="report_files/41467_2021_21246_Fig7_HTML.png"
                                alt="figure7"
                                loading="lazy"
                                width="685"
                                height="491" /></picture
                          ></a>
                        </div>
                        <div
                          class="c-article-section__figure-description"
                          data-test="bottom-caption"
                          id="figure-7-desc"
                        >
                          <p>
                            <b>a</b> Schematic illustration of scRNA-seq on
                            cells from nonlesional (NL, normal) and lesional
                            (LS, diseased) human skin from patients with atopic
                            dermatitis. <b>b</b> Jointly projecting and
                            clustering signaling pathways from NL and LS skin
                            onto shared two-dimensional manifold according to
                            functional similarity of the inferred networks.
                            Circle and square symbols represent the signaling
                            networks from NL and LS respectively. Each dot or
                            square represents the communication network of one
                            signaling pathway. Dot or square size is
                            proportional to the communication probability.
                            Different colors represent different groups of
                            signaling pathways. <b>c</b> Significant signaling
                            pathways were ranked based on differences in the
                            overall information flow within the inferred
                            networks between NL and LS skin. The overall
                            information flow of a signaling network is
                            calculated by summarizing all communication
                            probabilities in that network. The top signaling
                            pathways colored red are enriched in NL skin, and
                            these colored green were enriched in the LS skin.
                            <b>d</b> Comparison of the significant
                            ligand-receptor pairs between NL and LS skin, which
                            contribute to the signaling from Inflam.FIB (i.e.,
                            inflammatory fibroblasts) to dendritic cells (DC)
                            and T cells (TC) including cDC1, cDC2, LS,
                            Inflam.DC, TC, Inflam.TC, <i>CD40LG</i> + TC, and
                            NKT subpopulations. The highlighted
                            <i>CCL19-CCR7</i> signaling was previously validated
                            using immunofluorescence staining. Dot color
                            reflects communication probabilities and dot size
                            represents computed p-values. Empty space means the
                            communication probability is zero. <i>p</i>-values
                            are computed from one-sided permutation test.
                            <b>e</b> Expression distribution of ligand
                            <i>CCL19</i> and its receptor <i>CCR7</i> in NL
                            (red) and LS (green) skin. <b>f</b> Hierarchical
                            plot showing inferred intercellular communication
                            network of <i>CCL19-CCR7</i> signaling in LS skin.
                            Left and right portions show autocrine and paracrine
                            signaling to fibroblast and immune cells,
                            respectively. Circle sizes are proportional to the
                            number of cells in each cell group and edge width
                            represents the communication probability. Note that
                            CellChat predicted no significant
                            <i>CCL19-CCR7</i> signaling in NL skin. FIB:
                            fibroblasts; Inflam.FIB: inflammatory fibroblasts;
                            cDC: conventional dendritic cell; Inflam.DC:
                            inflammatory dendritic cell; LC: Langerhans cell;
                            TC: T cell; Inflam.TC: inflammatory T cell; NKT:
                            natural killer T cell.
                          </p>
                        </div>
                      </div>
                      <div class="u-text-right u-hide-print">
                        <a
                          class="c-article__pill-button"
                          data-test="article-link"
                          data-track="click"
                          data-track-label="button"
                          data-track-action="view figure"
                          href="https://www.nature.com/articles/s41467-021-21246-9/figures/7"
                          data-track-dest="link:Figure7 Full size image"
                          aria-label="Full size image figure 7"
                          rel="nofollow"
                          ><span>Full size image</span
                          ><svg
                            width="16"
                            height="16"
                            focusable="false"
                            role="img"
                            aria-hidden="true"
                            class="u-icon"
                          >
                            <use
                              xlink="http://www.w3.org/1999/xlink"
                              xlink:href="#global-icon-chevron-right"
                            ></use></svg></a>
                      </div>
                    </figure>
                  </div>
                  <p>
                    We inferred intercellular communication networks for the
                    nonlesional (NL) and lesional (LS) skin separately, and then
                    jointly mapped them onto a shared two-dimensional manifold
                    and clustered them into groups based on their functional
                    similarity. We identified four pathway groups (Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="figure anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#Fig7"
                      >7b</a
                    >). Almost all the same signaling pathways from NL and LS
                    were grouped together such as VEGF, GAS, LIGHT, CD40, and
                    MIF, suggesting that these pathways are essential for both
                    nonlesional and lesional skin and likely do not critically
                    contribute to disease pathogenesis. By comparing the overall
                    communication probability between nonlesional and lesional
                    skin, we found that 11 out of 16 signaling pathways were
                    highly active in lesional skin, including 9 pathways
                    involved in inflammatory and immune response, such as CXCL,
                    LIGHT, GLAECTIN, COMPLEMENT, MIF, CSF, IL4, CCL, and TNF
                    (Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="figure anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#Fig7"
                      >7c</a
                    >). Four pathways were specifically active in lesional skin,
                    including known inflammatory signals CSF, IL4, CCL, and TNF,
                    suggesting that these pathways might critically contribute
                    to disease progression. Specific to CCL signaling, CellChat
                    identified ligand-receptor pair <i>CCL19</i>-<i>CCR7</i> as
                    the most significant signaling, contributing to the
                    communication from Inflam.FIB to Inflam.DC (Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="figure anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#Fig7"
                      >7d–f</a
                    >). This is in agreement with a reported experimental
                    finding<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 70"
                        title="He, H. et al. Single-cell transcriptome analysis of human skin identifies novel fibroblast subpopulation and enrichment of immune subsets in atopic dermatitis. J. Allergy Clin. Immunol. 145, 1615–1628 (2020)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR70"
                        id="ref-link-section-d38771043e1879"
                        >70</a
                      ></sup
                    >. Ligand <i>MIF</i> and its multi-subunit receptor
                    <i>CD74/CD44</i> were found to act as major signaling from
                    Inflam.FIB to Inflam.TC in lesional skin compared to
                    nonlesional skin (Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="figure anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#Fig7"
                      >7d</a
                    >
                    and Supplementary Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="supplementary material anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#MOESM1"
                      >9a–c</a
                    >). Ligand <i>CXCL12</i> and its receptor <i>CXCR4</i> were
                    also found to be highly active in lesional skin, in
                    particular, for the signaling from Inflam.FIB to cDC2 and
                    Inflam.DC (Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="figure anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#Fig7"
                      >7d</a
                    >
                    and Supplementary Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="supplementary material anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#MOESM1"
                      >9a–c</a
                    >). Together, CellChat’s joint analysis using an example of
                    human lesional and nonlesional skin enables the discovery of
                    major signaling changes that might drive disease
                    pathogenesis.
                  </p>
                  <h3 class="c-article__sub-heading" id="Sec10">
                    Comparison with other cell–cell communication inference
                    tools
                  </h3>
                  <p>
                    We compared CellChat with three other tools for inferring
                    intercellular communications—SingleCellSignalR<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 9"
                        title="Cabello-Aguilar, S. et al. SingleCellSignalR: inference of intercellular networks from single-cell transcriptomics. Nucleic Acids Res. pii, gkaa183 (2020)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR9"
                        id="ref-link-section-d38771043e1916"
                        >9</a
                      ></sup
                    >, iTALK<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 10"
                        title="Wang, Y. et al. iTALK: an R Package to Characterize and Illustrate Intercellular Communication. Preprint at bioRxiv 507871 (2019)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR10"
                        id="ref-link-section-d38771043e1920"
                        >10</a
                      ></sup
                    >, and CellPhoneDB<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 16"
                        title="Efremova, M., Vento-Tormo, M., Teichmann, S. A. &amp; Vento-Tormo, R. CellPhoneDB: inferring cell–cell communication from combined expression of multi-subunit ligand–receptor complexes. Nat. Protoc. 15, 1484–1506 (2020)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR16"
                        id="ref-link-section-d38771043e1924"
                        >16</a
                      ></sup
                    >
                    using the same four mouse skin datasets analyzed by CellChat
                    (see “Methods” section). Currently existing tools, such as
                    SingleCellSignalR and iTALK, typically use only one
                    ligand/one receptor gene pairs, largely neglecting the
                    effect of multiple receptors. We computed the percentage of
                    false positive interactions caused by the above fact. False
                    positive interactions are defined as the interactions with
                    multi-subunits that are partially identified by these tools
                    (see “Methods” section). We found that the average rate of
                    false positive interactions identified by SingleCellSignalR
                    and iTALK was 10.6% and 14.3%, respectively (Supplementary
                    Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="supplementary material anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#MOESM1"
                      >10</a
                    >), suggesting the importance of accurate representation of
                    known ligand-receptor interactions. Of note, failed
                    detection of interactions with multi-subunits might be also
                    caused by low expression of multi-subunits of the receptors
                    that are not captured using scRNA-seq.
                  </p>
                  <p>
                    We also compared the performance of CellChat with
                    CellPhoneDB, which considers multi-subunit ligand–receptor
                    complexes. We reasoned that any given method can be regarded
                    as more accurate if its predictions more significantly
                    overlap with the predictions of more than one other method.
                    We found that CellChat predictions had more overlapping
                    interactions with both SingleCellSignalR and iTALK
                    predictions across all four scRNA-seq datasets
                    (Supplementary Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="supplementary material anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#MOESM1"
                      >11a</a
                    >). CellChat and CellPhoneDB shared ~50% predicted
                    interactions (Supplementary Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="supplementary material anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#MOESM1"
                      >11a</a
                    >). To assess the sensitivity of inferred communications to
                    the input data, we used subsampling of 90, 80, or 70% of the
                    total number of cells in each dataset, and then computed the
                    true positive rate (TPR), false positive rate (FPR), and
                    accuracy (ACC) by comparing subsampled datasets with the
                    original dataset. CellChat produced a slightly higher TPR,
                    lower FPR and higher ACC in comparison with CellPhoneDB
                    (Supplementary Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="supplementary material anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#MOESM1"
                      >11b</a
                    >). Both CellChat and CellPhoneDB were relatively robust to
                    subsampling, which is likely because both methods infer
                    cell–cell communication based on cell clusters. Such
                    robustness in terms of subsampling is very useful when
                    analyzing the rapidly growing volume of scRNA-seq data.
                  </p>
                  <p>
                    Next, we compared cell–cell communication networks inferred
                    by CellChat, CellPhoneDB, iTALK, and SingleCellSignalR using
                    an example of four spatially colocalized cell populations in
                    E14.5 embryonic mouse skin (Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="figure anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#Fig4"
                      >4</a
                    >). We compared the inferred significant ligand-receptor
                    (L-R) pairs for any two cell subpopulations between CellChat
                    and other methods. Here we only retained the top 10% of L-R
                    pairs (the most significant) inferred by iTALK and
                    SingleCellSignalR to ensure the comparable number of L-R
                    pairs with that by CellChat. The average numbers of L-R
                    pairs between two cell subpopulations inferred by the above
                    four methods were 12, 37, 14, and 12, respectively
                    (Supplementary Table&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="supplementary material anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#MOESM1"
                      >1</a
                    >). We found that CellChat shared more L-R pairs with
                    CellPhoneDB than with iTALK, likely due to the fact that
                    both CellChat and CellPhoneDB consider multi-subunit
                    complexes and determine the significant L-R pairs using a
                    statistical approach. SingleCellSignalR shared very few L-R
                    pairs with the other three methods, suggesting a potentially
                    different logic for quantifying and ranking L-R
                    interactions. Moreover, the majority of shared L-R pairs
                    between CellChat and CellPhoneDB were independently ranked
                    as top pairs by CellPhoneDB (Supplementary Data&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="supplementary material anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#MOESM4"
                      >1</a
                    >). This result suggests that although CellChat infers fewer
                    L-R pairs than CellPhoneDB, it captures the strongest (and
                    likely the most significant) L-R interactions.
                  </p>
                  <p>
                    We also systematically evaluated different methods based on
                    the assumption that spatially adjacent cell types should
                    have stronger cell–cell communication than spatially distant
                    cells. We have studied cell–cell communication for four
                    spatially colocalized cell populations in E14.5 embryonic
                    mouse skin, including Placodes, pre-DC, DC1, and DC2
                    (Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="figure anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#Fig4"
                      >4</a
                    >). We now added seven cell types that are likely not
                    spatially adjacent to the above four cell populations–FIB
                    (fibroblasts), MELA (melanocytes), Spinous (spinous
                    epithelial cells), MYL (myeloid cells), Immune (other immune
                    cells), ENDO (endothelial cells) and Muscle. We then
                    computed the number of inferred interactions, as well as the
                    sum of interaction probabilities or scores between each cell
                    type and the four spatially colocalized cell populations. We
                    found that CellChat consistently captures stronger
                    interactions in spatially adjacent cells than distant cells
                    both in terms of the number of interactions and the
                    interaction probabilities (Supplementary Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="supplementary material anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#MOESM1"
                      >12a-b</a
                    >). CellPhoneDB also performed well at discriminating
                    spatially-adjacent from distant cells. iTALK failed to
                    capture stronger interactions in spatially adjacent cells as
                    compared to spatially distant cells for FIB, MELA, MYL, and
                    ENDO. SingleCellSignalR also failed for FIB and ENDO. By
                    considering all seven cell types together, we found that
                    both CellChat and CellPhoneDB can significantly distinguish
                    spatially adjacent from distant cells, whereas iTALK and
                    SingleCellSignalR predicted stronger interactions in
                    spatially adjacent cells than distant cells with no
                    statistically significant differences (Supplementary
                    Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="supplementary material anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#MOESM1"
                      >12c</a
                    >). Since CellPhoneDB infers more interactions than
                    CellChat, we tested whether the top interactions predicted
                    by CellPhoneDB can also distinguish spatially adjacent from
                    distant cells. For the top 10%, top 20% and top 30%
                    interactions predicted by CellPhoneDB, the difference
                    between spatially adjacent and distant cells was not as
                    significant as with CellChat (Supplementary Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="supplementary material anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#MOESM1"
                      >13a-b</a
                    >), suggesting that CellChat performed better at capturing
                    stronger interactions. Together, our analyses show that
                    although CellChat produces fewer interactions, it performs
                    well at predicting stronger interactions.
                  </p>
                  <p>
                    The unique characteristics and capabilities of CellChat and
                    its comparison with other relevant tools are summarized in
                    Supplementary Table&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="supplementary material anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#MOESM1"
                      >2</a
                    >. First, CellChatDB database incorporates not only
                    multi-subunit structure of ligand-receptor complexes but
                    also soluble and membrane-bound stimulatory and inhibitory
                    cofactors, leading to a more comprehensive database than
                    those used by other tools. We also quantitatively showed the
                    differences and the strengths of CellChatDB in comparison to
                    other existing analogous databases, including CellTalkDB<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 71"
                        title="Shao, X. et al. CellTalkDB: a manually curated database of ligand-receptor interactions in humans and mice. Brief Bioinform. 
                  https://doi.org/10.1093/bib/bbaa269
                  
                 (2020)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR71"
                        id="ref-link-section-d38771043e1977"
                        >71</a
                      ></sup
                    >, CellPhoneDB<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 16"
                        title="Efremova, M., Vento-Tormo, M., Teichmann, S. A. &amp; Vento-Tormo, R. CellPhoneDB: inferring cell–cell communication from combined expression of multi-subunit ligand–receptor complexes. Nat. Protoc. 15, 1484–1506 (2020)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR16"
                        id="ref-link-section-d38771043e1981"
                        >16</a
                      ></sup
                    >, iTALK<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 10"
                        title="Wang, Y. et al. iTALK: an R Package to Characterize and Illustrate Intercellular Communication. Preprint at bioRxiv 507871 (2019)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR10"
                        id="ref-link-section-d38771043e1985"
                        >10</a
                      ></sup
                    >, SingleCellSignalR<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 9"
                        title="Cabello-Aguilar, S. et al. SingleCellSignalR: inference of intercellular networks from single-cell transcriptomics. Nucleic Acids Res. pii, gkaa183 (2020)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR9"
                        id="ref-link-section-d38771043e1989"
                        >9</a
                      ></sup
                    >, Ramilowski2015<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 72"
                        title="Ramilowski, J. A. et al. A draft network of ligand-receptor-mediated multicellular signalling in human. Nat. Commun. 6, 7866 (2015)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR72"
                        id="ref-link-section-d38771043e1994"
                        >72</a
                      ></sup
                    >, NicheNet<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 13"
                        title="Browaeys, R., Saelens, W. &amp; Saeys, Y. NicheNet: modeling intercellular communication by linking ligands to target genes. Nat. Methods 17, 159–162 (2020)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR13"
                        id="ref-link-section-d38771043e1998"
                        >13</a
                      ></sup
                    >, and ICELLNET<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 73"
                        title="Noël, F. et al. ICELLNET: a transcriptome-based framework to dissect intercellular communication. Preprint at bioRxiv 2020.2003.2005.976878 (2020)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR73"
                        id="ref-link-section-d38771043e2002"
                        >73</a
                      ></sup
                    >. Compared to the above databases, CellChatDB provides an
                    important resource for the community to study biologically
                    meaningful cell–cell communication (Supplementary
                    Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="supplementary material anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#MOESM1"
                      >1b</a
                    >
                    and Supplementary Note&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="supplementary material anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#MOESM1"
                      >1</a
                    >). Second, CellChat allows users to input a low-dimensional
                    representation of the data, a particularly useful function
                    when analyzing continuous states along pseudotime
                    trajectories. Third, CellChat can extract higher order
                    information from the inferred communications for
                    identification of major signaling sources, targets and
                    essential mediators, as well as the prediction of
                    coordinated responses among different cell types. Fourth,
                    CellChat can group signaling pathways based on similarity of
                    their communication patterns to identify signaling pathways
                    with similar architectures, and possibly functions. Finally,
                    CellChat can uncover conserved vs. context-specific
                    communication patterns through manifold learning of multiple
                    communication networks simultaneously.
                  </p>
                </div>
              </div>
            </section>
            <section data-title="Discussion">
              <div class="c-article-section" id="Sec11-section">
                <h2
                  class="
                    c-article-section__title
                    js-section-title js-c-reading-companion-sections-item
                  "
                  id="Sec11"
                >
                  Discussion
                </h2>
                <div class="c-article-section__content" id="Sec11-content">
                  <p>
                    In this work we report a database of signaling
                    ligand-receptor interactions that considers the multimeric
                    structure of ligand-receptor complexes and additional
                    effects on the core interaction by soluble and
                    membrane-bound stimulatory and inhibitory cofactors. The
                    ligand-receptor pairs are also classified into functionally
                    related signaling pathways via systematic manual curation
                    based on peer-reviewed literature. Comprehensive
                    recapitulation of known molecular interactions is essential
                    for developing biologically meaningful understanding of
                    intercellular communications from scRNA-seq data. For
                    example, signaling via BMP, IL, Interferon, TGFβ pathways
                    requires the presence of more than one membrane-bound
                    receptor subunits. Further, many pathways, such as BMP and
                    WNT, are prominently modulated by their cofactors, both
                    positively and negatively. To our knowledge, CellChatDB is
                    the first manually curated signaling interaction database in
                    mouse that considers multimeric structure. Although users
                    can map human genes to their mouse orthologues using
                    available tools such as biomaRt<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 74"
                        title="Durinck, S., Spellman, P. T., Birney, E. &amp; Huber, W. Mapping identifiers for the integration of genomic datasets with the R/Bioconductor package biomaRt. Nat. Protoc. 4, 1184–1191 (2009)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR74"
                        id="ref-link-section-d38771043e2021"
                        >74</a
                      ></sup
                    >, some molecular interactions are found in mouse but not in
                    human and vice-versa and these are typically lost during
                    such mapping. CellChatDB additionally provides the signaling
                    interactions in human by first automatically mapping to
                    human orthologues and then manually adding the interactions
                    specific to human.
                  </p>
                  <p>
                    Integration of all known molecular interactions when
                    studying intercellular communication requires new modeling
                    frameworks. To this end, we derived a mass action-based
                    model for quantifying the communication probability between
                    a given ligand and its cognate receptor. We modeled the
                    signaling communication probability between two cell groups
                    by considering the proportion of cells in each group across
                    all sequenced cells. This is important because abundant cell
                    populations tend to send collectively stronger signals than
                    the rare cell populations. With the increasing number of
                    datasets on unsorted single-cell transcriptomes in the Human
                    Cell Atlas, tools with such consideration will be
                    potentially in high demand. For the users who are interested
                    in analyzing sorting-enriched single cells, we provide an
                    option of removing the potential artifact of population size
                    when inferring cell–cell communication. In addition,
                    CellChat estimates the level of ligands by the geometric
                    mean of the subunits. Due to the low amounts of mRNA in
                    individual cells, dropout events often occur in scRNA-seq
                    data<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 75"
                        title="Kharchenko, P. V., Silberstein, L. &amp; Scadden, D. T. Bayesian approach to single-cell differential expression analysis. Nat. Methods 11, 740–742 (2014)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR75"
                        id="ref-link-section-d38771043e2028"
                        >75</a
                      ></sup
                    >, leading to possible zero expression of subunits. However,
                    dropouts are unlikely to affect strong signals predicted by
                    CellChat because dropouts commonly happen for genes with low
                    expression<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 75"
                        title="Kharchenko, P. V., Silberstein, L. &amp; Scadden, D. T. Bayesian approach to single-cell differential expression analysis. Nat. Methods 11, 740–742 (2014)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR75"
                        id="ref-link-section-d38771043e2032"
                        >75</a
                      >,<a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 76"
                        title="Zhang, L. &amp; Zhang, S. Comparison of computational methods for imputing single-cell RNA-sequencing data. IEEE/ACM Trans. Comput Biol. Bioinform. 17, 376–389 (2020)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR76"
                        id="ref-link-section-d38771043e2035"
                        >76</a
                      ></sup
                    >.
                  </p>
                  <p>
                    CellChat R package is a versatile and easy-to-use toolkit
                    for inferring, analyzing and visualizing cell–cell
                    communication from any given scRNA-seq data. It provides
                    several graphical outputs to facilitate different
                    post-analysis tasks. Of particular note is our customized
                    hierarchical plot that provides an intuitive way to
                    visualize oftentimes complex details of signaling by a given
                    pathway, including: (i) clear view of source and target cell
                    populations, (ii) easy-to-identified directionality and
                    probability of signaling, and (iii) paracrine vs. autocrine
                    signaling links. We demonstrated CellChat’s diverse
                    functionalities by applying it to finding continuous cell
                    lineage-associated signaling events, communications between
                    spatially colocalized cell populations, temporal changes in
                    time-course scRNA-seq data, and conserved and
                    context-specific communications between datasets from
                    distinct biological contexts.
                  </p>
                  <p>
                    A user-friendly web-based CellChat Explorer (<a
                      href="http://www.cellchat.org/"
                      >http://www.cellchat.org/</a
                    >) was also built, which contains two major components: (a)
                    Ligand-Receptor Interaction Explorer, which allows easy
                    exploration of our ligand-receptor interaction database
                    CellChatDB, and (b) Cell–Cell Communication Atlas Explorer,
                    which allows easy exploration of the cell–cell
                    communication. For any given scRNA-seq dataset that has been
                    processed by our CellChat R-package, we can host the
                    predicted results on our server, allowing easy exploration
                    and comparison of cell–cell communication. While at present
                    the Cell–Cell Communication Atlas only hosts the skin
                    scRNA-seq datasets analyzed in this study, we envision its
                    rapid growth to become a community-driven web portal for
                    cell–cell communication in a broad range of tissues at
                    single-cell resolution.
                  </p>
                  <p>
                    The successful performance of CellChat lies in utilizing a
                    mass action-based model to integrate all known molecular
                    interactions, including the core interaction between ligands
                    and receptors with multi-subunit structure, and additional
                    modulation by cofactors. While ligand-receptor interactions
                    and law-of-mass-action happen at the protein level, mRNA
                    levels are commonly used to approximate the protein level. A
                    higher level of molecular details (e.g., protein levels in
                    individual cells) could further improve the modeling
                    accuracy of CellChat and related tools. Due to the technical
                    difficulties of capturing single-cell proteomic information
                    at present time, a comprehensive modeling of ligand-receptor
                    interactions remains challenging. Determining a set of
                    biologically meaningful parameters in the mass action model
                    remains challenging, particularly when considering that
                    different pairs of ligands and receptors often have
                    different dissociation constants (i.e., the parameter
                    <i>Kh</i> in Hill function) and different degree of
                    cooperativity (i.e., the parameter <i>n</i> in Hill
                    function). Although these parameters lack explicit
                    biological connections in our current model, the Hill
                    function can be considered as a nonlinear approximation of
                    the ligand-receptor interactions. By computing the Jaccard
                    similarity between the interactions inferred using different
                    choices of the parameters <i>Kh</i> and <i>n</i>, we noticed
                    that the inferred ligand-receptor interactions by CellChat
                    are relatively robust to those parameters within certain
                    ranges for all four tested datasets (Supplementary
                    Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="supplementary material anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#MOESM1"
                      >14</a
                    >).
                  </p>
                  <p>
                    CellChat communication pattern analysis can uncover
                    coordinated responses among different cell types. Different
                    cell types may simultaneously activate the&nbsp;same cell
                    type-independent signaling patterns or different cell
                    type-specific signaling patterns. Different numbers of
                    patterns provide different resolutions when recovering
                    coordinated responses (Supplementary Note&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="supplementary material anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#MOESM1"
                      >2</a
                    >; Supplementary Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="supplementary material anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#MOESM1"
                      >15</a
                    >). This analysis can potentially help to derive general
                    cell–cell communication principles.
                  </p>
                  <p>
                    Cell clustering is a pre-requisite for cell–cell
                    communication analysis with CellChat and other tools, such
                    as CellPhoneDB, iTALK and SingleCellSignalR. While different
                    number of cell clusters may naturally affect the inferred
                    ligand-receptor interactions, with a fixed cluster number
                    the clustering results using different methods or parameters
                    will unlikely have major impact on the inferred
                    ligand-receptor interactions. This is because our cell–cell
                    communication is inferred at the cluster level, only
                    depending on estimation of the average gene expression in
                    each cell cluster. We demonstrated these two points using an
                    example of E14.5 mouse embryonic skin dataset with four
                    spatially colocalized cell subpopulations (Supplementary
                    Note&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="supplementary material anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#MOESM1"
                      >2</a
                    >; Supplementary Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="supplementary material anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#MOESM1"
                      >16</a
                    >). In general, cell clustering needs to be carried out
                    carefully in order to capture biologically meaningful cell
                    populations before cell–cell communication analysis.
                  </p>
                  <p>
                    The number of inferred ligand-receptor pairs clearly depends
                    on the method for calculating the average gene expression
                    per cell group. Here we systematically explored the inferred
                    ligand-receptor pairs using different methods by calculating
                    the average gene expression per cell group, including mean
                    (i.e., simply calculating the average gene expression), 5%
                    truncated mean (i.e., calculating the average gene
                    expression by discarding 5% from each end of the data), 10%
                    truncated mean, trimean (i.e., the method used in CellChat)
                    and median. For the four studied datasets, there are about
                    15% more dropped ligand-receptor pairs when calculating the
                    average gene expression using trimean compared to the 10%
                    truncated mean (Supplementary Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="supplementary material anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#MOESM1"
                      >17</a
                    >). Compared to other cell–cell communication tools, such as
                    CellPhoneDB, which uses a 10% truncated mean, CellChat
                    produces fewer ligand-receptor interactions. However, as
                    seen in our comparison study on spatially adjacent
                    subpopulations (Supplementary Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="supplementary material anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#MOESM1"
                      >13a, b</a
                    >), CellChat performs well at predicting stronger
                    interactions.
                  </p>
                  <p>
                    Although we found CellChat’s predictions can recapitulate
                    known biology to a substantial degree, systematic evaluation
                    of predicted cell–cell communication networks is challenging
                    due to the lack of ground truth<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 7"
                        title="Armingol, E., Officer, A., Harismendy, O. &amp; Lewis, N. E. Deciphering cell-cell interactions and communication from gene expression. Nat. Rev. Genet. 22, 71–88 (2021)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR7"
                        id="ref-link-section-d38771043e2101"
                        >7</a
                      ></sup
                    >. Here we employed three strategies to compare the
                    performance of different computational methods. First, we
                    reason that a more accurate method will have a larger
                    proportion of overlapped predictions with other methods.
                    However, such assumption has the following two limitations:
                    (1) Similar methods tend to generate similar results
                    regardless of accuracy; and (2) Different ligand-receptor
                    databases used in each method could contribute to the
                    variety of predicted interactions. Second, we
                    comprehensively compared the inferred interactions for any
                    two cell subpopulations on a specific dataset. We found that
                    the shared interactions between CellChat and other methods
                    were independently ranked as top pairs by other methods
                    including CellPhoneDB. Third, we reason that spatially
                    adjacent cell types should have stronger cell–cell
                    communication than spatially distant cells. CellChat
                    performs better in distinguishing spatially adjacent from
                    distant cells both in terms of the number of interactions
                    and the interaction strengths. Together, our analyses show
                    that although CellChat produces fewer interactions than
                    other methods, it performs well at predicting stronger
                    interactions, which is helpful for narrowing down on
                    interactions for further experimental validations. Other
                    types of single-cell data such as proteomics<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 77"
                        title="Marx, V. A dream of single-cell proteomics. Nat. Methods 16, 809–812 (2019)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR77"
                        id="ref-link-section-d38771043e2105"
                        >77</a
                      ></sup
                    >
                    and spatial transcriptomics<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 78"
                        title="Burgess, D. J. Spatial transcriptomics coming of age. Nat. Rev. Genet. 20, 317–317 (2019)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR78"
                        id="ref-link-section-d38771043e2109"
                        >78</a
                      ></sup
                    >
                    when available are also helpful and important to benchmark
                    and optimize these cell–cell communication methods in future
                    studies.
                  </p>
                  <p>
                    Recent advances in spatially resolved transcriptomic
                    techniques offer an opportunity to explore spatial
                    organization of cells in tissues<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 78"
                        title="Burgess, D. J. Spatial transcriptomics coming of age. Nat. Rev. Genet. 20, 317–317 (2019)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR78"
                        id="ref-link-section-d38771043e2116"
                        >78</a
                      ></sup
                    >. The integration of spatial information with scRNA-seq
                    data will likely offer new insights into cellular
                    crosstalk<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 79"
                        title="Dries, R. et al. Giotto, a pipeline for integrative analysis and visualization of single-cell spatial transcriptomic data. Preprint at bioRxiv 701680 (2019)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR79"
                        id="ref-link-section-d38771043e2120"
                        >79</a
                      >,<a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 80"
                        title="Cang, Z. &amp; Nie, Q. Inferring spatial and signaling relationships between cells from single cell transcriptomic data. Nat. Commun. 11, 2084 (2020)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR80"
                        id="ref-link-section-d38771043e2123"
                        >80</a
                      ></sup
                    >. The present version of CellChat provides an easy-to-use
                    tool for intercellular communication analysis on
                    conventional, non-spatially resolved scRNA-seq data. While
                    it remains to be tested, we believe it can be relatively
                    easily adjusted, such as via introduction of spatial
                    constrains on cell–cell signaling, to build intercellular
                    communication networks on spatially resolved transcriptomic
                    datasets. As single-cell multi-omics data is becoming more
                    common<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 81"
                        title="Stuart, T. &amp; Satija, R. Integrative single-cell analysis. Nat. Rev. Genet. 20, 257–272 (2019)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR81"
                        id="ref-link-section-d38771043e2127"
                        >81</a
                      >,<a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 82"
                        title="Jin, S., Zhang, L. &amp; Nie, Q. scAI: an unsupervised approach for the integrative analysis of parallel single-cell transcriptomic and epigenomic profiles. Genome Biol. 21, 25 (2020)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR82"
                        id="ref-link-section-d38771043e2130"
                        >82</a
                      ></sup
                    >, we anticipate that methods like CellChat, which are able
                    to perform system-level analyses, will serve as useful
                    hypothesis-generating tools whose predictive power will
                    extend beyond the ability to classify cell populations and
                    establish their lineage relationships, which currently
                    dominate single-cell genomics studies.
                  </p>
                </div>
              </div>
            </section>
            <section data-title="Methods">
              <div class="c-article-section" id="Sec12-section">
                <h2
                  class="
                    c-article-section__title
                    js-section-title js-c-reading-companion-sections-item
                  "
                  id="Sec12"
                >
                  Methods
                </h2>
                <div class="c-article-section__content" id="Sec12-content">
                  <h3 class="c-article__sub-heading" id="Sec13">
                    Database construction for ligand-receptor interactions
                  </h3>
                  <p>
                    To construct a database of ligand-receptor interactions that
                    comprehensively represents the current state of knowledge,
                    we manually reviewed other publicly available signaling
                    pathway databases, as well as peer-reviewed literature and
                    developed CellChatDB. CellChatDB is a database of
                    literature-supported ligand-receptor interactions in both
                    mouse and human. The majority of ligand–receptor
                    interactions in CellChatDB were manually curated on the
                    basis of KEGG (Kyoto Encyclopedia of Genes and Genomes)
                    signaling pathway database (<a
                      href="https://www.genome.jp/kegg/pathway.html"
                      >https://www.genome.jp/kegg/pathway.html</a
                    >). Additional signaling molecular interactions were
                    gathered from recent peer-reviewed experimental studies. We
                    took into account not only the structural composition of
                    ligand-receptor interactions, that often involve multimeric
                    receptors, but also cofactor molecules, including soluble
                    agonists and antagonists, as well as co-stimulatory and
                    co-inhibitory membrane-bound receptors that can prominently
                    modulate ligand-receptor mediated signaling events. The
                    detailed steps for how CellChatDB was built and how to
                    update CellChatDB by adding user-defined ligand-receptor
                    pairs were provided in Supplementary Note&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="supplementary material anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#MOESM1"
                      >1</a
                    >. To further analyze cell–cell communication in a more
                    biologically meaningful way, we grouped all of the
                    interactions into 229 signaling pathway families, such as
                    WNT, ncWNT, TGFβ, BMP, Nodal, Activin, EGF, NRG, TGFα, FGF,
                    PDGF, VEGF, IGF, chemokine and cytokine signaling pathways
                    (CCL, CXCL, CX3C, XC, IL, IFN), Notch and TNF. The
                    supportive evidences for each signaling interaction is
                    included within the database.
                  </p>
                  <h3 class="c-article__sub-heading" id="Sec14">
                    Inference of intercellular communications
                  </h3>
                  <p>
                    a) Identification of differentially expressed signaling
                    genes. To infer the cell state-specific communications, we
                    first identified differentially expressed signaling genes
                    across all cell groups within a given scRNA-seq dataset,
                    using the Wilcoxon rank sum test with the significance level
                    of 0.05.
                  </p>
                  <p>
                    b) Calculation of ensemble average expression. To account
                    for the noise effects, we calculated the ensemble average
                    expression of signaling genes in a given cell group using a
                    statistically robust mean method:
                  </p>
                  <div id="Equ1" class="c-article-equation">
                    <div class="c-article-equation__content">
                      <span class="mathjax-tex"
                        >$${{\mathrm{EM}} = \frac{1}{2}Q_2 + \frac{1}{4}\left(
                        {Q_1 + Q_3} \right)}$$</span
                      >
                    </div>
                    <div class="c-article-equation__number">(1)</div>
                  </div>
                  <p>
                    where <i>Q</i><sub>1</sub>, <i>Q</i><sub>2</sub>, and
                    <i>Q</i><sub>3</sub> is the first, second and third quartile
                    of the expression levels of a signaling gene in a cell
                    group.
                  </p>
                  <p>
                    c) Calculation of intercellular communication probability.
                    We modeled ligand-receptor mediated signaling interactions
                    using the law of mass action. Since the physical process of
                    ligand-receptor binding involves protein-protein
                    interactions, we used a random walk based network
                    propagation technique<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 83"
                        title="Cowen, L., Ideker, T., Raphael, B. J. &amp; Sharan, R. Network propagation: a universal amplifier of genetic associations. Nat. Rev. Genet. 18, 551–562 (2017)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR83"
                        id="ref-link-section-d38771043e2280"
                        >83</a
                      >,<a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 84"
                        title="Ronen, J. &amp; Akalin, A. netSmooth: Network-smoothing based imputation for single cell RNA-seq. F1000Res 7, 8–8 (2018)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR84"
                        id="ref-link-section-d38771043e2283"
                        >84</a
                      ></sup
                    >
                    to project the gene expression profiles onto a
                    high-confidence experimentally validated protein-protein
                    network from STRINGdb<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 83"
                        title="Cowen, L., Ideker, T., Raphael, B. J. &amp; Sharan, R. Network propagation: a universal amplifier of genetic associations. Nat. Rev. Genet. 18, 551–562 (2017)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR83"
                        id="ref-link-section-d38771043e2287"
                        >83</a
                      >,<a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 85"
                        title="Szklarczyk, D. et al. STRING v11: protein-protein association networks with increased coverage, supporting functional discovery in genome-wide experimental datasets. Nucleic Acids Res. 47, D607–D613 (2019)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR85"
                        id="ref-link-section-d38771043e2290"
                        >85</a
                      ></sup
                    >. Based on the projected ligand and receptor profiles, the
                    communication probability <i>P</i><sub>i,j</sub> from cell
                    groups <i>i</i> to <i>j</i> for a particular ligand-receptor
                    pair <i>k</i> was modeled by:
                  </p>
                  <div id="Equ2" class="c-article-equation">
                    <div class="c-article-equation__content">
                      <span class="mathjax-tex"
                        >$$P_{i,j}^k = \frac{{L_iR_j}}{{K_h + L_iR_j}} \times
                        \left( {1 + \frac{{AG_i}}{{K_h + AG_i}}} \right) \cdot
                        \left( {1 + \frac{{AG_j}}{{K_h + AG_j}}} \right) \\
                        \times \frac{{K_h}}{{K_h + AN_i}} \cdot
                        \frac{{K_h}}{{K_h + AN_j}} \times
                        \frac{{n_in_j}}{{n^2}}, \\ {L_i = \root {{m1}} \of
                        {{L_{i,1} \cdots L_{i,m1}}},\,R_j = \root {{m2}} \of
                        {{R_{j,1} \cdots R_{j,m2}}} \cdot \frac{{1 + RA_j}}{{1 +
                        RI_j}}.}$$</span
                      >
                    </div>
                    <div class="c-article-equation__number">(2)</div>
                  </div>
                  <p>
                    Here <i>L</i><sub><i>i</i></sub> and <i>R</i
                    ><sub><i>j</i></sub> represent the expression level of
                    ligand <i>L</i> and receptor <i>R</i> in cell group
                    <i>i</i> and cell group <i>j</i>, respectively. The
                    expression level of ligand <i>L</i> with <i>m1</i> subunits
                    (i.e.,
                    <span class="mathjax-tex"
                      >\(L_{i,1}, \cdots ,L_{i,m1}\)</span
                    >) was approximated by their geometric mean, implying that
                    the zero expression of any subunit leads to an inactive
                    ligand. Similarly, we computed the expression level of
                    receptor <i>R</i> with <i>m2</i> subunits. In addition,
                    co-stimulatory and co-inhibitory membrane-bound receptors
                    are capable of modulating signaling via the control of
                    receptor activation<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 86"
                        title="Chang, C. Agonists and antagonists of TGF-beta family ligands. Cold Spring Harb. Perspect. Biol. 8, a021923 (2016)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR86"
                        id="ref-link-section-d38771043e3006"
                        >86</a
                      ></sup
                    >. For the ligand-receptor pair with multiple co-stimulatory
                    receptors, we computed the average expression of these
                    co-stimulatory receptors (denoted by RA) and then used a
                    linear function to model the positive modulation of the
                    receptor expression. For each ligand-receptor pair with
                    multiple co-inhibitory receptors, we modeled them using the
                    same approach. A Hill function was used to model the
                    interactions between <i>L</i> and <i>R</i> with a parameter
                    <i>K</i><sub><i>h</i></sub> whose default value was set to
                    be 0.5 as the input data has a normalized range from 0 to 1.
                    The extracellular agonists and antagonists from both sender
                    and receiver cells are able to directly or indirectly
                    modulate the ligand-receptor interaction<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 86"
                        title="Chang, C. Agonists and antagonists of TGF-beta family ligands. Cold Spring Harb. Perspect. Biol. 8, a021923 (2016)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR86"
                        id="ref-link-section-d38771043e3023"
                        >86</a
                      ></sup
                    >. For the ligand-receptor pair with multiple soluble
                    agonists, we computed the average expression of these
                    agonists (denoted by AG) and then used a Hill function to
                    model the positive modulation of the ligand-receptor
                    interaction. For the ligand-receptor pair with multiple
                    soluble antagonists, we modeled them using the same
                    approach. The effect of cell proportion in each cell group
                    was also included in the probability calculation when
                    analyzing unsorted single-cell transcriptomes, where <i>n</i
                    ><sub><i>i</i></sub> a<i>n</i>d <i>n</i
                    ><sub><i>j</i></sub> are the numbers of cells in cell groups
                    <i>i</i> and <i>j</i>, respectively, a<i>n</i>d <i>n</i> is
                    the total number of cells in a given dataset. Together, the
                    communication probabilities among all pairs of cell groups
                    across all pairs of ligand-receptor were represented by a
                    three-dimensional array
                    <b>P</b> (<i>K</i> × <i>K</i> × <i>N</i>), where <i>K</i> is
                    the number of cell groups and <i>N</i> is the number of
                    ligand-receptor pairs or signaling pathways. The
                    communication probability of a signaling pathway was
                    computed by summarizing the probabilities of its associated
                    ligand-receptor pairs. It should be noted that we did not
                    perform normalization along the second dimension of
                    <b>P</b> such that
                    <span class="mathjax-tex"
                      >\(\mathop {\sum }\limits_j P_{i,j}^k = 1\)</span
                    >&nbsp;because the normalized data are not suitable for
                    comparing the communication probability between different
                    cell groups&nbsp;across multiple signaling pathways. The
                    communication probability here only represents the
                    interaction strength and is not exactly a probability.
                  </p>
                  <p>
                    d) Identification of statistically significant intercellular
                    communications. The significant interactions between two
                    cell groups are identified using a permutation test by
                    randomly permuting the group labels of cells, and then
                    recalculating the communication probability <i>P</i
                    ><sub><i>i,j</i></sub> between cell group <i>i</i> and cell
                    group <i>j</i> through a pair of ligand <i>L</i> and
                    receptor <i>R</i>. The <i>p</i>-value of each <i>P</i
                    ><sub>i,j</sub> is computed by:
                  </p>
                  <div id="Equ3" class="c-article-equation">
                    <div class="c-article-equation__content">
                      <span class="mathjax-tex"
                        >$${p{\hbox{-}}value = \frac{{\left\{ {\#
                        m{\mathrm{|}}P_{i,j}^{(m)} \le P_{i,j},m = 1,2, \cdots
                        ,M} \right\}}}{M}}$$</span
                      >
                    </div>
                    <div class="c-article-equation__number">(3)</div>
                  </div>
                  <p>
                    where the probability <i>P</i><sub>i,j</sub
                    ><sup><i>(m)</i></sup> is the communication probability for
                    the <i>m</i>-th permutation. <i>M</i>&nbsp;is the total
                    number of permutations (<i>M</i> = 100 by default). The
                    interactions with <i>p</i>-value &lt;0.05 are considered
                    significant.
                  </p>
                  <h3 class="c-article__sub-heading" id="Sec15">
                    Discovery of dominant senders, receivers, mediators, and
                    influencers in the intercellular communication networks
                  </h3>
                  <p>
                    To allow ready identification of major signaling sources,
                    targets, essential mediators and key influencers, as well as
                    other high-order information in intercellular
                    communications, the centrality metrics from graph theory,
                    previously used for social network analysis, were
                    adopted<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 17"
                        title="Mário Cordeiro, R. P. S. Pavel Brazdil and João Gama in Social Media and Journalism-Trends, Connections, Implications (Intech Open, London, UK; 2018)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR17"
                        id="ref-link-section-d38771043e3321"
                        >17</a
                      ></sup
                    >. Specifically, we used measures in weighted-directed
                    networks, including out-degree, in-degree, flow betweenesss
                    and information centrality, to respectively identify
                    dominant senders, receivers, mediators and influencers for
                    the intercellular communications. In a weighted-directed
                    network with the weights as the computed communication
                    probabilities, the out-degree, computed as the sum of
                    communication probabilities of the outgoing signaling from a
                    cell group, and the in-degree, computed as the sum of the
                    communication probabilities of the incoming signaling to a
                    cell group, can be used to identify the dominant cell
                    senders and receivers of signaling networks, respectively.
                    Flow betweenness score<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 87"
                        title="Butts, C. T. Social network analysis with sna. J. Stat. Softw. 1, 1–51 (2008)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR87"
                        id="ref-link-section-d38771043e3325"
                        >87</a
                      ></sup
                    >
                    measures a group of cells’ capability as gatekeeper to
                    control communication flow between any two cell groups.
                    Information centrality score provides a hybrid measure, for
                    example by combining closeness and ﻿eigenvector, for
                    information flow within a signaling network, and a higher
                    value indicates greater control on the information flow<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 87"
                        title="Butts, C. T. Social network analysis with sna. J. Stat. Softw. 1, 1–51 (2008)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR87"
                        id="ref-link-section-d38771043e3329"
                        >87</a
                      ></sup
                    >. Other popular centrality metrics, such as hub, authority,
                    EigenCentrality&nbsp;and PageRank<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 88"
                        title="Landherr, A., Friedl, B. &amp; Heidemann, J. A critical review of centrality measures in social networks. Bus. Inf. Syst. Eng. 2, 371–385 (2010)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR88"
                        id="ref-link-section-d38771043e3333"
                        >88</a
                      ></sup
                    >, can be also used to identify highly influential cell
                    groups in the intercellular communications. The flow
                    betweenness and information centrality are calculated by the
                    package sna<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 87"
                        title="Butts, C. T. Social network analysis with sna. J. Stat. Softw. 1, 1–51 (2008)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR87"
                        id="ref-link-section-d38771043e3337"
                        >87</a
                      ></sup
                    >. Other measures are computed by the package igraph (<a
                      href="https://igraph.org/"
                      >https://igraph.org/</a
                    >).
                  </p>
                  <h3 class="c-article__sub-heading" id="Sec16">
                    Identification of major signals for specific cell groups and
                    global communication patterns
                  </h3>
                  <p>
                    To identify key signals and latent communication patterns
                    among all signaling pathways, CellChat uses an unsupervised
                    learning method non-negative matrix factorization that has
                    been successfully applied in pattern recognition<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 18"
                        title="Lee, D. D. &amp; Seung, H. S. Learning the parts of objects by non-negative matrix factorization. Nature 401, 788–791 (1999)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR18"
                        id="ref-link-section-d38771043e3357"
                        >18</a
                      >,<a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 19"
                        title="Zhang, L. &amp; Zhang, S. A general joint matrix factorization framework for data integration and its systematic algorithmic exploration. IEEE T Fuzzy Syst. 28, 1971–1983 (2020)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR19"
                        id="ref-link-section-d38771043e3360"
                        >19</a
                      >,<a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 82"
                        title="Jin, S., Zhang, L. &amp; Nie, Q. scAI: an unsupervised approach for the integrative analysis of parallel single-cell transcriptomic and epigenomic profiles. Genome Biol. 21, 25 (2020)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR82"
                        id="ref-link-section-d38771043e3363"
                        >82</a
                      >,<a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 89"
                        title="Zhang, L. &amp; Zhang, S. Learning common and specific patterns from data of multiple interrelated biological scenarios with matrix factorization. Nucleic Acids Res. 47, 6606–6617 (2019)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR89"
                        id="ref-link-section-d38771043e3366"
                        >89</a
                      ></sup
                    >. First, the latent patterns were found for sending cells
                    by summarizing the communication probability array
                    <b>P</b> (three-dimensional) along the second dimension to
                    obtain a two-dimensional matrix <b>P</b><sub><b>j</b></sub
                    >. A non-negative matrix factorization was then carried out
                    via:
                  </p>
                  <div id="Equ4" class="c-article-equation">
                    <div class="c-article-equation__content">
                      <span class="mathjax-tex"
                        >$${{\mathrm{min}}_{W,H &gt; 0}\left\| {P_j - WH}
                        \right\|,}$$</span
                      >
                    </div>
                    <div class="c-article-equation__number">(4)</div>
                  </div>
                  <p>
                    where the two low-dimensional matrices <b>W</b> and
                    <b>H</b> are the cell loading and signaling loading matrices
                    with sizes <i>K</i> × <i>R</i> and <i>R</i> × <i>N</i>,
                    respectively. Each of the <i>R</i> columns in <b>W</b> and
                    the corresponding rows in <b>H</b> is considered as a
                    communication pattern. <i>W</i><sub><i>ir</i></sub> is the
                    loading values of cell group <i>i</i> in pattern <i>r</i>,
                    representing the contributions of cell group <i>i</i> in
                    pattern <i>r</i>. <i>H</i><sub><i>rk</i></sub> represents
                    the contributions of ligand-receptor pair or signaling
                    pathway <i>k</i> in pattern <i>r</i>. As the number of
                    patterns increases, there might be redundant patterns,
                    making it difficult to interpret the communication patterns.
                    We chose five patterns as the initial default because the
                    number of cell groups and significant signaling pathways are
                    relatively small. In addition, we inferred the number of
                    patterns based on two metrics that have been implemented in
                    the NMF R package, including Cophenetic and Silhouette<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 90"
                        title="Gaujoux, R. &amp; Seoighe, C. A flexible R package for nonnegative matrix factorization. BMC Bioinform. 11, 367 (2010)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR90"
                        id="ref-link-section-d38771043e3506"
                        >90</a
                      ></sup
                    >. Both metrics measure the stability for a particular
                    number of patterns based on a hierarchical clustering of the
                    consensus matrix. For a range of the number of patterns, a
                    suitable number of patterns is the one at which Cophenetic
                    and Silhouette values begin to drop suddenly.
                  </p>
                  <p>
                    In sum, the matrix <b>W</b> represents the <i>R</i> latent
                    patterns of cell groups, indicating how these cell groups
                    coordinate to send signals; the matrix <b>H</b> represents
                    the <i>R</i> latent patterns of ligand-receptor pairs or
                    signaling pathways, indicating how these ligand-receptor
                    pairs or signaling pathways work together to send signals;
                    the connection of <b>W</b> with <b>H</b> predicts the key
                    signals sent from certain cell groups. Similarly, we
                    summarized the communication probability array
                    <b>P</b> along the first dimension to infer the key signals
                    received by certain cell groups, as well as their latent
                    patterns. Together, outgoing patterns reveal how the sender
                    cells (i.e., cells as signal sources) coordinate with each
                    other, as well as how they coordinate with certain signaling
                    pathways to drive communication. Incoming patterns show how
                    the target cells (i.e., cells as signal receivers)
                    coordinate with each other, as well as how they coordinate
                    with certain signaling pathways to respond to incoming
                    signals.
                  </p>
                  <p>
                    To intuitively show the associations of latent patterns with
                    cell groups and ligand-receptor pairs or signaling pathways,
                    we used alluvial plots implemented in the ggalluvial package
                    (<a
                      href="https://cran.r-project.org/web/packages/ggalluvial/index.html"
                      >https://cran.r-project.org/web/packages/ggalluvial/index.html</a
                    >). We first normalized each row of <b>W</b> and each column
                    of <b>H</b> to be [0,1], and then set the elements in
                    <b>W</b> and <b>H</b> to be zero if they are less than 0.5.
                    Such thresholding allows to uncover the most enriched cell
                    groups and signaling pathways associated with each inferred
                    pattern, that is, each cell group or signaling pathway is
                    associated with only one inferred pattern. These thresholded
                    matrices <b>W</b> and <b>H</b> are used as inputs for
                    creating alluvial plots. To directly relate cell groups with
                    their enriched signaling pathways, we set the elements in
                    <b>W</b> and <b>H</b> to be zero if they are less than 1/<i
                      >R</i
                    >
                    where <i>R</i> is the number of latent patterns. By using a
                    less strict threshold, more enriched signaling pathways
                    associated each cell group might be obtained. Using a
                    contribution score of each cell group to each signaling
                    pathway computed by multiplying <b>W</b> by <b>H</b>, we
                    constructed a dot plot in which the dot size is proportion
                    to the contribution score to show association between cell
                    group and their enriched signaling pathways.
                  </p>
                  <h3 class="c-article__sub-heading" id="Sec17">
                    Quantification of similarity among intercellular
                    communication networks
                  </h3>
                  <p>
                    Two different similarity measures were used to quantify the
                    similarity among intercellular communication networks. A
                    functional similarity <i>S</i> was calculated based on the
                    overlap of communications via the Jaccard similarity defined
                    by:
                  </p>
                  <div id="Equ5" class="c-article-equation">
                    <div class="c-article-equation__content">
                      <span class="mathjax-tex"
                        >$${S = \frac{{E\left( G \right) \cap E\left(
                        {G^{\prime}} \right)}}{{E\left( G \right) \cup E\left(
                        {G^{\prime}} \right) - E\left( G \right) \cap E\left(
                        {G^{\prime}} \right)}},}$$</span
                      >
                    </div>
                    <div class="c-article-equation__number">(5)</div>
                  </div>
                  <p>
                    where <i>G</i> and <i>G’</i> are two signaling networks and
                    <i>E</i>(<i>G</i>) is the set of communications in signaling
                    network <i>G</i>. High degree of functional similarity
                    indicates major senders and receivers are similar, and it
                    can be interpreted as the two signaling pathways or two
                    ligand-receptor pairs exhibit similar and/or redundant
                    roles.
                  </p>
                  <p>
                    A structural similarity was used to compare their signaling
                    network structure, without considering the similarity of
                    senders and receivers, using a previously developed measure
                    for structural topological differences<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 91"
                        title="Schieber, T. A. et al. Quantification of network structural dissimilarities. Nat. Commun. 8, 13928 (2017)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR91"
                        id="ref-link-section-d38771043e3748"
                        >91</a
                      ></sup
                    >. The dissimilarity measure between signaling networks
                    <i>G</i> and <i>G’</i> with the number of cell groups being
                    <i>N</i> and <i>M</i>, respectively, is calculated by:
                  </p>
                  <div id="Equ6" class="c-article-equation">
                    <div class="c-article-equation__content">
                      <span class="mathjax-tex"
                        >$${D\left( {G,\,G{\prime}} \right) = w_1\sqrt
                        {{\mathrm{JSD}}(u_G,\,u_{G{\prime}})/{\mathrm{log}}2} +
                        w_2\left| {\sqrt {{\mathrm{NND}}\left( G \right)} -
                        \sqrt {{\mathrm{NND}}\left( {G{\prime}} \right)} }\,\,
                        \right|} \\ +\, {\frac{{w_3}}{2}\left( {\sqrt
                        {{\mathrm{JSD}}(P_{\alpha G},\,P_{\alpha
                        G{\prime}})/{\mathrm{log}}2} + \sqrt
                        {{\mathrm{JSD}}(P_{\alpha G^c},\,P_{\alpha
                        G^{c{\prime}}})/{\mathrm{log}}2} } \right)}$$</span
                      >
                    </div>
                    <div class="c-article-equation__number">(6)</div>
                  </div>
                  <p>
                    where <i>G</i><sup><i>c</i></sup> indicates the complement
                    of <i>G</i>, and JSD is the Jensen–Shannon divergence and
                    <i>NND</i> is defined as:
                  </p>
                  <div id="Equ7" class="c-article-equation">
                    <div class="c-article-equation__content">
                      <span class="mathjax-tex"
                        >$${{\mathrm{NND}}\left( G \right) =
                        \frac{{{\mathrm{JSD}}\left( {P_1, \ldots ,P_N}
                        \right)}}{{{\mathrm{log}}\left( {d + 1}
                        \right)}}}$$</span
                      >
                    </div>
                    <div class="c-article-equation__number">(7)</div>
                  </div>
                  <p>
                    with
                    <span class="mathjax-tex"
                      >\({\mathrm{JSD}}\left( {P_1, \ldots ,P_N} \right) =
                      \frac{1}{N}\mathop {\sum }\limits_{i,j}^N
                      p_i(j){\mathrm{log}}( {\frac{{p_i(j)}}{{u_j}}} )\)</span
                    >
                    and
                    <span class="mathjax-tex"
                      >\(u_j = ( {\mathop {\sum }\limits_{i = 1}^N p_i( j )}
                      )/N\)</span
                    >
                    being the Jensen–Shannon divergence and the average of the
                    <i>N</i> distributions, respectively.
                    <span class="mathjax-tex"
                      >\(P_i = \left\{ {p_i\left( j \right)} \right\}\)</span
                    >
                    is the distance distribution in each cell group <i>i</i>,
                    where <i>p</i><sub><i>i</i></sub
                    >(<i>j</i>) is the fraction of cell groups connected to cell
                    group <i>i</i> at distance <i>j</i>. <i>d</i> is the
                    signaling network’s diameter.
                    <span class="mathjax-tex"
                      >\({\mathrm{JSD}}(u_G,u_{G{\prime}})\)</span
                    >
                    measures the difference between the signaling networks’
                    averaged cell group-distance distributions, <i>u</i
                    ><sub><i>G</i></sub> and
                    <span class="mathjax-tex">\(u_{G{\prime}}\)</span>, and
                    <span class="mathjax-tex"
                      >\({\mathrm{JSD}}(P_{\alpha G},\,P_{\alpha
                      G{\prime}})\)</span
                    >
                    measures the difference between the <i>α</i>-centrality
                    values of the signaling networks. <i>w</i><sub>1</sub>,
                    <i>w</i><sub>2</sub>, and <i>w</i><sub>3</sub> are the
                    weights of each term with <i>w</i><sub>1</sub> + <i>w</i
                    ><sub>2</sub> + <i>w</i><sub>3</sub> = 1. Similar to a
                    previous study<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 91"
                        title="Schieber, T. A. et al. Quantification of network structural dissimilarities. Nat. Commun. 8, 13928 (2017)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR91"
                        id="ref-link-section-d38771043e4768"
                        >91</a
                      ></sup
                    >, we selected <i>w</i><sub>1</sub> = 0.45, <i>w</i
                    ><sub>2</sub> = 0.45, and <i>w</i><sub>3</sub> = 0.1. The
                    structural similarity <i>S</i> was computed by one minus
                    dissimilarity measure <i>D</i>.
                  </p>
                  <h3 class="c-article__sub-heading" id="Sec18">
                    Manifold and classification learning of intercellular
                    communication networks
                  </h3>
                  <p>
                    The manifold learning of the inferred intercellular
                    communication networks consists of three steps. First, we
                    built a shared nearest neighbor (SNN) similarity network
                    <i>Gs</i> of all signaling pathways, which was constructed
                    by calculating the <i>k</i>-nearest signaling pathways of
                    each signaling pathway using the calculated functional or
                    structural similarity matrix <i>S</i> of intercellular
                    communication networks. The fraction of shared nearest
                    signaling pathways between a given signaling pathway and its
                    neighbors was used as weights of the SNN network. The number
                    of nearest neighbors <i>k</i> was chosen as the square root
                    of the total number of signaling pathways. Second, we
                    smoothed the similarity matrix <i>S</i> using
                    <i>Gs</i> × <i>S</i>. This smooth process provides a better
                    representation of the similarity between signaling pathways
                    to allow filtering of the weak similarity (potentially
                    noise-induced) and enhancing the strong similarity<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 82"
                        title="Jin, S., Zhang, L. &amp; Nie, Q. scAI: an unsupervised approach for the integrative analysis of parallel single-cell transcriptomic and epigenomic profiles. Genome Biol. 21, 25 (2020)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR82"
                        id="ref-link-section-d38771043e4821"
                        >82</a
                      ></sup
                    >. Finally, we performed uniform manifold approximation and
                    projection (UMAP)<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 92"
                        title="Leland McInnes, JohnHealy &amp; Melville, J. UMAP: uniform manifold approximation and projection for dimension reduction. arXiv 1802, 03426 (2018)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR92"
                        id="ref-link-section-d38771043e4825"
                        >92</a
                      ></sup
                    >
                    on the smoothed similarity matrix. To better visualize the
                    similarity of intercellular communication networks, we used
                    the first two dimensions of the learned manifold, where each
                    dot in this two-dimensional space represents an individual
                    signaling pathway or ligand-receptor pair.
                  </p>
                  <p>
                    Moreover, to group the signaling pathways based on their
                    similarity of intercellular communication networks in an
                    unsupervised manner, we performed <i>k</i>-means clustering
                    of the first two components of the learned manifold. The
                    number of signaling groups was determined according to the
                    eigenvalue spectrum by analyzing the Laplacian matrix
                    derived from a consensus matrix<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 5"
                        title="Jin, S., MacLean, A. L., Peng, T. &amp; Nie, Q. scEpath: energy landscape-based inference of transition probabilities and cellular trajectories from single-cell transcriptomic data. Bioinformatics 34, 2077–2086 (2018)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR5"
                        id="ref-link-section-d38771043e4835"
                        >5</a
                      >,<a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 12"
                        title="Wang, S., Karikomi, M., MacLean, A. L. &amp; Nie, Q. Cell lineage and communication network inference via optimization for single-cell transcriptomics. Nucleic Acids Res. 47, e66 (2019)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR12"
                        id="ref-link-section-d38771043e4838"
                        >12</a
                      ></sup
                    >. First, we performed <i>k</i>-means clustering multiple
                    times for different values of <i>k</i> (e.g., 2–10). Second,
                    we constructed a consensus matrix representing the
                    probability of two signaling pathways being in the same
                    group across multiple values of <i>k</i>. We then pruned the
                    consensus matrix by setting the elements to be zero if they
                    are less than 0.3 to ensure better robustness to noise.
                    Third, we estimated the number of signaling groups by
                    computing the eigenvalues of the associated Laplacian matrix
                    of the constructed consensus matrix. More generally, the
                    number of signaling groups is usually determined by the
                    first or second largest eigenvalue gap (i.e., the difference
                    between consecutive eigenvalues) based on the spectral graph
                    theory<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 93"
                        title="Luxburg, Uv. A. Tutorial on spectral clustering. Stat. Comput. 17, 1–32 (2007)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR93"
                        id="ref-link-section-d38771043e4852"
                        >93</a
                      ></sup
                    >.
                  </p>
                  <h3 class="c-article__sub-heading" id="Sec19">
                    Classification of cells into groups
                  </h3>
                  <p>
                    CellChat provides built-in functions to classify cells into
                    groups. Briefly, a SNN graph of all cells is first
                    constructed via the calculation of the <i>k</i>-nearest
                    neighbors (20 by default) for each cell based on the
                    low-dimensional representation space (e.g., via principle
                    component analysis and diffusion map analysis) of the
                    scRNA-seq data. The low-dimensional representation space can
                    be either provided by user or computed by CellChat. Next,
                    cells are clustered into groups by applying the Louvain
                    community detection algorithm<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 94"
                        title="Blondel, V. D., Guillaume, J.-L., Lambiotte, R. &amp; Lefebvre, E. Fast unfolding of communities in large networks. J. Stat. Mech. 2008, P10008 (2008)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR94"
                        id="ref-link-section-d38771043e4867"
                        >94</a
                      ></sup
                    >
                    to the constructed SNN graph. The number of cell groups is
                    determined either by user-input resolution parameter in the
                    Louvain algorithm or by an eigenvalue spectrum by analyzing
                    the Laplacian matrix derived from multiple runs of Louvain
                    algorithm with different resolution parameters.
                  </p>
                  <h3 class="c-article__sub-heading" id="Sec20">
                    Single-cell RNA-seq datasets, data preprocessing, and
                    analysis
                  </h3>
                  <p>
                    Mouse skin wound dataset. We used our recently published
                    scRNA-seq dataset from mouse skin wounds<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 23"
                        title="Guerrero-Juarez, C. F. et al. Single-cell analysis reveals fibroblast heterogeneity and myeloid-derived adipocyte progenitors in murine skin wounds. Nat. Commun. 10, 650 (2019)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR23"
                        id="ref-link-section-d38771043e4879"
                        >23</a
                      ></sup
                    >. This dataset included 21,819 cells and was generated via
                    10X Genomics platform (GEO accession code: GSE113854).
                    Briefly, scRNA-seq was performed on unsorted cells isolated
                    from mouse skin wound dermis from day 12 post-wounding.
                    Unsupervised clustering identified fibroblasts (FIB, ~65%),
                    immune cell populations, including myeloid cells (MYL, 15%),
                    T lymphocytes (TCELL, 4%), B lymphocytes (BCELL, 3%),
                    dendritic cells (DC, 1%), endothelial cells (ENDO, 9%),
                    lymphatic endothelial cells (LYME, 1%), Schwann cells (SCH,
                    1%) and red blood cells (RBC, 1%). For the intercellular
                    communication analysis, we excluded red blood cells and used
                    the remaining 21,557 cells. The digital data matrices were
                    normalized by a global method, in which the expression value
                    of each gene was divided by the total expression in each
                    cell and multiplied by a scale factor (10,000 by default).
                    These values were then log-transformed with a pseudocount of
                    1. Normalized data were used for all the analyses. To
                    investigate the heterogeneity of intercellular
                    communications among different cell subpopulations, we
                    performed subclustering analysis on the cell types, whose
                    abundance in the dataset was greater than 5% using the
                    Louvain community detection method. The number of cell
                    groups was determined by the eigengap approach.
                  </p>
                  <p>
                    Embryonic mouse skin dataset. Recently published embryonic
                    mouse skin datasets<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 22"
                        title="Gupta, K. et al. Single-cell analysis reveals a hair follicle dermal niche molecular differentiation trajectory that begins prior to morphogenesis. Dev. Cell 48, 17–31 e16 (2019)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR22"
                        id="ref-link-section-d38771043e4886"
                        >22</a
                      ></sup
                    >
                    were downloaded from GEO (accession codes: GSM3453535,
                    GSM3453536, GSM3453537, and GSM3453538) and included two
                    Embryonic day E13.5 biological replicates and two Embryonic
                    day E14.5 biological replicates. These samples contain
                    unsorted whole-skin cells captured via 10X Genomics
                    platform. For both E13.5 and E14.5 scRNA-seq datasets, we
                    removed the cells with the amount of UMI count less than
                    2500 and greater than 50000, as well as the cells with the
                    number of genes less than 1000 and the fraction of
                    mitochondrial counts greater than 20%. 12,951 cells at E13.5
                    and 12,197 cells at E14.5 were used for downstream analyses.
                    First, we performed clustering analysis of cells from E13.5
                    and E14.5 using the Louvain community detection method,
                    respectively. The values of the resolution parameter in the
                    Louvain community detection method were explored to produce
                    the major cell populations in embryonic skin<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 22"
                        title="Gupta, K. et al. Single-cell analysis reveals a hair follicle dermal niche molecular differentiation trajectory that begins prior to morphogenesis. Dev. Cell 48, 17–31 e16 (2019)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR22"
                        id="ref-link-section-d38771043e4890"
                        >22</a
                      >,<a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 95"
                        title="Ge, W. et al. Single-cell transcriptome profiling reveals dermal and epithelium cell fate decisions during embryonic hair follicle development. Theranostics 10, 7581–7598 (2020)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR95"
                        id="ref-link-section-d38771043e4893"
                        >95</a
                      ></sup
                    >. Thus, 11 and 13 cell populations were identified at E13.5
                    and E14.5, respectively (Supplementary Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="supplementary material anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#MOESM1"
                      >3a–d</a
                    >). The cell populations were annotated based on the known
                    markers<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 22"
                        title="Gupta, K. et al. Single-cell analysis reveals a hair follicle dermal niche molecular differentiation trajectory that begins prior to morphogenesis. Dev. Cell 48, 17–31 e16 (2019)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR22"
                        id="ref-link-section-d38771043e4900"
                        >22</a
                      >,<a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 95"
                        title="Ge, W. et al. Single-cell transcriptome profiling reveals dermal and epithelium cell fate decisions during embryonic hair follicle development. Theranostics 10, 7581–7598 (2020)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR95"
                        id="ref-link-section-d38771043e4903"
                        >95</a
                      ></sup
                    >. Compared to E13.5, there were two specific populations at
                    E14.5, including dermal condensate (DC) cells and pericytes.
                    Second, we performed subclustering analysis of DC cells,
                    basal cells and melanocytes at E14.5, respectively. This
                    analysis identified three DC states including pre-DC, DC1,
                    and DC2, three basal state including basal, proliferative
                    basal and placode cells, and three melanocyte subpopulations
                    including MELA-A, MELA-B, and MELA-C (Supplementary
                    Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="supplementary material anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#MOESM1"
                      >3e-g</a
                    >). Third, we performed pseudotime analysis on epidermal and
                    dermal cells at E14.5 using diffusion map, respectively.
                  </p>
                  <p>
                    Human disease skin dataset. The processed transcriptomic
                    data of 17,349 cells from four lesional and four
                    non-lesional human skin samples (patient ID: S1, S2, S3, S5,
                    S7, S11, S14, and S15) from patients with atopic dermatitis
                    was downloaded from GEO database under accession code
                    GSE147424<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 70"
                        title="He, H. et al. Single-cell transcriptome analysis of human skin identifies novel fibroblast subpopulation and enrichment of immune subsets in atopic dermatitis. J. Allergy Clin. Immunol. 145, 1615–1628 (2020)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR70"
                        id="ref-link-section-d38771043e4913"
                        >70</a
                      ></sup
                    >. We performed the integration analysis of these eight
                    samples using Seurat V3 package based on the tutorial from
                    <a
                      href="https://satijalab.org/seurat/v3.2/immune_alignment.html"
                      >https://satijalab.org/seurat/v3.2/immune_alignment.html</a
                    >. Unsupervised clustering analysis segregated these
                    combined cells into 10 broad cell types (Supplementary
                    Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="supplementary material anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#MOESM1"
                      >8a-b</a
                    >), including fibroblasts (FIB), dendritic cells (DC), and T
                    cells (TC). The original study highlighted the cell–cell
                    communication among fibroblasts, dendritic cells and T
                    cells<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 70"
                        title="He, H. et al. Single-cell transcriptome analysis of human skin identifies novel fibroblast subpopulation and enrichment of immune subsets in atopic dermatitis. J. Allergy Clin. Immunol. 145, 1615–1628 (2020)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR70"
                        id="ref-link-section-d38771043e4927"
                        >70</a
                      ></sup
                    >. Therefore, following the analysis from the original
                    study<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 70"
                        title="He, H. et al. Single-cell transcriptome analysis of human skin identifies novel fibroblast subpopulation and enrichment of immune subsets in atopic dermatitis. J. Allergy Clin. Immunol. 145, 1615–1628 (2020)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR70"
                        id="ref-link-section-d38771043e4931"
                        >70</a
                      ></sup
                    >, we performed the second-level clustering analysis of FIB,
                    DC, and TC. FIB was clustered into five subgroups with
                    distinct markers, including <i>APOE</i> high FIB
                    (<i>APOE</i> + FIB), <i>FBN1</i> + FIB, <i>COL11A</i> + FIB,
                    Inflam.FIB (inflammatory FIB expressing chemokines such as
                    <i>CCL19</i>) and a small T cell group expressing
                    <i>CD3D</i> (Supplementary Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="supplementary material anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#MOESM1"
                      >8c</a
                    >). This contaminated TC population was removed for further
                    analysis. DC was also separated into five subgroups,
                    including cDC1 (type A DC), cDC2 (type B DC), LC (Langerhans
                    cells), Inflam.DC (inflammatory DC) and other immune cells
                    that does not express DC markers such as CD1A and CD1C
                    (Supplementary Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="supplementary material anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#MOESM1"
                      >8d</a
                    >). This contaminated immune cell group was also removed for
                    further analysis. TC was clustered into four subgroups,
                    including TC, Inflam.TC (inflammatory TC),
                    <i>CD40LG</i> + TC and NKT (NK T cells) (Supplementary
                    Fig.&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="supplementary material anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#MOESM1"
                      >8e</a
                    >). Together, CellChat was applied to 7563 cells from
                    lesional and nonlesional skin involved in twelve cell
                    groups, including <i>APOE</i> + FIB, <i>FBN1</i> + FIB,
                    <i>COL11A</i> + FIB, Inflam.FIB, cDC1, cDC2, LC, Inflam.DC,
                    TC, Inflam.TC, <i>CD40LG</i> + TC, and NKT.
                  </p>
                  <h3 class="c-article__sub-heading" id="Sec21">
                    Method comparisons
                  </h3>
                  <p>
                    We compare the performance of CellChat with three other
                    tools, including SingleCellSignalR<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 9"
                        title="Cabello-Aguilar, S. et al. SingleCellSignalR: inference of intercellular networks from single-cell transcriptomics. Nucleic Acids Res. pii, gkaa183 (2020)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR9"
                        id="ref-link-section-d38771043e4988"
                        >9</a
                      ></sup
                    >, iTALK<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 10"
                        title="Wang, Y. et al. iTALK: an R Package to Characterize and Illustrate Intercellular Communication. Preprint at bioRxiv 507871 (2019)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR10"
                        id="ref-link-section-d38771043e4992"
                        >10</a
                      ></sup
                    >, and&nbsp;CellPhoneDB<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 16"
                        title="Efremova, M., Vento-Tormo, M., Teichmann, S. A. &amp; Vento-Tormo, R. CellPhoneDB: inferring cell–cell communication from combined expression of multi-subunit ligand–receptor complexes. Nat. Protoc. 15, 1484–1506 (2020)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR16"
                        id="ref-link-section-d38771043e4996"
                        >16</a
                      ></sup
                    >
                    . We compare our database CellChatDB with other existing
                    analogous databases, including CellTalkDB<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 71"
                        title="Shao, X. et al. CellTalkDB: a manually curated database of ligand-receptor interactions in humans and mice. Brief Bioinform. 
                  https://doi.org/10.1093/bib/bbaa269
                  
                 (2020)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR71"
                        id="ref-link-section-d38771043e5000"
                        >71</a
                      ></sup
                    >, CellPhoneDB<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 16"
                        title="Efremova, M., Vento-Tormo, M., Teichmann, S. A. &amp; Vento-Tormo, R. CellPhoneDB: inferring cell–cell communication from combined expression of multi-subunit ligand–receptor complexes. Nat. Protoc. 15, 1484–1506 (2020)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR16"
                        id="ref-link-section-d38771043e5004"
                        >16</a
                      ></sup
                    >, iTALK<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 10"
                        title="Wang, Y. et al. iTALK: an R Package to Characterize and Illustrate Intercellular Communication. Preprint at bioRxiv 507871 (2019)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR10"
                        id="ref-link-section-d38771043e5009"
                        >10</a
                      ></sup
                    >, SingleCellSignalR<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 9"
                        title="Cabello-Aguilar, S. et al. SingleCellSignalR: inference of intercellular networks from single-cell transcriptomics. Nucleic Acids Res. pii, gkaa183 (2020)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR9"
                        id="ref-link-section-d38771043e5013"
                        >9</a
                      ></sup
                    >, Ramilowski2015<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 72"
                        title="Ramilowski, J. A. et al. A draft network of ligand-receptor-mediated multicellular signalling in human. Nat. Commun. 6, 7866 (2015)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR72"
                        id="ref-link-section-d38771043e5017"
                        >72</a
                      ></sup
                    >, NicheNet<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 13"
                        title="Browaeys, R., Saelens, W. &amp; Saeys, Y. NicheNet: modeling intercellular communication by linking ligands to target genes. Nat. Methods 17, 159–162 (2020)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR13"
                        id="ref-link-section-d38771043e5021"
                        >13</a
                      ></sup
                    >&nbsp;and ICELLNET<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 73"
                        title="Noël, F. et al. ICELLNET: a transcriptome-based framework to dissect intercellular communication. Preprint at bioRxiv 2020.2003.2005.976878 (2020)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR73"
                        id="ref-link-section-d38771043e5025"
                        >73</a
                      ></sup
                    >.&nbsp;SingleCellSignalR scores a given ligand-receptor
                    interaction between two cell populations using a regularized
                    product score approach based on average expression levels of
                    a ligand and its receptor and an ad hoc approach for
                    estimating an appropriate score threshold. iTALK identifies
                    differentially expressed ligands and receptors among
                    different cell populations and accounts for the matched
                    ligand-receptor pairs as significant interactions.
                    CellPhoneDB v2.0 predicts enriched signaling interactions
                    between two cell populations by considering the minimum
                    average expression of the members of the heteromeric complex
                    and performing empirical shuffling to calculate which
                    ligand–receptor pairs display significant cell-state
                    specificity. The detailed description of how these methods
                    were performed is available in Supplementary Note&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="supplementary material anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#MOESM1"
                      >3</a
                    >.
                  </p>
                  <p>
                    Both CellChat and CellPhoneDB, but not SingleCellSignalR,
                    and&nbsp;iTALK, consider multi-subunit structure of ligands
                    and receptors to represent heteromeric complexes accurately.
                    To evaluate the effect of neglecting multi-subunit structure
                    of ligands and receptors, we compute false positive rates
                    for the tools that use only one ligand and one receptor gene
                    pairs. The false positive interactions are defined by the
                    interactions with multi-subunits that are partially
                    identified by iTALK and SingleCellSignalR. The ground truth
                    of the interactions with multi-subunits is based on our
                    curated CellChatDB database. For example, for
                    <i>Tgfb1</i> ligand and its heteromeric receptor
                    <i>Tgfbr1/Tgfbr2</i> curated in CellChatDB, if the method
                    only identifies one of the two pairs (Tgfb1–Tgfbr1 and
                    Tgfb1–Tgfbr2), then we consider this prediction as one false
                    positive interaction.
                  </p>
                  <p>
                    We performed subsampling of scRNA-seq datasets using a
                    ‘geometric sketching’ approach, which maintains the
                    transcriptomic heterogeneity within a dataset with a smaller
                    subset of cells<sup
                      ><a
                        data-track="click"
                        data-track-action="reference anchor"
                        data-track-label="link"
                        data-test="citation-ref"
                        aria-label="Reference 96"
                        title="Hie, B., Cho, H., DeMeo, B., Bryson, B. &amp; Berger, B. Geometric sketching compactly summarizes the single-cell transcriptomic landscape. Cell Syst. 8, 483–493 e487 (2019)."
                        href="https://www.nature.com/articles/s41467-021-21246-9#ref-CR96"
                        id="ref-link-section-d38771043e5044"
                        >96</a
                      ></sup
                    >. We evaluated the robustness of inferred interactions from
                    subsampled datasets using three measures, including TPR,
                    FPR, and ACC, which were defined in Supplementary
                    Note&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="supplementary material anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#MOESM1"
                      >3</a
                    >. Note that such subsampling analysis was used to evaluate
                    the consistency rather than accuracy.
                  </p>
                  <h3 class="c-article__sub-heading" id="Sec22">
                    RNAscope in situ assay
                  </h3>
                  <p>
                    Frozen E14.5 mouse skin tissue sections were used for RNA in
                    situ hybridization using RNAscope® kit v2 (323100, Advanced
                    Cell Diagnostics). The following mouse probes from Advanced
                    Cell Diagnostics were used: <i>Dct</i> probe (460461-C2),
                    <i>Edn3</i> (505841), <i>Ednrb</i> (473801-C3),
                    <i>Axl</i> (450931-C2), <i>Thy1</i> (430661-C3). We have
                    complied with all relevant ethical regulations for animal
                    testing and research. All animal experiments have been
                    approved by the International Animal Care and Use Committee
                    (IACUC) of the University of California, Irvine.
                  </p>
                  <h3 class="c-article__sub-heading" id="Sec23">
                    Reporting summary
                  </h3>
                  <p>
                    Further information on research design is available in
                    the&nbsp;<a
                      data-track="click"
                      data-track-label="link"
                      data-track-action="supplementary material anchor"
                      href="https://www.nature.com/articles/s41467-021-21246-9#MOESM6"
                      >Nature Research Reporting Summary</a
                    >
                    linked to this article.
                  </p>
                </div>
              </div>
            </section>

            <section data-title="Data availability">
              <div class="c-article-section" id="data-availability-section">
                <h2
                  class="
                    c-article-section__title
                    js-section-title js-c-reading-companion-sections-item
                  "
                  id="data-availability"
                >
                  Data availability
                </h2>
                <div
                  class="c-article-section__content"
                  id="data-availability-content"
                >
                  <p>
                    CellChatDB is included in the CellChat repository (<a
                      href="https://github.com/sqjin/CellChat"
                      >https://github.com/sqjin/CellChat</a
                    >). KEGG pathway database is available at
                    <a href="https://www.genome.jp/kegg/pathway.html"
                      >https://www.genome.jp/kegg/pathway.html</a
                    >. The datasets analyzed in this study are available from
                    the Gene Expression Omnibus (GEO) repository under the
                    following accession numbers:
                    <a
                      href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE113854"
                      >GSE113854</a
                    >,
                    <a
                      href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE122043"
                      >GSE122043</a
                    >
                    (including four samples GSM3453535, GSM3453536, GSM3453537,
                    GSM3453538;) and
                    <a
                      href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE147424"
                      >GSE147424</a
                    >.
                  </p>
                </div>
              </div>
            </section>
            <section data-title="Code availability">
              <div class="c-article-section" id="code-availability-section">
                <h2
                  class="
                    c-article-section__title
                    js-section-title js-c-reading-companion-sections-item
                  "
                  id="code-availability"
                >
                  Code availability
                </h2>
                <div
                  class="c-article-section__content"
                  id="code-availability-content"
                >
                  <p>
                    CellChat is publicly available as an R package. Source
                    codes, as well as tutorials have been deposited at the
                    GitHub repository (<a
                      href="https://github.com/sqjin/CellChat"
                      >https://github.com/sqjin/CellChat</a
                    >). The web-based CellChat Explorer, including
                    Ligand-Receptor Interaction Explorer for exploring the
                    ligand-receptor interaction database and Cell–Cell
                    Communication Atlas Explorer for exploring the intercellular
                    communications in tissues, is available at
                    <a href="http://www.cellchat.org/"
                      >http://www.cellchat.org/</a
                    >.
                  </p>
                </div>
              </div>
            </section>
            <div id="MagazineFulltextArticleBodySuffix">
              <section aria-labelledby="Bib1" data-title="References">
                <div class="c-article-section" id="Bib1-section">
                  <h2
                    class="
                      c-article-section__title
                      js-section-title js-c-reading-companion-sections-item
                    "
                    id="Bib1"
                  >
                    References
                  </h2>
                  <div class="c-article-section__content" id="Bib1-content">
                    <div data-container-section="references">
                      <ol class="c-article-references">
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">1.</span>
                          <p class="c-article-references__text" id="ref-CR1">
                            Stubbington, M. J. T., Rozenblatt-Rosen, O., Regev,
                            A. &amp; Teichmann, S. A. Single-cell
                            transcriptomics to explore the immune system in
                            health and disease. <i>Science</i> <b>358</b>, 58–63
                            (2017).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2sXhsF2jtLrI"
                              aria-label="CAS reference 1"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28983043"
                              aria-label="PubMed reference 1"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5654495"
                              aria-label="PubMed Central reference 1"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1126%2Fscience.aan6828"
                              aria-label="Article reference 1"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2017Sci...358...58S"
                              aria-label="ADS reference 1"
                              >ADS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 1"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Single-cell%20transcriptomics%20to%20explore%20the%20immune%20system%20in%20health%20and%20disease&amp;journal=Science&amp;volume=358&amp;pages=58-63&amp;publication_year=2017&amp;author=Stubbington%2CMJT&amp;author=Rozenblatt-Rosen%2CO&amp;author=Regev%2CA&amp;author=Teichmann%2CSA"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">2.</span>
                          <p class="c-article-references__text" id="ref-CR2">
                            Vento-Tormo, R. et al. Single-cell reconstruction of
                            the early maternal-fetal interface in humans.
                            <i>Nature</i> <b>563</b>, 347–353 (2018).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC1cXit1SnsbzI"
                              aria-label="CAS reference 2"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30429548"
                              aria-label="PubMed reference 2"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1038%2Fs41586-018-0698-6"
                              aria-label="Article reference 2"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2018Natur.563..347V"
                              aria-label="ADS reference 2"
                              >ADS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC30429548"
                              aria-label="PubMed Central reference 2"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 2"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Single-cell%20reconstruction%20of%20the%20early%20maternal-fetal%20interface%20in%20humans&amp;journal=Nature&amp;volume=563&amp;pages=347-353&amp;publication_year=2018&amp;author=Vento-Tormo%2CR"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">3.</span>
                          <p class="c-article-references__text" id="ref-CR3">
                            Giladi, A. et al. Dissecting cellular crosstalk by
                            sequencing physically interacting cells.
                            <i>Nat. Biotechnol.</i> <b>38</b>, 629–637 (2020).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BB3cXksFKmu74%253D"
                              aria-label="CAS reference 3"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32152598"
                              aria-label="PubMed reference 3"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1038%2Fs41587-020-0442-2"
                              aria-label="Article reference 3"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC32152598"
                              aria-label="PubMed Central reference 3"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 3"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Dissecting%20cellular%20crosstalk%20by%20sequencing%20physically%20interacting%20cells&amp;journal=Nat.%20Biotechnol.&amp;volume=38&amp;pages=629-637&amp;publication_year=2020&amp;author=Giladi%2CA"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">4.</span>
                          <p class="c-article-references__text" id="ref-CR4">
                            Yuan, G. C. et al. Challenges and emerging
                            directions in single-cell analysis.
                            <i>Genome Biol.</i> <b>18</b>, 84 (2017).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28482897"
                              aria-label="PubMed reference 4"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5421338"
                              aria-label="PubMed Central reference 4"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1186%2Fs13059-017-1218-y"
                              aria-label="Article reference 4"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC1cXpvVWksbc%253D"
                              aria-label="CAS reference 4"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 4"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Challenges%20and%20emerging%20directions%20in%20single-cell%20analysis&amp;journal=Genome%20Biol.&amp;volume=18&amp;publication_year=2017&amp;author=Yuan%2CGC"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">5.</span>
                          <p class="c-article-references__text" id="ref-CR5">
                            Jin, S., MacLean, A. L., Peng, T. &amp; Nie, Q.
                            scEpath: energy landscape-based inference of
                            transition probabilities and cellular trajectories
                            from single-cell transcriptomic data.
                            <i>Bioinformatics</i> <b>34</b>, 2077–2086 (2018).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC1MXlsFKqur8%253D"
                              aria-label="CAS reference 5"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29415263"
                              aria-label="PubMed reference 5"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6658715"
                              aria-label="PubMed Central reference 5"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1093%2Fbioinformatics%2Fbty058"
                              aria-label="Article reference 5"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 5"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=scEpath%3A%20energy%20landscape-based%20inference%20of%20transition%20probabilities%20and%20cellular%20trajectories%20from%20single-cell%20transcriptomic%20data&amp;journal=Bioinformatics&amp;volume=34&amp;pages=2077-2086&amp;publication_year=2018&amp;author=Jin%2CS&amp;author=MacLean%2CAL&amp;author=Peng%2CT&amp;author=Nie%2CQ"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">6.</span>
                          <p class="c-article-references__text" id="ref-CR6">
                            Shao, X., Lu, X. Y., Liao, J., Chen, H. J. &amp;
                            Fan, X. H. New avenues for systematically inferring
                            cell-cell communication: through single-cell
                            transcriptomics data. <i>Protein Cell</i> <b>11</b>,
                            866–880 (2020).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32435978"
                              aria-label="PubMed reference 6"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7719148"
                              aria-label="PubMed Central reference 6"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1007%2Fs13238-020-00727-5"
                              aria-label="Article reference 6"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BB3cXhtVaitrfP"
                              aria-label="CAS reference 6"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 6"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=New%20avenues%20for%20systematically%20inferring%20cell-cell%20communication%3A%20through%20single-cell%20transcriptomics%20data&amp;journal=Protein%20Cell&amp;volume=11&amp;pages=866-880&amp;publication_year=2020&amp;author=Shao%2CX&amp;author=Lu%2CXY&amp;author=Liao%2CJ&amp;author=Chen%2CHJ&amp;author=Fan%2CXH"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">7.</span>
                          <p class="c-article-references__text" id="ref-CR7">
                            Armingol, E., Officer, A., Harismendy, O. &amp;
                            Lewis, N. E. Deciphering cell-cell interactions and
                            communication from gene expression.
                            <i>Nat. Rev. Genet.</i> <b>22</b>, 71–88 (2021).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BB3cXit1yht7vJ"
                              aria-label="CAS reference 7"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33168968"
                              aria-label="PubMed reference 7"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1038%2Fs41576-020-00292-x"
                              aria-label="Article reference 7"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC33168968"
                              aria-label="PubMed Central reference 7"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 7"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Deciphering%20cell-cell%20interactions%20and%20communication%20from%20gene%20expression&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=22&amp;pages=71-88&amp;publication_year=2021&amp;author=Armingol%2CE&amp;author=Officer%2CA&amp;author=Harismendy%2CO&amp;author=Lewis%2CNE"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">8.</span>
                          <p class="c-article-references__text" id="ref-CR8">
                            Kumar, M. P. et al. Analysis of single-cell RNA-seq
                            identifies cell-cell communication associated with
                            tumor characteristics. <i>Cell Rep.</i> <b>25</b>,
                            1458–1468 e1454 (2018).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC1cXitFWlu7jJ"
                              aria-label="CAS reference 8"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30404002"
                              aria-label="PubMed reference 8"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7009724"
                              aria-label="PubMed Central reference 8"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1016%2Fj.celrep.2018.10.047"
                              aria-label="Article reference 8"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 8"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Analysis%20of%20single-cell%20RNA-seq%20identifies%20cell-cell%20communication%20associated%20with%20tumor%20characteristics&amp;journal=Cell%20Rep.&amp;volume=25&amp;pages=1458-1468%20e1454&amp;publication_year=2018&amp;author=Kumar%2CMP"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">9.</span>
                          <p class="c-article-references__text" id="ref-CR9">
                            Cabello-Aguilar, S. et al. SingleCellSignalR:
                            inference of intercellular networks from single-cell
                            transcriptomics. <i>Nucleic Acids Res.</i>
                            <b>pii</b>, gkaa183 (2020).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 9"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=SingleCellSignalR%3A%20inference%20of%20intercellular%20networks%20from%20single-cell%20transcriptomics&amp;journal=Nucleic%20Acids%20Res.&amp;volume=pii&amp;publication_year=2020&amp;author=Cabello-Aguilar%2CS"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">10.</span>
                          <p class="c-article-references__text" id="ref-CR10">
                            Wang, Y. et al. iTALK: an R Package to Characterize
                            and Illustrate Intercellular Communication. Preprint
                            at <i>bioRxiv</i> 507871 (2019).
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">11.</span>
                          <p class="c-article-references__text" id="ref-CR11">
                            Raredon, M. S. B. et al. Single-cell connectomic
                            analysis of adult mammalian lungs. <i>Sci. Adv.</i>
                            <b>5</b>, eaaw3851 (2019).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BB3cXhtlalur3I"
                              aria-label="CAS reference 11"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31840053"
                              aria-label="PubMed reference 11"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6892628"
                              aria-label="PubMed Central reference 11"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1126%2Fsciadv.aaw3851"
                              aria-label="Article reference 11"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2019SciA....5.3851R"
                              aria-label="ADS reference 11"
                              >ADS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 11"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Single-cell%20connectomic%20analysis%20of%20adult%20mammalian%20lungs&amp;journal=Sci.%20Adv.&amp;volume=5&amp;publication_year=2019&amp;author=Raredon%2CMSB"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">12.</span>
                          <p class="c-article-references__text" id="ref-CR12">
                            Wang, S., Karikomi, M., MacLean, A. L. &amp; Nie, Q.
                            Cell lineage and communication network inference via
                            optimization for single-cell transcriptomics.
                            <i>Nucleic Acids Res.</i> <b>47</b>, e66 (2019).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC1MXisV2htLzE"
                              aria-label="CAS reference 12"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30923815"
                              aria-label="PubMed reference 12"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6582411"
                              aria-label="PubMed Central reference 12"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1093%2Fnar%2Fgkz204"
                              aria-label="Article reference 12"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 12"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Cell%20lineage%20and%20communication%20network%20inference%20via%20optimization%20for%20single-cell%20transcriptomics&amp;journal=Nucleic%20Acids%20Res.&amp;volume=47&amp;publication_year=2019&amp;author=Wang%2CS&amp;author=Karikomi%2CM&amp;author=MacLean%2CAL&amp;author=Nie%2CQ"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">13.</span>
                          <p class="c-article-references__text" id="ref-CR13">
                            Browaeys, R., Saelens, W. &amp; Saeys, Y. NicheNet:
                            modeling intercellular communication by linking
                            ligands to target genes. <i>Nat. Methods</i>
                            <b>17</b>, 159–162 (2020).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC1MXitlCht7%252FP"
                              aria-label="CAS reference 13"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31819264"
                              aria-label="PubMed reference 13"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1038%2Fs41592-019-0667-5"
                              aria-label="Article reference 13"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC31819264"
                              aria-label="PubMed Central reference 13"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 13"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=NicheNet%3A%20modeling%20intercellular%20communication%20by%20linking%20ligands%20to%20target%20genes&amp;journal=Nat.%20Methods&amp;volume=17&amp;pages=159-162&amp;publication_year=2020&amp;author=Browaeys%2CR&amp;author=Saelens%2CW&amp;author=Saeys%2CY"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">14.</span>
                          <p class="c-article-references__text" id="ref-CR14">
                            Ren, X. et al. Reconstruction of cell spatial
                            organization based on ligand-receptor mediated
                            self-assembly. <i>Cell Res.</i> <b>30</b>, 763–778
                            (2020).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BB3cXht1WmsL%252FE"
                              aria-label="CAS reference 14"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32541867"
                              aria-label="PubMed reference 14"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7608415"
                              aria-label="PubMed Central reference 14"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1038%2Fs41422-020-0353-2"
                              aria-label="Article reference 14"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 14"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Reconstruction%20of%20cell%20spatial%20organization%20based%20on%20ligand-receptor%20mediated%20self-assembly&amp;journal=Cell%20Res.&amp;volume=30&amp;pages=763-778&amp;publication_year=2020&amp;author=Ren%2CX"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">15.</span>
                          <p class="c-article-references__text" id="ref-CR15">
                            Derynck, R. &amp; Budi, E. H. Specificity,
                            versatility, and control of TGF-beta family
                            signaling. <i>Sci. Signal.</i> <b>12</b>, eaav5183
                            (2019).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC1MXhtlantrbJ"
                              aria-label="CAS reference 15"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30808818"
                              aria-label="PubMed reference 15"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6800142"
                              aria-label="PubMed Central reference 15"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1126%2Fscisignal.aav5183"
                              aria-label="Article reference 15"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 15"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Specificity%2C%20versatility%2C%20and%20control%20of%20TGF-beta%20family%20signaling&amp;journal=Sci.%20Signal.&amp;volume=12&amp;publication_year=2019&amp;author=Derynck%2CR&amp;author=Budi%2CEH"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">16.</span>
                          <p class="c-article-references__text" id="ref-CR16">
                            Efremova, M., Vento-Tormo, M., Teichmann, S. A.
                            &amp; Vento-Tormo, R. CellPhoneDB: inferring
                            cell–cell communication from combined expression of
                            multi-subunit ligand–receptor complexes.
                            <i>Nat. Protoc.</i> <b>15</b>, 1484–1506 (2020).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BB3cXjvFynsL0%253D"
                              aria-label="CAS reference 16"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32103204"
                              aria-label="PubMed reference 16"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1038%2Fs41596-020-0292-x"
                              aria-label="Article reference 16"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC32103204"
                              aria-label="PubMed Central reference 16"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 16"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=CellPhoneDB%3A%20inferring%20cell%E2%80%93cell%20communication%20from%20combined%20expression%20of%20multi-subunit%20ligand%E2%80%93receptor%20complexes&amp;journal=Nat.%20Protoc.&amp;volume=15&amp;pages=1484-1506&amp;publication_year=2020&amp;author=Efremova%2CM&amp;author=Vento-Tormo%2CM&amp;author=Teichmann%2CSA&amp;author=Vento-Tormo%2CR"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">17.</span>
                          <p class="c-article-references__text" id="ref-CR17">
                            Mário Cordeiro, R. P. S.
                            <i
                              >Pavel Brazdil and João Gama in Social Media and
                              Journalism-Trends, Connections, Implications</i
                            >
                            (Intech Open, London, UK; 2018).
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">18.</span>
                          <p class="c-article-references__text" id="ref-CR18">
                            Lee, D. D. &amp; Seung, H. S. Learning the parts of
                            objects by non-negative matrix factorization.
                            <i>Nature</i> <b>401</b>, 788–791 (1999).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADyaK1MXntFartrg%253D"
                              aria-label="CAS reference 18"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=10548103"
                              aria-label="PubMed reference 18"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.emis.de/MATH-item?1369.68285"
                              aria-label="MATH reference 18"
                              >MATH</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1038%2F44565"
                              aria-label="Article reference 18"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=1999Natur.401..788L"
                              aria-label="ADS reference 18"
                              >ADS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10548103"
                              aria-label="PubMed Central reference 18"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 18"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Learning%20the%20parts%20of%20objects%20by%20non-negative%20matrix%20factorization&amp;journal=Nature&amp;volume=401&amp;pages=788-791&amp;publication_year=1999&amp;author=Lee%2CDD&amp;author=Seung%2CHS"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">19.</span>
                          <p class="c-article-references__text" id="ref-CR19">
                            Zhang, L. &amp; Zhang, S. A general joint matrix
                            factorization framework for data integration and its
                            systematic algorithmic exploration.
                            <i>IEEE T Fuzzy Syst.</i> <b>28</b>, 1971–1983
                            (2020).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1109%2FTFUZZ.2019.2928518"
                              aria-label="Article reference 19"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 19"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=A%20general%20joint%20matrix%20factorization%20framework%20for%20data%20integration%20and%20its%20systematic%20algorithmic%20exploration&amp;journal=IEEE%20T%20Fuzzy%20Syst.&amp;volume=28&amp;pages=1971-1983&amp;publication_year=2020&amp;author=Zhang%2CL&amp;author=Zhang%2CS"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">20.</span>
                          <p class="c-article-references__text" id="ref-CR20">
                            Moon, K. R. et al. Manifold learning-based methods
                            for analyzing single-cell RNA-sequencing data.
                            <i>Curr. Opin. Syst. Biol.</i> <b>7</b>, 36–46
                            (2018).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1016%2Fj.coisb.2017.12.008"
                              aria-label="Article reference 20"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 20"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Manifold%20learning-based%20methods%20for%20analyzing%20single-cell%20RNA-sequencing%20data&amp;journal=Curr.%20Opin.%20Syst.%20Biol.&amp;volume=7&amp;pages=36-46&amp;publication_year=2018&amp;author=Moon%2CKR"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">21.</span>
                          <p class="c-article-references__text" id="ref-CR21">
                            Kanehisa, M., Furumichi, M., Tanabe, M., Sato, Y.
                            &amp; Morishima, K. KEGG: new perspectives on
                            genomes, pathways, diseases and drugs.
                            <i>Nucleic Acids Res.</i> <b>45</b>, D353–D361
                            (2017).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC1cXhslWgsrw%253D"
                              aria-label="CAS reference 21"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27899662"
                              aria-label="PubMed reference 21"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1093%2Fnar%2Fgkw1092"
                              aria-label="Article reference 21"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC27899662"
                              aria-label="PubMed Central reference 21"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 21"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=KEGG%3A%20new%20perspectives%20on%20genomes%2C%20pathways%2C%20diseases%20and%20drugs&amp;journal=Nucleic%20Acids%20Res.&amp;volume=45&amp;pages=D353-D361&amp;publication_year=2017&amp;author=Kanehisa%2CM&amp;author=Furumichi%2CM&amp;author=Tanabe%2CM&amp;author=Sato%2CY&amp;author=Morishima%2CK"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">22.</span>
                          <p class="c-article-references__text" id="ref-CR22">
                            Gupta, K. et al. Single-cell analysis reveals a hair
                            follicle dermal niche molecular differentiation
                            trajectory that begins prior to morphogenesis.
                            <i>Dev. Cell</i> <b>48</b>, 17–31 e16 (2019).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC1MXntlyl"
                              aria-label="CAS reference 22"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30595533"
                              aria-label="PubMed reference 22"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1016%2Fj.devcel.2018.11.032"
                              aria-label="Article reference 22"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC30595533"
                              aria-label="PubMed Central reference 22"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 22"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Single-cell%20analysis%20reveals%20a%20hair%20follicle%20dermal%20niche%20molecular%20differentiation%20trajectory%20that%20begins%20prior%20to%20morphogenesis&amp;journal=Dev.%20Cell&amp;volume=48&amp;pages=17-31%20e16&amp;publication_year=2019&amp;author=Gupta%2CK"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">23.</span>
                          <p class="c-article-references__text" id="ref-CR23">
                            Guerrero-Juarez, C. F. et al. Single-cell analysis
                            reveals fibroblast heterogeneity and myeloid-derived
                            adipocyte progenitors in murine skin wounds.
                            <i>Nat. Commun.</i> <b>10</b>, 650 (2019).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC1MXns1CqsLY%253D"
                              aria-label="CAS reference 23"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30737373"
                              aria-label="PubMed reference 23"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6368572"
                              aria-label="PubMed Central reference 23"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1038%2Fs41467-018-08247-x"
                              aria-label="Article reference 23"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2019NatCo..10..650G"
                              aria-label="ADS reference 23"
                              >ADS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 23"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Single-cell%20analysis%20reveals%20fibroblast%20heterogeneity%20and%20myeloid-derived%20adipocyte%20progenitors%20in%20murine%20skin%20wounds&amp;journal=Nat.%20Commun.&amp;volume=10&amp;publication_year=2019&amp;author=Guerrero-Juarez%2CCF"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">24.</span>
                          <p class="c-article-references__text" id="ref-CR24">
                            Peters, T. et al. Wound-healing defect of CD18(-/-)
                            mice due to a decrease in TGF-beta1 and
                            myofibroblast differentiation. <i>EMBO J.</i>
                            <b>24</b>, 3400–3410 (2005).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD2MXhtVKns7zP"
                              aria-label="CAS reference 24"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16148944"
                              aria-label="PubMed reference 24"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1276170"
                              aria-label="PubMed Central reference 24"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1038%2Fsj.emboj.7600809"
                              aria-label="Article reference 24"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 24"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Wound-healing%20defect%20of%20CD18%28-%2F-%29%20mice%20due%20to%20a%20decrease%20in%20TGF-beta1%20and%20myofibroblast%20differentiation&amp;journal=EMBO%20J.&amp;volume=24&amp;pages=3400-3410&amp;publication_year=2005&amp;author=Peters%2CT"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">25.</span>
                          <p class="c-article-references__text" id="ref-CR25">
                            Rappolee, D. A., Mark, D., Banda, M. J. &amp; Werb,
                            Z. Wound macrophages express TGF-alpha and other
                            growth factors in vivo: analysis by mRNA
                            phenotyping. <i>Science</i> <b>241</b>, 708–712
                            (1988).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADyaL1cXlt1Krs74%253D"
                              aria-label="CAS reference 25"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=3041594"
                              aria-label="PubMed reference 25"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1126%2Fscience.3041594"
                              aria-label="Article reference 25"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=1988Sci...241..708R"
                              aria-label="ADS reference 25"
                              >ADS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3041594"
                              aria-label="PubMed Central reference 25"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 25"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Wound%20macrophages%20express%20TGF-alpha%20and%20other%20growth%20factors%20in%20vivo%3A%20analysis%20by%20mRNA%20phenotyping&amp;journal=Science&amp;volume=241&amp;pages=708-712&amp;publication_year=1988&amp;author=Rappolee%2CDA&amp;author=Mark%2CD&amp;author=Banda%2CMJ&amp;author=Werb%2CZ"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">26.</span>
                          <p class="c-article-references__text" id="ref-CR26">
                            Shah, M., Foreman, D. M. &amp; Ferguson, M. W.
                            Neutralising antibody to TGF-beta 1,2 reduces
                            cutaneous scarring in adult rodents.
                            <i>J. Cell Sci.</i> <b>107</b>, 1137–1157 (1994).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADyaK2cXltVKhsLw%253D"
                              aria-label="CAS reference 26"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=7929624"
                              aria-label="PubMed reference 26"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7929624"
                              aria-label="PubMed Central reference 26"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 26"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Neutralising%20antibody%20to%20TGF-beta%201%2C2%20reduces%20cutaneous%20scarring%20in%20adult%20rodents&amp;journal=J.%20Cell%20Sci.&amp;volume=107&amp;pages=1137-1157&amp;publication_year=1994&amp;author=Shah%2CM&amp;author=Foreman%2CDM&amp;author=Ferguson%2CMW"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">27.</span>
                          <p class="c-article-references__text" id="ref-CR27">
                            Wu, L., Yu, Y. L., Galiano, R. D., Roth, S. I. &amp;
                            Mustoe, T. A. Macrophage colony-stimulating factor
                            accelerates wound healing and upregulates TGF-beta1
                            mRNA levels through tissue macrophages.
                            <i>J. Surg. Res.</i> <b>72</b>, 162–169 (1997).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADyaK2sXnvFaisr4%253D"
                              aria-label="CAS reference 27"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=9356238"
                              aria-label="PubMed reference 27"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1006%2Fjsre.1997.5178"
                              aria-label="Article reference 27"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9356238"
                              aria-label="PubMed Central reference 27"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 27"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Macrophage%20colony-stimulating%20factor%20accelerates%20wound%20healing%20and%20upregulates%20TGF-beta1%20mRNA%20levels%20through%20tissue%20macrophages&amp;journal=J.%20Surg.%20Res.&amp;volume=72&amp;pages=162-169&amp;publication_year=1997&amp;author=Wu%2CL&amp;author=Yu%2CYL&amp;author=Galiano%2CRD&amp;author=Roth%2CSI&amp;author=Mustoe%2CTA"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">28.</span>
                          <p class="c-article-references__text" id="ref-CR28">
                            Ishida, Y., Gao, J. L. &amp; Murphy, P. M. Chemokine
                            receptor CX3CR1 mediates skin wound healing by
                            promoting macrophage and fibroblast accumulation and
                            function. <i>J. Immunol.</i> <b>180</b>, 569–579
                            (2008).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD2sXhsVGjurvE"
                              aria-label="CAS reference 28"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18097059"
                              aria-label="PubMed reference 28"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.4049%2Fjimmunol.180.1.569"
                              aria-label="Article reference 28"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC18097059"
                              aria-label="PubMed Central reference 28"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 28"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Chemokine%20receptor%20CX3CR1%20mediates%20skin%20wound%20healing%20by%20promoting%20macrophage%20and%20fibroblast%20accumulation%20and%20function&amp;journal=J.%20Immunol.&amp;volume=180&amp;pages=569-579&amp;publication_year=2008&amp;author=Ishida%2CY&amp;author=Gao%2CJL&amp;author=Murphy%2CPM"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">29.</span>
                          <p class="c-article-references__text" id="ref-CR29">
                            Border, W. A. &amp; Noble, N. A. Transforming growth
                            factor beta in tissue fibrosis.
                            <i>N. Engl. J. Med.</i> <b>331</b>, 1286–1292
                            (1994).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADyaK2MXitlGksrs%253D"
                              aria-label="CAS reference 29"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=7935686"
                              aria-label="PubMed reference 29"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1056%2FNEJM199411103311907"
                              aria-label="Article reference 29"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7935686"
                              aria-label="PubMed Central reference 29"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 29"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Transforming%20growth%20factor%20beta%20in%20tissue%20fibrosis&amp;journal=N.%20Engl.%20J.%20Med.&amp;volume=331&amp;pages=1286-1292&amp;publication_year=1994&amp;author=Border%2CWA&amp;author=Noble%2CNA"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">30.</span>
                          <p class="c-article-references__text" id="ref-CR30">
                            Abbasi, S. et al. Distinct regulatory programs
                            control the latent regenerative potential of dermal
                            fibroblasts during wound healing.
                            <i>Cell Stem Cell</i> <b>27</b>, 396–412 e396
                            (2020).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BB3cXhsFGit7zI"
                              aria-label="CAS reference 30"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32755548"
                              aria-label="PubMed reference 30"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1016%2Fj.stem.2020.07.008"
                              aria-label="Article reference 30"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC32755548"
                              aria-label="PubMed Central reference 30"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 30"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Distinct%20regulatory%20programs%20control%20the%20latent%20regenerative%20potential%20of%20dermal%20fibroblasts%20during%20wound%20healing&amp;journal=Cell%20Stem%20Cell&amp;volume=27&amp;pages=396-412%20e396&amp;publication_year=2020&amp;author=Abbasi%2CS"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">31.</span>
                          <p class="c-article-references__text" id="ref-CR31">
                            Lee, Y. S. et al. STAT3 signalling pathway is
                            implicated in keloid pathogenesis by preliminary
                            transcriptome and open chromatin analyses.
                            <i>Exp. Dermatol.</i> <b>28</b>, 480–484 (2019).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC1MXoslyis7g%253D"
                              aria-label="CAS reference 31"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30916811"
                              aria-label="PubMed reference 31"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6488383"
                              aria-label="PubMed Central reference 31"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1111%2Fexd.13923"
                              aria-label="Article reference 31"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 31"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=STAT3%20signalling%20pathway%20is%20implicated%20in%20keloid%20pathogenesis%20by%20preliminary%20transcriptome%20and%20open%20chromatin%20analyses&amp;journal=Exp.%20Dermatol.&amp;volume=28&amp;pages=480-484&amp;publication_year=2019&amp;author=Lee%2CYS"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">32.</span>
                          <p class="c-article-references__text" id="ref-CR32">
                            Feng, Y. et al. The signaling protein Wnt5a promotes
                            TGFbeta1-mediated macrophage polarization and kidney
                            fibrosis by inducing the transcriptional regulators
                            Yap/Taz. <i>J. Biol. Chem.</i> <b>293</b>,
                            19290–19302 (2018).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC1cXisFWitbrL"
                              aria-label="CAS reference 32"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30333225"
                              aria-label="PubMed reference 32"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6302175"
                              aria-label="PubMed Central reference 32"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1074%2Fjbc.RA118.005457"
                              aria-label="Article reference 32"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 32"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=The%20signaling%20protein%20Wnt5a%20promotes%20TGFbeta1-mediated%20macrophage%20polarization%20and%20kidney%20fibrosis%20by%20inducing%20the%20transcriptional%20regulators%20Yap%2FTaz&amp;journal=J.%20Biol.%20Chem.&amp;volume=293&amp;pages=19290-19302&amp;publication_year=2018&amp;author=Feng%2CY"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">33.</span>
                          <p class="c-article-references__text" id="ref-CR33">
                            Abraityte, A. et al. Wnt5a is elevated in heart
                            failure and affects cardiac fibroblast function.
                            <i>J. Mol. Med.</i> <b>95</b>, 767–777 (2017).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2sXlt12ktLs%253D"
                              aria-label="CAS reference 33"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28357477"
                              aria-label="PubMed reference 33"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1007%2Fs00109-017-1529-1"
                              aria-label="Article reference 33"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC28357477"
                              aria-label="PubMed Central reference 33"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 33"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Wnt5a%20is%20elevated%20in%20heart%20failure%20and%20affects%20cardiac%20fibroblast%20function&amp;journal=J.%20Mol.%20Med.&amp;volume=95&amp;pages=767-777&amp;publication_year=2017&amp;author=Abraityte%2CA"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">34.</span>
                          <p class="c-article-references__text" id="ref-CR34">
                            Vuga, L. J. et al. WNT5A is a regulator of
                            fibroblast proliferation and resistance to
                            apoptosis. <i>Am. J. Respir. Cell Mol. Biol.</i>
                            <b>41</b>, 583–589 (2009).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD1MXhtlCktbzK"
                              aria-label="CAS reference 34"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19251946"
                              aria-label="PubMed reference 34"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2778165"
                              aria-label="PubMed Central reference 34"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1165%2Frcmb.2008-0201OC"
                              aria-label="Article reference 34"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 34"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=WNT5A%20is%20a%20regulator%20of%20fibroblast%20proliferation%20and%20resistance%20to%20apoptosis&amp;journal=Am.%20J.%20Respir.%20Cell%20Mol.%20Biol.&amp;volume=41&amp;pages=583-589&amp;publication_year=2009&amp;author=Vuga%2CLJ"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">35.</span>
                          <p class="c-article-references__text" id="ref-CR35">
                            Saxena, N., Mok, K. W. &amp; Rendl, M. An updated
                            classification of hair follicle morphogenesis.
                            <i>Exp. Dermatol</i> <b>28</b>, 332–344 (2019).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30887615"
                              aria-label="PubMed reference 35"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7137758"
                              aria-label="PubMed Central reference 35"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1111%2Fexd.13913"
                              aria-label="Article reference 35"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 35"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=An%20updated%20classification%20of%20hair%20follicle%20morphogenesis&amp;journal=Exp.%20Dermatol&amp;volume=28&amp;pages=332-344&amp;publication_year=2019&amp;author=Saxena%2CN&amp;author=Mok%2CKW&amp;author=Rendl%2CM"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">36.</span>
                          <p class="c-article-references__text" id="ref-CR36">
                            Mok, K. W. et al. Dermal condensate niche fate
                            specification occurs prior to formation and is
                            placode progenitor dependent. <i>Dev. Cell</i>
                            <b>48</b>, 32–48 e35 (2019).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC1MXntlym"
                              aria-label="CAS reference 36"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30595537"
                              aria-label="PubMed reference 36"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1016%2Fj.devcel.2018.11.034"
                              aria-label="Article reference 36"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC30595537"
                              aria-label="PubMed Central reference 36"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 36"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Dermal%20condensate%20niche%20fate%20specification%20occurs%20prior%20to%20formation%20and%20is%20placode%20progenitor%20dependent&amp;journal=Dev.%20Cell&amp;volume=48&amp;pages=32-48%20e35&amp;publication_year=2019&amp;author=Mok%2CKW"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">37.</span>
                          <p class="c-article-references__text" id="ref-CR37">
                            Glover, J. D. et al. Hierarchical patterning modes
                            orchestrate hair follicle morphogenesis.
                            <i>PLoS Biol.</i> <b>15</b>, e2002117 (2017).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28700594"
                              aria-label="PubMed reference 37"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5507405"
                              aria-label="PubMed Central reference 37"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1371%2Fjournal.pbio.2002117"
                              aria-label="Article reference 37"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC1cXhs1WrtbjF"
                              aria-label="CAS reference 37"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 37"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Hierarchical%20patterning%20modes%20orchestrate%20hair%20follicle%20morphogenesis&amp;journal=PLoS%20Biol.&amp;volume=15&amp;publication_year=2017&amp;author=Glover%2CJD"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">38.</span>
                          <p class="c-article-references__text" id="ref-CR38">
                            Tsai, S. Y. et al. Wnt/beta-catenin signaling in
                            dermal condensates is required for hair follicle
                            formation. <i>Dev. Biol.</i> <b>385</b>, 179–188
                            (2014).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC3sXhvFOhsr%252FK"
                              aria-label="CAS reference 38"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24309208"
                              aria-label="PubMed reference 38"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1016%2Fj.ydbio.2013.11.023"
                              aria-label="Article reference 38"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC24309208"
                              aria-label="PubMed Central reference 38"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 38"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Wnt%2Fbeta-catenin%20signaling%20in%20dermal%20condensates%20is%20required%20for%20hair%20follicle%20formation&amp;journal=Dev.%20Biol.&amp;volume=385&amp;pages=179-188&amp;publication_year=2014&amp;author=Tsai%2CSY"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">39.</span>
                          <p class="c-article-references__text" id="ref-CR39">
                            Fu, J. &amp; Hsu, W. Epidermal Wnt controls hair
                            follicle induction by orchestrating dynamic
                            signaling crosstalk between the epidermis and
                            dermis. <i>J. Invest. Dermatol.</i> <b>133</b>,
                            890–898 (2013).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC38XhsleksbvN"
                              aria-label="CAS reference 39"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23190887"
                              aria-label="PubMed reference 39"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1038%2Fjid.2012.407"
                              aria-label="Article reference 39"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC23190887"
                              aria-label="PubMed Central reference 39"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 39"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Epidermal%20Wnt%20controls%20hair%20follicle%20induction%20by%20orchestrating%20dynamic%20signaling%20crosstalk%20between%20the%20epidermis%20and%20dermis&amp;journal=J.%20Invest.%20Dermatol.&amp;volume=133&amp;pages=890-898&amp;publication_year=2013&amp;author=Fu%2CJ&amp;author=Hsu%2CW"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">40.</span>
                          <p class="c-article-references__text" id="ref-CR40">
                            Chen, D., Jarrell, A., Guo, C., Lang, R. &amp; Atit,
                            R. Dermal beta-catenin activity in response to
                            epidermal Wnt ligands is required for fibroblast
                            proliferation and hair follicle initiation.
                            <i>Development</i> <b>139</b>, 1522–1533 (2012).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC38Xnt1GrurY%253D"
                              aria-label="CAS reference 40"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22434869"
                              aria-label="PubMed reference 40"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3308184"
                              aria-label="PubMed Central reference 40"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1242%2Fdev.076463"
                              aria-label="Article reference 40"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 40"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Dermal%20beta-catenin%20activity%20in%20response%20to%20epidermal%20Wnt%20ligands%20is%20required%20for%20fibroblast%20proliferation%20and%20hair%20follicle%20initiation&amp;journal=Development&amp;volume=139&amp;pages=1522-1533&amp;publication_year=2012&amp;author=Chen%2CD&amp;author=Jarrell%2CA&amp;author=Guo%2CC&amp;author=Lang%2CR&amp;author=Atit%2CR"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">41.</span>
                          <p class="c-article-references__text" id="ref-CR41">
                            Andl, T., Reddy, S. T., Gaddapara, T. &amp; Millar,
                            S. E. WNT signals are required for the initiation of
                            hair follicle development. <i>Dev. Cell</i>
                            <b>2</b>, 643–653 (2002).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD38XktVGjs7o%253D"
                              aria-label="CAS reference 41"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12015971"
                              aria-label="PubMed reference 41"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1016%2FS1534-5807%2802%2900167-3"
                              aria-label="Article reference 41"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC12015971"
                              aria-label="PubMed Central reference 41"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 41"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=WNT%20signals%20are%20required%20for%20the%20initiation%20of%20hair%20follicle%20development&amp;journal=Dev.%20Cell&amp;volume=2&amp;pages=643-653&amp;publication_year=2002&amp;author=Andl%2CT&amp;author=Reddy%2CST&amp;author=Gaddapara%2CT&amp;author=Millar%2CSE"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">42.</span>
                          <p class="c-article-references__text" id="ref-CR42">
                            Biggs, L. C. et al. Hair follicle dermal
                            condensation forms via Fgf20 primed cell cycle exit,
                            cell motility, and aggregation. <i>Elife</i>
                            <b>7</b>, e36468 (2018).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30063206"
                              aria-label="PubMed reference 42"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6107334"
                              aria-label="PubMed Central reference 42"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.7554%2FeLife.36468"
                              aria-label="Article reference 42"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 42"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Hair%20follicle%20dermal%20condensation%20forms%20via%20Fgf20%20primed%20cell%20cycle%20exit%2C%20cell%20motility%2C%20and%20aggregation&amp;journal=Elife&amp;volume=7&amp;publication_year=2018&amp;author=Biggs%2CLC"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">43.</span>
                          <p class="c-article-references__text" id="ref-CR43">
                            Huh, S. H. et al. Fgf20 governs formation of primary
                            and secondary dermal condensations in developing
                            hair follicles. <i>Genes Dev.</i> <b>27</b>, 450–458
                            (2013).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC3sXktlymtbY%253D"
                              aria-label="CAS reference 43"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23431057"
                              aria-label="PubMed reference 43"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3589561"
                              aria-label="PubMed Central reference 43"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1101%2Fgad.198945.112"
                              aria-label="Article reference 43"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 43"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Fgf20%20governs%20formation%20of%20primary%20and%20secondary%20dermal%20condensations%20in%20developing%20hair%20follicles&amp;journal=Genes%20Dev.&amp;volume=27&amp;pages=450-458&amp;publication_year=2013&amp;author=Huh%2CSH"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">44.</span>
                          <p class="c-article-references__text" id="ref-CR44">
                            Jamora, C. et al. A signaling pathway involving
                            TGF-beta2 and snail in hair follicle morphogenesis.
                            <i>PLoS Biol.</i> <b>3</b>, e11 (2005).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15630473"
                              aria-label="PubMed reference 44"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1371%2Fjournal.pbio.0030011"
                              aria-label="Article reference 44"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD2MXptVOisA%253D%253D"
                              aria-label="CAS reference 44"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC15630473"
                              aria-label="PubMed Central reference 44"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 44"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=A%20signaling%20pathway%20involving%20TGF-beta2%20and%20snail%20in%20hair%20follicle%20morphogenesis&amp;journal=PLoS%20Biol.&amp;volume=3&amp;publication_year=2005&amp;author=Jamora%2CC"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">45.</span>
                          <p class="c-article-references__text" id="ref-CR45">
                            Foitzik, K., Paus, R., Doetschman, T. &amp; Dotto,
                            G. P. The TGF-beta2 isoform is both a required and
                            sufficient inducer of murine hair follicle
                            morphogenesis. <i>Dev. Biol.</i> <b>212</b>, 278–289
                            (1999).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADyaK1MXkvVOktbY%253D"
                              aria-label="CAS reference 45"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=10433821"
                              aria-label="PubMed reference 45"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1006%2Fdbio.1999.9325"
                              aria-label="Article reference 45"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10433821"
                              aria-label="PubMed Central reference 45"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 45"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=The%20TGF-beta2%20isoform%20is%20both%20a%20required%20and%20sufficient%20inducer%20of%20murine%20hair%20follicle%20morphogenesis&amp;journal=Dev.%20Biol.&amp;volume=212&amp;pages=278-289&amp;publication_year=1999&amp;author=Foitzik%2CK&amp;author=Paus%2CR&amp;author=Doetschman%2CT&amp;author=Dotto%2CGP"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">46.</span>
                          <p class="c-article-references__text" id="ref-CR46">
                            Scutera, S. et al. Survival and migration of human
                            dendritic cells are regulated by an
                            IFN-alpha-inducible Axl/Gas6 pathway.
                            <i>J. Immunol.</i> <b>183</b>, 3004–3013 (2009).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD1MXhtVWrs7zO"
                              aria-label="CAS reference 46"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19657094"
                              aria-label="PubMed reference 46"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.4049%2Fjimmunol.0804384"
                              aria-label="Article reference 46"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC19657094"
                              aria-label="PubMed Central reference 46"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 46"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Survival%20and%20migration%20of%20human%20dendritic%20cells%20are%20regulated%20by%20an%20IFN-alpha-inducible%20Axl%2FGas6%20pathway&amp;journal=J.%20Immunol.&amp;volume=183&amp;pages=3004-3013&amp;publication_year=2009&amp;author=Scutera%2CS"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">47.</span>
                          <p class="c-article-references__text" id="ref-CR47">
                            Abu-Thuraia, A. et al. AXL confers cell migration
                            and invasion by hijacking a PEAK1-regulated focal
                            adhesion protein network. <i>Nat. Commun.</i>
                            <b>11</b>, 3586 (2020).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32681075"
                              aria-label="PubMed reference 47"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7368075"
                              aria-label="PubMed Central reference 47"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1038%2Fs41467-020-17415-x"
                              aria-label="Article reference 47"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2020NatCo..11.3586A"
                              aria-label="ADS reference 47"
                              >ADS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BB3cXhsVaktrfE"
                              aria-label="CAS reference 47"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 47"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=AXL%20confers%20cell%20migration%20and%20invasion%20by%20hijacking%20a%20PEAK1-regulated%20focal%20adhesion%20protein%20network&amp;journal=Nat.%20Commun.&amp;volume=11&amp;publication_year=2020&amp;author=Abu-Thuraia%2CA"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">48.</span>
                          <p class="c-article-references__text" id="ref-CR48">
                            Tieche, C. C. et al. Tumor initiation capacity and
                            therapy resistance are differential features of
                            EMT-related subpopulations in the NSCLC Cell Line
                            A549. <i>Neoplasia</i> <b>21</b>, 185–196 (2019).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC1cXisFektbfF"
                              aria-label="CAS reference 48"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30591423"
                              aria-label="PubMed reference 48"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1016%2Fj.neo.2018.09.008"
                              aria-label="Article reference 48"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC30591423"
                              aria-label="PubMed Central reference 48"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 48"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Tumor%20initiation%20capacity%20and%20therapy%20resistance%20are%20differential%20features%20of%20EMT-related%20subpopulations%20in%20the%20NSCLC%20Cell%20Line%20A549&amp;journal=Neoplasia&amp;volume=21&amp;pages=185-196&amp;publication_year=2019&amp;author=Tieche%2CCC"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">49.</span>
                          <p class="c-article-references__text" id="ref-CR49">
                            Kumar, A., Bhanja, A., Bhattacharyya, J. &amp;
                            Jaganathan, B. G. Multiple roles of CD90 in cancer.
                            <i>Tumour Biol.</i> <b>37</b>, 11611–11622 (2016).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC28XhtVyiu7rF"
                              aria-label="CAS reference 49"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27337957"
                              aria-label="PubMed reference 49"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1007%2Fs13277-016-5112-0"
                              aria-label="Article reference 49"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC27337957"
                              aria-label="PubMed Central reference 49"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 49"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Multiple%20roles%20of%20CD90%20in%20cancer&amp;journal=Tumour%20Biol.&amp;volume=37&amp;pages=11611-11622&amp;publication_year=2016&amp;author=Kumar%2CA&amp;author=Bhanja%2CA&amp;author=Bhattacharyya%2CJ&amp;author=Jaganathan%2CBG"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">50.</span>
                          <p class="c-article-references__text" id="ref-CR50">
                            Rege, T. A. et al. Thy-1, via its GPI anchor,
                            modulates Src family kinase and focal adhesion
                            kinase phosphorylation and subcellular localization,
                            and fibroblast migration, in response to
                            thrombospondin-1/hep I. <i>Exp. Cell Res.</i>
                            <b>312</b>, 3752–3767 (2006).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD28XhtFCjs7fK"
                              aria-label="CAS reference 50"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17027000"
                              aria-label="PubMed reference 50"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1016%2Fj.yexcr.2006.07.029"
                              aria-label="Article reference 50"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2006cca..book.....R"
                              aria-label="ADS reference 50"
                              >ADS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC17027000"
                              aria-label="PubMed Central reference 50"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 50"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Thy-1%2C%20via%20its%20GPI%20anchor%2C%20modulates%20Src%20family%20kinase%20and%20focal%20adhesion%20kinase%20phosphorylation%20and%20subcellular%20localization%2C%20and%20fibroblast%20migration%2C%20in%20response%20to%20thrombospondin-1%2Fhep%20I&amp;journal=Exp.%20Cell%20Res.&amp;volume=312&amp;pages=3752-3767&amp;publication_year=2006&amp;author=Rege%2CTA"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">51.</span>
                          <p class="c-article-references__text" id="ref-CR51">
                            Meshcheryakova, A. et al. Exploring the role of
                            sphingolipid machinery during the epithelial to
                            mesenchymal transition program using an integrative
                            approach. <i>Oncotarget</i> <b>7</b>, 22295–22323
                            (2016).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26967245"
                              aria-label="PubMed reference 51"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5008362"
                              aria-label="PubMed Central reference 51"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.18632%2Foncotarget.7947"
                              aria-label="Article reference 51"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 51"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Exploring%20the%20role%20of%20sphingolipid%20machinery%20during%20the%20epithelial%20to%20mesenchymal%20transition%20program%20using%20an%20integrative%20approach&amp;journal=Oncotarget&amp;volume=7&amp;pages=22295-22323&amp;publication_year=2016&amp;author=Meshcheryakova%2CA"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">52.</span>
                          <p class="c-article-references__text" id="ref-CR52">
                            Botchkareva, N. V., Botchkarev, V. A. &amp;
                            Gilchrest, B. A. Fate of melanocytes during
                            development of the hair follicle pigmentary unit.
                            <i>J. Investig. Dermatol Symp. Proc.</i> <b>8</b>,
                            76–79 (2003).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12894999"
                              aria-label="PubMed reference 52"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1046%2Fj.1523-1747.2003.12176.x"
                              aria-label="Article reference 52"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC12894999"
                              aria-label="PubMed Central reference 52"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 52"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Fate%20of%20melanocytes%20during%20development%20of%20the%20hair%20follicle%20pigmentary%20unit&amp;journal=J.%20Investig.%20Dermatol%20Symp.%20Proc.&amp;volume=8&amp;pages=76-79&amp;publication_year=2003&amp;author=Botchkareva%2CNV&amp;author=Botchkarev%2CVA&amp;author=Gilchrest%2CBA"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">53.</span>
                          <p class="c-article-references__text" id="ref-CR53">
                            Garcia, R. J. et al. Endothelin 3 induces skin
                            pigmentation in a keratin-driven inducible mouse
                            model. <i>J. Invest Dermatol</i> <b>128</b>, 131–142
                            (2008).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD2sXhsVSksbjJ"
                              aria-label="CAS reference 53"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17611578"
                              aria-label="PubMed reference 53"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1038%2Fsj.jid.5700948"
                              aria-label="Article reference 53"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC17611578"
                              aria-label="PubMed Central reference 53"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 53"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Endothelin%203%20induces%20skin%20pigmentation%20in%20a%20keratin-driven%20inducible%20mouse%20model&amp;journal=J.%20Invest%20Dermatol&amp;volume=128&amp;pages=131-142&amp;publication_year=2008&amp;author=Garcia%2CRJ"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">54.</span>
                          <p class="c-article-references__text" id="ref-CR54">
                            Rognoni, E. &amp; Watt, F. M. Skin cell
                            heterogeneity in development, wound healing, and
                            cancer. <i>Trends Cell Biol.</i> <b>28</b>, 709–722
                            (2018).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29807713"
                              aria-label="PubMed reference 54"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6098245"
                              aria-label="PubMed Central reference 54"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1016%2Fj.tcb.2018.05.002"
                              aria-label="Article reference 54"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 54"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Skin%20cell%20heterogeneity%20in%20development%2C%20wound%20healing%2C%20and%20cancer&amp;journal=Trends%20Cell%20Biol.&amp;volume=28&amp;pages=709-722&amp;publication_year=2018&amp;author=Rognoni%2CE&amp;author=Watt%2CFM"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">55.</span>
                          <p class="c-article-references__text" id="ref-CR55">
                            Bazzi, H., Fantauzzo, K. A., Richardson, G. D.,
                            Jahoda, C. A. &amp; Christiano, A. M.
                            Transcriptional profiling of developing mouse
                            epidermis reveals novel patterns of coordinated gene
                            expression. <i>Dev. Dyn.</i> <b>236</b>, 961–970
                            (2007).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD2sXltlWnsLg%253D"
                              aria-label="CAS reference 55"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17330888"
                              aria-label="PubMed reference 55"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1002%2Fdvdy.21099"
                              aria-label="Article reference 55"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC17330888"
                              aria-label="PubMed Central reference 55"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 55"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Transcriptional%20profiling%20of%20developing%20mouse%20epidermis%20reveals%20novel%20patterns%20of%20coordinated%20gene%20expression&amp;journal=Dev.%20Dyn.&amp;volume=236&amp;pages=961-970&amp;publication_year=2007&amp;author=Bazzi%2CH&amp;author=Fantauzzo%2CKA&amp;author=Richardson%2CGD&amp;author=Jahoda%2CCA&amp;author=Christiano%2CAM"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">56.</span>
                          <p class="c-article-references__text" id="ref-CR56">
                            Byrne, C., Tainsky, M. &amp; Fuchs, E. Programming
                            gene expression in developing epidermis.
                            <i>Development</i> <b>120</b>, 2369–2383 (1994).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADyaK2MXntFersw%253D%253D"
                              aria-label="CAS reference 56"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=7525178"
                              aria-label="PubMed reference 56"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7525178"
                              aria-label="PubMed Central reference 56"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 56"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Programming%20gene%20expression%20in%20developing%20epidermis&amp;journal=Development&amp;volume=120&amp;pages=2369-2383&amp;publication_year=1994&amp;author=Byrne%2CC&amp;author=Tainsky%2CM&amp;author=Fuchs%2CE"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">57.</span>
                          <p class="c-article-references__text" id="ref-CR57">
                            Mukhopadhyay, A. et al. Negative regulation of Shh
                            levels by Kras and Fgfr2 during hair follicle
                            development. <i>Dev. Biol.</i> <b>373</b>, 373–382
                            (2013).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC38XhslymsLnP"
                              aria-label="CAS reference 57"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23123965"
                              aria-label="PubMed reference 57"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1016%2Fj.ydbio.2012.10.024"
                              aria-label="Article reference 57"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC23123965"
                              aria-label="PubMed Central reference 57"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 57"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Negative%20regulation%20of%20Shh%20levels%20by%20Kras%20and%20Fgfr2%20during%20hair%20follicle%20development&amp;journal=Dev.%20Biol.&amp;volume=373&amp;pages=373-382&amp;publication_year=2013&amp;author=Mukhopadhyay%2CA"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">58.</span>
                          <p class="c-article-references__text" id="ref-CR58">
                            Richardson, G. D. et al. KGF and EGF signalling
                            block hair follicle induction and promote
                            interfollicular epidermal fate in developing mouse
                            skin. <i>Development</i> <b>136</b>, 2153–2164
                            (2009).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD1MXps1ehs7s%253D"
                              aria-label="CAS reference 58"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19474150"
                              aria-label="PubMed reference 58"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2729337"
                              aria-label="PubMed Central reference 58"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1242%2Fdev.031427"
                              aria-label="Article reference 58"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 58"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=KGF%20and%20EGF%20signalling%20block%20hair%20follicle%20induction%20and%20promote%20interfollicular%20epidermal%20fate%20in%20developing%20mouse%20skin&amp;journal=Development&amp;volume=136&amp;pages=2153-2164&amp;publication_year=2009&amp;author=Richardson%2CGD"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">59.</span>
                          <p class="c-article-references__text" id="ref-CR59">
                            Petiot, A. et al. A crucial role for Fgfr2-IIIb
                            signalling in epidermal development and hair
                            follicle patterning. <i>Development</i> <b>130</b>,
                            5493–5501 (2003).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD3sXps12qtL8%253D"
                              aria-label="CAS reference 59"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=14530295"
                              aria-label="PubMed reference 59"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1242%2Fdev.00788"
                              aria-label="Article reference 59"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC14530295"
                              aria-label="PubMed Central reference 59"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 59"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=A%20crucial%20role%20for%20Fgfr2-IIIb%20signalling%20in%20epidermal%20development%20and%20hair%20follicle%20patterning&amp;journal=Development&amp;volume=130&amp;pages=5493-5501&amp;publication_year=2003&amp;author=Petiot%2CA"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">60.</span>
                          <p class="c-article-references__text" id="ref-CR60">
                            du Cros, D. L. Fibroblast growth factor and
                            epidermal growth factor in hair development.
                            <i>J. Invest. Dermatol.</i> <b>101</b>, 106S–113S
                            (1993).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=8326142"
                              aria-label="PubMed reference 60"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1016%2F0022-202X%2893%2990509-G"
                              aria-label="Article reference 60"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8326142"
                              aria-label="PubMed Central reference 60"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 60"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Fibroblast%20growth%20factor%20and%20epidermal%20growth%20factor%20in%20hair%20development&amp;journal=J.%20Invest.%20Dermatol.&amp;volume=101&amp;pages=106S-113S&amp;publication_year=1993&amp;author=Cros%2CDL"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">61.</span>
                          <p class="c-article-references__text" id="ref-CR61">
                            Buback, F., Renkl, A. C., Schulz, G. &amp; Weiss, J.
                            M. Osteopontin and the skin: multiple emerging roles
                            in cutaneous biology and pathology.
                            <i>Exp. Dermatol.</i> <b>18</b>, 750–759 (2009).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD1MXhtFCktLjM"
                              aria-label="CAS reference 61"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19558497"
                              aria-label="PubMed reference 61"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1111%2Fj.1600-0625.2009.00926.x"
                              aria-label="Article reference 61"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC19558497"
                              aria-label="PubMed Central reference 61"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 61"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Osteopontin%20and%20the%20skin%3A%20multiple%20emerging%20roles%20in%20cutaneous%20biology%20and%20pathology&amp;journal=Exp.%20Dermatol.&amp;volume=18&amp;pages=750-759&amp;publication_year=2009&amp;author=Buback%2CF&amp;author=Renkl%2CAC&amp;author=Schulz%2CG&amp;author=Weiss%2CJM"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">62.</span>
                          <p class="c-article-references__text" id="ref-CR62">
                            Mori, R., Shaw, T. J. &amp; Martin, P. Molecular
                            mechanisms linking wound inflammation and fibrosis:
                            knockdown of osteopontin leads to rapid repair and
                            reduced scarring. <i>J. Exp. Med.</i> <b>205</b>,
                            43–51 (2008).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD1cXht1ykt7o%253D"
                              aria-label="CAS reference 62"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18180311"
                              aria-label="PubMed reference 62"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2234383"
                              aria-label="PubMed Central reference 62"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1084%2Fjem.20071412"
                              aria-label="Article reference 62"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 62"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Molecular%20mechanisms%20linking%20wound%20inflammation%20and%20fibrosis%3A%20knockdown%20of%20osteopontin%20leads%20to%20rapid%20repair%20and%20reduced%20scarring&amp;journal=J.%20Exp.%20Med.&amp;volume=205&amp;pages=43-51&amp;publication_year=2008&amp;author=Mori%2CR&amp;author=Shaw%2CTJ&amp;author=Martin%2CP"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">63.</span>
                          <p class="c-article-references__text" id="ref-CR63">
                            Liaw, L. et al. Altered wound healing in mice
                            lacking a functional osteopontin gene (spp1).
                            <i>J. Clin. Invest.</i> <b>101</b>, 1468–1478
                            (1998).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADyaK1cXitleksbw%253D"
                              aria-label="CAS reference 63"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=9525990"
                              aria-label="PubMed reference 63"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC508725"
                              aria-label="PubMed Central reference 63"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1172%2FJCI2131"
                              aria-label="Article reference 63"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 63"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Altered%20wound%20healing%20in%20mice%20lacking%20a%20functional%20osteopontin%20gene%20%28spp1%29&amp;journal=J.%20Clin.%20Invest.&amp;volume=101&amp;pages=1468-1478&amp;publication_year=1998&amp;author=Liaw%2CL"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">64.</span>
                          <p class="c-article-references__text" id="ref-CR64">
                            Gilliver, S. C., Emmerson, E., Bernhagen, J. &amp;
                            Hardman, M. J. MIF: a key player in cutaneous
                            biology and wound healing. <i>Exp. Dermatol.</i>
                            <b>20</b>, 1–6 (2011).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC3MXhsVOjsrk%253D"
                              aria-label="CAS reference 64"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21158933"
                              aria-label="PubMed reference 64"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1111%2Fj.1600-0625.2010.01194.x"
                              aria-label="Article reference 64"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC21158933"
                              aria-label="PubMed Central reference 64"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 64"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=MIF%3A%20a%20key%20player%20in%20cutaneous%20biology%20and%20wound%20healing&amp;journal=Exp.%20Dermatol.&amp;volume=20&amp;pages=1-6&amp;publication_year=2011&amp;author=Gilliver%2CSC&amp;author=Emmerson%2CE&amp;author=Bernhagen%2CJ&amp;author=Hardman%2CMJ"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">65.</span>
                          <p class="c-article-references__text" id="ref-CR65">
                            Ashcroft, G. S. et al. Estrogen modulates cutaneous
                            wound healing by downregulating macrophage migration
                            inhibitory factor. <i>J. Clin. Invest.</i>
                            <b>111</b>, 1309–1318 (2003).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD3sXjsFCrsrw%253D"
                              aria-label="CAS reference 65"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12727922"
                              aria-label="PubMed reference 65"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC154440"
                              aria-label="PubMed Central reference 65"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1172%2FJCI16288"
                              aria-label="Article reference 65"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 65"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Estrogen%20modulates%20cutaneous%20wound%20healing%20by%20downregulating%20macrophage%20migration%20inhibitory%20factor&amp;journal=J.%20Clin.%20Invest.&amp;volume=111&amp;pages=1309-1318&amp;publication_year=2003&amp;author=Ashcroft%2CGS"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">66.</span>
                          <p class="c-article-references__text" id="ref-CR66">
                            Danchin, A. The specification of the immune response
                            revisited. <i>Surv. Immunol. Res.</i> <b>1</b>,
                            173–183 (1982).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADyaL38Xls1Slu7k%253D"
                              aria-label="CAS reference 66"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=6314467"
                              aria-label="PubMed reference 66"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6314467"
                              aria-label="PubMed Central reference 66"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 66"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=The%20specification%20of%20the%20immune%20response%20revisited&amp;journal=Surv.%20Immunol.%20Res.&amp;volume=1&amp;pages=173-183&amp;publication_year=1982&amp;author=Danchin%2CA"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">67.</span>
                          <p class="c-article-references__text" id="ref-CR67">
                            Mirza, R. E., Fang, M. M., Ennis, W. J. &amp; Koh,
                            T. J. Blocking interleukin-1beta induces a
                            healing-associated wound macrophage phenotype and
                            improves healing in type 2 diabetes.
                            <i>Diabetes</i> <b>62</b>, 2579–2587 (2013).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC3sXhtFWjsb%252FK"
                              aria-label="CAS reference 67"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23493576"
                              aria-label="PubMed reference 67"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3712034"
                              aria-label="PubMed Central reference 67"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.2337%2Fdb12-1450"
                              aria-label="Article reference 67"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 67"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Blocking%20interleukin-1beta%20induces%20a%20healing-associated%20wound%20macrophage%20phenotype%20and%20improves%20healing%20in%20type%202%20diabetes&amp;journal=Diabetes&amp;volume=62&amp;pages=2579-2587&amp;publication_year=2013&amp;author=Mirza%2CRE&amp;author=Fang%2CMM&amp;author=Ennis%2CWJ&amp;author=Koh%2CTJ"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">68.</span>
                          <p class="c-article-references__text" id="ref-CR68">
                            Thomay, A. A. et al. Disruption of interleukin-1
                            signaling improves the quality of wound healing.
                            <i>Am. J. Pathol.</i> <b>174</b>, 2129–2136 (2009).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD1MXmvFOqu70%253D"
                              aria-label="CAS reference 68"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19389930"
                              aria-label="PubMed reference 68"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2684178"
                              aria-label="PubMed Central reference 68"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.2353%2Fajpath.2009.080765"
                              aria-label="Article reference 68"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 68"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Disruption%20of%20interleukin-1%20signaling%20improves%20the%20quality%20of%20wound%20healing&amp;journal=Am.%20J.%20Pathol.&amp;volume=174&amp;pages=2129-2136&amp;publication_year=2009&amp;author=Thomay%2CAA"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">69.</span>
                          <p class="c-article-references__text" id="ref-CR69">
                            Sauder, D. N. et al. Interleukin-1 enhances
                            epidermal wound healing. <i>Lymphokine Res.</i>
                            <b>9</b>, 465–473 (1990).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADyaK3cXmt12gsbc%253D"
                              aria-label="CAS reference 69"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=2151047"
                              aria-label="PubMed reference 69"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2151047"
                              aria-label="PubMed Central reference 69"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 69"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Interleukin-1%20enhances%20epidermal%20wound%20healing&amp;journal=Lymphokine%20Res.&amp;volume=9&amp;pages=465-473&amp;publication_year=1990&amp;author=Sauder%2CDN"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">70.</span>
                          <p class="c-article-references__text" id="ref-CR70">
                            He, H. et al. Single-cell transcriptome analysis of
                            human skin identifies novel fibroblast subpopulation
                            and enrichment of immune subsets in atopic
                            dermatitis. <i>J. Allergy Clin. Immunol.</i>
                            <b>145</b>, 1615–1628 (2020).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BB3cXksFGrs7c%253D"
                              aria-label="CAS reference 70"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32035984"
                              aria-label="PubMed reference 70"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1016%2Fj.jaci.2020.01.042"
                              aria-label="Article reference 70"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC32035984"
                              aria-label="PubMed Central reference 70"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 70"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Single-cell%20transcriptome%20analysis%20of%20human%20skin%20identifies%20novel%20fibroblast%20subpopulation%20and%20enrichment%20of%20immune%20subsets%20in%20atopic%20dermatitis&amp;journal=J.%20Allergy%20Clin.%20Immunol.&amp;volume=145&amp;pages=1615-1628&amp;publication_year=2020&amp;author=He%2CH"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">71.</span>
                          <p class="c-article-references__text" id="ref-CR71">
                            Shao, X. et al. CellTalkDB: a manually curated
                            database of ligand-receptor interactions in humans
                            and mice. <i>Brief Bioinform.</i>
                            <a href="https://doi.org/10.1093/bib/bbaa269"
                              >https://doi.org/10.1093/bib/bbaa269</a
                            >
                            (2020).
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">72.</span>
                          <p class="c-article-references__text" id="ref-CR72">
                            Ramilowski, J. A. et al. A draft network of
                            ligand-receptor-mediated multicellular signalling in
                            human. <i>Nat. Commun.</i> <b>6</b>, 7866 (2015).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2MXhtlWhu7%252FM"
                              aria-label="CAS reference 72"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26198319"
                              aria-label="PubMed reference 72"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4525178"
                              aria-label="PubMed Central reference 72"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1038%2Fncomms8866"
                              aria-label="Article reference 72"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2015NatCo...6.7866R"
                              aria-label="ADS reference 72"
                              >ADS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 72"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=A%20draft%20network%20of%20ligand-receptor-mediated%20multicellular%20signalling%20in%20human&amp;journal=Nat.%20Commun.&amp;volume=6&amp;publication_year=2015&amp;author=Ramilowski%2CJA"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">73.</span>
                          <p class="c-article-references__text" id="ref-CR73">
                            Noël, F. et al. ICELLNET: a transcriptome-based
                            framework to dissect intercellular communication.
                            Preprint at bioRxiv 2020.2003.2005.976878 (2020).
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">74.</span>
                          <p class="c-article-references__text" id="ref-CR74">
                            Durinck, S., Spellman, P. T., Birney, E. &amp;
                            Huber, W. Mapping identifiers for the integration of
                            genomic datasets with the R/Bioconductor package
                            biomaRt. <i>Nat. Protoc.</i> <b>4</b>, 1184–1191
                            (2009).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD1MXpt12ksrk%253D"
                              aria-label="CAS reference 74"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19617889"
                              aria-label="PubMed reference 74"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3159387"
                              aria-label="PubMed Central reference 74"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1038%2Fnprot.2009.97"
                              aria-label="Article reference 74"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 74"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Mapping%20identifiers%20for%20the%20integration%20of%20genomic%20datasets%20with%20the%20R%2FBioconductor%20package%20biomaRt&amp;journal=Nat.%20Protoc.&amp;volume=4&amp;pages=1184-1191&amp;publication_year=2009&amp;author=Durinck%2CS&amp;author=Spellman%2CPT&amp;author=Birney%2CE&amp;author=Huber%2CW"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">75.</span>
                          <p class="c-article-references__text" id="ref-CR75">
                            Kharchenko, P. V., Silberstein, L. &amp; Scadden, D.
                            T. Bayesian approach to single-cell differential
                            expression analysis. <i>Nat. Methods</i> <b>11</b>,
                            740–742 (2014).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2cXotFCjs70%253D"
                              aria-label="CAS reference 75"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24836921"
                              aria-label="PubMed reference 75"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4112276"
                              aria-label="PubMed Central reference 75"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1038%2Fnmeth.2967"
                              aria-label="Article reference 75"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 75"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Bayesian%20approach%20to%20single-cell%20differential%20expression%20analysis&amp;journal=Nat.%20Methods&amp;volume=11&amp;pages=740-742&amp;publication_year=2014&amp;author=Kharchenko%2CPV&amp;author=Silberstein%2CL&amp;author=Scadden%2CDT"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">76.</span>
                          <p class="c-article-references__text" id="ref-CR76">
                            Zhang, L. &amp; Zhang, S. Comparison of
                            computational methods for imputing single-cell
                            RNA-sequencing data.
                            <i>IEEE/ACM Trans. Comput Biol. Bioinform.</i>
                            <b>17</b>, 376–389 (2020).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BB3cXitVyqtL%252FM"
                              aria-label="CAS reference 76"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29994128"
                              aria-label="PubMed reference 76"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC29994128"
                              aria-label="PubMed Central reference 76"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 76"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Comparison%20of%20computational%20methods%20for%20imputing%20single-cell%20RNA-sequencing%20data&amp;journal=IEEE%2FACM%20Trans.%20Comput%20Biol.%20Bioinform.&amp;volume=17&amp;pages=376-389&amp;publication_year=2020&amp;author=Zhang%2CL&amp;author=Zhang%2CS"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">77.</span>
                          <p class="c-article-references__text" id="ref-CR77">
                            Marx, V. A dream of single-cell proteomics.
                            <i>Nat. Methods</i> <b>16</b>, 809–812 (2019).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC1MXhsFKhtbfK"
                              aria-label="CAS reference 77"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31406385"
                              aria-label="PubMed reference 77"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1038%2Fs41592-019-0540-6"
                              aria-label="Article reference 77"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC31406385"
                              aria-label="PubMed Central reference 77"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 77"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=A%20dream%20of%20single-cell%20proteomics&amp;journal=Nat.%20Methods&amp;volume=16&amp;pages=809-812&amp;publication_year=2019&amp;author=Marx%2CV"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">78.</span>
                          <p class="c-article-references__text" id="ref-CR78">
                            Burgess, D. J. Spatial transcriptomics coming of
                            age. <i>Nat. Rev. Genet.</i> <b>20</b>, 317–317
                            (2019).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC1MXosFKgur8%253D"
                              aria-label="CAS reference 78"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30980030"
                              aria-label="PubMed reference 78"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1038%2Fs41576-019-0129-z"
                              aria-label="Article reference 78"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC30980030"
                              aria-label="PubMed Central reference 78"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 78"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Spatial%20transcriptomics%20coming%20of%20age&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=20&amp;pages=317-317&amp;publication_year=2019&amp;author=Burgess%2CDJ"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">79.</span>
                          <p class="c-article-references__text" id="ref-CR79">
                            Dries, R. et al. Giotto, a pipeline for integrative
                            analysis and visualization of single-cell spatial
                            transcriptomic data. Preprint at bioRxiv 701680
                            (2019).
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">80.</span>
                          <p class="c-article-references__text" id="ref-CR80">
                            Cang, Z. &amp; Nie, Q. Inferring spatial and
                            signaling relationships between cells from single
                            cell transcriptomic data. <i>Nat. Commun.</i>
                            <b>11</b>, 2084 (2020).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BB3cXosVOmu7k%253D"
                              aria-label="CAS reference 80"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32350282"
                              aria-label="PubMed reference 80"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7190659"
                              aria-label="PubMed Central reference 80"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1038%2Fs41467-020-15968-5"
                              aria-label="Article reference 80"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2020NatCo..11.2084C"
                              aria-label="ADS reference 80"
                              >ADS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 80"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Inferring%20spatial%20and%20signaling%20relationships%20between%20cells%20from%20single%20cell%20transcriptomic%20data&amp;journal=Nat.%20Commun.&amp;volume=11&amp;publication_year=2020&amp;author=Cang%2CZ&amp;author=Nie%2CQ"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">81.</span>
                          <p class="c-article-references__text" id="ref-CR81">
                            Stuart, T. &amp; Satija, R. Integrative single-cell
                            analysis. <i>Nat. Rev. Genet.</i> <b>20</b>, 257–272
                            (2019).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC1MXmtFKjtbs%253D"
                              aria-label="CAS reference 81"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30696980"
                              aria-label="PubMed reference 81"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1038%2Fs41576-019-0093-7"
                              aria-label="Article reference 81"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC30696980"
                              aria-label="PubMed Central reference 81"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 81"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Integrative%20single-cell%20analysis&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=20&amp;pages=257-272&amp;publication_year=2019&amp;author=Stuart%2CT&amp;author=Satija%2CR"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">82.</span>
                          <p class="c-article-references__text" id="ref-CR82">
                            Jin, S., Zhang, L. &amp; Nie, Q. scAI: an
                            unsupervised approach for the integrative analysis
                            of parallel single-cell transcriptomic and
                            epigenomic profiles. <i>Genome Biol.</i> <b>21</b>,
                            25 (2020).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32014031"
                              aria-label="PubMed reference 82"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6996200"
                              aria-label="PubMed Central reference 82"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1186%2Fs13059-020-1932-8"
                              aria-label="Article reference 82"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 82"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=scAI%3A%20an%20unsupervised%20approach%20for%20the%20integrative%20analysis%20of%20parallel%20single-cell%20transcriptomic%20and%20epigenomic%20profiles&amp;journal=Genome%20Biol.&amp;volume=21&amp;publication_year=2020&amp;author=Jin%2CS&amp;author=Zhang%2CL&amp;author=Nie%2CQ"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">83.</span>
                          <p class="c-article-references__text" id="ref-CR83">
                            Cowen, L., Ideker, T., Raphael, B. J. &amp; Sharan,
                            R. Network propagation: a universal amplifier of
                            genetic associations. <i>Nat. Rev. Genet.</i>
                            <b>18</b>, 551–562 (2017).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2sXps1Kgur4%253D"
                              aria-label="CAS reference 83"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28607512"
                              aria-label="PubMed reference 83"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1038%2Fnrg.2017.38"
                              aria-label="Article reference 83"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC28607512"
                              aria-label="PubMed Central reference 83"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 83"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Network%20propagation%3A%20a%20universal%20amplifier%20of%20genetic%20associations&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=18&amp;pages=551-562&amp;publication_year=2017&amp;author=Cowen%2CL&amp;author=Ideker%2CT&amp;author=Raphael%2CBJ&amp;author=Sharan%2CR"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">84.</span>
                          <p class="c-article-references__text" id="ref-CR84">
                            Ronen, J. &amp; Akalin, A. netSmooth:
                            Network-smoothing based imputation for single cell
                            RNA-seq. <i>F1000Res</i> <b>7</b>, 8–8 (2018).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29511531"
                              aria-label="PubMed reference 84"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5814748"
                              aria-label="PubMed Central reference 84"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.12688%2Ff1000research.13511.3"
                              aria-label="Article reference 84"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 84"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=netSmooth%3A%20Network-smoothing%20based%20imputation%20for%20single%20cell%20RNA-seq&amp;journal=F1000Res&amp;volume=7&amp;pages=8-8&amp;publication_year=2018&amp;author=Ronen%2CJ&amp;author=Akalin%2CA"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">85.</span>
                          <p class="c-article-references__text" id="ref-CR85">
                            Szklarczyk, D. et al. STRING v11: protein-protein
                            association networks with increased coverage,
                            supporting functional discovery in genome-wide
                            experimental datasets. <i>Nucleic Acids Res.</i>
                            <b>47</b>, D607–D613 (2019).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC1MXhs1Gqtr%252FM"
                              aria-label="CAS reference 85"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30476243"
                              aria-label="PubMed reference 85"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1093%2Fnar%2Fgky1131"
                              aria-label="Article reference 85"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC30476243"
                              aria-label="PubMed Central reference 85"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 85"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=STRING%20v11%3A%20protein-protein%20association%20networks%20with%20increased%20coverage%2C%20supporting%20functional%20discovery%20in%20genome-wide%20experimental%20datasets&amp;journal=Nucleic%20Acids%20Res.&amp;volume=47&amp;pages=D607-D613&amp;publication_year=2019&amp;author=Szklarczyk%2CD"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">86.</span>
                          <p class="c-article-references__text" id="ref-CR86">
                            Chang, C. Agonists and antagonists of TGF-beta
                            family ligands.
                            <i>Cold Spring Harb. Perspect. Biol.</i> <b>8</b>,
                            a021923 (2016).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27413100"
                              aria-label="PubMed reference 86"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4968162"
                              aria-label="PubMed Central reference 86"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1101%2Fcshperspect.a021923"
                              aria-label="Article reference 86"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC1cXns1ymu7w%253D"
                              aria-label="CAS reference 86"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 86"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Agonists%20and%20antagonists%20of%20TGF-beta%20family%20ligands&amp;journal=Cold%20Spring%20Harb.%20Perspect.%20Biol.&amp;volume=8&amp;publication_year=2016&amp;author=Chang%2CC"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">87.</span>
                          <p class="c-article-references__text" id="ref-CR87">
                            Butts, C. T. Social network analysis with sna.
                            <i>J. Stat. Softw.</i> <b>1</b>, 1–51 (2008).
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">88.</span>
                          <p class="c-article-references__text" id="ref-CR88">
                            Landherr, A., Friedl, B. &amp; Heidemann, J. A
                            critical review of centrality measures in social
                            networks. <i>Bus. Inf. Syst. Eng.</i> <b>2</b>,
                            371–385 (2010).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1007%2Fs12599-010-0127-3"
                              aria-label="Article reference 88"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 88"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=A%20critical%20review%20of%20centrality%20measures%20in%20social%20networks&amp;journal=Bus.%20Inf.%20Syst.%20Eng.&amp;volume=2&amp;pages=371-385&amp;publication_year=2010&amp;author=Landherr%2CA&amp;author=Friedl%2CB&amp;author=Heidemann%2CJ"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">89.</span>
                          <p class="c-article-references__text" id="ref-CR89">
                            Zhang, L. &amp; Zhang, S. Learning common and
                            specific patterns from data of multiple interrelated
                            biological scenarios with matrix factorization.
                            <i>Nucleic Acids Res.</i> <b>47</b>, 6606–6617
                            (2019).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BB3cXktVyrtro%253D"
                              aria-label="CAS reference 89"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31175825"
                              aria-label="PubMed reference 89"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6649783"
                              aria-label="PubMed Central reference 89"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1093%2Fnar%2Fgkz488"
                              aria-label="Article reference 89"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 89"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Learning%20common%20and%20specific%20patterns%20from%20data%20of%20multiple%20interrelated%20biological%20scenarios%20with%20matrix%20factorization&amp;journal=Nucleic%20Acids%20Res.&amp;volume=47&amp;pages=6606-6617&amp;publication_year=2019&amp;author=Zhang%2CL&amp;author=Zhang%2CS"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">90.</span>
                          <p class="c-article-references__text" id="ref-CR90">
                            Gaujoux, R. &amp; Seoighe, C. A flexible R package
                            for nonnegative matrix factorization.
                            <i>BMC Bioinform.</i> <b>11</b>, 367 (2010).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1186%2F1471-2105-11-367"
                              aria-label="Article reference 90"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC3cXoslGiurY%253D"
                              aria-label="CAS reference 90"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 90"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=A%20flexible%20R%20package%20for%20nonnegative%20matrix%20factorization&amp;journal=BMC%20Bioinform.&amp;volume=11&amp;publication_year=2010&amp;author=Gaujoux%2CR&amp;author=Seoighe%2CC"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">91.</span>
                          <p class="c-article-references__text" id="ref-CR91">
                            Schieber, T. A. et al. Quantification of network
                            structural dissimilarities. <i>Nat. Commun.</i>
                            <b>8</b>, 13928 (2017).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2sXpsVOmtw%253D%253D"
                              aria-label="CAS reference 91"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28067266"
                              aria-label="PubMed reference 91"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5227707"
                              aria-label="PubMed Central reference 91"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1038%2Fncomms13928"
                              aria-label="Article reference 91"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2017NatCo...813928S"
                              aria-label="ADS reference 91"
                              >ADS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 91"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Quantification%20of%20network%20structural%20dissimilarities&amp;journal=Nat.%20Commun.&amp;volume=8&amp;publication_year=2017&amp;author=Schieber%2CTA"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">92.</span>
                          <p class="c-article-references__text" id="ref-CR92">
                            Leland McInnes, JohnHealy &amp; Melville, J. UMAP:
                            uniform manifold approximation and projection for
                            dimension reduction. <i>arXiv</i> <b>1802</b>, 03426
                            (2018).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 92"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=UMAP%3A%20uniform%20manifold%20approximation%20and%20projection%20for%20dimension%20reduction&amp;journal=arXiv&amp;volume=1802&amp;publication_year=2018&amp;author=Leland%20McInnes%2CJohnHealy&amp;author=Melville%2CJ"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">93.</span>
                          <p class="c-article-references__text" id="ref-CR93">
                            Luxburg, Uv. A. Tutorial on spectral clustering.
                            <i>Stat. Comput.</i> <b>17</b>, 1–32 (2007).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ams.org/mathscinet-getitem?mr=2370963"
                              aria-label="MathSciNet reference 93"
                              >MathSciNet</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1007%2Fs11222-006-9001-z"
                              aria-label="Article reference 93"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 93"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Tutorial%20on%20spectral%20clustering&amp;journal=Stat.%20Comput.&amp;volume=17&amp;pages=1-32&amp;publication_year=2007&amp;author=Luxburg%2CUvA"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">94.</span>
                          <p class="c-article-references__text" id="ref-CR94">
                            Blondel, V. D., Guillaume, J.-L., Lambiotte, R.
                            &amp; Lefebvre, E. Fast unfolding of communities in
                            large networks. <i>J. Stat. Mech.</i> <b>2008</b>,
                            P10008 (2008).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.emis.de/MATH-item?07120580"
                              aria-label="MATH reference 94"
                              >MATH</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1088%2F1742-5468%2F2008%2F10%2FP10008"
                              aria-label="Article reference 94"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 94"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Fast%20unfolding%20of%20communities%20in%20large%20networks&amp;journal=J.%20Stat.%20Mech.&amp;volume=2008&amp;publication_year=2008&amp;author=Blondel%2CVD&amp;author=Guillaume%2CJ-L&amp;author=Lambiotte%2CR&amp;author=Lefebvre%2CE"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">95.</span>
                          <p class="c-article-references__text" id="ref-CR95">
                            Ge, W. et al. Single-cell transcriptome profiling
                            reveals dermal and epithelium cell fate decisions
                            during embryonic hair follicle development.
                            <i>Theranostics</i> <b>10</b>, 7581–7598 (2020).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BB3cXitVWlsbnI"
                              aria-label="CAS reference 95"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32685006"
                              aria-label="PubMed reference 95"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7359078"
                              aria-label="PubMed Central reference 95"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.7150%2Fthno.44306"
                              aria-label="Article reference 95"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 95"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Single-cell%20transcriptome%20profiling%20reveals%20dermal%20and%20epithelium%20cell%20fate%20decisions%20during%20embryonic%20hair%20follicle%20development&amp;journal=Theranostics&amp;volume=10&amp;pages=7581-7598&amp;publication_year=2020&amp;author=Ge%2CW"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                        <li
                          class="
                            c-article-references__item
                            js-c-reading-companion-references-item
                          "
                        >
                          <span class="c-article-references__counter">96.</span>
                          <p class="c-article-references__text" id="ref-CR96">
                            Hie, B., Cho, H., DeMeo, B., Bryson, B. &amp;
                            Berger, B. Geometric sketching compactly summarizes
                            the single-cell transcriptomic landscape.
                            <i>Cell Syst.</i> <b>8</b>, 483–493 e487 (2019).
                          </p>
                          <p class="c-article-references__links u-hide-print">
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC1MXht1Kgu7zN"
                              aria-label="CAS reference 96"
                              >CAS</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31176620"
                              aria-label="PubMed reference 96"
                              rel="nofollow"
                              >PubMed</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6597305"
                              aria-label="PubMed Central reference 96"
                              rel="nofollow"
                              >PubMed Central</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              href="https://doi.org/10.1016%2Fj.cels.2019.05.003"
                              aria-label="Article reference 96"
                              >Article</a
                            >&nbsp;
                            <a
                              data-track="click"
                              data-track-action="outbound reference"
                              data-track-label="link"
                              aria-label="Google Scholar reference 96"
                              href="http://scholar.google.com/scholar_lookup?&amp;title=Geometric%20sketching%20compactly%20summarizes%20the%20single-cell%20transcriptomic%20landscape&amp;journal=Cell%20Syst.&amp;volume=8&amp;pages=483-493%20e487&amp;publication_year=2019&amp;author=Hie%2CB&amp;author=Cho%2CH&amp;author=DeMeo%2CB&amp;author=Bryson%2CB&amp;author=Berger%2CB"
                            >
                              Google Scholar</a
                            >&nbsp;
                          </p>
                        </li>
                      </ol>
                      <p class="c-article-references__download u-hide-print">
                        <a
                          data-track="click"
                          data-track-action="download citation references"
                          data-track-label="link"
                          href="https://citation-needed.springer.com/v2/references/10.1038/s41467-021-21246-9?format=refman&amp;flavour=references"
                          >Download references<svg
                            width="16"
                            height="16"
                            focusable="false"
                            role="img"
                            aria-hidden="true"
                            class="u-icon"
                          >
                            <use
                              xlink="http://www.w3.org/1999/xlink"
                              xlink:href="#global-icon-download"
                            ></use></svg></a>
                      </p>
                    </div>
                  </div>
                </div>
              </section>
            </div>


            <section data-title="Comments">
              <div class="c-article-section" id="article-comments-section">
                <h2
                  class="
                    c-article-section__title
                    js-section-title js-c-reading-companion-sections-item
                  "
                  id="article-comments"
                >
                  Comments
                </h2>
                <div
                  class="c-article-section__content"
                  id="article-comments-content"
                >
                  <p>
                    By submitting a comment you agree to abide by our
                    <a href="https://www.nature.com/info/tandc.html">Terms</a>
                    and
                    <a
                      href="https://www.nature.com/info/community-guidelines.html"
                      >Community Guidelines</a
                    >. If you find something abusive or that does not comply
                    with our terms or guidelines please flag it as
                    inappropriate.
                  </p>
                  <div
                    id="disqus_thread"
                    class="pb20"
                    data-object="disqus"
                    data-disqus-s3="e30= 1cfdcc37e243cd11e06aa51bcd5288081ddd679f 1633747204"
                    data-disqus-key="ZZDdHrSs8UQRH0LAFbG4KmcMfQdvUwYDEyYczGZhznVSuj9QDfXFgWxFA1tz53jo"
                    data-disqus-sso-name="Nature"
                    data-disqus-sso-button="//www.nature.com/static/images/buttons/disqus-sso-login-button.e9d0ed0791.png"
                    data-disqus-sso-icon="//www.nature.com/static/images/favicons/nature/favicon.62367f778b.ico"
                    data-disqus-sso-url="https://idp.nature.com/authorize/natureuser?client_id=grover&amp;redirect_uri=http%3A%2F%2Fwww.nature.com%2Farticles%2Fs41467-021-21246-9%2Fdisqus"
                    data-disqus-sso-logout="https://idp.nature.com/logout/natureuser?redirect_uri=https%3A%2F%2Fwww.nature.com%2Farticles%2Fs41467-021-21246-9"
                    data-disqus-shortname="ncomms"
                    data-disqus-identifier="10.1038/s41467-021-21246-9"
                    data-disqus-url="https://www.nature.com/articles/s41467-021-21246-9"
                  ></div>
                </div>
              </div>
            </section>

            <span data-recommended="jobs"></span>
          </div>
        </article>
      </main>

      <aside
        class="c-article-extras u-hide-print"
        aria-label="Article navigation"
        data-component-reading-companion=""
        data-container-type="reading-companion"
        data-track-component="reading companion"
      >
        <div class="c-pdf-download u-clear-both">
          <a
            href="https://www.nature.com/articles/s41467-021-21246-9.pdf"
            class="c-pdf-download__link"
            data-article-pdf="true"
            data-readcube-pdf-url="true"
            data-test="download-pdf"
            data-draft-ignore="true"
            data-track="click"
            data-track-action="download pdf"
            data-track-label="link"
            data-track-external=""
          >
            <span>Download PDF</span>
            <svg
              aria-hidden="true"
              focusable="false"
              width="16"
              height="16"
              class="u-icon"
            >
              <use xlink:href="#global-icon-download"></use>
            </svg>
          </a>
        </div>

        <div class="c-reading-companion">
          <div
            class="c-reading-companion__sticky"
            data-component="reading-companion-sticky"
            data-test="reading-companion-sticky"
            style="top: 40px"
          >
            <div
              class="
                c-reading-companion__panel
                c-reading-companion__sections
                c-reading-companion__panel--active
              "
              id="tabpanel-sections"
              aria-labelledby="tab-sections"
            >
              <div class="u-mt-16" data-component-mpu="">
              </div>
            </div>
            <div
              class="
                c-reading-companion__panel
                c-reading-companion__figures
                c-reading-companion__panel--full-width
              "
              id="tabpanel-figures"
              aria-labelledby="tab-figures"
            >
            </div>
            <div
              class="
                c-reading-companion__panel
                c-reading-companion__references
                c-reading-companion__panel--full-width
              "
              id="tabpanel-references"
              aria-labelledby="tab-references"
            >
           </div>
          </div>
        </div>
      </aside>
    </div>

    <footer role="contentinfo" class="composite-layer">
      <div class="u-mt-16 u-mb-16">
        <div class="u-container">
          <div
            class="u-display-flex u-flex-wrap u-justify-content-space-between"
          >
            <p class="c-meta u-ma-0 u-flex-shrink">
              <span class="c-meta__item">
                Nature Communications (<i>Nat Commun</i>)
              </span>

              <span class="c-meta__item">
                <abbr title="International Standard Serial Number">ISSN</abbr>
                <span itemprop="onlineIssn">2041-1723</span> (online)
              </span>
            </p>
          </div>
        </div>
      </div>


      <div class="c-corporate-footer">
        <div class="u-container">
          <img
            src="report_files/sn-logo-white.svg"
            alt="Springer Nature"
            loading="lazy"
            width="140"
            height="14"
          />
          <p class="c-corporate-footer__legal" data-test="copyright">
            © 2021 Springer Nature Limited
          </p>
        </div>
      </div>

      <svg class="u-hide hide">
        <symbol id="global-icon-chevron-right" viewBox="0 0 16 16">
          <path
            d="M7.782 7L5.3 4.518c-.393-.392-.4-1.022-.02-1.403a1.001 1.001 0 011.417 0l4.176 4.177a1.001 1.001 0 010 1.416l-4.176 4.177a.991.991 0 01-1.4.016 1 1 0 01.003-1.42L7.782 9l1.013-.998z"
            fill-rule="evenodd"
          ></path>
        </symbol>
        <symbol id="global-icon-download" viewBox="0 0 16 16">
          <path
            d="M2 14c0-.556.449-1 1.002-1h9.996a.999.999 0 110 2H3.002A1.006 1.006 0 012 14zM9 2v6.8l2.482-2.482c.392-.392 1.022-.4 1.403-.02a1.001 1.001 0 010 1.417l-4.177 4.177a1.001 1.001 0 01-1.416 0L3.115 7.715a.991.991 0 01-.016-1.4 1 1 0 011.42.003L7 8.8V2c0-.55.444-.996 1-.996.552 0 1 .445 1 .996z"
            fill-rule="evenodd"
          ></path>
        </symbol>
        <symbol id="global-icon-email" viewBox="0 0 18 18">
          <path
            d="M1.995 2h14.01A2 2 0 0118 4.006v9.988A2 2 0 0116.005 16H1.995A2 2 0 010 13.994V4.006A2 2 0 011.995 2zM1 13.994A1 1 0 001.995 15h14.01A1 1 0 0017 13.994V4.006A1 1 0 0016.005 3H1.995A1 1 0 001 4.006zM9 11L2 7V5.557l7 4 7-4V7z"
            fill-rule="evenodd"
          ></path>
        </symbol>
        <symbol id="global-icon-institution" viewBox="0 0 18 18">
          <path
            d="M14 8a1 1 0 011 1v6h1.5a.5.5 0 01.5.5v.5h.5a.5.5 0 01.5.5V18H0v-1.5a.5.5 0 01.5-.5H1v-.5a.5.5 0 01.5-.5H3V9a1 1 0 112 0v6h8V9a1 1 0 011-1zM6 8l2 1v4l-2 1zm6 0v6l-2-1V9zM9.573.401l7.036 4.925A.92.92 0 0116.081 7H1.92a.92.92 0 01-.528-1.674L8.427.401a1 1 0 011.146 0zM9 2.441L5.345 5h7.31z"
            fill-rule="evenodd"
          ></path>
        </symbol>
        <symbol id="global-icon-search" viewBox="0 0 22 22">
          <path
            fill-rule="evenodd"
            d="M21.697 20.261a1.028 1.028 0 01.01 1.448 1.034 1.034 0 01-1.448-.01l-4.267-4.267A9.812 9.811 0 010 9.812a9.812 9.811 0 1117.43 6.182zM9.812 18.222A8.41 8.41 0 109.81 1.403a8.41 8.41 0 000 16.82z"
          ></path>
        </symbol>
        <symbol id="icon-info" viewBox="0 0 18 18">
          <path
            d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm0 7h-1.5l-.11662113.00672773c-.49733868.05776511-.88337887.48043643-.88337887.99327227 0 .47338693.32893365.86994729.77070917.97358929l.1126697.01968298.11662113.00672773h.5v3h-.5l-.11662113.0067277c-.42082504.0488782-.76196299.3590206-.85696816.7639815l-.01968298.1126697-.00672773.1166211.00672773.1166211c.04887817.4208251.35902055.761963.76398144.8569682l.1126697.019683.11662113.0067277h3l.1166211-.0067277c.4973387-.0577651.8833789-.4804365.8833789-.9932723 0-.4733869-.3289337-.8699473-.7707092-.9735893l-.1126697-.019683-.1166211-.0067277h-.5v-4l-.00672773-.11662113c-.04887817-.42082504-.35902055-.76196299-.76398144-.85696816l-.1126697-.01968298zm0-3.25c-.69035594 0-1.25.55964406-1.25 1.25s.55964406 1.25 1.25 1.25 1.25-.55964406 1.25-1.25-.55964406-1.25-1.25-1.25z"
            fill-rule="evenodd"
          ></path>
        </symbol>
        <symbol id="icon-success" viewBox="0 0 18 18">
          <path
            d="M9 0a9 9 0 110 18A9 9 0 019 0zm3.486 4.982l-4.718 5.506L5.14 8.465a.991.991 0 00-1.423.133 1.06 1.06 0 00.13 1.463l3.407 2.733a1 1 0 001.387-.133l5.385-6.334a1.06 1.06 0 00-.116-1.464.991.991 0 00-1.424.119z"
            fill-rule="evenodd"
          ></path>
        </symbol>
        <symbol id="icon-chevron-down" viewBox="0 0 16 16">
          <path
            d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z"
            fill-rule="evenodd"
            transform="matrix(0 1 -1 0 11 1)"
          ></path>
        </symbol>
      </svg>
    </footer>

    <div
      class="
        c-site-messages
        message
        u-hide-print
        c-site-messages--nature-briefing
        c-site-messages--nature-briefing-email-variant
        c-site-messages--nature-briefing-redesign-2020
        sans-serif
      "
      data-component-id="nature-briefing-banner"
      data-component-expirydays="30"
      data-component-trigger-scroll-percentage="15"
      data-track="in-view"
      data-track-action="in-view"
      data-track-category="nature briefing"
      data-track-label="redesign banner visible"
    >
      <div class="c-site-messages__banner-large">
        <div class="c-site-messages__close-container">
          <button
            class="c-site-messages__close"
            data-track="click"
            data-track-category="nature briefing"
            data-track-label="redesign banner dismiss"
          >
            <!--?xml version="1.0" encoding="UTF-8" standalone="no"?-->
            <svg
              width="25px"
              height="25px"
              focusable="false"
              aria-hidden="true"
              viewBox="0 0 25 25"
              version="1.1"
              xmlns="http://www.w3.org/2000/svg"
              xlink="http://www.w3.org/1999/xlink"
            >
              <title>Close banner</title>
              <defs></defs>
              <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
                <rect opacity="0" x="0" y="0" width="25" height="25"></rect>
                <path
                  d="M6.29679575,16.2772478 C5.90020818,16.6738354 5.90240728,17.3100587 6.29617427,17.7038257 C6.69268654,18.100338 7.32864195,18.0973145 7.72275218,17.7032043 L12,13.4259564 L16.2772478,17.7032043 C16.6738354,18.0997918 17.3100587,18.0975927 17.7038257,17.7038257 C18.100338,17.3073135 18.0973145,16.671358 17.7032043,16.2772478 L13.4259564,12 L17.7032043,7.72275218 C18.0997918,7.32616461 18.0975927,6.68994127 17.7038257,6.29617427 C17.3073135,5.89966201 16.671358,5.90268552 16.2772478,6.29679575 L12,10.5740436 L7.72275218,6.29679575 C7.32616461,5.90020818 6.68994127,5.90240728 6.29617427,6.29617427 C5.89966201,6.69268654 5.90268552,7.32864195 6.29679575,7.72275218 L10.5740436,12 L6.29679575,16.2772478 Z"
                  fill="#ffffff"
                ></path>
              </g>
            </svg>
            <span class="visually-hidden">Close</span>
          </button>
        </div>

        <div class="c-site-messages__form-container">
          <div class="grid grid-12 last">
            <div class="grid grid-4">
              <img
                alt="Nature Briefing"
                src="report_files/nature-briefing-logo-n150-white.svg"
                width="250"
                height="40"
              />
              <p
                class="
                  c-site-messages--nature-briefing__strapline
                  extra-tight-line-height
                "
              >
                Sign up for the <em>Nature Briefing</em> newsletter — what
                matters in science, free to your inbox daily.
              </p>
            </div>
            <div class="grid grid-8 last">
              <form
                action="/briefing/signup/formfeedback"
                method="post"
                data-location="banner"
                data-track="submit"
                data-track-action="transmit-form"
              >
                <input
                  id="briefing-banner-signup-form-input-track-originReferralPoint"
                  type="hidden"
                  name="track_originReferralPoint"
                  value="DirectEmailBannerRedesign2020"
                />
                <input
                  id="briefing-banner-signup-form-input-track-formType"
                  type="hidden"
                  name="track_formType"
                  value="DirectEmailBanner"
                />
                <label
                  class="nature-briefing-banner__email-label"
                  for="banner-EmailAddressInput"
                  >Email address</label
                >

                <div class="nature-briefing-banner__email-wrapper">
                  <input
                    class="
                      nature-briefing-banner__email-input
                      box-sizing
                      text14
                    "
                    type="email"
                    id="banner-EmailAddressInput"
                    name="email"
                    placeholder="e.g. jo.smith@university.ac.uk"
                    required="true"
                    aria-required="true"
                    data-test-element="briefing-emailbanner-email-input"
                  />
                  <button
                    type="submit"
                    class="
                      nature-briefing-banner__submit-button
                      box-sizing
                      text14
                    "
                    data-test-element="briefing-emailbanner-signup-button"
                  >
                    Sign up
                  </button>
                </div>

                <div
                  class="
                    nature-briefing-banner__checkbox-wrapper
                    grid grid-12
                    last
                  "
                >
                  <input
                    class="nature-briefing-banner__checkbox-checkbox"
                    id="gdpr-briefing-banner-checkbox"
                    type="checkbox"
                    name="gdpr"
                    value="1"
                    data-test-element="briefing-emailbanner-gdpr-checkbox"
                    required=""
                  />
                  <label
                    class="
                      nature-briefing-banner__checkbox-label
                      box-sizing
                      text13
                      sans-serif
                      block
                      tighten-line-height
                    "
                    for="gdpr-briefing-banner-checkbox"
                    >I agree my information will be processed in accordance with
                    the <em>Nature</em> and Springer Nature Limited
                    <a href="https://www.nature.com/info/privacy"
                      >Privacy Policy</a
                    >.</label
                  >
                </div>
              </form>
            </div>
          </div>
        </div>
      </div>

      <div class="c-site-messages__banner-small">
        <div class="c-site-messages__close-container">
          <button
            class="c-site-messages__close"
            data-track="click"
            data-track-category="nature briefing"
            data-track-label="redesign banner dismiss"
          >
            <!--?xml version="1.0" encoding="UTF-8" standalone="no"?-->
            <svg
              width="25px"
              height="25px"
              focusable="false"
              aria-hidden="true"
              viewBox="0 0 25 25"
              version="1.1"
              xmlns="http://www.w3.org/2000/svg"
              xlink="http://www.w3.org/1999/xlink"
            >
              <title>Close banner</title>
              <defs></defs>
              <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
                <rect opacity="0" x="0" y="0" width="25" height="25"></rect>
                <path
                  d="M6.29679575,16.2772478 C5.90020818,16.6738354 5.90240728,17.3100587 6.29617427,17.7038257 C6.69268654,18.100338 7.32864195,18.0973145 7.72275218,17.7032043 L12,13.4259564 L16.2772478,17.7032043 C16.6738354,18.0997918 17.3100587,18.0975927 17.7038257,17.7038257 C18.100338,17.3073135 18.0973145,16.671358 17.7032043,16.2772478 L13.4259564,12 L17.7032043,7.72275218 C18.0997918,7.32616461 18.0975927,6.68994127 17.7038257,6.29617427 C17.3073135,5.89966201 16.671358,5.90268552 16.2772478,6.29679575 L12,10.5740436 L7.72275218,6.29679575 C7.32616461,5.90020818 6.68994127,5.90240728 6.29617427,6.29617427 C5.89966201,6.69268654 5.90268552,7.32864195 6.29679575,7.72275218 L10.5740436,12 L6.29679575,16.2772478 Z"
                  fill="#ffffff"
                ></path>
              </g>
            </svg>
            <span class="visually-hidden">Close</span>
          </button>
        </div>
      </div>
    </div>

    <noscript>
      <img
        hidden
        src="https://verify.nature.com/verify/nature.png"
        border="0"
        width="0"
        height="0"
        style="display: none"
        alt=""
      />
    </noscript>

    <script src="report_files/ping" async=""></script>
    <img
      src="report_files/s41467-021-21246-9.avif"
      alt=""
      class="u-visually-hidden"
      width="1"
      height="1"
    />


  </body>
</html>
